#  A Novel Protocol for the Direct Identification of the Major Mitochondrial Oxidase in Fonsecaea pedrosoi
Quaglee Dragontacos


## Abstract
Here, we report the synthesis of glucoamylase GlcGlcA2 in the mycelia of a phytopathogenic and saprophytic fungus, . pedrosoi. The glucoamylase forms a homodimeric tetramate containing a -helical fold near the catalytic center. Strikingly, the fold is flanked by three putative tandemly overlapping domains, one of which is located between 5.8S and 5.8S islands in an adjacent region. Phylogenetic analysis of the a -helical fold reveals that the F. pedrosoi genome sequence was derived from a single point of deletion, and that the gene responsible for its synthesis is highly conserved. In addition to glucoamylase, two other domains are differentially folded, both affecting the transport of essential compounds. Together, these results reveal a novel mechanism for GlcGlcA2, which has previously been associated with secondary metabolism, transport, and iron homeostasis.

IMPORTANCE Fungal infections kill more than one million people per year, and more than a half a million cases of fungal disease occur in the United States alone. The fungus Fusarium graminearum is an important component of the human microbiome, and in 1999, the FDA approved F. graminearum f-box strain F107, which contains a gene for glucoamylase. Glucosylceramide (Glc) is a major sugar molecule that is produced by filamentous fungi, allowing the accumulation of the complexly-assembled glucoronic acid in their cell walls. To gain insight into the genetic mechanisms GlcGcA2, we identified the F. graminearum mutant F107 and showed that the mutant contains a smaller copy of GlcGcA2. Whole genome duplication events, loss of GlcGcA2 homologs, and upregulation of the enzyme in the glucoamylase promoter resulted in a total of nine mutants, three of which showed phenotypes similar to those of the wild type. The remaining mutants were either not identified or lost their GlcGcA2 homologs.


## Introduction
The Fonsecaea testis (hereafter called the Fonsecaea testis) is a unique organelle that forms a complex matrix of organelles and mediators, which comprise the oxidative phosphoryl group. The Fonsecaea testis has several different roles in the life of the plant, including the production of carotenoids, oxidases and antimicrobial substances [10], [11].

Many biological processes require oxidative phosphorylation to fulfill cellular functions, including in the biotrophic growth of various phytopathogens, such as C. albicans [12], [13], [14], [15]. The oxidative phosphorylation of the Fonsecaea testis has been reported to be important in the virulence of Candida albicans [16]. The Fonsecaea testis consists of two chambers, which are distinguished by the degree of phospholipid content in each chamber [13]. The first chamber contains lipid chains and the second chamber has protein chains containing fatty acids and cholesteryl esters. F. pedrosoi is a representative example of a phytopathogen that is able to survive inside lipid droplets, which are associated with lipid droplets [17]. As the lipid droplet environment is different from the lipid droplet environment, it has the potential to develop resistance to oxidative stress [17].

The oxidative phosphorylation of the Fonsecaea testis has been reported to be critical for its pathogenesis. Several studies have demonstrated that oxidative phosphorylation of the Fonsecaea testis is important for the virulence of C. albicans [18], [19], [20], [21], [22]. In particular, the phospholipase B phospholipase A enzyme oxidizes the fatty acids to form the oxidized polyunsaturated fatty acids (PUFAs) [23], and the C. albicans phospholipase B enzyme oxidizes PUFAs to form the isoprenyl group in the fatty acyl groups of the fatty acyl groups [24]. The NADP^+-dependent oxidase enzyme oxidizes urea and mannose to form D-N-acetyl-glucosamine (GlcNAc) and UDP-N-acetyl-glucosamine (UDP-GlcNAc) [25], [26].


## Methods
 2011.

High-throughput sequencing and data analysis
Sequence data analysis was performed by Macrogen (Aurora, CA, USA). The following primer pairs were used for the phylogenetic analysis: F. pedrosoi-nucleotide (CNAG_00014) (Vecchamber et al., 1997), F. lusitaniae-nucleotide (CNAG_00039) (O’Donnell et al., 1997), F. syriafinii-nucleotide (CNAG_00043) (Müller et al., 2006), F. terreus-nucleotide (CNAG_00061) (O’Donnell et al., 1996), F. fujica-nucleotide (CNAG_00085) (O’Donnell et al., 1995), F. niger-nucleotide (CNAG_00053) (O’Donnell et al., 1995), F. syriafinii-nucleotide (CNAG_0054) (O’Donnell et al., 1997), F. terreus-nucleotide (CNAG_00060) (O’Donnell et al., 1995), F. nostroides-nucleotide (CNAG_0027) (O’Donnell et al., 1994), F. norvegicus-nucleotide (CNAG_0141) (O’Donnell et al., 2000), F. parapsilosis-nucleotide (CNAG_0013) (O’Donnell et al., 2001), F. pyogenes-nucleotide (CNAG_0012) (O’Donnell et al., 2001) and F. thermophila-nucleotide (CNAG_0029) (O’Donnell et al., 2003).

The sequences were deposited in GenBank (accession numbers KP86699, KP86600, KP86601, KP86600, KP86600, KP86601, KP86601, and KP86600, respectively). The sequences have been deposited in GenBank (accession numbers KP86699, KP86600, KP86601, and KP86600, respectively).

The phylogenetic tree was constructed using Mega 6.


## Results
Strains were first constructed by the use of a single knockout strain by homologous recombination in each of the two knockout strains to generate the fonsecaea-pathogenic Fonsecaea proteae strai. As shown in Fig 1, the Fonsecaea proteae strain grew well and produced a single large mitochondrial pigment in the dark when grown in the dark with KOH. In contrast, the knockout strain produced a single large pigment in the dark when grown in the dark with KOH (Fig 1). In order to establish the nature of the pigmentation in the dark, the rxss mutant strain was constructed using a single knockout strain. A single knockout strain was generated using the dcp-a knockout strain with the corresponding deletion construct (Fig 1).

The major mitochondrial oxidase in Fonsecaea pedrosoi was identified in the genome of F. pedrosoi NIH-1 strain. Fonsecaea proteae strain, which is a new pathogenic strain, had been shown to be the cause of several fatal fusarium infections [9], [10]. To test whether the knockout strain produces an oxidase in the dark, we first confirmed the presence of the oxidase in the wild type strain of Fonsecaea pedrosoi by co-inoculation assay. The oxidase in the mutant strain was identified in the genome of Fonsecaea proteae strain. The presence of the oxidase in the mutant strain was confirmed by co-inoculation assay (Fig 2). The deletion strain of the fonsecaea-pathogenic Fonsecaea proteae strain had a single large pigment in the dark when grown in the dark with KOH. In contrast, the knockout strain produced an oxidase in the dark when grown in the dark with KOH. The same result was obtained with the dcp-a mutant strain with the deletion construct. These results indicated that the knockout strain produces an oxidase in the dark in Fonsecaea pedrosoi.


## Discussion
Among the mitochondrial DNA databases, the most studied are the mitochondrial DNA databases (MDBs) [10], [11], [12], [13], [14], [15. Although the detection of mitochondrial DNA in S. cerevisiae is relatively low compared to S. pombe, the level of significance of the identification in this species is still significant [11]. The high level of significance of the mitochondrial DNA in S. cerevisiae is even greater, with the level of significance of 1.6 in S. pombe and 2.6 in S. cerevisiae having remained unchanged for the most part (Figure 2, Supplementary File S2). The lowest level of significance of the mitochondrial DNA in the other species is in S. uvarum (2.6) [11], with the level of significance of 1.6 in S. pombe having remained unchanged for the most part (Figure 2, Supplementary File S2). The highest level of significance of the mitochondrial DNA in S. cerevisiae is in S. uvarum (2.3) [11], with the level of significance of 2.4 in S. cerevisiae having remained unchanged for the most part (Figure 2, Supplementary File S2). The lower level of significance of the mitochondrial DNA in S. uvarum is in S. uvarum (2.1) [11], with the level of significance of 1.5 in S. cerevisiae having remained unchanged for the most part (Figure 2, Supplementary File S2). In contrast, the highest level of significance of the mitochondrial DNA in S. uvarum (2.7) has remained unchanged for the most part (Figure 2, Supplementary File S2).

The most significant difference between the mitochondrial DNA databases in S. cerevisiae and S. pombe was the lowest level of significance of the mitochondrial DNA in the other species (Figure 2, Supplementary File S2). The lower level of significance of the mitochondrial DNA in the other species is in S. uvarum (2.2) [11], with the level of significance of 1.7 in S. cerevisiae having remained unchanged for the most part (Figure 2, Supplementary File S2).
