#  Biochemical Characterization of Human Papulocyte Metabolites According to Molecular Modeling and Transcriptomic Analysis
Quaglee Dragontacos


## Abstract
The surface of fungal colonies on surfaces is a structure that uniquely (but not always) adapts to changing densities, thereby allowing them to survive in hostile environments. The UV-dependent dimorphic transition from the typical hyphal to smooth transition is typically observed in many fungal species. Because of the interaction with the environment, a variety of phenotypic and phenotypic plasticity phenotypes have been described. In this study, we characterized two putative model proteins (PP-Epig1 and HD-Mpk2) in the ascomycete Dictyostelium discoideum. These proteins act as putative UV-dependent dimorphic signal transduction (NTL) signals for hyphal growth, asexual development, and smoothness. The HD-Mpk2 PEG forms a tight subpopulation around the NTL and a wide subpopulation around the UV-dependent dimorphic transition.


## Introduction
Its role in cell metabolism was also investigated in vivo [21.

MyoMet is a large protein family consisting of about 70 protein-coding genes. It is highly conserved among other proteins of the major classes (e.g., the ATP binding protein [22]), and is present in eukaryotic cells in the mitochondria and in the cytoplasm of all eukaryotes. It is present in many organisms, including the yeast and mammals (for review see [23]). MyoMet plays a key role in the production of energy and other components for cell metabolism, and is involved in the evolution of eukaryotic cells. The MyoMet protein family has been characterized in detail in [24].

MyoMet proteins are found in all eukaryotes, and they are classified according to the physiological role. In the model yeast Saccharomyces cerevisiae, the MyoMet protein family is found in the core proteins of the core metabolic network, i.e., the glyoxylate cycle, fatty acid metabolism, and protein catabolism. In the pathogenic fungi Candida albicans, the MyoMet protein family is present in the core metabolic network and in the cytoplasm of the main metabolic modules, including lipid metabolism, amino acid metabolism, and protein catabolism. The MyoMet family is not found in other eukaryotic species, and they are thought to be not regulated by the core metabolic network [24].

In the case of eukaryotes, there is growing evidence that some myoMet proteins regulate important metabolic processes in a molecular manner, e.g., in response to stress and nutrient sensing, and may be involved in the formation of metabolites for metabolic processes [25]. The MyoMet complex is involved in the regulation of gene expression, gene architecture, and other cellular processes. In the case of fungi, the MyoMet proteins are found in the nucleus, in the mitochondria, in the mitochondria of DSC and in the cytoplasm of various eukaryotes [26]. The MyoMet complex regulates the sexual reproduction system of fungi.


## Methods
digitatum was incubated with 2 mM H_2O_2 at 37°C for 3 h and then centrifuged at 12,000 × g at 4°C for 15 mi. The pellet was re-suspended in 1 mL of water and centrifuged at 12,000 × g for 15 min. The pellet was re-suspended in 1 mL of ice-cold 1× PBS and centrifuged at 12,000 × g for 15 min. The pellet was resuspended in 0.9 mL of 0.9% NaCl, and centrifuged at 12,000 × g for 15 min. The supernatant was removed and the pellet was re-suspended in 1 mL of water. The absorbance was measured at 490 nm in a spectrophotometer at a wavelength of 570 nm. The proteins were detected in p-nitrophenyl phosphate (pNP) at pH 4.0 with 0.1% Tween 80, and in p-nitrophenyl phosphatidyl phosphate (pNPP) at pH 3.5 with 0.1% Tween 80. The p-nitrophenyl phosphatidyl phosphate ratio was calculated using the following formula:where M_pNP was the protein concentration, PNP was the protein concentration in 1 mL of water, and NPP was the protein concentration in 1 mL of water. All measurements were repeated three times.

To detect the amount of nucleotide substitutions, the peptides of pNP, pNP, and NPP were excised from the surface of the protein using a NucleoSpin Protein Kit (Sigma-Aldrich Co., St. Louis, MO, USA). The peptides were subsequently subjected to immobilized gel electrophoresis (IPG) and transferred to an Immobiline-Sepharose membrane (Millipore, Billerica, MA, USA). The membrane was blocked with 1% bovine serum albumin (BSA) for 1 h at room temperature. The membrane was incubated with 0.2% Triton X-100 for 1 h at room temperature and then incubated with the anti-P. digitatum M-64 polyclonal antibody (1:1000 dilution) at room temperature for 1 h. The membrane was washed with PBS and blocked with 1% bovine serum albumin (BSA) for 1 h at room temperature.


## Results
All the enzymes involved in the synthesis of cell wall components, such as chitin synthase, glucan and glucuronoxylomannan, ß-glucan synthase, glucan synthase and glucan deacetylase were annotated in the NCBI databas. We found that the production of chitin in the cell walls of N. fischeri is regulated by the ß-1,3-glucan synthase (Fig 3A). The production of chitin is mediated by a conserved regulator of the ß-1,3-glucan synthase, phosphofructokinase (Fig 3B). The phosphofructokinase (PK) is a molecular target of phosphatidylinositol 3-kinase (PI3K) and is found in the cell wall of fungi and is required for the synthesis of ß-1,3-glucan (Fig 3C). The degradation of ß-1,3-glucan by phosphofructokinase is dependent on the production of ß-1,3-glucan (Fig 3D). In addition, it is possible that the synthesis of chitin may be regulated by phosphatidylinositol 3-kinase, which is an important and well-characterized enzyme for chitin deacetylase production (Fig 3E). As a result, we hypothesized that the production of chitin in the cell wall of N. fischeri may be regulated by the ß-1,3-glucan synthase.

The expression of the inducible phosphofructokinase (PfK1) in N. fischeri and other fungi was analyzed. The expression of PfK1 in N. fischeri was induced in response to the glucan extract, while the PfK1 expression in N. fischeri was only induced in response to the glucan extract (Fig 4). The PfK1 expression in N. fischeri was not induced in the presence of the cell wall inhibitor 2-deoxy-glucose (2-DG) at the same time point (Fig 4). The PfK1 expression in N.


## Discussion
The results of RNA-seq and protein-coding gene analysis revealed that human Papulocytes have a low abundance of protein-coding genes (Pigs: P4-p19, P5-p19, P8-p20, P10-p20, P9-p21, P14-p22, P17-p23, P14-p18, P20-p20, P8-p20, P11-p20, P12-p20, P15-p20, P13-p22, P15-p21, P17-p23, P15-p22, P15-p23, P15-p24 and P17-p23. The most abundant family is the P7-p19 family (P4-p20, P5-p20, P8-p20, P10-p20, P9-p20, P11-p20, P12-p20, P15-p20, P16-p20, P17-p23, P15-p22, P15-p23, P17-p23, P17-p23, P15-p23, P15-p23, P15-p24 and P17-p23, respectively), which are present in other human families (Table 1).

The low abundance of protein-coding genes in human Papulocytes suggests that human papulocytes are primarily derived from non-specific exocrine glands. The expression of Papulocytes has been shown to be regulated by exogenous cortisone, which is thought to stimulate the differentiation of Papulocytes in vivo [35]. Moreover, human papulocytes were recently shown to express corticosteroids and a transcription factor that controls the expression of immunoglobulins in response to immunization [36]. The human papulocytes in this study were analyzed using RT-qPCR and RT-PCR were performed by means of RT-qPCR in vitro. The results showed that the human papulocytes in this study were classified into 11 groups according to their composition (Table 1).

In summary, these findings suggest that human Papulocytes are distinct from other phagocytes.
