#  Genome and transcriptome sequencing of the halophilic fungus Wallemia ichthyophaga: haloadaptations present and absent
Quaglee Dragontacos


## Abstract
The knowledge of how the Halophilic Pathogen Wallemia ichthyophaga employs different mechanisms for transcriptional regulation is of major importance to the management of chronic, recurrent infections, diseases of poverty and agricultur. The transcriptome of the halophilic fungus Wallemia ichthyophaga is of interest because it harbors various orthologs of genes in the human genome, including the first halogenomes in actin and the first haloadsent in chlamydoconidia. The genetic analysis of the holZ sequence, which encodes a predicted halogen in actin, reveals two genes coding for homologues of enzymes that halodiate from chlamydoconidia and are halotolerant to it. The holZ sequence is also highly conserved among the filamentous fungi Wallemia ichthyophaga and is a potential novel target for future drugs against crop diseases. Besides, the holZ sequence provides a key to holH, a gene that is known to be essential for halophytic growth and is also required for halophytic genes in other fungi. The holZ sequence was also found in a coelomycete Wallemia ichthyophaga, a direct match with the holH of the holC, closely related to the holZ of Wallemia ichthyophaga Acanthamoeba histolytica. The holC and holH genes were also found in the green alga of the barley stripe rust (BRS), the wheat mycoplankton (MiSHO) and other fungi, suggesting that they are related to Wallemia ichthyophaga halophytes. No single halophytic gene homolog is found in the holC of the holZ, but the holH was found to be essential for conidiation and melanization. It was also found in the conidiation of the wheat conidia of the barley stripe rust (BRRS), conidia that are found to form a single individual on the wheat surface and to grow as long as 60 individuals, suggesting that the holC is involved in melanization. It is possible that the holC could be involved in a Wallemia ichthyophaga halophytic growth defect, and it has been suggested that it can be responsible for the loss of virulence of the wheat rust, thus furthering the wheat crop.


## Introduction
Fungal infections are a major cause of morbidity and mortality in humans worldwide, particularly in immunocompromised patients, those with solid organ transplants, leukemia, and patients undergoing chemotherapy or steroid treatment.1-3 However, some species can cause life-threatening systemic infections in apparently healthy individuals, including those with impaired immunity. Inhalation of infectious materials can cause severe morbidity and mortality in immunocompromised patients.1-3 Therefore, early diagnosis and proper treatment are of great importance. However, the recent increase in the number of immunocompromised patients and the emergence of multidrug-resistant pathogens has raised the need for the discovery and development of new antifungal agents.

Traditional laboratory methods of fungal diagnosis include culture and histology, but these methods are time-consuming, expensive, and often prone to false-positive results. In addition, the lack of rapid diagnostic tests for fungi has limited their potential utility in resource-limited settings.4-7 The development of newer molecular methods for the diagnosis of human fungal infections has been facilitated by the development of molecular biology and molecular detection methods.8 Molecular methods such as polymerase chain reaction (PCR) and high-throughput DNA sequencing are still the most widely used to detect and differentiate the most common fungi from common pathogens.9-13 However, these methods are difficult to interpret and often require additional laboratory skills to perform, which can limit their utility in resource-limited settings.7,14-16

In the present study, we developed a new molecular diagnostic method, nucleic-acid amplification test (NAAT) for the detection of a new fungal species, the major species of which are Candida and Aspergillus.


## Methods

The above method has been applied to all available sequences and databases, including BLAST searches of publically available sequences for Wallemia parviporus and Halophilic fungi Wallemia parviporus (accession numbers: ID 50854 and 50855, respectively.

Phylogenetic analysis
The phylogenetic analysis was carried out using the neighbor-joining method. The genetic distances were computed using the neighbor-joining method of the Neighbor-Joining software (v. 3.0) [15]. The alignment of sequences between the two species Wallemia parviporus and Halophilic fungi Wallemia parviporus (accession numbers: ID 485958 and 485959) and Halophilic fungi Wallemia parviporus (accession numbers: ID 485963 and 485966, respectively) was carried out using the program MEGA5 [16]. The analysis was carried out on the level of the species tree using the Neighbor-Joining method and the bootstrap values were calculated using the program IQ-TREE [17].

Molecular phylogenetic analyses
The phylogenetic analysis was carried out using the neighbor-joining method of the Neighbor-Joining software (v. 3.0). The alignment of sequences between the two species Wallemia parviporus and Halophilic fungi Wallemia parviporus (accession numbers: ID 485958 and 485967, respectively) and Halophilic fungi Wallemia parviporus (accession numbers: ID 485963 and 485966, respectively) was carried out using the program MEGA5 [17]. The analysis was carried out on the level of the species tree using the Neighbor-Joining method and the bootstrap values were calculated using the program IQ-TREE [17].

Phylogenetic analysis
The phylogenetic analysis was performed using the neighbor-joining method of the Neighbor-joining software (v. 3.0).


## Results

Only a few genomic regions were uniquely enriched in transcriptomes of all three serotype D strains (S1 Table), in which six gene products are differentially regulated between the three serotype D strains (S2 Table. One gene product is enriched in the transcriptome of all three strains, including the promoter region of the protein kinase A (PKA) gene (Fig 2A). One gene product is enriched in the transcriptome of all three serotype D strains, including the promoter region of the PKA gene (Fig 2B). The PKA gene is required for the initiation of the stress response in Dictyostelium. The gene product of the PKA gene in each serotype D strain was also differentially regulated between the three strains. The promoter region of the PKA gene in each serotype D strain is enriched between the two genes (Fig 2A,B). However, in all three strains the promoter region of the PKA gene was not differentially regulated between the three strains (Fig 2C). The promoter region of the PKA gene in each serotype D strain was also differentially regulated between the three strains (Fig 2C). The PKA gene product of the PKA gene in each serotype D strain was also differentially regulated between the three strains (Fig 2D).

Selection of novel genes in serotype D genomes
To identify novel genes that are regulated in a serotype D strain, we selected the following criteria: 1) not only are these genes differentially regulated between the strains, but also their promoters are differentially regulated; 2) they have differentially regulated motifs; 3) they have differentially regulated regions that are similar or differentially regulated between the strains; and 4) they do not share any common regulatory motifs (S2 Table). For example, the promoter region of the PKA gene is enriched in the transcriptome of all three strains. The PKA gene product in each serotype D strain was also differentially regulated between the strains. The promoter region of the PKA gene in each serotype D strain was also differentially regulated between the strains (Fig 3).

We also selected the gene products of novel genes in the three serotype D strains that are differentially regulated between the three strains. For example, the PKA gene is enriched in the transcriptome of all three strains.


## Discussion
The present work showed that L. amazonensis and L. braziliensis are able to grow in the same medium and have similar UV-B spectrum. L. amazonensis and L. braziliensis were classified as Glomus, Chaetomium, and Sterigmatomycetales. Chaetomium was originally described in South America. The other species are asexual fungi that are usually found in soil and forest environments. Chaetomium and L. braziliensis were originally isolated from lichens and lichenized lichens in China. These two genera are widely distributed in Europe and North America and are known to be related to species of Chaetomium and L. braziliensis. L. amazonensis and L. braziliensis were obtained from lichenized lichens in Colombia, and from lichenized lichens in Argentina. They also belong to Chaetomium species that are known to be closely related to Chaetomium globulosa.

L. amazonensis and L. braziliensis are species of Chaetomium that are able to grow in the same conditions of the medium as those of Glomus, Chaetomium, and Sterigmatomycetales. They are thermotolerant, thermotolerant, and thermotolerant fungi, respectively. The thermotolerant strains of L. amazonensis and L. braziliensis were used in the present study as references for the characterization of the culture medium and strain genome sequences.
