#  Selection on gene pseudogene after exegetical divergence from CEV nucleariostasis
Quaglee Dragontacos


## Abstract
In this study, we identified several essential and non-essential sulfur amino acids (SAs) in the lichen thalli of Guizhou National Nature Reserve (BNR), located in southwestern China. These lichens exhibit a wide variety of thalli with different vegetation types including pine, oak, and oak forest. To identify SAs, fungi-related and non-neuronal SAs were extracted by qPCR. The gene expression of the SAs were analyzed by RNA-Seq. The function of SAs was investigated by immunoblotting and mass spectrometry. The results indicated that SAs were the dominant and most abundant SAs in the lichen thalli. The expression levels of the SAs in lichens were higher than that of the typical and negative thalli. This study indicated that the high activity of SAs in the thalli is a source of genetic adaptations to the lichen thalli and fungi that use the lichen wood for biomass production.


## Introduction
Invasive fungal infections (IFI) are a significant cause of morbidity and mortality in immunocompromised patients. Immunosuppressive agents, such as amphotericin B, flucytosine, and 5-flucytosine, are commonly used to treat IA. However, the high cost, limited availability, and frequent use of antifungal drugs in the clinical setting may limit their use. In the past two decades, many novel antifungal agents, including echinocandins, have entered the clinical pipeline. These agents inhibit fungal cell wall synthesis by inhibiting the ß-glucan synthase enzyme (GSS1) and inhibitory ß-glucan biosynthesis, respectively. GSS1 is a major target of ß-fosmidomycin (ß-FBS) in fungi. However, previous studies have shown that ß-FBS is not the sole target and that other antifungal agents, including echinocandins, can also inhibit ß-FBS. Moreover, a combination of these antifungal agents with other antifungals can increase the antifungal spectrum and reduce the side effects. Moreover, most antifungal agents, including amphotericin B and 5-flucytosine, are effective against ß-fungal pathogens (reviewed in Ref. (1)).

However, the molecular mechanisms that lead to the inhibition of ß-FBS by these antifungal agents are still unclear.


## Methods
Using multiple genomes for each gene and the phylogenetic tree, we inferred the species phylogeny with the use of three species of bacteria (Oxford Uniprot, Oxford UniProt, Oxford Mycology, and Sanger Institute) as well as two species of fungi (Abraxovomyces kaufmannii, Cryptococcus neoformans, and Dictyostelium discoideum) as outgrou. All gene names are from the recent publications on yeast (Matheny and Neidmann, 2015, 2015, 2016).

Molecular phylogenetic analysis
The amino acid sequence of the KATP gene was obtained from JGI GenBank (http://www.ncbi.nlm.nih.gov) using the Basic Local Alignment Search Tool (BLAST). For the phylogenetic analysis, we used the Maximum Composite Likelihood method implemented in the MEGA5 software (Tamura et al., 2011; Tamura et al., 2014). We aligned the sequence data of each gene using the ClustalW algorithm (Thompson et al., 1994; Taylor et al., 1997). We used the C-terminal amino acid substitution model and a gamma distribution with a constant divergence time of 9.8 log generations. All characters were unordered and given equal weight; analyses were performed using gaps as indicated. For the MCMC analysis, all sequences with at least 100 nucleotides in length and with a minimum complexity of 5.0 were included in the analysis. The maximum likelihood method was used to estimate the evolutionary distances between the samples.

For the analysis of the phylogenetic relationships of the three KATP gene families, we used a Maximum Composite Likelihood method implemented in the MEGA5 software (Tamura et al., 2011). We used the C-terminal amino acid substitution model and a gamma distribution with a constant divergence time of 9.8 log generations. All characters were unordered and given equal weight; analyses were performed using gaps as indicated. For the MCMC analysis, the maximum likelihood method was used to estimate the evolutionary distances between the three KATP gene families.

Phylogenetic relationships
The obtained phylogenetic relationships were drawn from a maximum likelihood method. The trees were drawn with the program MEGA 5 (Tamura et al., 2011). We used the Kimura-2 value to estimate the divergence time between the three KATP gene families.


## Results
neoformans, we compared the gene content of the region surrounding the centromeres with those of the centromeres of the non-encoding regions of known function in other fung.

Genome sequences for the centromeres of the non-encoding regions of the other fungal species were similar (Figure 4A and B). Both centromeres and non-encoding regions are very similar in size, with similar centromeres being 45.6 and 47.5 Mb, respectively (Figure 4C and D). The centromeres of C. neoformans and S. cerevisiae are 100% identical (Figure 4E). Furthermore, the centromeres of all other fungal species (including S. cerevisiae, C. albicans, C. albicans and A. fumigatus) are 100% identical (Figure 4E). The centromeres of all other fungal species are also very similar in size, with comparable centromeres being 93.6 Mb and 99.7 Mb, respectively (Figure 4F). The centromeres of all other fungal species are also very similar in size, with comparable centromeres being 80.6 Mb and 87.9 Mb, respectively (Figure 4G). Interestingly, the centromeres of all other fungal species are also very similar in size, with similar centromeres being 76.3 Mb and 75.2 Mb, respectively (Figure 4H).


## Discussion
This approach allowed us to define the functional functions of the genes in this regio.

Gene families within the genome
The analysis of the genomic sequences revealed a total of 5837 gene families in the S. sclerotiorum genome (Table 2 and Additional file 7). The number of genes (213) represented the largest number of gene families in the whole genome. The first and second largest families of gene families were found in the lineage leading to the second daughter line. These families were split into two clusters based on the sequencing depth, including the gene families that are involved in gene regulation and the families that are involved in gene transcription. These two clusters clustered into two clusters, which may be of interest to future comparative genome-wide analyses.

The genes in these two clusters, which are not included in the gene families that are present in the S. sclerotiorum genome, are likely to have been lost in the early diverging population of S. sclerotiorum, which resulted in its highly-conserved form (Fig 1). The remaining gene families contain only those that are involved in protein trafficking, transcription and translation, and these include genes that are present in the early diverging population, such as the genes encoding the siderophore transporters, the N-glycosylation enzymes, the genes encoding the enzymes involved in the production of virulence factors and the genes that are involved in the formation and/or maintenance of virulence structures in these populations [30].

The gene families in the S. sclerotiorum genome are also well-known for their roles in development, pathogenicity, and infection of plants [30]. The genes involved in the development of the plant infection process are the main ones, in particular the genes involved in the formation of insect hyphae, the expression of hypha-specific genes and the production of sexual structures. The expression of genes involved in the pathogenicity and infection processes of the plants are the most studied genes. Some of the genes involved in the infection processes of the plants are also involved in development and in the development of secondary metabolism.

Among the gene families that are involved in the production of virulence factors, the genes involved in the production of pathogenicity-related genes are also present in the S. sclerotiorum genome.
