# A Comprehensive Analysis of Transcripts Orchestrates Formation and Phosphorylation in Fungi
Quaglee Dragontacos


## Abstract
Although little is known about the structure and function of the F-box protein Fb01 for intracellular localization and synthesis in filamentous fungi, it is thought that Fb01 is an important component of the extracellular compartment of the fungus. To gain insight into Fb01 function, two Fb01 homologs were fused to a homologous protein in Saccharomyces cerevisiae Fc (CBS 78580), a highly abundant downstream component of the extracellular matrix. Both homologs were found to reside within the fungal envelope and form double-membrane non-coding RNA molecules (DNMs). As a proof of principle, analysis of C2H-imaging showed that both homologs encoded mature forms of the Fb01 protein, and their double-membrane counterparts, including trimethylated forms, homologous integron forms, and un-mated integron forms, were found. In addition, a Fb01-like HSP70 protein was found as only one of eight Fb01-like proteins present in the Saccharomyces cerevisiae genome. The Fb01-like proteins were also found in only two of the three dimorphic Cryptococcus species, indicating that they are not surrounded by an external cell wall. Taken together, these results indicate that Fb01 is a promising target for the development of antifungal drugs.


## Introduction
o.org/10.1016/j.cell.2014.09.01, (Seo et al., 2014, 2015).

In this work, we use the biomass-based approach to investigate the functional diversity of the FungiBank database (http://afmb.org/fungi/bank/, accessed June 2015), including FungiBank1 (www.fungibank.org), FungiBank2 (www.fungibank.org/fungibank-2), and FungiBank3 (www.fungibank.org/fungibank-3). We find that the functional diversity of the FungiBank database is skewed toward the more active and functional PhosphorylTerminal (Ptc) and the presence of two non-overlapping amino acid sequences, F_20170204/F_20170206, F_20170207/F_20170207, F_20170208/F_20170209, indicate that there is little diversity at the periplasmic space, and that there is a high amount of protein which is translocated across the periplasmic space. Furthermore, we find that PhosphorylTerminalFungus (PUF) and PhosphorylTerminalFungusFusA (PUF-FUSA) are important for F_20170204 localization and localization in vivo, respectively.

PhosphorylTerminalFungus (PUF) and PhosphorylTerminalFungusFusA (PUF-FUSA) are found in the FungiBank database, which was first established in March 2013 (Seo et al., 2014, 2015). PhosphorylTerminalFungus (PUF) and PhosphorylTerminalFungusFusA (PUF-FUSA) are found in the FungiBank database (www.fungibank.org/fungibank-db/phosphorylTerminalFungus/), and are found in the Resources database (https://www.fungibank.org/resources/).


## Methods
cinerea
To characterize the transcriptome of the fungi in the Fungal Isolator Project, we used a complete transcriptome of . cinerea [B. cinerea (B. cinerea) (http://www.plantbiology.duke.edu/fungi/bioinformatics_server/index.html)] and the corresponding transcriptome of B. cinerea (http://www.plantbiology.duke.edu/fungi/bioinformatics_server/index.html). All transcripts were aligned using MAFFT (version 7.8; [26]) and aligned with Jalview (version 1.30; [27]). The alignment was manually corrected in Jalview and all transcriptomic data were aligned using SAM format (http://www.z-server.org/SAM_form.html).

Data sets were selected based on their functional annotation and assigned functional annotation by BLASTp analysis against the Pfam database (http://pfam.xfam.org/) [28]. The only exception was B. cinerea, which was used for the analysis. The annotation of B. cinerea was performed using the Blast2GO software (http://blast2go.sourceforge.net/) [29] and used to distinguish the functional categories of the taxa.

Sequence data set was further analyzed with a non-redundant set of taxonomic groups (OTU) from the abovementioned bacterial and fungal genomes, which were extracted from the UniProt database (http://www.uniprot.org/). The OTU sequences were compared with the different species of B. cinerea [B. cinerea (B. cinerea) (http://www.plantbiology.duke.edu/fungi/bioinformatics_server/index.html]) using BLASTn search. The OTU sequences were aligned with the BLASTn alignment program (version 2.2.8; [30]). All data were grouped with species and with the other species in order of phylogeny or genus and then the data were classified as “endophytic”.

Identification of species and the overall taxonomic groups
In order to determine the overall phylogenetic groups of the fungal genera of B. cinerea, we used the Unite database (http://unite.ut.


## Results
niger

A comprehensive analysis of the transcriptome data of . niger was performed using the RNA-Seq analysis software. The data were normalized to the log_2-fold change in transcript level of the total eukaryotic transcript (TEM) (Fig 3A) by using the expression analysis software program (http://pmc.csu.edu/pmc/). The expression of the gene was analyzed using the GO, HOG, and ERM (FunCat. pathway analysis) pathways. The GO and HOG pathway analyses revealed that the transcript level of the genes was significantly increased in the mutant strain (P<0.05, Fisher's exact test).

A total of 142 genes were significantly upregulated in the mutant strain (Fig 3B). The gene expression level was also significantly increased in the strain with an increase in the gene expression level in the mutant strain. The genes that were upregulated in the mutant strain were tested against the M. tuberculosis proteome database (http://www.medun.org/plat_db/). The analysis revealed that the gene expression level of the genes that were significantly upregulated in the mutant strain was up-regulated by 22-fold (P<0.05).

In order to further investigate the gene expression changes of the genes in the mutant strain, the expression levels of the genes in the mutant strain were compared with those of the wild-type strain. The expression levels of the genes in the mutant strain were significantly increased by 5-fold (P<0.05). The expression levels of the genes in the wild-type strain were significantly increased by 32-fold (P<0.05). The expression levels of the genes in the mutant strain were significantly increased by 17-fold (P<0.05) (Fig 3C).

In order to evaluate the role of the M. tuberculosis proteome in the expression of the genes in the mutant strain, the expression levels of the genes in the mutant strain were compared with those of the wild-type strain. The expression levels of the genes in the mutant strain were significantly increased by 12-fold (P<0.05) (Fig 3D). The expression levels of the genes in the mutant strain were significantly increased by 6-fold (P<0.05) (Fig 3D).

In order to evaluate the effect of the M.


## Discussion
We identified genes involved in the biosynthesis of various secondary metabolite. In addition to secondary metabolites, some genes involved in the synthesis of a secondary metabolite, secondary metabolite biosynthesis, and secondary metabolite metabolism were also found in the genome. Among these genes, five putative secondary metabolites were identified, including some novel metabolites. The biosynthesis of secondary metabolites is a complex process. For instance, C. albicans and C. neoformans have been reported to synthesize aldehyde dehydrogenase, which is the main enzyme in the secondary metabolite biosynthesis, and to reduce the production of secondary metabolites [33], [34]. In addition, the biosynthesis of secondary metabolites is also a complex process. For instance, A. fumigatus and A. terreus have been reported to produce aldehyde dehydrogenase, which is the main enzyme in the biosynthesis of aldehydes, and to reduce the production of secondary metabolites [35]. It is also important to emphasize that the biosynthesis of secondary metabolites was not found in the genome of A. fumigatus, which is a species of the Aspergillus genus. However, it is interesting to note that the second most abundant secondary metabolite in A.
