# A mutated BTK1 MAPK transcription factor regulates autophagy and pathogenesis in filamentous fungi
Quaglee Dragontacos


## Abstract
Two new species, Rhizoscyphus sp. (L.) Hentgen and R. sp. (L.) Barreto is described and illustrated. The new species is characterised by its ability to grow on submerged substrates and its exceptional tolerance to the toxicological effects of silica. The study also reveals that this fungus may also be able to tolerate high temperatures and high salt concentrations in addition to its innate immunity. The results of this study are based on phenotypical screening and comparison with other fungi of the order Basidiomycota, Cryptomycota, and Pucciniomycotina. All the available mycelial phenotypical data is provided for R. sp. (L.) Hentgen and R. sp. (L.) Barreto. A phylogenomic study of the new species and allied species, based on a combined dataset of partial LSU, ITS, tef1 and rpb2 sequence data, is provided. In the revision of this study, a total of 35.98% (195 of its 196 questions) were asked of R. sp. (L.) Hentgen, 27.50% (135 of its 232), R. sp. (L.) Barreto and 27.50% (198 of its 331) for the majority of the questions and the last two questions for all the ITS regions of the rpb2 and tef1 regions. For the tef1 region, 68.50% (362 of its 364) answered and for the 271 tef1 regions, 88.


## Introduction
An emerging pathogen, Cryptococcus neoformans, is a global threat to human and animal life [1,2]. C. neoformans is a facultative intracellular pathogen that causes life-threatening meningoencephalitis, mainly in immunocompromised hosts such as those with AIDS, cancer, and other chronic diseases, and it is a major cause of death in both immunocompetent and immunocompromised individuals [3]. Although Cryptococcus neoformans can persist in the air and soil, it can also cause a lethal meningoencephalitis in the presence of serum and cerebrospinal fluid (CSF), which can then be recovered by medical practitioners and the general population [4].

The ability of Cryptococcus to persist within macrophages is essential for the pathogen to penetrate the phagosome and cause infection [5,6]. Macrophages are classically activated by environmental stimuli such as cytokines and chemokines and are able to counterbalance the pathogen through the production of various mediators, including cytokines and chemokines [7]. In contrast, macrophages also exhibit an anti-inflammatory capacity, which can negatively affect the host defense mechanisms.


## Methods
The pYES4 vector was transformed into FB1 (a ura3<U+0394>::URA3 auxotroph) or FB2 (a ura3<U+0394>::URA3 auxotroph) transformant. The pYES4 vector was transformed into E. coli DH5a competent cells (Invitrogen) and used for the subsequent experiments. The pYES4 vector was transformed into FB1, and the pyrG-DG-strep-seq gene was inserted into E. coli DH5a competent cells.

Construction of a pyrG-DG-strep-seq library
The pYES4 vector was transformed into E. coli BL21. Plasmid pYES4 contains a pyrG-DG-strep-seq gene (a ura3<U+0394>::URA3 auxotroph) using lithium acetate. The pYES4 vector was transformed into E. coli BL21. Plasmid pYES4 contains a pyrG-DG-strep-seq gene (a ura3<U+0394>::URA3 auxotroph) using lithium acetate. The pYES4 vector was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli DH5a competent cells. The pYES4 vector was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli DH5a competent cells. The pYES4 vector was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli BL21. Plasmid pYES4 was transformed into E. coli BL21.


## Results
However, we also detected a trend toward a growth defect in the BTK1 mutant (Fig 4B. We next assessed the impact of the BTK1 gene on the autophagic pathway in an autophagic pathway mutant of the BTK1 strain (Fig 4C). Autophagic cells are heterochromatic cells that undergo autophagic cell-to-cell autophagy reactions (12). The BTK1 mutant exhibited a phenotype that is similar to that of the wild-type strain (Fig 4C). Next, we compared the growth of the BTK1 mutant in the presence of rapamycin or FK506 and compared the growth of the BTK1 mutant in the presence of autophagy inhibitors. As shown in Fig 4D, the BTK1 mutant was unable to grow in the absence of rapamycin or FK506, while the BTK1 mutant exhibited increased sensitivity to autophagy inhibitors in the absence of rapamycin or FK506. The growth defect of the BTK1 mutant was consistent with the growth defects in the wild-type strain (Fig 4C), and thus suggests that the autophagic pathway of the BTK1 mutant is impaired in the presence of autophagy inhibitors.

It is important to highlight that although autophagy is not impaired in the presence of autophagy inhibitors, the growth defect of the BTK1 mutant was associated with a growth defect. Moreover, growth defects in the BTK1 mutant were not associated with an impaired growth of the wild-type strain, indicating that the growth defect in the BTK1 mutant was not due to impaired growth of the wild-type strain. These results indicate that the loss of the BTK1 gene impairs autophagy, and not the impaired growth of the wild-type strain. These results suggest that the loss of the BTK1 gene impairs the autophagic pathway of the BTK1 mutant.

The growth defect in the BTK1 mutant was associated with a reduced number of cells in the autophagosome (Fig 4E). As shown in Fig 4E, the number of cells in the BTK1 mutant was reduced compared to that in the wild-type strain.


## Discussion
In . cerevisiae, a Hog1p-regulated MAPK pathway is required for regulating autophagy in the presence of glucose [41]. In contrast to the HOG pathway, autophagy is regulated by the MAPK pathway in fungi.

In the present study, we identified a TATA-binding protein, that has been reported to be associated with autophagy in the C. neoformans genome [32]. It is the third TATA-binding protein in C. neoformans and is involved in regulating the autophagy in S. cerevisiae [32], [43]. In C. albicans, a TATA-binding protein (TATAG_01763) is involved in regulating the autophagy in the presence of glucose [44]. In C. neoformans, we showed that TATAG_01763 is required for autophagy in the presence of glucose. In S. cerevisiae, the Ssa1-LsA/Tsa3-LsA homologue in C. neoformans is required for autophagy in the presence of glucose [30], [31]. In C. albicans, the Cca1-LsA/Tca2-LsA homologue, in C. albicans, is required for autophagy in the presence of glucose [30], [31]. In contrast, the Cca1-LsA/Tca2-LsA homologue, in C. neoformans, is required for autophagy in the presence of glucose [30], [31]. In C. albicans, the Cca1-LsA homologue, in C. albicans, is required for autophagy in the presence of glucose [30], [31]. In addition to autophagy, the Cca1-LsA homologue, in C. albicans, is required for autophagy in the presence of glucose [30], [31]. In C. neoformans, the Cca1 homologue, in C. albicans, is required for autophagy in the presence of glucose [30], [31].

In the present study, we found that the C. neoformans MAPK pathway, which is involved in autophagy, is regulated by the MAPK pathway.
