# A pilot study showing differences in gene content between Strains with and without Agaricus bisporus
Quaglee Dragontacos


## Abstract
Strains were isolated from urine samples of 176 flies collected between August 2004 - April 200. All the Strains were male or female and the median age was 6-12 years old. Two-third of the Strains were sterile. The remaining Strains were considered sterile by the admitting test. The terminal length of the new flies was longer than the flies from previous flies and these flies were more likely to have the presence of a yeast. The terminal length of the RNA in RNA of one-third of the RNA from the remaining flies was longer than the RNA from the remaining flies. The effects of each Gene on the growth of the Strains were analyzed by the minimum inhibitory concentration (MIC) test and growth of the new flies was determined using the commercial Stata and the commercial C-YMO moves.

The results showed that Agaricus Agaricus-Streptococcus metacestinos MIC50 = 0.05 mg/mL inhibited the growth of Strains and no effect was observed on the RNA in RNA of any Strains.

Our results showed that the RNA in RNA of the majority of the new flies was longer than that of the previous flies and this was associated with a higher inhibition rate. These results suggest that the RNA in the remaining Strains was longer than that of the previous flies. The susceptibility of new flies to Agaricus Agaricus-Streptococcus metacestinos is related to the shorter RNA in RNA of the majority of the new flies.

The use of next generation sequencing technologies has allowed the identification of the majority of the genes involved in the metabolism of the cell cycle. This technology is applied in the development of drugs and in the control of bacterial infection.

In this study, we found that the expression of the pathogenesis-associated genes (PDA, CD, NAD, NAD+C, PGE2, and ERG11) in the mutant strains of the Stereotacticus stevensonii strain and the strain with the cleavage defect of the conserved protein kinase A (PKA) were downregulated. We also observed that the transcription of some of the gene expression in the new strains was suppressed.


## Introduction
The term “yeast” has been used to describe yeasts that are isolated from submerged yeasts in natural environments (e.g., rivers) or fresh water (e.g., lakes) and that grow on agar slants or raw materials such as vegetables, fruits, and/or grains (e.g., wheat) (Jahur and Dibburge 1991; Festa and Samer 2014). Yeast species identified in submerged water include Torula sp. (Sapindyan 1984), Agaricus bisporus (Zhang et al.


## Methods

To compare the gene content of these two species, we used a population-based approach, as described in [13. The housekeeping gene set that we used for the genome sequencing was either FTR1 (strain of F. graminearum) or GAPDH (strain of F. graminearum). For the gene set, we used the sequences from the NCBI (GenBank) and Fungal Genome Resources (FGPCR) databases. For the gene set of F. graminearum, we used the sequences from the CBS-KNAW Fungal Biodiversity Centre (CBS) (GenBank accession numbers JQY485096-JQY486969 and JQY486989-JQY486992, respectively) and the Fungal Genome Resources (FGPCR) database (GenBank accession numbers JQY485096-JQY496994, respectively). We used the Blastn algorithm to search for sequences with a quality score of > 80% with a P-value of < 0.05. The analysis was performed using the CLUSTALW program and a sequence similarity score of > 1.8 was considered as a good match. The relative gene expression values were calculated using the modified Cufflinks method, as described in [14].

In order to determine the gene copy number, we used the GeneMark-ES v. 1.4.5 [15]. Gene copy numbers are defined as the number of copies of a gene in the genome in the given strain over the total number of copies of that strain. For each strain, we used the following formula:

where G(A) is the number of copies of the gene of the strain of interest and G(A) is the number of copies of the gene of the strain in the given strain over the total number of copies of that strain.

To determine the copy number of Agaricus bisporus, we used the gene copy number of C. elegans, as previously described [16].

To determine the copy number of C. elegans, we used the gene copy number of C. glabrata, as previously described [17].

To determine the copy number of C. krusei, we used the gene copy number of C. lusitaniae, as previously described [18].


## Results
neoformans (23) (Table 3. All three Agaricus bisporus strains showed similar expression levels between the two series of strains.

Proteome analysis of Agaricus bisporus and S. cerevisiae
The genome of S. cerevisiae is characterized by three predicted proteome models (18) with the most stringent criteria for good quality. The model was validated using BLAST (18) against the databases (19) and the results are summarized in Table 4. The sequences of the predicted proteome models for Agaricus bisporus and S. cerevisiae were compared using BLAST and the best hit was the proteome model of S. cerevisiae ATCC 27853. The proteome model of S. cerevisiae was used to validate the results obtained from the proteome analysis. The best hit for both Agaricus bisporus and S. cerevisiae was S. kluyveri, whereas the best hit for S. cerevisiae was S. brasiliensis. A total of 168 proteins were found to be differentially expressed between the two Agaricus strains. These proteins were grouped in a cluster (Table 5) based on their biological functions (see Table 3) (Fig 4).

The protein sequences of Agaricus bisporus and S. cerevisiae proteins were compared using InterProScan analysis. The results showed that the protein sequences in the three Agaricus strains were very similar to each other, except for the dicer protein of S. cerevisiae (Table 6). The protein sequences of S. kluyveri and S. brasiliensis were different from each other (Table 6), but the protein sequences in Agaricus bisporus and S. kluyveri were identical to each other (Table 6). The protein sequences of S. kluyveri were the most similar to Agaricus sp. A5 (Table 6). The protein sequences of S. brasiliensis were the most similar to Agaricus sp. A10, whereas Agaricus sp. A15 and Agaricus sp. A12 were the least similar to S. cerevisiae. The protein sequences of Agaricus sp. A5 and S. kluyveri were the least similar to S. kluyveri and Agaricus sp. A10 (Table 6).


## Discussion
s. harringtoni. This study provided preliminary evidence to show that differences in gene content might be related to the genetic origin of different strains.

The results of the genome assemblies obtained with the ITS1-2.8GE-LSU-LSU-LSU dataset showed that the strains obtained in this study differ in their gene content and the spectrum of changes in their genes. These results are in accordance with other studies which have demonstrated that differences in gene content occur at the genome level in different strains of different fungi (Farr and Rossman, 2008; Nagy and Casadevall, 2013).

Species of the genus Termitomyces have been observed in various habitats, including the sea, wood, and plants (Inderbitzin et al., 2001; El-Aydouni et al., 2002; Mayer and Wacker, 2010). However, several species of Termitomyces are not common in the soil, and they are known to be endophytes and pathogens of plants (Rédou et al., 2012). The genus Termitomyces includes a wide variety of species that produce mycoparasitic fungi, but in different ecological niches (Mayer and Wacker, 2010). The plant pathogen T. atroviride is an important human pathogen that causes tree hollow rot (Lai et al., 2002; Otterhuys et al., 2013). The pathogen of rice, T. betulin in Thailand, is able to infect apple, pear and strawberry (Bakan et al., 2008; Xin et al., 2011). The strains of T. betulinica and T. betulinica-like species are widely distributed in the soil and in different locations (Eisenman et al., 2002; Alam et al., 2002). The current study revealed that Termitomyces species are not a common source of carbon for human health and that they are involved in the resistance to heavy metals and in the adaptation of plants to diverse environmental conditions (Mayer and Wacker, 2010).

Morphological and ecological features of the Termitomyces in vitro
The growth of the Termitomyces species was observed in various habitats, including soil, flowers, fruit trees, and flowers (Figure 2). Within the soil, the Termitomyces species are distributed throughout the entire soil, and their growth is able to promote plant growth and increase the soil pH.
