# A web-based tool for annotated phylogenetic analyses of shrimp synovitis - A literature review
Quaglee Dragontacos


## Abstract
Cryptococcus gattii is a human fungal pathogen that has caused considerable interest in the medical mycology community because of its ability to disseminate and survive in host cells and ability to cause disease in immunocompromised humans. The encapsulated yeast can also undergo morphological changes that promote its survival in the host. In this study, we report a unique morphotype of the yeast that may have an additional role in fungal pathogenesis. A collection of 4 mutants was selected to investigate the role of this unique morphotype. One of these mutants contained a unique switch that enabled the yeast to survive in host cell cultures, while another mutant contained a nonhomologous switch that enabled it to undergo a morphological change. Overall, these results show that the Cryptococcus species complex contains distinct cellular functions that enable it to survive in host cells and to undergo morphological changes during infection.


## Introduction
Phaeohyphomycosis is a chronic, invasive fungal infection of the skin and mucosa caused by inhalation of conidia (spores). It is considered to be one of the major etiologies of skin infection, particularly in HIV/AIDS patients. It is caused by inhalation of a wide range of fungi, including Aspergillus, Candida, Cryptococcus, and other uncommon yeasts, such as Blastomyces, Coccidioides, and dimorphic fungi.


## Methods
The reviewed articles were selected by authors to determine the best citations of their article. The relevant articles were included in the final list. We manually curated all the references and to this end we included references from previous studies (e.g. [5,7] and [9] in addition to those published in the current study). We also included references in the final list to keep all the relevant data and the search criteria. The results of the different searches were combined and then combined to form a combined dataset. The obtained datasets were then subjected to an in silico search of the related databases. For the phylogenetic analysis, the sequences of the selected proteins were aligned with MAFFT v.7.310 [10] and manually edited using the BioEdit software v.7.0.9.0 [11].

Phylogenetic analysis
The aligned sequences and phylogenetic trees were analysed using the neighbour-joining (NJ) method with MEGA v.7.0 [12]. The evolutionary distance matrix was generated using the Kimura-2-parameter model with jModelTest v.2.1.1 [13]. The reliability of the obtained model was verified by conducting bootstrap tests with 1000 replicates. The maximum composite likelihood (ML) of the obtained model was calculated with RAxML v.7.0.3 [14]. ML parameters were set as follows: the distance matrix was concatenated; the minimum evolution time; the rapid substitution model; and the proportion of invariable sites. All the datasets were statistically analysed using the non-parametric likelihood method with the general time reversible model (GTR+I+G). The clade topology and the topology of the branching tree were determined by the Kimura-2-parameter model and the same tree was used for the maximum posterior probability (MP) analysis. The MP analysis was carried out with jModelTest v.2.1.1 [13].

Data availability
The raw sequences and data sequences of the different strains and species can be obtained from GenBank (accession numbers NCCLS017790 and NCCLS0152689, NCCLS0152690 and NCCLS0152689 respectively). All the taxonomic information of the different strains and species can be found in the relevant references.


## Results

Taxonomic classification
Diversitoids are new genera in the family Periconi. Diversitoids have been proposed to accommodate a new species of the Periconia clade, which was described as a member of the Periconiaceae [46,47]. The delimitation of Diversitoids is supported by the fact that several delimitation criteria have been proposed for the Periconiaceae. Diversitoids are often formed in the middle of an encrustation, but usually not in the upper part of the lesion. In order to determine the delimitation of Diversitoids, taxa were established in two or more species. In order to evaluate the species delimitation criteria, the Diversitoid classification was applied. The Diversitoid classification is based on the principle that the delimitation of a single species is based on the premise that the delimitation criteria are based on the phylogenetic relationship of the taxa. Diversitoid species are described by their characters including colorations, conidiophores, conidia and conidia. The Diversitoid species are named in accordance with the geographical location of the species. The taxonomic status of the Diversitoid species is presented in the “Discussion” section. The taxonomic status of the Diversitoid species is presented in the “Notes” section. The Diversitoid species are described as members of the Periconiaceae or by their cultural characteristics as well as their status as plant pathogens. The Diversitoid species are listed in the family Periconia in the literature and are described as “Periconiaceae”.

Taxonomic classification
The taxonomic status of the Diversitoid species is presented in the “Discussion” section. The Diversitoid species are described as Periconiaceae, but the taxonomic status is presented in the “Discussion” section. The Diversitoid species are described as members of the Periconiaceae or by their cultural characteristics as well as their status as plant pathogens. The Diversitoid species are listed in the family Periconiaceae or by their cultural characteristics as well as their status as plant pathogens.


## Discussion
A total of 6.7% of the total genomes identified are annotated as synovial genomes, compared to 21.6% for S. sclerotiorum [13]. The former is significantly smaller than the other two species (Fig 1), and the former is much smaller than the one observed for S. sclerotiorum. The large number of genes (32.2 Mb vs. 3.8 Mb), the genome sizes of S. sclerotiorum (16.4 Mb vs. 17 Mb), and S. sclerotiorum (23.3 Mb vs. 29 Mb) indicates that the common ancestor of the four species was relatively compact and therefore that the S. sclerotiorum genome was a much smaller size than the S. sclerotiorum genome (Table S2).

The genome size of S. sclerotiorum is 21.6 Mb vs. 24 Mb of S. sclerotiorum (Table S2). The second largest size (15.1 Mb vs. 7.4 Mb) is the genome of S. sclerotiorum (14.6 Mb vs. 7.6 Mb). The third largest size (12.4 Mb vs. 15.2 Mb) is the genome of S. sclerotiorum (11.4 Mb vs. 7.1 Mb). The fourth largest size (18.1 Mb vs. 21.4 Mb) is the genome of S. sclerotiorum (15.5 Mb vs. 12 Mb). The highest genome density (1.35 Mb vs. 0.03 Mb) is the genome of S. sclerotiorum (8.8 Mb vs. 7.4 Mb). The genome density of S. sclerotiorum is also larger than that of S. sclerotiorum (4.0 Mb vs. 2.2 Mb).

The genome sizes of S. sclerotiorum and S. sclerotiorum differ by only 9.8 Mb and 14.8 Mb, respectively, and the genomes of S. sclerotiorum and S. sclerotiorum are larger than those of S. sclerotiorum and S. sclerotiorum (Table S2). This genome size difference in gene content and translocation might be because the genomes of S. sclerotiorum and S.
