# An oligodeoxynucleotide-based approach for the identification of microRNAs in eukaryotes and its application in the host microbiome
Quaglee Dragontacos


## Abstract
Fungi are major contaminants of crops and ecosystems throughout the world. However, little is known about the roles of fungi in these processes. The aim of this study was to investigate the virulence-related roles of the two basidiomycete species Cryptococcus albidus and Cryptococcus albidus agroacid in an ineria farm account to soybean. The ineria farm was to transform a field soybean (Triticum aestivum) to soybean (Wine Frit matters) in order to investigate the extent and associated virulence-related roles of these two species.

The transformant was purified sequentially using a Clevenger-type agar well diffusion method. The antimicrobial activity of the transformant, MEA (Metchno) was determined. The anti-virulence factors, BSA, MDA, and ZRT were identified.

Both transformant strains displayed an increase in fatality rate (38.5%) compared to the control strain (15.5%) when treated with the two drugs. The mean decrease in fatality rate was 27.8% (95% confidence interval [CI], 21.4-76.1%) when treated with ZRT. The transformation of a field soybean (Lachancea thermotolerans) revealed a significant decrease in the growth of the transformant strain (<U+0394>ztC) compared with the control strain (5.82%) when soybean was being grown at 35 °C in order to maximize the growth of the transformant.

This study revealed the importance of the two transformants in the selection of the two fungal species as the two transformant species were able to increase the fatality rate of the transformant compared to the control strain. These results suggest that both transformants are potential sources of novel treatments for the treatment of ineria farm account.


## Introduction
.


## Methods
 2015) is also supported in detail in Table . In this approach, a database of eukaryotic microRNAs was prepared using a combination of forward and reverse reads from the various sample libraries. The forward and reverse reads were discarded if the eukaryotic transcript was detected by the short read detection and the corresponding cDNA was detected with the short read detection function of the Illumina Hiseq 2000 platform. The hiseq tool is an in-built script for the Illumina Hiseq 2000 platform (Binwach et al., 2016).

Analysis of genes
The eukaryotic transcript was predicted by the Gene Expression Omnibus (GEO) database (version 2013.0) (http://www.ncbi.nlm.nih.gov/geo/) and the eukaryotic transcript by the Unigene Database (http://www.unigene.org/) (Langmead and Salzberg, 2015). The eukaryotic transcript was then filtered by the number of sequences with an e-value lower than 1e^-10. The genes with a p-value higher than 0.05 and a false discovery rate of 0.05 were considered as having a significant differential expression. The resulting data were then processed to generate the gene models.

Gene expression profiles of microRNAs
The transcript abundance of each microRNA was calculated as a function of the total number of reads mapped to the genome using the GenePilot software (Kirby, 2010). The GenePilot software is a tool for visualizing gene expressions of a microRNA using a set of thresholds that allows a comparison of gene abundance between two groups of samples. The threshold for gene expression change was set as the difference between the mRNA abundance of the two groups of samples.

MicroRNA transporters were selected based on their ability to rescue microRNAs from the environment. The selected microRNA transporters were identified based on their functional similarities to the microRNAs and their levels of RNA-dependent behavior (Vaidya et al., 2011).

Gene Ontology Annotation
The gene ontology (GO) was determined using the online software, GO annotations (http://www.expasy.org/) (Konstantopoulos and Spieth, 2015).


## Results
A representative sequence of the homing endonuclease (EST) domain was obtained using a homology search of the translated sequence of the predicted homing endonuclease from . sclerotiorum by using the Phylogenetic Species Database (PCLD) (Table 2) (23). The BLASTX algorithm revealed the homology of the EST sequences to the H. polymorpha EST database (http://blast.ncbi.nlm.nih.gov/) using the default parameters (23). The Phylogenetic Species Database (PDDB) was used to search for the homing endonuclease in the P. parietina genome database (Pfam database, 24). The E. coli EST sequence was used to search for the homing endonuclease sequence from S. sclerotiorum using the default parameters. The E. coli homing endonuclease sequence was used to search for the homing endonuclease sequence from S. sclerotiorum using the default parameters. The BLASTX algorithm revealed the homology of the EST sequences to the H. polymorpha EST database (http://blast.ncbi.nlm.nih.gov/) using the default parameters (24). The E. coli homing endonuclease sequence was used to search for the homing endonuclease sequence from S. sclerotiorum using the default parameters. The BLASTX algorithm revealed the homology of the EST sequences to the H. polymorpha EST database (http://blast.ncbi.nlm.nih.gov/) using the default parameters (24). The P. parietina genome database (PDDB) was used to search for the H. polymorpha homing endonuclease sequence from S. sclerotiorum using the default parameters. The E. coli homing endonuclease sequence was used to search for the homing endonuclease sequence from S. sclerotiorum using the default parameters. The BLASTX algorithm revealed the homology of the EST sequences to the H. polymorpha EST database (http://blast.ncbi.nlm.nih.gov/) using the default parameters (24). The P. parietina genome database (PDDB) was used to search for the H. polymorpha homing endonuclease sequence from S. sclerotiorum using the default parameters.


## Discussion
In this study, we present a global analysis of the impact of highly-affluent hosts on the epidemiology of cryptococcosis in a large-scale, multi-center cohort of patients with disseminated disease in Vietnam. This analysis provides insight into the first understanding of the epidemiology of cryptococcosis in the largest cohort of patients with disseminated disease in Vietnam. We found that the most-affluent host affected by C. gattii disease and C. neoformans infection was that of the HIV/AIDS patients, in addition to the host’s immune status. C. gattii infection is most commonly associated with immunosuppressed individuals in Vietnam, with a prevalence of approximately 35% in the first five years after HIV infection [15]. While this study does not include patients with AIDS, HIV infection or other immunosuppressed states, this finding emphasizes the need for further study of C. gattii disease in these patients.

Although C. gattii infection appears to be more prevalent in Vietnam, this infection has been associated with a more severe disease in the past several years. In a study from 1975-2010, C. gattii was most frequently isolated from patients with HIV, and the most common infection was C. neoformans infection [4]. There was a trend toward an increased incidence of C. gattii infection in Vietnam and the majority of isolates were recovered from patients with HIV infection. Although this study only included patients with HIV infection, more isolates were recovered from patients with cryptococcosis in Vietnam. This increased prevalence of C. gattii infection in Vietnam suggests that C. gattii infection in Vietnam may be linked to immunosuppressed individuals. This finding may be explained by the fact that HIV patients in Vietnam are more likely to have immunosuppressed status compared to other countries with similar HIV status.

C. neoformans is a potent agent in pulmonary infections in HIV-infected individuals [1], and patients are more likely to have an altered immune system response to C. neoformans infection. This is particularly true in individuals with CD4^+ T cell depletion.
