# Approaching the gut microbiome during pathogenesis: Lessons from human chronic phases and translational research
Quaglee Dragontacos


## Abstract
Current Bio. Chem. 276 : E6261s. https://doi.org/10.7554/ncbi.uif.06261s

Microbiome during pathogenesis: lessons from human chronic phases and translational research in rodents. Current Biol. Chem. 276 : E6261s.

Microbiome during pathogenesis: lessons from human chronic phases and translational research in rodents. Current Biol. Chem. 276 : E6261s.

Oomycete: A mammalian model of invasive fungal disease. Nippon & Folin.

A mammalian model of invasive fungal disease. Nippon & Folin.

The fungal pathogen C. neoformans is the major causative agent of fatal fungal meningitis worldwide. In this study, we investigated the gut mycobiome of C. neoformans and found that this host-pathogen interface had the strongest influence on the pathogenesis of C. neoformans infection. Our data suggest that gut mycobiome composition shapes the mode of interaction of C. neoformans with human pathogens and contributes to the development of complex multicellular organisms.

Most C. neoformans live-camo- or basidiomycete life-cycle are derived from the intestine, where they adhere and proliferate inside host cells. In contrast, the necrotrophic life-cycle derived from the gut mycobiome is produced mainly in the respiratory tract. Studies of fungal mycobiome have mainly focused on the respiratory tract, which is the primary site of interaction between fungal and host cells. However, the role of gut mycobiome in fungal pathogenesis of C. neoformans has been largely unexplored. In this study, we found that the in vitro interaction between C. neoformans and host mycobiome is required for mediating host-pathogen association and to contribute to the pathogenesis of this disease. Our study highlights the importance of the host mycobiome for the outcome of C. neoformans infection and suggests that modulation of the host mycobiome by specific antibodies could have a beneficial effect in the host's control of the disease.

IMPORTANCE The human gut microbiome is a dynamic mixture of microorganisms that establish a mutual interaction with the host cell.


## Introduction
Tuberculosis (TB) is one of the leading causes of morbidity and mortality in the world. In 2007, an estimated 10 million new cases and 1.5 million deaths were reported in the world [1]. The incidence of TB remains high, with an estimated annual incidence of 10,000 per 100,000 population. The World Health Organization estimates that approximately 10% of the world’s population is chronically infected with TB, and it is estimated that 10% of the TB patients in the United States die of the disease [2]. The complex interplay between the host immune system and the pathogen is an important aspect of the outcome of TB disease. The availability of tools for studying the immune response to infectious agents has greatly advanced our understanding of the host immune response to TB. In vitro assays using the monoclonal antibody (mAb) mAb mAb are routinely used to study the immune response to TB. The mAb mAb is a molecule that specifically recognizes the pathogen and induces the antibody response in the infected host. The antibody response to the TB infection varies depending on the mAb used. In vitro studies show that the mAb response is protective, while the antibody response is detrimental. In contrast, the antibody response is detrimental, leading to decreased disease severity and decreased disease progression [3].

The mAb mAb is highly specific and non-specific. Thus, mAb mAb can be used to study the immune response to TB. The mAb mAb has been tested against two types of bacterial pathogens, namely, Staphylococcus aureus, which causes infections in the upper and lower airways and, and Escherichia coli, which causes infections in the lower airway and central nervous system [4]. The mAb mAb has been approved by the U.S. Food and Drug Administration (FDA) for the treatment of adults with a cutaneous lesion and for the treatment of individuals with a cutaneous infection [5]. The mAb mAb has been approved by Food and Drug Administration (FDA) for the treatment of invasive pulmonary aspergillosis (IPA) in patients with a fever of up to 34°C, which is associated with the fungemia associated with IPA [6]. The mAb mAb is effective against the following Candida species: C. albicans, C. glabrata, and C. tropicalis [7].


## Methods

.2. Bacterial Bacterial Isolates
A standard bacterial community was selected from six bacterial strains belonging to the Gram-negative bacteria (Brugia, Bacillus, Bacillus, Bacillus, Bifida, Bifida, Bifida, and Bifida-resistant bacteria) and the Gram-positive bacteria (Escherichia coli, Salmonella, Staphylococcus, S. aureus, and Streptococcus). All bacterial isolates were cultured in Escherichia coli (G-20S), Lactobacillus (LB10), Streptococcus (LB10), and Salmonella (S10). Overnight culture of each culture was diluted and plated on LB medium to enumerate colonies. Isolates were identified by visual observation and identification of the lowest concentration that gave growth.

4.3. Antimicrobial Activity Assays
Antimicrobial activity of B. subtilis strains were assessed on LB broth medium, following the methods of Almeida-Marini et al. [22]. B. subtilis strains were inoculated onto potato dextrose agar (PDA) plates and incubated at 28°C for 24 h. After incubation, the plates were transferred to a sterile 96-well microtitre plate and washed three times with sterile distilled water to remove the bacteria. Next, B. subtilis strains were inoculated on the LB agar and incubated at 28°C for 24 h. The bacterial colony-forming units per gram of dry matter (CFU/g) was determined according to the method of Liu et al. [23]. The activity of the selected antibacterial compounds was also evaluated in a 96-well microtitre plate. The bacterial colony-forming units per gram of dry matter (CFU/g) were determined according to the method of Käm and Schiestl [24].

4.4. Antifungal Activity of B. subtilis Strains
The antifungal activity of B. subtilis strains was assessed on B. subtilis G1508, B. subtilis G1341, and B. subtilis G1339. B. subtilis G1341 was a kind gift from G. H. Man, United States Department of Agriculture, USA.


## Results
Our analysis also identified a total of 844 OTUs, including those for which only one sample was collected, and the number of OTUs per gut microbiome was low (Fig S3. Our sequencing and quality control procedures identified a total of 4,629 bacterial OTUs that were present in all three cohorts. The number of total bacterial reads per sample was considerably higher than in other studies that have reported a higher number of bacterial reads in the gut (15-25 OTUs) (17-24 OTUs) (18-24 OTUs) (19, 20). Our sequencing and sequencing quality control procedures also identified 6,356 bacterial OTUs that were present in at least one of the three cohorts. The number of total bacterial reads per sample was much higher than that reported in the literature for other pathogens (16-21 OTUs) (19, 20). The number of total bacterial reads per sample was almost identical between the two cohorts (Fig S4), and the difference in the number of total bacterial reads per sample was significant in all three cohorts (P < 0.001).

Bacterial Bacteriophage-Specific Mapping
The bacterial Bacteriophage-specific Mapping tool identified a total of 13,963 bacterial OTUs, including those for which only one sample was collected, and the number of OTUs per gut microbiome was low (Fig S5). The number of bacterial OTUs present in all three cohorts was high (Fig S5), and the number of OTUs in the three cohorts was low (Fig S5). All three cohorts had a high proportion of uncharacterized OTUs, and the proportion of uncharacterized OTUs was higher in the two cohorts than in the other two cohorts (Fig S5). The proportion of OTUs present in the gut was higher in the two cohorts than in the other two cohorts (Fig S5). The proportion of uncharacterized OTUs in the gut was higher in the two cohorts than in the other two cohorts (Fig S5). The proportion of OTUs present in the gut microbiota was higher in the two cohorts than in the other two cohorts (Fig S5).

We performed a multivariate analysis of the abundance of each microbial OTU using a Kaplan-Meier log rank test.


## Discussion
However, in order to investigate the impact of gut microbiota on the gut microbiota of humans, we used the mouse in vivo model for the bacterial community in the mous. To our knowledge, this is the first study that has shown that the gut microbiota of mice is not differentially regulated during chronic human chronic phases in vivo [40].

We found that the gut microbiota of mice is significantly more complex and diverse than the human gut microbiota. This is in agreement with previous studies that have shown that the gut microbiota of mice is significantly more complex and diverse than the human gut microbiota [40]. Additionally, our results also suggest that the gut microbiota of mice is more diverse than the gut microbiota of humans. The reason for this is that the gut microbiota of mice is more diverse than the gut microbiota of humans. These results are consistent with previous studies that showed that the gut microbiota of mice is significantly more diverse than that of mice [40].

The gut microbiota of mice is significantly more complex and diverse than the human gut microbiota [40]. We found that the gut microbiota of mice is significantly more complex and diverse than that of humans. These results are in agreement with previous studies that have shown that the gut microbiota of mice is significantly more complex and diverse than that of mice [40]. Our results also suggest that the gut microbiota of mice is significantly more diverse than that of humans. Our results also suggest that the gut microbiota of mice is significantly more diverse than that of humans. These results are in agreement with previous studies that have shown that the gut microbiota of mice is significantly more diverse than that of humans [40].

In this study, we found that the gut microbiota of mice is significantly more complex and diverse than that of humans. This is in agreement with previous studies that have shown that the gut microbiota of mice is significantly more complex and diverse than that of humans [40]. We found that the gut microbiota of mice is significantly more diverse than that of humans. This is in agreement with previous studies that have shown that the gut microbiota of mice is significantly more complex and diverse than that of humans. These results are in agreement with previous studies that have shown that the gut microbiota of mice is significantly more complex and diverse than that of humans. These results are in agreement with previous studies that have shown that the gut microbiota of mice is significantly more complex and diverse than that of humans [40].
