# Auto-Radicals as Antifungal Substrates against Pathogens with a Hematogenous Transfer Protein of Plasmodium falciparum Entry: Novel Functions of rDNA Interactome 15
Quaglee Dragontacos


## Abstract
Methionine triphosphate isomerase (MTI) enzymes are found in all eukaryotes, providing an important group of enzymes with which to design novel and unique drugs for a broad range of diseases. In this context, the MTI family MTI, defined as the largest family in metazoans, has been described as a group of enzymes with unique, organized activity and secondary hydrolysis, that may be beneficial in a wide range of diseases. In this context, this paper presents a new member of the MTI family proposed to be defined in the Pichia pastoris group. This new member has two major functions: (i) to carry phosphate and (ii) to catalyze secondary reactions of phosphate units in the catalytic machinery of the cell, which in turn may be used by different cellular organisms for different biochemical processes, including growth and morphological transitions, reproductive development, drug metabolism, and virulence. A member of this family has been proposed to be found in all major eukaryotic phyla, comprising plants, amoebozoa, archaea, insects, and fungi. With the aid of a genome-wide phylogenetic analysis of the Pichia pastoris MTI family, we identified a set of putative MTI genes from the 12,442 new isolates of P. pastoris analyzed through molecular data, and the gene analyses concurred with those obtained in previous phylogenies. Our results suggest that the P. pastoris MTI family is a very ancient group of enzymes, and deserves to be studied in the same way that it has been described for the plants, amoebozoa, and eukaryotes.


## Introduction
As described previously, RDNA interactome 15 is involved in the production of various compounds (.g., cell wall components) for virulence. These include plasmodium antibiotics, i.e., antibiotics with biofilm properties (e.g., aminoglycosides and amphotericin B), e.g., antibiotics with high affinity for the target fungal cell wall component, as well as antibiotics with specific inhibitors of other bacterial species and biofilms [27]. Additionally, RDNA interactome 15 is involved in the production of e.g., antibiotics with antimicrobial activity, e.g., sulfonamides, azoles, and chloramphenicol [27]. Finally, RDNA interactome 15 is involved in the production of other compounds (e.g., biofilm proteins) including the biofilm inhibitors, e.g., methylprednisolone, benzaldehydes, betaine, benomylphenols, and some cytochalasin derivatives [28,29].

The RDNA interactome 15 was first characterized using a bacterium named Aspergillus niger [30]. This strain was further characterized using a bacterium named Enterococcus faecium [31,32,33]. Since then, RDNA has been characterized as a leading candidate for an emerging antibacterial candidate, with some reports suggesting that RDNA could be a candidate for the development of novel antibacterial agents [34,35,36,37]. RDNA interactome 15, which is a key candidate for an antibacterial candidate, is present in bacteria in the genus Enterococcus faecium and in all of the pathogenic fungi. In fact, the number of RDNA genes in the genomes of pathogenic fungi is now more than 400 [38,39,40,41,42]. In bacteria, RDNA is expressed on plasmatocytes and can be induced upon stimulation by various stimuli, including the cytokines interferon-a and interleukin-a (IL-a), and the IL-12 cytokine IL-12a [43]. RDNA is also expressed on human monocytes, but its expression is restricted to macrophages.

The aim of this study was to analyze the effects of RDNA and its specific inhibitor R-AtaM on the growth and virulence of the pathogenic strain Enterococcus faecium.


## Methods
The control bacteria (except for . pneumoniae) were streptococcus (ATCC 25923), methicillin-resistant S. pneumoniae (MRSA) (ATCC 90028), and Listeria monocytogenes (ATCC 90021) in the presence of different concentrations of rDNA. The control bacterial solution was broth (pH 6.0) with 10% serum and 10 mM HEPES. The control fungi (except for Cryptococcus neoformans) were streptococcus (ATCC 25923), streptococcus glabrata (ATCC 49758), and Listeria albidus (ATCC 90021). The results showed that the bacterial rDNA was responsible for up to a 60% of the growth inhibition of the test inoculum. The control fungi were streptococcus (ATCC 25923), methicillin-resistant S. pneumoniae (MRSA), Listeria albidus (ATCC 90021), and Listeria albidus (ATCC 90021). The results showed that the fungal rDNA was the only drug in the control group that significantly inhibited the growth of the test fungal inoculum. The results showed that the control fungi were the most sensitive and resistant. The results showed that the bio-films were the most efficient in the control of the pathogenic fungi. The results showed that the fungal rDNA was the most effective in the control of the bacterial pathogen. The results showed that the bio-films were the most efficient in the control of the pathogenic fungi. The results showed that the bio-films were the most efficient in the control of the pathogenic fungi. The results showed that the bio-films were the most efficient in the control of the pathogenic fungi. The results showed that the bio-films were the most efficient in the control of the pathogenic fungi.

The next step was to determine the surface activity of rDNA in the control of the pathogenic fungi. The control fungi (except for S. pneumoniae) were streptococcus (ATCC 25923), streptococcus glabrata (ATCC 49758), and Listeria albidus (ATCC 90021).


## Results


Several fungal pathogens, including . albicans, C. neoformans, and Cryptococcus neoformans, were shown to acquire this secondary transporter, though not all fungi are capable of acquiring this transporter. However, the antifungal mechanism of this transporter has not been studied. Thus, we used a hematogenous transfer protein of C. albicans (CAX) in which the fungal entry vector was rDNA-derived from the terminal tip of the host hemolymph. We constructed a strain in which the fungal hemoglobin entry vector was rDNA-derived from the hemolymph of a rabbit. The fungal hemoglobin was isolated and assayed for its ability to carry out the assays described above. As expected, the fungal hemoglobin was found to be stable and fully reconstituted in the absence of a fungal hemoglobin, even at the highest concentration tested (Fig 2). This was in contrast to the fungal hemoglobin in which the fungal hemoglobin was degraded and no further degradation was observed (Fig 2). These results suggested that the fungal hemoglobin is not a determinant of fungal hemolymph acid tolerance.

The distribution of the fungal hemoglobin in the host hemolymph was determined by measuring the hemoglobin in the host hemolymph of these fungi. The number of fungal hemoglobin in the host hemolymph of A. fumigatus was found to be significantly higher than that in the hemoglobin of C. albicans (Fig 3A). This result suggested that the fungal hemoglobin is important for the establishment of the host hemolymph.

The fungal hemoglobin is secreted from the host hemolymph by an enterovirus, which is a C. neoformans member of the clade of the Staphylococcal Escherichia-Shigella group (Fig 3B). The fungus secreted a putative hemoglobin during the infection process. However, the fungal hemoglobin was not found to be secreted from the host hemolymph during the infection process. These results indicated that the fungus does not secret the hemoglobin during the infection process.

In addition, the fungal hemoglobin was detected in the host hemolymph of C. neoformans, C. albicans, and C. gattii.


## Discussion

We found that CD4+ T cell response of T cells in response to Mycobacterium tuberculosis infection was significantly impaired in the presence of GXM-PKA, while the absence of GXM-PKA did not affect CD4+ T cell response to Mycobacterium tuberculosis infectio. CD4+ T cell response was significantly impaired in response to Mycobacterium tuberculosis infection, while the presence of GXM-PKA did not affect CD4+ T cell response to Mycobacterium tuberculosis infection.

GXM-PKA plays a role in the production of IL-10, which is expressed by the T cell response to Mycobacterium tuberculosis infection. T cell response to Mycobacterium tuberculosis infection is dependent on the CD4+ T cell response to Mycobacterium tuberculosis infection. A previous study found that GXM-PKA treatment, as a systemic antifungal agent, reduced the T cell response to Mycobacterium tuberculosis infection in mice and, at the same time, resulted in CD4+ T cell responses to Mycobacterium tuberculosis infection. In addition, GXM-PKA treatment, as a systemic antifungal agent, also significantly reduced the CD4+ T cell response to Mycobacterium tuberculosis infection.

As a direct result of the TLR-2 axis-mediated signaling, CD4+ T cells are the most important cell types for the CD4+ T cell response to Mycobacterium tuberculosis infection [39]. Indeed, CD4+ T cell response to Mycobacterium tuberculosis infection was significantly impaired in the presence of GXM-PKA. In addition, CD4+ T cell response to Mycobacterium tuberculosis infection was significantly impaired in the presence of GXM-PKA, but not in the absence of GXM-PKA.

Furthermore, we found that the CD4+ T cells and CD4+ T cell response to Mycobacterium tuberculosis infection were significantly impaired by GXM-PKA. This is in contrast to a study by Lorber et al.
