# Biomarker-guided detection of novel molecular burdensome gene contents in a library of Aureobasidium pullulans cerebri fruit-derived spore particles
Quaglee Dragontacos


## Abstract
Aneuploidy and polyploidy is common in eukaryotes but is also found in unicellular organisms, including animals, plants and fungi. Neokitivuploids occur most frequently in yeast and are characterized by phenotypes that are similar to those of diploid progenitors but differ from those of any other yeast.

We report on the occurrence of aneuploidy and polyploidy in a yeast collected in the laboratory of the Schizosaccharomyces pombe. We identified a subset of 150 isolates that were considered to be aneuploid unless specifically noted otherwise.

Aneuploidy and polyploidy occurred more frequently in unicellular organisms than in eukaryotes. Both, however, have higher frequencies in metazoans and choanoflagellates. The majority of yeast isolates were considered to have a polyploid parent organism even though this was not true in other eukaryotes.

It is unclear whether aneuploidy and polyploidy exist in the same species or in different organisms. Considerable differences in gene content were found among yeast and metazoans in the occurrence of aneuploidy and polyploidy.

The online version of this article (doi:10.


## Introduction
Malassezia is a polyextrem ménage à trois genus of fungal plant pathogens, which causes a range of skin and nail infections in healthy and immunocompromised people. [1] The genus includes about 150 species, most of which cause superficial infections of mostly soft tissue and soft tissue infection, such as nails, cutaneous and visceral diseases. [2] Skin infections are rarely severe, with the exception of anemia, emaciation, diabetes, folliculitis, osteomyelitis, osteomyelitis, mucormycosis, rheumatism, rheumatism, mycetoma, verrucosis, wound infections, and pityriasis versicolor. [3]

A number of studies have investigated the phylogenetic relationships within Malassezia. The genus Malassezia includes several closely related fungi, which cause a range of skin and nail infections in healthy and immunocompromised people.


## Methods
The library was obtained from the Microbial Genomics Laboratory, Faculty of Veterinary Medicine, Aziz University, Mexic. The DNA library was sequenced by the Institute of Biosciences, University of São Paulo using a sequencing platform Agilent 3730xl Genetic Analyzer (Agilent Technologies, Palo Alto, CA, USA). The data was processed with MacClade v.3.05 [49].

Libraries were sequenced on an Illumina MiSeq platform (Illumina, San Diego, CA, USA). The reads were first trimmed to remove the adapter sequences, trimmed to a minimum length of 100 bases, and then trimmed to the same length. The trimmed reads were then mapped to the dataset using the TopHat program [50] with default parameters. The mapped reads were then binned using a pipeline developed by PUBG version 3.0 [51].

For the analysis of the transcriptome data, the resulting transcripts were analyzed using the Mothur 2.14.0 software [52] with default parameters. The program was used to compare the transcriptome data with those available in the National Center for Biotechnology Information (NCBI) database.

Phylogenetic Analysis
The sequences generated in this study have been deposited in the National Center for Biotechnology Information (NCBI) with the accession numbers PRJNA206755 (C. pullulans), PRJNA206894 (C. albicans), PRJNA206894 (C. tropicalis), PRJNA206947 (C. bracarensis), PRJNA206940 (C. guilliermondii), PRJNA206941 (C. guilliermondii), PRJNA206941 (C. parapsilosis). The phylogenetic tree was constructed using the neighbor-joining method with the maximum-likelihood method. The neighbor-joining analysis was performed using MEGA v.6.05 [53] and the pairwise deletion method with the “cluster” method. The general time reversible model was used with the “-lm=2” option. The phylogenetic tree was visualized in FigTree v.1.4.3.


## Results
We observed significant increases in the number of novel molecular burdensome gene contents in these experimental strains, with the most pronounced increase in strains with high yeast colony counts (Table 3. Consistent with this trend, our results also found that the most pronounced increase in the number of novel molecular burdensome gene contents was observed in strains with high yeast colony counts, with the most pronounced increase in strains with high yeast colony counts (Fig 1B). In addition, the most pronounced increase in the number of novel molecular burdensome gene contents was observed in strains with low yeast colony counts (Fig 1B). The most pronounced increase in the number of novel molecular burdensome gene contents was observed in strains with low yeast colony counts (Fig 1B). We also observed significant increases in the number of novel molecular burdensome gene contents in strains with high yeast colony counts. However, our analysis did not detect significant increases in the number of novel molecular burdensome gene contents in strains with high yeast colony counts. Thus, the major differences in the number of novel molecular burdensome gene contents between strains with high yeast colony counts and strains with low yeast colony counts may be related to the ability of these strains to grow in high nutrient-rich media.

Aureobasidium pullulans cerebri fruit-derived spore particles serve as host phagolysosomes for fungi
To elucidate the functions of Aureobasidium pullulans spore particles in host phagolysosome biogenesis, we first characterized the interactions of these particles with host fungi. In contrast to previous studies, which have demonstrated that fungal host phagolysosomes have a direct role in fungal phagocytosis [23], we found that the Aureobasidium pullulans spore particles and fungal host phagolysosomes were not only strongly dependent on host phagolysosome biogenesis, but also shared a similar number of phagolysosomal enzymes (Fig 2). We then tested the ability of these particles to efficiently degrade fungal cell wall polysaccharides and also to assemble and assemble fungal phagosomes (Fig 2).


## Discussion
The use of natural products as anti-infectants has emerged as a powerful means of combating bacterial infections and could be a strategy to reduce morbidity and mortality caused by many infectious diseases. A new approach to treating infectious diseases using natural products has emerged and has the potential to improve the management of many infectious diseases by providing a protective protective effect against infectious diseases caused by microbial pathogens. The use of natural products in treating infectious diseases has been studied, but the mechanisms of their action are still poorly understood. In this study, we focused on a single compound, named tCOO^·, which has been found to be a potent antibacterial agent that exerts a broad and reversible effect in vitro against several bacteria and fungi (12). Although this compound was originally identified as an anti-bacterial compound, it was not identified as a natural product. Here, we used an analytical strategy to identify a compound that could be utilized in the treatment of infectious diseases caused by bacteria. We found that tCOO^· was able to inhibit the growth of E. coli S1 at concentrations of 10 to 20 µM and inhibit the growth of E. coli L15 at concentrations of 1.3 to 10.3 µM. These concentrations were substantially higher than the concentrations that were used in the present work. However, tCOO^· was not able to inhibit the growth of E. coli L15 at concentrations of 1.3 to 10.3 µM. This suggests that this compound might be considered as a potential candidate as a potential anti-bacterial compound that could be used in combination with antibiotics or other natural products to treat infectious diseases caused by microbial pathogens.

The bioactive compound tCOO^· was initially identified as an antibiotic and was used in the treatment of bacterial infections (14). However, the activity of this compound against S. aureus was inhibited by drug concentrations of 4 to 20 µM and its concentration increased to the level of 1.3 µM in the present work.
