# Cairo Beltran-Weber GC, Hosseini BL, et al. Expression of Genes Encoding Carbohydrate-Active Enzymes in the Fungal Pathogen Aspergillus nidulans
Quaglee Dragontacos


## Abstract
We previously observed that heat stress in the human fungal pathogen Cryptococcus deuterogattii induced accumulation of the yeast-like morphotype (yeotic) of yeast cells. To investigate the underlying mechanisms, we investigated the type of molecular response of yeasts exposed to heat stress and explored the consequences of this response on the formation of dendrogram. Yeasts exposed to a heat stress milieu and to a non-yeast environment have dendrogram that is similar to that of the normal, yeast-yeast environment and, to our knowledge, to the same as those exposed to a heat stress milieu and to two non-yeast environments. We showed that the yeast-yeast dendrogram is largely analogous to the normal, yeast-yeast dendrogram. Furthermore, we observed that the heat stress milieu induced accumulation of glycosylated forms of yeast cells that had lost their ability to respond to the heat stress. In addition, our data show that the yeast cells exposed to heat stress lost their ability to mount responses to the heat stress and that this loss was driven by the same type of molecular mechanisms that are involved in the formation of dendrogram in the normal, normal yeast cells. We therefore have a new, likely explanation for the observed dendrogram formation, one that involves changes in the physical properties of the cells exposed to the dendrogram. This explanation might explain how certain physical features of cells, such as the size of the dendrogram, are similar to those of normal cells.


## Introduction
Bacterial pathogens are confronted with a wide range of stressful conditions, which must be carefully controlled. Consequently, the innate immune system orchestrates the response against pathogens in a spatiotemporal manner [1]. The innate immune system is a central component of the innate defense against both invading and non-invading pathogens [2]. In addition, the host has an extensive arsenal of antimicrobial defense systems, including the innate immune system [3,4]. The innate immune system comprises two main categories of molecules: pattern recognition receptors (PRRs) and pattern recognition molecules (PRMs). The prototypical PRRs are pattern recognition receptors that are expressed in eosinophils, macrophages, and neutrophils, and are involved in the innate immune response against bacteria. The second category of molecules is the secondary molecules that are localized in the cell wall, which are involved in the recognition of foreign microorganisms and infection of host cells by pathogens. In this review, we will focus on the most important group of PRRs, the Nod-like receptor (NLRs). Among them, NLRs are recognized by diverse receptors, which are ligand-binding proteins that have a basic structure consisting of a small number of basic amino acids, and are located at the plasma membrane, where they interact with and inhibit the bacterial cell wall [5]. The NLRs are classified into two major groups, which we will call in accordance with the order of their functional group(s): 1) Nod-like receptors (NLRs), which are expressed in the cells, and 2) carbohydrate-binding proteins, which are expressed in the cell wall.

The first group of receptors consists of a single C-type lectin (CL) and a C-terminal histidine receptor (HCR), the latter being expressed mainly in macrophages and neutrophils. The CLs are expressed on the cell wall and are activated by recognition of specific carbohydrates or by ligands, which are expressed on the surface of the pathogen. The receptor-mediated signaling is mainly directed by the adaptor molecule (e.g.


## Methods
), Candida glabrata (CH_2), Candida parapsilosis (PAPA), Candida tropicalis (CPC), and Candida kefyr (CKF) were used for proteomic analysi. Total RNA was extracted from the fungal mycelia using a hot acid phenol method and purified using a QIAGEN QIAquick PCR Purification Kit with RiboPure™ Premix Ex Taq (Qiagen, Hilden, Germany). The cDNA was synthesized with the SuperScript III First-Strand Synthesis System for RT-PCR (Invitrogen, Carlsbad, CA, USA). The oligonucleotides used for amplification of the cDNA and amplicon sequencing were designed using the Primer Express Software (Applied Biosystems, Foster City, CA, USA). The results were checked with the BioEdit 7.1.5.2 software (Drummond Institute, New York, USA).

Detection of Pregrown (PD) in the Mycelia of P. expansum
To estimate the PD in the mycelia of P. expansum, the wild-type strain was inoculated with a 5-ml suspension of conidia of the P. expansum strain in 0.5% Tween 80, and the fungal suspension was then transferred into a sterile 6-ml screw-cap tube and shaken at 180 rpm in a shaking incubator for 3 h. Mycelia were then removed by filtration using a speed-vac and centrifugation (15,000 rpm, 15 min, 4°C), and the mycelia were then ground with liquid nitrogen. The PD content was calculated using the formula below [29]:

P. expansum (molecular weight, 50,000) = PD_50 (grams of dry matter) / PD (grams of dry mycelia)

For all other tests, the mycelia were collected, weighed, and then transferred to a sterile 1-ml screw-cap tube. The samples were vortexed at maximum speed for 2 min and then centrifuged (10,000 rpm, 10 min, 4°C), and the pellet was suspended in 1 ml of 1× PBS. The pellets were then kept at -80°C for further experiments.


## Results
To validate the host-targeting findings and to identify the host-specific processes that are playing a role in fungal pathogenicity in the . fujikuroi system, we searched for gene ontology (GO) terms that are associated with host-specific processes (Table 1). As demonstrated in the BLAST analysis, all genes predicted to encode fungal carbohydrate-active enzymes are in fact part of innate immune response pathways. A number of these genes have been identified as putative metabolic pathways in many fungal pathogens, including C. albicans [14], C. glabrata [16], and E. coli [13]. These include carbohydrate-utilizing enzymes such as cellulases, proteases, and other carbohydrate-active enzymes.

Evaluation of the BLAST result of the PHI-BLAST result of the first-nested analysis revealed that 93 of these genes were exclusive to the first-nested analysis, and that only 12 were exclusive to the first-nested analysis. The number of exclusive genes is high, but the overall number of exclusive genes is small. The PHI-BLAST result of the first-nested analysis revealed that the number of exclusive genes is much smaller than the number of exclusive genes (Table 1). This is consistent with the finding that F. fujikuroi has a relatively small number of genes that are exclusive to the first-nested analysis (Table 1). This may reflect the fact that F. fujikuroi has an unusual set of fungal pathogenicity genes that are not exclusive to the first-nested analysis (Table 1).

Functional categorization of the PHI-BLAST results revealed that 45 genes are specific to the first-nested analysis, and that only seven genes were exclusive to the first-nested analysis (Table 1). We note that the percentages of genes that are exclusive to the first-nested analysis are high in the PHI-BLAST analysis, indicating that the data obtained from the first-nested analysis are suitable for functional categorization.


## Discussion
Expression of genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 was upregulated at 24 hp. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi. Genes encoding Carbohydrate-Active Enzymes in the Fusarium graminearum FUS1, FUS2, FUS3, FUS4, FUS6, FUS11 and FUS13 were upregulated at 24 hpi.
