# Characterization and Evolution of the Haliscophane Synthetase from Rice Blast Fungus
Quaglee Dragontacos


## Abstract
We present a case of disseminated cryptococcosis in a 40-year-old woman. Prior to admission, she was attended at two hospitals for hypertension and clinically diagnosed malaria. Cryptococcosis was diagnosed by cerebrospinal fluid Cryptococcus antigen detection from blood and urine. She had mild malaria and was immunocompetent. Serum Cryptococcus antigen was positive. After antifungal treatment, she subsequently developed meningitis and was diagnosed with cryptococcal meningitis. Despite the treatment with amphotericin and flucytosine, she died from fulminant intracranial hypertension.

•We present a case of disseminated cryptococcosis in a 40-year-old HIV-infected male with suspected Cryptococcus infection. •Herpes was considered as the most likely cause of disseminated cryptococcosis. •The diagnosis of cryptococcosis was confirmed by Cryptococcus antigen detection from blood and urine.

We present a case of disseminated cryptococcosis in a 40-year-old HIV-infected male with suspected Cryptococcus infection.

Herpes was considered as the most likely cause of disseminated cryptococcosis.

The diagnosis of cryptococcosis was confirmed by Cryptococcus antigen detection from blood and urine.


## Introduction
Invasive fungal infections are a major cause of morbidity and mortality among immunocompromised patients.


## Methods
glaucescens H1 gene (see Table S1 in the supplemental material. The amplification of the gene was performed using the PCR reaction mixture (10 µl) containing 1 µl of forward primer, 5 µl of reverse primer, 0.5 µl of template DNA, 1.25 µl of Taq DNA polymerase, 0.4 µl of each forward and reverse primers, 0.4 µl of template DNA, 0.5 µl of 25 pmol of each of the internal transcribed spacer (ITS) and 1.25 µl of sterile water. The PCR reaction was performed as described previously [18] using the following thermal profile: denaturation at 94 °C for 10 min, followed by 35 cycles of denaturation at 94 °C for 30 s, annealing at 52 °C for 30 s and elongation at 72 °C for 1 min. The PCR products were purified with QIAquick PCR purification kit (Qiagen, Hilden, Germany). The purified PCR products were sequenced by PCR using the BigDye® Terminator v. 3.1 Cycle Sequencing kit (Life Technologies, Carlsbad, CA, USA). The sequences obtained from the sequencing run were compared with those deposited in GenBank using a BLASTn algorithm.

To perform the phylogenetic analyses, the C. glaucescens H1 gene sequence was aligned with the sequences of a representative Arabidopsis species using the program MUSCLE [35] with default parameters. The phylogenetic tree was constructed using the maximum likelihood method implemented in the program MEGA 5 [36].

Analysis of Clustering and Motif Sequence Analysis
To perform the clustering analysis, the sequences were clustered in a concatenated database by the neighbor-joining method and were aligned with MUSCLE [36]. The phylogenetic tree was constructed using the maximum likelihood method implemented in MEGA 5 [36].

The first-order differential clustering was performed in the concatenated sequences from rice blast sequences with the CLUSTALW method, with default parameters.


## Results
japonicu. The results shown in Figure 4A and B show that the catalytic activity of the enzymes from the three enzymatic complexes was significantly increased when the hydrolysis reaction with the 3-BP from the haliscophane biosynthesis pathway was included. The enzyme from the haliscophane biosynthesis pathway is considered as a key enzyme in the Haliscophane synthesis pathway in rice [21]. The activity of the enzymes from the three enzymatic complexes was also found to be increased when the hydrolysis reaction with the 3-BP from the haliscophane biosynthesis pathway was included, which is in agreement with the results of the preliminary characterization of the catalytic activity of the enzymes from the three enzymatic complexes. The enzymatic complex from the Haliscophane biosynthesis pathway was found to be an ideal for the growth of rice blast fungus.


## Discussion
The sequence data showed that the protein had an acetylated version (DPGF0072) and an N-acetylated version (DPGF0071. This was confirmed by the proteomic analysis. The DPGF0072 and DPGF0071 sequences showed similarity to the recently identified fungal acetyltransferase (ATG_03372) from S. cerevisiae (Table 2). The proteins were predicted to be involved in the regulation of acetylation of fatty acids, fatty acid binding and fatty acid biosynthesis, respectively.

The Fungal Acetyltransferase (DPGF0072) was previously characterized as a fungal acetyltransferase from Saccharomyces cerevisiae (Shukla et al., 2017). The C. neoformans DPGF0072 was characterized by the presence of a ribonucleotide (RNP) domain containing the PheUA-CysG motif (Madsen, 2018). The ribonucleotide was subsequently identified as the major component of the pathway in C. neoformans (Madsen, 2018). Our results showed that the DPGF0072 protein from C. neoformans was a homologue of the C. albicans CaD-1 (Figure 3). This protein was also identified as an acetyltransferase (DPGF0072) from S. cerevisiae (Madsen, 2018). In addition, DPGF0072 showed similarity to the putative acetyltransferase (DPGF0071) from Cryptococcus neoformans (Madsen, 2018). The acetyltransferase from Cryptococcus neoformans was characterized by the presence of an exo-acetyl group (DPGF0071) and the presence of a carbohydrate esterase (DPGF0072) (Madsen, 2018). The DPGF0072 protein was also identified as a fungal glucosylceramides-acetyltransferase (DPGF0072) from Candida albicans (Madsen, 2018).
