# Characterization of Exosomes and Their Antimicrobial Activity against Multidrug Resistant Pathogens
Quaglee Dragontacos


## Abstract
The aim of this study was to evaluate the influence of 20 whole plant species grown in the state of Pievella on the immune response of wild-type (WT) mice. Wild-type (WT) mice were infected with either the fungal or bacterial pathogens, to evaluate the responses to infection, and to determine whether the immune responses are induced by the same fungal pathogens. Also, mice were infected with either a non-pathogenic (NK) yeast or a pathogenic (GS) yeast in the form of mutated copies of IgG1 and IgG2a, respectively. The immunological responses to infection were assessed by expression of anti-NK and anti-GS epitopes, and by the expression of cytokines, or expression of antigenic specific T-cell responses. NK mice had significantly higher levels of reactive oxygen species, as well as their CD4+ and CD8+ T-cell responses to infection, as compared to WT mice. CD4+ and CD8+ T-cell responses to infection were also reduced significantly by both NK and non-pathogenic IgG mice. In conclusion, the results of this study demonstrated that NK and non-pathogenic IgG mice can induce significantly higher immunological responses than WT mice, and these responses were associated with enhanced production of Th1, Th17, or IFN<U+03B3> and production of anti-inflammatory cytokines.


## Introduction
Candida albicans is a fungal pathogen, frequently colonizing the skin, lungs, and gastrointestinal tract. This species can cause superficial and/or deep-seated candidiasis in both immunocompromised and immunocompetent patients [1-3]. C. albicans can cause disease in both healthy individuals and patients with underlying conditions [4]. The disease can be disseminated and may be life-threatening, especially in patients with underlying diseases. The ability of C. albicans to sense environmental changes such as temperature, pH, and hypoxemia may be critical for the ability of the fungus to colonize and cause disease [5-7]. Therefore, it is critical for C.


## Methods
Briefly, the sample was washed with 2% TFA (isopropyl alcohol) and centrifuged at 3000 rpm for 10 min at 4 °. The supernatant was transferred into a 1.5 mL microtube and the bacterial suspension was filtered through a 0.45 µm nylon filter. The filtered supernatant was incubated with 10 µL of TRIzol reagent (Invitrogen Life Technologies, Carlsbad, CA, USA) at 37 °C for 10 min in order to precipitate the supernatant. The supernatant was then removed and the bacterial suspension was added to a 1.5 mL microtube containing 1 µL of TRIzol reagent. The mixture was incubated at 37 °C for 20 min in order to precipitate the supernatant. The precipitated supernatant was removed, and the bacterial suspension was transferred to a new microtube containing 1 µL of TRIzol reagent. The resulting solution was incubated at 37 °C for 15 min to obtain the final concentration of 0.1% of the concentration used for the quantification of bacterial exosome concentrations. The amount of the bacterial exosome suspension was determined using a spectrophotometer (ELX-800; Molecular devices, Sunnyvale, CA, USA).

Statistical Analysis
The comparative effectiveness of different antibiotics was evaluated by the difference in the relative MIC values. A reference set was prepared for each of the antibiotics tested in the study by dividing the number of bacteria present in the sample by the number of bacteria present in the control group. For the comparison of the treatment groups, the mean values of the antibiotic concentrations were used, whereas the MICs were calculated by the comparison of the concentration of the tested sample with that of the control group.

The efficacy of different antibiotics on the pathogenic fungi was evaluated by the difference in the relative survival of the different antibiotics. The reference set was prepared for each of the antibiotics tested in the study by dividing the number of bacteria present in the sample by the number of bacteria present in the control group.

The relative survival of the different antibiotics was evaluated by the difference in the number of CFU per mL of the bacterial suspension and the number of colony forming units (CFU/mL) per mL of the bacterial suspension.


## Results

To examine whether the expression of the genes involved in cellular defense against pathogens was differentially regulated between the two libraries, we compared the gene expression of the GO terms, which represent biological processes such as apoptosis, DNA replication, and DNA damage repair, as well as the terms, which are associated with cellular immunity and immune function, as well as the terms, which are associated with pathogens and their targets, which are associated with cancer and their targets, and as well as the terms, which are related to immune system functions, such as response to pathogen.

In order to identify genes whose expression patterns were differentially regulated between the two libraries, we performed real-time RT-PCR analysis. In this study, we used an Illumina HiSeq 2500 platform, which is able to amplify and profile the transcriptome of the promoter of genes involved in immune response and response to disease [31]. We performed this analysis using real-time RT-PCR in order to determine the expression levels of the genes that were differentially regulated between the two libraries.

We found that gene expression of the GO terms (including response to pathogens) and the terms (including response to pathogens) were significantly different between the two libraries (P<0.001, Fisher’s exact test, Figure 4A, Supplementary Table S3), suggesting that these genes were differentially expressed between the two libraries. Gene expression was also significantly different between the libraries (P<0.001, Fisher’s exact test, Figure 4B, Supplementary Table S3).

Additionally, we found that the expression of the GO terms (including response to pathogens) and the terms (including response to pathogens) were significantly different between the two libraries (P<0.001, Fisher’s exact test, Figure 4C, Supplementary Table S3).

Transcriptome Analysis and Gene Expression in the Yeast Extract
We next performed transcriptome sequencing and comparative transcriptomics (Tables S1, S2) to examine the transcriptome profile of yeast extracts isolated from different pathogenic fungi.


## Discussion
In the current study, we focused on the identification of exosomes of . albicans and C. tropicalis. In addition, we also investigated the exosome function of C. albicans, which has been reported to be involved in the pathogenesis of this yeast. In the current study, we found that the exosome is highly enriched in C. albicans and C. tropicalis. The exosome was enriched in C. albicans and C. tropicalis. These results indicated that C. albicans and C. tropicalis have the ability to generate exosomes in response to antifungal drugs. C. albicans is known to produce exosomes in response to ergosterol depletion in the cell membrane, which may be the reason for the depletion of ergosterol in the cell membrane of C. albicans (23).

The higher abundance of C. albicans exosome was also observed in C. tropicalis. In the current study, we also found that C. albicans exosome was enriched in C. tropicalis. In previous studies, we also reported that C. albicans cells are very sensitive to azole antifungal drugs (24). The high C. albicans levels in the cells of C. albicans is a result of the genetic alteration of the cells which is a result of the action of azole drugs (25). The low C. albicans levels in the cells of C. albicans has been previously reported (26). C. tropicalis, which is a common pathogen causing PCM, is also a common pathogen causing infectious diseases (27). In our study, we found that C. tropicalis exosome was enriched in C. albicans and C. tropicalis. It has been reported that C. albicans is a common pathogen causing PCM, which is a common cause of PCM. The result suggested that C. albicans may be a common cause of PCM, which is a common cause of PCM.

In the current study, we found that the exosome of C. albicans was enriched in C. tropicalis. This result was in line with previous studies that the high C. albicans levels in the cells of C.
