# Characterization of newly isolated oleaginous yeasts - Cryptococcus podzolicus, Trichosporon porosum and Pichia segobiensis
Quaglee Dragontacos


## Abstract
In our previous study, Microdochium unilateralis JCM1000P yeasts showed antagonism toward Candida species in vitro. However, the inhibitory effect of the microdochium yeast Pbukuroffema waksmanii isolated from the same microdilution control strain against microdochium species was not clear. In this study, we further investigated the antagonistic effect of P. unilateralis JCM1000P and the inhibitory activity of Pbukuroffema waksmanii by the methods of agar well diffusion and agar slant analysis. We found that the antagonistic effect of P. unilateralis JCM1000P against microdochium species was inhibited by 24 h incubation in the culture medium. The inhibition of microdochium by Pbukuroffema waksmanii occurred on a cell density platform and was inhibited by 24 h culture. This study demonstrates that microdochium species, including microdochium proliferating yeasts, are antagonistic microdochial yeasts with potent activity against microdochium species. We conclude that microdochium is a promising host for the microdochium biotransformation inhibitor Microd9000 and is a promising target for further development as a method for the control of microdochium infections.


## Introduction
Rhodotorula spp. are saprophytes that are ubiquitous saprophytes on plant material. They are an important component of soil and root exudation in forests and forests deciduous trees, but also on crops such as fruits, vegetables, and cereals [1]. The genus R. stelgyrus (Fonsecaea) is represented by more than 10 species distributed worldwide [2]. It is one of the most important forest-dwelling saprophytes, and can also cause diseases on many other trees including leaves, trees of warm-blooded animals, insects, and termites, as well as on many other plants [3,4]. The rhizosphere microbiome of R. stelgyrus is thought to be important for the rhizome-microbiota interactions [5]. Rhizosporella species are widely distributed on the plant world with more than 200 species distributed in the upper and lower parts of the world [6,7]. R. stelgyrus rhizospores can be produced from two different stages and penetrate the rhizosphere by a process called phloem formation [8].


## Methods
The oleaginous yeast strains were isolated from different parts of the phylum: mycelium (.g. asphalchus), mycelium (sporocarpus), mycelium (sporocarpus), mycelium (phycoerythricum), mycelium (terreola), mycelium (sporis), mycelium (spores) and other fungi (Cryptococcus gattii, Candida albicans, Coccidioides brasiliensis, Penicillium expansum, Penicillium avenaceum and Cryptococcus neoformans).

DNA extraction, PCR and sequencing
The oleaginous yeast strains were isolated from several kinds of fungi and from different kinds of yeast species: Candida albicans (Alb1), Coccidioides brasiliensis (Cb), Saccharomyces cerevisiae (Sc), Cryptococcus neoformans (Cne1), Eurotiomyces variotii (Eu), Capsaspora prolificans (Cpt), Ustilago maydis (Um) and Aspergillus niger (Au). The PCR amplification of the ITS region was carried out using universal primers ITS5 and ITS4 [18]. The fungal internal transcribed spacer (ITS) region was amplified using ITS-2.1 and ITS-3 primers [19] and the fungal partial 28S region using RPB1-RPB2-5F and RPB1-RPB2-7cR primers [20]. The PCR products were purified using the QIAquick PCR Purification Kit (Qiagen, Hilden, Germany).

PCR was performed with the dNTPs according to the manufacturer’s protocol with the following modifications: One cycle of PCR was carried out in a total volume of 25 µl, using 1 µl of each primer in a 20 µl reaction mixture (Applied Biosystems, Foster City, CA, USA). The PCR products were purified with the QIAquick PCR Purification Kit and then sequenced by the University of São Paulo Laboratory of Molecular Biosciences.


## Results
The genus Cryptococcus was not used to identify newly isolated species of the genus Cryptococcu. However, it has been recently discovered that the genus Cryptococcus is not a good fit for existing species of the genus Cryptococcus. The present study was focused on the identification of three new strains of Cryptococcus by DNA sequencing (see Section 3).

The identification of Cryptococcus podzolicus, Trichosporon porosum and Pichia segobiensis was complicated by the fact that the yeast strains were identified as P. porosum by using PCR amplification methods that were previously used to identify new species of P. porosum [21]. However, the DNA sequences of the P. porosum strains did not match the sequences of previously identified species, and P. porosum strains were not identified by PCR. Therefore, P. porosum was not considered a new species by PCR amplification methods. Therefore, P. porosum was introduced as the new species, although the DNA sequences of the new species were not found to match the sequences of the previously identified species.

The identification of new species of P. porosum was also complicated by the fact that some of the strains in this study were from Brazil [22]. The strain of P. porosum CEA17 from Brazil was identified as P. porosum by PCR amplification methods that were previously used to identify species of P. porosum [21]. The strain of P. porosum CEA17 from Brazil was identified as P. porosum by PCR amplification methods that were previously used to identify species of P. porosum [22]. Therefore, P. porosum was transferred to the new species name P. porosum. The strains of P. porosum CEA17 and P. porosum CEA17 were identified as P. porosum by PCR amplifications of DNA sequences of the P. porosum strains, which were not found to match the sequences of the previously identified species. Therefore, P. porosum was transferred to the new species name P. porosum.

Although the DNA sequence of the strains of P. porosum CEA17 and P.


## Discussion
podzolicus and . segobiensis, and have provided only a single example of such an isolate. In this study, we chose to specifically analyze isolates of Cryptococcus podzolicus and C. segobiensis, since they are part of the same population and are likely closely related to each other. Although the isolation of C. podzolicus and C. segobiensis was based on the same strains, we have also compared the strain profiles of the two species. We have identified two new isolates, which were of C. podzolicus and C. segobiensis.

We have previously demonstrated that the mycobiome of S. maritima is dominated by strains from other species in the genus Cryptococcus
The composition of the genus Cryptococcus is dominated by strains from the genera Candida, Aspergillus, Cryptococcus and Aspergillus
The results of the present study also demonstrate that the mycobiome of S. maritima is dominated by strains from other species in the genus Cryptococcus, a pattern consistent with previous studies from other regions [7, 10]. This pattern has been previously observed in other species of the genus Cryptococcus [13]. A previous study from the same region of Germany also observed the dominance of strains of C. podzolicus in S. maritima, and found that the species had the highest relative abundance in the soil [14]. These findings are in agreement with the previous study from São Paulo, Brazil [15].

The first isolate from this region of S. maritima was designated as C. podzolicus and named as C. podzolicus (C. podzolicus). In the present study, the first isolate was designated as C. podzolicus (C. podzolicus). The fungus was found to be pathogenic in a variety of soil and plant hosts, and was found to be able to colonize a wide variety of plant species and to cause disease in plant hosts [16]. C.
