# Complete DNA Sequence of Neutrophils: From Deficits to Cycle-Related Proteins in Fungi
Quaglee Dragontacos


## Abstract
Cryptococcus gattii is the etiological agent of the life-threatening human disease cryptococcosis. The mechanisms underlying its susceptibility to multiple pathogens are poorly understood. This study investigated the response of C. gattii to four different doses of the Bruton’s tyrosine kinase inhibitor BK-TOS by flow cytometry. Fourteen cells were fixed in 1% fetal bovine serum to simulate heat, and four differently dosed strains of C. gattii (isolated from two different groups of animals), C. deuterogattii, C. neoformans, C. bacillis, and C. bacillis, were used in this study. Three different cell populations were used, and the results were compared to those obtained with cultured cells. The results showed that the isolated cells were able to survive and replicate within macrophages and spleen of all treated animals. Pretreatment with low doses of BK-TOS reduced the number of viable cells and induced a more pronounced Th1-type inflammatory response in the lungs. The isolated cells were not able to form a persistent Th2-type inflammatory mass in the lungs.


## Introduction
The human body is inhabited by diverse microbial communities, which constitute the human microbiome. However, studies on the microbiome of the human body have mainly focused on the bacterial members of the human microbiome (e.g., Bacteria, archaea, viruses) [1,2,3,4]. The major bacterial groups found in the human microbiome are Actinobacteria, Proteobacteria, and Firmicutes [1,2,3,4]. The Firmicutes are generally considered to be classically associated with plants, animals, and higher plants. They include members of Firmicutes genus including members of Firmicutes subgenus (F.


## Methods
The strain PD-3 was grown on the Fava Netto and Spheriospores Agar (FNA) slants (Rosalind et a., 2007) at 28°C. Fungal spores were harvested from the slants, washed with sterile water and counted with a hemacytometer. The number of spores was calculated by subtracting the spore count of the slant from the spore count of the FNA slants. The results were expressed as mean percentage of fungi per fungus in the test samples.

Total RNA Isolation and Reverse-transcription-PCR
Total RNA was isolated from the spores using the Trizol Reagent (Invitrogen, Carlsbad, CA, USA) according to the manufacturer’s protocol. The integrity and quantity of RNA was examined using an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA) and an RNA Nano 6000 Nano Kit (RNA 6000, Nanjing, China). The quality of the extracted RNA was assessed using an Agilent 2100 Bioanalyzer and an RNA 6000 Nano Kit. The RNA integrity was determined using an Agilent 2100 Bioanalyzer and an RNA 6000 Nano Kit. The mRNA expression levels were calculated using the 2^-<U+0394><U+0394>Ct method (Yuan et al., 2009).

Measurement of Gene Expression Levels
Total RNA was extracted from the following fungi: Candida albicans, Candida glabrata, C. parapsilosis, C. tropicalis, C. krusei, C. parapsilosis sensu stricto, C. parapsilosis krusei, C. tropicalis vitellogena, C. parapsilosis bacilli, C. krusei, C. krusei vitellogena, C. krusei vitellogena and C. krusei vitellogena. The integrity and quantity of RNA were analyzed by an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA) and an RNA Nano 6000 Nano Kit (RNA 6000). The concentration and purity of the extracted RNA was estimated using an Agilent 2100 Bioanalyzer and an RNA 6000 Nano Kit.


## Results
Transcriptome analysis of iron-responsive and iron-deficient transcription factors
We have previously shown that the transcriptional response to iron is associated with iron acquisition and that these transcription factors are involved in iron homeostasis (9). Therefore, we generated a transcriptome of human iron-responsive and iron-deficient transcription factors using RNA-Seq, which was used to identify common transcription factors of interest.


## Discussion
 2017. The results of this analysis provide a new perspective on the functions of eukaryotic cyclins and, in particular, the detection of in vivo cyclin-like proteins. The first cell cycle-related protein identified in the yeast and human cyclins, NAC1 and NAC2, has been identified in the fungal cyclin-like proteins (CRM) family, which is present in a set of nine homologs, one of which is required for a specific survival in the human cell (Nakase et al., 2016). A second cyclin, SAC1, was also identified in the yeast and human cyclins, respectively (Chai et al., 2017). The presence of SAC1 in the human cyclins indicates that the human cyclins are not the only cyclins in their family. In addition, SAC1 is also present in the other fungal cyclins (SAC2, TAC1 and TAC2). The presence of SAC1 in the yeast and human cyclins suggests that the cyclin-like proteins are not only in-competent but also responsible for the successful survival of the yeast and mammalian hosts. This conclusion is supported by the fact that SAC1 has been found in other fungal species, such as Neurospora crassa and Cryptococcus neoformans (Tiller et al., 2007; Tiller et al., 2008). Although we have only begun to address the cyclin-like protein family in fungi, it will be interesting to characterize the cyclin-like protein family in a mammalian species.

It has been shown that cyclins act as active members of a cyclin-like protein family in Cryptococcus neoformans (Zhou et al., 2011). In addition, we have shown that the cyclin-like protein family in the Cryptococcus neoformans species is the result of the cyclin-like protein family, which is present in the C. neoformans and C. gattii species-specific cyclins (Zhou et al., 2011). In C. gattii, the C. neoformans cyclin-like protein family is present in the C. gattii genome, which suggests that the C. gattii cyclin-like proteins might be involved in a new phase of cyclin-like protein evolution in C. neoformans.
