# Comparative genomics and mating between the opportunistic pathogenic fungi Cladosporium putrescarpa and Cladosporium perniciosa
Quaglee Dragontacos


## Abstract
In a macropin experiment, yeasts were treated with a mixture of copper (1.4 mM) and copper and then treated with it for 7 days. The experiments were performed at day 7, when yeasts were treated with copper and copper-rich foods, and at 7 days, when yeasts were treated with copper-rich foods and then treated with it for another 7 days. The effects of treatment were compared between yeast treated with copper- and copper-rich foods and it respectively. The results indicated that treatment with copper- and copper-rich foods resulted in a lower susceptibility to infection. Treatment with copper- and copper-rich foods also induced faster secreted aspartic-type protease (SS) formation, which was related to the lower survival of yeast cells in the gastrointestinal tract. In conclusion, it was found that copper- and copper-rich foods stimulated the expression of fungal genes involved in the host defense against pathogens and promoted yeast colonization in the gastrointestinal tract.


## Introduction
The incidence of fungal infections in patients with a hematological malignancy has increased dramatically in the past few decades, partly due to advances in immunosuppression and chemotherapy, which resulted in more susceptible patients and patients with comorbidities receiving intensive chemotherapy, to develop invasive fungal infections (IFIs). [1] The most common type of IFI is candidiasis. However, a few rare, opportunistic fungal pathogens have also been described, including Histoplasma capsulatum, Blastomyces dermatitidis, Coccidioides immitis, Cryptococcus neoformans, and Paracoccidioides brasiliensis. [2] The occurrence of other IFIs has also increased in the past few years, such as cryptococcosis, cytomegal infections, and even leishmaniasis, a rare but serious infection of the central nervous system. [3], [4]

The epidemiology of IFI in patients with different types of malignancies is very complex and has been described in several different types of malignancy. The clinical presentation varies according to the type of organ involved and the clinical stage of the patient. In some patients, the etiology is not clearly described but the pathophysiology is complex. We present a case of an immunosuppressed patient with cryptococcal meningitis in whom the history of chemotherapy and antifungal therapy led to a favorable outcome.


## Methods
Charles . Schässl (Institute for Molecular Mycology and Toxicology, University of Arvada).

2.2. Sequence Analysis
The ITS rDNA region was amplified with the universal ITS primers ITS1F (5'-CTTGGTCATTTAGAGGAAGTAA-3') and ITS4 (5'-TCCTCCGCTTATTGATATGC-3') and the ITS rDNA region with the primers ITS1F (5'-GGACTACHVGGTTACG-3') and ITS4 (5'-TCCTCCGCTTATTGATATGC-3'). The amplified PCR mixture (10 µl) contained 2.5 µl of 10 × PCR buffer, 1.5 µl of 5 mM MgCl_2, 0.5 µl of each 10 µM of each dNTPs and 1.5 µl of the appropriate size of PCR reaction (2 × PCR buffer, 1 × FastStart Master Mix, 1 × FastStart Taq Master Mix and 1 × FastStart DNA Polymerase (Roche Diagnostics, Mannheim, Germany). The PCR was performed in a Bio-Rad CFX384 Real-Time PCR Detection System (Bio-Rad, Hercules, CA, USA). The PCR conditions were as follows: 95 °C for 5 min, followed by 40 cycles of 95 °C for 30 s, 55 °C for 30 s, and 72 °C for 1 min, with a final extension step at 72 °C for 5 min. The PCR products were then purified using the Qiaquick PCR Purification Kit (Qiagen, Hilden, Germany). The purified PCR product was sequenced with the BigDye Terminator v. 3.1 Cycle Sequencing Kit (Life Technologies, Carlsbad, CA, USA). The sequences obtained were subjected to an automated BLAST search using the following databases: NCBI (http://www.ncbi.nlm.nih.gov), UNITE (http://unite.ut.ee), Blast2GO (http://blast2go.org/) and I-TASSER (http://ihasy.


## Results
Phylogenetic analysis
The ITS (GenBank accession number: KJ201940) and LSU (GenBank accession number: KJ201940) dataset were generated using the Neighbor-Joining method. The resulting alignment contained 1529 characters with a median of 2175 characters per alignment.


## Discussion
The combined analyses of both approaches, using the combined results of sequencing and phylogenomic analysis, provide strong evidence that Cladosporium is a distinct lineage in the phylogenetic analysis of opportunistic pathogenic fung.

In addition to its small genome, Cladosporium is one of the most well-studied fungi and the majority of the isolates in this study are well-characterized for their large genome size. Furthermore, the sequencing of the human fungal pathogens C. neoformans, the human fungal pathogens C. gattii, and the fungal pathogens Cryptococcus neoformans and Aspergillus fumigatus also provide valuable information regarding their genomes and phylogeny. The genome sizes of the fungi studied here are comparable to that of the human fungal pathogens and are thus comparable to the human fungal pathogens. Moreover, the genome sizes of the closely related human fungal pathogens A. fumigatus and Candida albicans were comparable to the genomes of the two closely related human fungal pathogens, Candida glabrata and C. gattii. The genome sizes of the fungi studied here are comparable to the genomes of the closely related human fungal pathogens Cryptococcus neoformans and Aspergillus fumigatus, respectively. The genome sizes of Cladosporium, C. gattii, and C. parapsilosis are similar to those of C. neoformans and A. fumigatus. However, the genome sizes of Cladosporium and C. gattii are comparable to those of C. neoformans and A. fumigatus. The genome sizes of the three closely related human fungal pathogens C. neoformans and C. gattii are similar to those of A. fumigatus.

Comparative genomic analysis of opportunistic pathogens is a powerful technique for the identification of novel species and novel species-specific gene families. For instance, the genomic analysis of C. albicans identified the gene family CglaA and the gene family CglaB [25]. Another way to study the potential for gene family expansion is to determine the gene family expansion frequency. The genomic analysis of C.
