# Computational Ability of CRISPR/Cas9 Gene Family in Botrytis cinerea Involves Modulation of Morphogenesis, Sporulation, and Virulence
Quaglee Dragontacos


## Abstract
Human histidine biosynthesis is essential for the maintenance of heme biosynthesis in the Aspergillus fumigatus complex. H+-ATPase has been suggested as a target for antifungal therapy.

A2F3 is an essential histidine phosphatase. A2F3 deficiency results in inappropriate activation of the MAPK cascade in A. fumigatus. We investigated whether a2f3 could be essential for asexual development in A. fumigatus in two different strains. We found that a2f3 deficient A2F3 strains produced maleish, with typical fruiting body characteristics. A2F3 expression was restored by treatment with the Amphotericin B deoxycholate Df-lipoxygenase (ABMD) Bafilmed. A2F3 expression was significantly up-regulated in the A2F3 null mutant lacking the Df-lipoxygenase (DfLox) Bafilmed.


## Introduction
A number of medically important fungi such as Aspergillus spp., Candida spp., Cryptococcus spp., and Pneumocystis jirovecii are known to be pathogenic to humans [1], [2]. Aspergillus spp. are the major cause of invasive aspergillosis (IA) in immunocompromised patients, especially in hematologic and solid organ transplant patients [3]. Aspergillus fumigatus is the most common species and causes over 400,000 infections/year worldwide, with over 350,000 deaths/year [4]. A. fumigatus is the most common fungal pathogen in immunocompromised patients and is responsible for over 70% of IA [5].


## Methods
The Cas9 gene family includes the following target genes: CAP1 (. cinerea), CAAG_0123 (C. cinerea), GPD1 (C. cinerea), CACG_006 (C. cinerea), GPD2 (C. cinerea), and CACG_009 (C. cinerea). The target gene CAS9 was chosen based on previous studies using Cas9 gene family as a functional regulator of bacterial and fungal virulence. The CRISPR/Cas9 gene family was selected based on the ability of the target gene to interact with other proteins in the genome. To construct the CRISPR/Cas9 gene family in the target gene, the Cas9 gene was used as the template. The genes of the target gene were amplified from C. cinerea, C. cinerea, and C. cinerea genome, and the amplified gene was cloned into pBluescript SK(+) [31].


## Results
We therefore investigated the CRISPR/Cas9-dependent gene expression in the . crassa and A. fumigatus genomes. We observed that only CRISPR/Cas9-dependent gene expression could be detected in N. crassa (Table 1) and A. fumigatus (Table 2). In contrast, CRISPR/Cas9-dependent gene expression could be detected in N. crassa (Table 1), A. fumigatus (Table 2), and P. striiformis (Table 2). In contrast, the expression of CRISPR/Cas9-dependent gene expression could not be detected in A. fumigatus (Table 1) and P. striiformis (Table 2) (Fig 1A).

To elucidate the mechanism of CRISPR/Cas9-mediated stress response in N. crassa and A. fumigatus, we employed a series of stress-inducing compounds that inhibit the expression of genes that regulate stress tolerance. To test whether CRISPR/Cas9-mediated stress responses were affected by A. fumigatus, we examined the sensitivity of the nro1<U+0394> and nro2<U+0394> strains to the stressor compounds and the sensitivity of nro1<U+0394> and nro2<U+0394> to the stressor compounds and the sensitivity of nro1<U+0394> to the stressor compounds and to the exposure to the stressor compounds (Fig 2). The sensitivity of the nro1<U+0394> strain to the stressor compounds and the sensitivity of nro2<U+0394> to the stressor compounds were both reduced compared to the nro1<U+0394> strain (Fig 2). In contrast, the sensitivity of the nro2<U+0394> strain to the stressor compounds was reduced compared to the nro1<U+0394> strain (Fig 2).

We found that CRISPR/Cas9-mediated stress-induced gene expression could be detected in all three N. crassa and A. fumigatus genomes (Table 2).


## Discussion
cingulata are significantly enriched for introns that are not present in the upstream gene of the gen. Our analysis of the intron sequences from all the genes from P. cingulata revealed an overall enrichment of introns in the genomes of the three species (Table S2). This suggests that intron distribution is inversely proportional to gene size. This is consistent with the model that, if introns are abundant, they would be contained within the genes, rather than being transcribed from the genome (21).

The intron sequences of P. cingulata are significantly enriched for introns that are not present in the upstream gene of the gene. This indicates that, if introns are abundant, they would be contained within the genes, rather than being transcribed from the genome. It also suggests that, if intron distribution is inversely proportional to gene size, the gene would be transcribed from the genome, rather than being encoded from the genome. Our analysis of the introns in the genes of the two species showed that the introns are not distributed in the genome, but are distributed between the genes. This suggests that the gene distribution is inversely proportional to gene size.

The intron sequences in the genomes of P. cingulata and P. cingulata, P. fluorescens and P. fuckelii, are significantly enriched for introns that are present in the upstream gene, rather than the genes, as shown in Table S3. This suggests that introns are not present in the genes.

The intron sequences in the genomes of P. cingulata and P. fluorescens, P. cingulata, P. cingulata, and P. fuckelii are significantly enriched for introns that are not present in the upstream gene, rather than the genes, as shown in Table S4. This suggests that introns are not present in the genes.

The introns in the genes of the three species of P. cingulata are enriched for introns that are present in the upstream gene, rather than the genes, as shown in Table S5. This suggests that introns are not present in the genes.

The intron sequences in the genes of the three species of P.
