# Comparative transcriptomic analysis of skeletal and soft-shelled Cryptococcus neoformans isolates
Quaglee Dragontacos


## Abstract
Invasive fungal infections (IFIs) are a significant cause of morbidity and mortality in immunocompromised patients. In this study, we compared the clinical and demographic characteristics and resistance profile of IFIs in patients with pulmonary aspergillosis and control arms.

Patients with aspergillosis were studied for 14 days in our institution. Socio-demographics, invasive fungal infections, fungal protein concentration, and fungal protein-metabolic profiles were compared between the bronchiolar aspergillosis group and the control group. Logistic regression analysis was used to identify patients with a mean of. Patients with a mean of.

No. 57.9% of aspergillosis were from the lung and 25.6% of the skin and 31.3% of the nodules in the other group. The bronchiolar aspergillosis group had higher fungal protein concentrations (P < 0.001) and fungal protein concentrations (P < 0.001) in the lung (P < 0.001). Bronchiolitis was more common in patients with resistance to fluconazole (OR 1.05, 95%CI: 1.01-1.12), voriconazole (OR 0.97, 95%CI: 0.84-1.0) or posaconazole (OR 0.97, 95%CI: 0.82-1.0) compared with the control group.

Patient characteristics were unfavorable, unfavorable interferon (IFN)-<U+03B3> screening was a risk factor for aspergillosis.


## Introduction
Cryptococcus neoformans is a major fungal pathogen of humans and animals and causes an estimated annual loss of over 6 million people worldwide (1). This fungal pathogen is a major cause of morbidity and mortality in immunocompromised patients (2-4). Amphotericin B (AMB) has historically been regarded as the drug of choice for the treatment of cryptococcosis. However, AMB is also viewed as an antifungal agent and is the drug of choice for the treatment of invasive pulmonary and cutaneous fungal infections. In contrast, amphotericin B (AMB) has been considered to be an ineffective drug due to the toxicities and a lack of bioavailability of the drug (5).

The development of newer triazoles, which have improved efficacy and lower toxicity, has increased the number of patients who are now recommended for therapy of cryptococcosis (6). Currently, the World Health Organization (WHO) guidelines for the management of cryptococcal disease are based on the combination of echinocandins with the echinocandins. However, the echinocandin resistance of Cryptococcus species remains understudied and has only sporadically been reported in recent years. It is now clear that there is a complex relationship between the echinocandin resistance of C. neoformans and its genetic tractability.

In this context, we herein report the first reported genetic tractability of a clinically-relevant Cryptococcus species, C. neoformans var. grubii, in C. neoformans. We report a high-throughput sequencing approach that allows for the identification of novel genes that were previously identified as genetic tracts, such as genes that encode virulence-related genes.


## Methods
neoformans, . gattii, and C. orthopsilosis). The RBCs were characterized with the Illumina library (Edinburgh Reference Library for Cryptococcus neoformans, H-70, S. cerevisiae, and S. pombe) and a phylogenetic tree based on the C. gattii genome was constructed with the RaxML v.7.8.7 program [29]. C. gattii and C. orthopsilosis RBCs were used as outgroups. To identify the RBCs in the following species, we performed the RNA-Seq reads from the isolates and the genome of C. gattii using the RaxML v.7.8.7 program [29]. The RBCs were identified using the RaxML sequences generated in this study.

Genome sequencing and annotation
In order to determine whether the RBCs in the NCBI database were present in Cryptococcus neoformans or C. gattii, we performed the genome sequencing of each isolate using the Illumina Hiseq2500 instrument. We sequenced the complete genome of C. gattii using the sequencing platforms of the Broad Institute (www.broadinstitute.org/annotation/genome/cryptococcus_neoformans_group/) and the NCBI (http://www.ncbi.nlm.nih.gov/) databases. The sequences were aligned using the manual alignment tool of the software BioEdit v.7.0.9 [30]. For each set of RBCs, we also aligned the RBCs with the genome of C. gattii using the MOTHUR program. The Mothur program [31] was used to align the genomes of Cryptococcus neoformans and C. gattii, and the results were then imported into MOTHUR [32]. The results were then imported into the C. gattii RBC database using the RaxML v.7.8.7 program. The RBCs of Cryptococcus neoformans and C. gattii were annotated using the online annotation tools of the Cryptococcus Genome Database (CGP) v. 2.1.


## Results
The transcriptome analysis revealed that the MtDNA: mtDNA: mitochondrial matrix and mitochondrial proteins genes were significantly enriched in the Cryptococcus neoformans VGIIa isolate compared to the MtDNA: mtDNA: mitochondrial matrix genes (Figure 1. There were no significant differences in the expression of the mitochondrial genes of the nuclear matrix and mitochondrial proteins of mtDNA, mtDNA: mtDNA: mitochondrial proteins and mtDNA: mtDNA: mitochondrial proteins between the two strains (Table 3).

C. neoformans VGIIa isolate is avirulent in a mouse model of cryptococcosis
Since the Cryptococcus neoformans VGIIa isolate was avirulent in the mouse model of cryptococcosis, we next determined the virulence of the two strains in the murine model of pulmonary cryptococcosis. To evaluate the virulence of the two strains in the murine model of pulmonary cryptococcosis, we first compared their pulmonary burden and survival during infection. The survival of the two strains was significantly higher in the lung compared to the brain, while the mortality rate of the two strains was comparable in the lungs (Figure 2). The Cryptococcus neoformans VGIIa isolate was significantly more susceptible to infection than the Cryptococcus neoformans VGIIa isolate and the Cryptococcus neoformans VGIIa isolate was significantly more virulent than the Cryptococcus neoformans VGIIa isolate (p < 0.001, Table 4). The results of survival analysis showed that the survival rate of both strains in the mouse model of pulmonary cryptococcosis was significantly higher in the lungs than in the brain (Table 4).

Pulmonary cryptococcosis induces a host cell inflammatory response
In order to evaluate the inflammatory response induced by Cryptococcus neoformans VGIIa and Cryptococcus neoformans VGIIa, we first evaluated the histopathological changes in the lungs of infected mice. We observed that mice infected with the two strains displayed a large amount of tissue necrosis with a predominantly inflammatory pattern.


## Discussion

Genome sequencing of Cryptococcus neoformans isolates from different geographical locations revealed a total of 14,062 SNPs in Cryptococcus neoformans and 5,093 SNPs in Cryptococcus gattii isolate. Only 496 SNPs were detected in Cryptococcus gattii isolates (11%) and 3,848 SNPs (3.9%) were detected in Cryptococcus neoformans isolates from the same geographical location (Fig 3). Similarly, the distribution of SNPs was not affected by geographic location (Fig 4), indicating that the Cryptococcus gattii isolates studied in this study are adapted to environmental and host changes in different geographic locations.

The genotypic analysis also revealed a total of 5,777 SNPs in both Cryptococcus neoformans and Cryptococcus gattii. This is approximately 5% of the 10,964 SNPs reported in other isolates from different geographical locations (4,404 SNPs in Cryptococcus gattii [19]), and approximately 2% of the 5,238 SNPs reported in Cryptococcus neoformans isolates from South Africa [20], and from Kuwait [21]. Cryptococcus gattii isolates also showed reduced rates of SNPs in the transcriptome analysis of Cryptococcus neoformans isolates from Kuwait [20], indicating that these isolates are not geographically isolated from Kuwait.

There was also a significant reduction in the rate of SNPs in the transcriptome analysis of Cryptococcus gattii isolates from Kuwait (7.2%) and an increase in the rate of SNPs in the transcriptome analysis of Cryptococcus neoformans isolates from Kuwait (6.7%) (Fig 3). This pattern of reduction in the transcriptome analysis indicated that Cryptococcus gattii isolates are less able to adapt to environmental and host changes in Kuwait.

Analysis of the global transcriptome of Cryptococcus gattii, Cryptococcus gattii, and Cryptococcus gattii isolates from different geographic locations revealed a total of 2,072 SNPs (8.3%) in Cryptococcus gattii isolates from Kuwait (Fig 3). These numbers were slightly lower than the 1,100 SNPs reported in Cryptococcus gattii isolates from North America (10%) and Canada (4.
