# Comparison of quantitative real time PCR with Sequencing and ribosomal RNA expression for the identification of Aspergillus fumigatus species in a 2011 case series
Quaglee Dragontacos


## Abstract
Aspergillus fumigatus is a filamentous fungus that is the most common causative agent of invasive aspergillosis in humans. Due to its ubiquitous nature, this fungus has a particularly adapted repertoire of antimicrobial and antioxidant strategies that it uses to disrupt host defenses. We compared transcriptomes between strains grown in the presence of A. fumigatus or when they were isolated from dead aspergilli and found 70 unique transcripts that were differentially expressed between the two strains. Surprisingly, we found significant differences (P < 0.05) in gene expression profiles between the two strains and these differentially expressed transcripts were enriched in networks involved in carbohydrate metabolism. Further, we found those differently expressed transcripts were related to processes that are conserved in different Aspergilli and those involved in plant defense. These results suggest that differences in gene expression between the two Aspergilli may be part of a convergent process that is under constant evolution in both lineages. These results could represent valuable information on the evolution of resistance to modern aspergilli.


## Introduction
We have rephrased the previously mentioned sentence in the tex.

We thank the reviewer for their kind comments. We clarified the previous sentence by correcting the use of the word genus and species in the last sentence.

We have addressed this issue in our revision of A. fumigatus and clarified the taxonomy of the new species. In the revised manuscript, the new species was identified based on morphological characteristics, and an Aspergillus species name was provided.

The genus Aspergillus is morphologically similar to A. fumigatus, and the A. fumigatus strains used for this study were identified based on morphological characteristics and the ability of the isolates to grow in a medium with Aspergillus conidia. A. fumigatus was chosen as the initial species, and strains were selected by phylogenetic analysis based on phenotypic characteristics and the ability of the isolates to grow in a medium with Aspergillus conidia.

We have revised the Discussion and Introduction and revised the Introduction and revised Introduction.

We have further reviewed the literature regarding Aspergillus species and added references from several publications. We have also reviewed the literature regarding Aspergillus species and clarified that the A. fumigatus strains used for this study are identified based on phenotypic characteristics and the ability of the isolates to grow in a medium with Aspergillus conidia.

We have rephrased the sentence mentioned in the Introduction and revised the Introduction.

The authors should address the major points in the subsection headed “Aspergillus species,”

In this subsection, the authors mention Aspergillus fumigatus, and they include the term “aspergillus”. However, A. fumigatus can be divided into two varieties, A and D. The authors should clarify this point.

In this subsection, the authors mention A. terreus, and they include the term “terreus”. However, they should also mention A. niger.

The authors should clarify the other species in the genus, N. crassa, which has been isolated from a variety of environments, such as soil, air, air conditioning units, and water? This would be important and informative.


## Methods
In this study, we used the same primers as for PCR amplification to amplify aspergillosis from the 2009 case serie. The PCR amplification was done in a 96-well plate (Greiner Bio-One, UK) in a total volume of 50 µl. The primers are listed in Table S2 in File S1.

The PCR reaction mixture (Takara Bio-Takara, Shiga, Japan) contained 2.5 µl of the forward primer (5’-GAGAATCCCTCCAGCGTCTC-3’ and 3’-GGAAGAGAAGCATCCCTAATC-3’), 1.25 µl of ExTaq Master Mix, 0.25 µl of the forward primer (5’-GAGAATCCCTCCCTCTCACG-3’) and 0.5 µl of ExTaq DNA polymerase (Vazyme Biotech, Co., Ltd., Beijing, China). PCR reaction was carried out in a Bio-Tek Synergy HT Real-Time PCR Detection System (Bio-Tek, Inc., Winooski, VT, USA). The gene-specific primers were used as internal reference for the amplification of Aspergillus fumigatus as previously described.


## Results
In vitro antifungal activity
Fourteen of the 58 isolates tested against 1.5 mg/mL FK506, 6 of those isolates showed moderate to strong activity against FK506 and 13 showed moderate to strong activity against FK506. No activity against the azole antifungals was detected. The MIC for FK506 was 0.5 µg/ml for all of the tested isolates. All of the isolates showed activity against C. albicans, C. glabrata, C. krusei, C. krusei-staphylococcus, C. dubliniensis, C. dubliniensis-staphylococcal, C. krusei, C. parapsilosis, C. albicans, C. glabrata, C. krusei-staphylococcal, and C. albicans (Table 1).

Antifungal activity
All of the isolates were found to be susceptible to FK506 by MIC. The MIC values for FK506 for C. albicans and C. glabrata are in Table 1. The MIC value for C. glabrata is 0.25 µg/ml. The MIC values for C. krusei are 0.5 µg/ml and 0.5 µg/ml. MIC values for C. albicans are 0.25 µg/ml and 0.5 µg/ml.


## Discussion

In a clinical setting where invasive candidiasis is common, isolation of . fumigatus species is challenging, and DNA is detected only by specific primers for the Aspergillus species. In a large-scale, prospective, clinical study, the results of the present study revealed that Candida spp. were the most frequently detected species in the bloodstream specimens of invasive candidiasis patients (82.5% of patients) compared to 17.1% (4/5) for A. fumigatus. Only A. fumigatus was detected in the bloodstream specimens of patients with invasive candidiasis.

The Cryptococcus species was the most commonly detected species in all patients with candidiasis and in patients with severe invasive candidiasis. However, only one of the patients was also infected by Cryptococcus spp.

The Candida spp. species was the most commonly detected species in the bloodstream of patients with candidiasis. However, only a single patient was infected by Candida spp.

C. neoformans was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only one patient was infected by C. neoformans.

The Candida spp. species was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only a single patient was infected by Candida spp.

The Candida spp. species was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only one patient was infected by C. neoformans.

The Candida spp. species was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only a single patient was infected by C. neoformans.

The Candida spp. species was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only a single patient was infected by C. neoformans.

The Candida spp. species was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only a single patient was infected by C. neoformans.

The Candida spp. species was the most commonly detected species in the bloodstream samples of patients with candidiasis. However, only a single patient was infected by C. neoformans.

The Candida spp.
