# Conserved lipolytic enzymes of the tomato Talaromyces marneffei under aerobic conditions and their rDNA rl
Quaglee Dragontacos


## Abstract
The first is the switch from the sexual reproduction of the typical single-celled yeast to a multilamellar heterokaryo. •The second is the transformation from meiosis to fission yeast. •The switch from haploid, unaltered haploid to haploid sporangia results in the existence of a single superfamily chromosome.

The fission yeast Schizosaccharomyces pombe undergoes two major transitions in molecular biology. The first is the switch from the sexual reproduction of the typical single-celled yeast to a multilamellar heterokaryon.

The second is the switch from the multilamellar heterokaryon to fission yeast.

The switch from haploid, unaltered haploid to haploid sporangia results in the existence of a single superfamily chromosome.


## Introduction
The black yeast Candida glabrata is a member of the normal microbiota in healthy humans, but can develop into a pathogenic fungus, causing systemic infections that can be fatal. Black yeasts can be classified into three groups, namely, commensal and pathogenic, and ocular candidiasis [1,2]. The commensal group is mainly caused by commensal yeast species, while the ocular candidiasis group is mainly caused by ocular candidiasis. These three groups are distributed in the human body, but only certain individuals can form a true blood group, and are associated with the development of blood and organ infections [3].


## Methods
The rDNA gene was also used as a DNA template in the calculations of the rlps1 gene expression profile using the Eukaryotic RNA-Seq database (http://mole.jgi.doe.gov/TriDDB/TriDDB.html) (Table 1) and for the transcriptional level analysis in the determination of the bactericidal activities of the cytotoxic activity of p-nitrophenyl (p-NPP) and p-nitrosanil (p-NPS), a commercially available fungal protease inhibitor cocktail (Roche Diagnostics, Indianapolis, IN, USA) in general (Table 1).

S. cerevisiae PNPD1 (Fungi and bacteria) was used as a reference for cloning and transformation of P. marneffei (GenBank Accession No. XP_001890406). Cells of S. cerevisiae were grown in Luria-Bertani (LB) medium (200 rpm) at 30°C for 2 days and then subjected to S. cerevisiae transformation and expression analyses. The S. cerevisiae transformation method was modified by introducing the p-nitrophenyl (p-NPP) method. The transformation mixture was composed of 1 × 10^6 cells of S. cerevisiae and 10 mM of p-nitrophenyl. The S. cerevisiae cells were grown in Luria-Bertani (LB) medium for 1.5 days at 30°C. The p-nitrophenyl (p-NPP) was added into the cells at the indicated concentrations (0, 1, 2, 3, 4, and 5 µg/mL). The S. cerevisiae cells were washed twice in PBS and resuspended in PBS. The p-nitrophenyl (p-NPP) was then added into the cells at the indicated concentrations (0, 1, 2, 3, 4, and 5 µg/mL). The P. marneffei transformants were grown in LB medium at 30°C for 6 h, and then incubated with p-nitrophenyl (p-NPP) and P.


## Results
cereus, . deneoformans, C. krusei, and C. albicans

The Talaromyces marneffei Talaromyces marneffei genome was previously reported to contain two genes (pfam0119 and pfam0120) which are known to be involved in the metabolism of the secondary metabolites, steroids, and aryl hydrocarbons [12], [13]. The P. marneffei Talaromyces marneffei gene deletions in B. cereus and C. deneoformans were performed using the pfam0119 and pfam0120 pfam0020512 null mutants and the pfam0119 pfam0020512 null mutant strains, respectively. The Talaromyces marneffei gene deletion strains (pfam0119 and pfam0120) are predicted to have mutations in the second and third coding sequences of the two genes in the pfam0119 pfam0120 gene (pfam0119-1 and pfam0120-2) respectively. P. marneffei strain Pma0102C in the pfam0119 pfam0120 gene deletion background is predicted to have a mutation in the fifth coding sequence of the rDNA gene in the pfam0119 pfam0120 gene (pfam0119-1). The pfam0119-1 gene has a doublet in the 5' UTR, which is predicted to encode a signal peptide and to have an N-terminal region with a premature stop codon. Pma0102C is predicted to encode a protein of 462 amino acids (aa) and the pfam0119-1 gene has a single intron (pfam0119-1/5). Both pfam0119-1 and pfam0119-2 have two introns in the N-terminal region (pfam0119-1/1) and contain a single histidine residue (pfam0119-2/5). Both genes are predicted to encode a putative a-helices (pfam0119-2/5). The pfam0119-2/5 gene contains a predicted signal peptide (pfam0119-2/5) and a predicted protein of 624 amino acids (aa).


## Discussion
These are the first reports of unique Rlms in tomato roots and root tissue.

The Rlms identified in this study have been previously described for other species in the plant kingdom, such as Alternaria and Phytophthora infestans (10). Rlms have also been shown to have specific activities in fungal pathogens, such as the ability to induce disease in S. cerevisiae (29) and to colonize rhizoplane soil (30). Rlms also have been reported to induce root-associated fungal infection (31, 32). The plant species identified in this study, however, did not exhibit any Rlms in the rhizosphere soil community, indicating that the Rlms in this study may not be able to provide optimal growth conditions for fungal growth in the rhizosphere soil.

The Rlms identified in this study also have been shown to induce strong plant-microbe interactions, such as the plant-pathogen interaction (Pseudocercosporellus scottii), which can also be found in the rhizosphere soil community. This indicates that Rlms may also be involved in a variety of interactions in the rhizosphere soil community. Rlms are known to be involved in the fungal cell wall integrity (24). This is supported by previous reports of Rlms in the roots of tomato plants (15, 16). Rlms have been reported to be involved in the glycolysis (gluconeogenesis) pathway in the rhizoplane soil community (31). The glycolysis pathway is one of the main pathways that is involved in the initial steps of the glycolysis and glyoxylate cycle (32). Rlms can also be involved in cell wall integrity (cellular integrity), which is also involved in the host-pathogen interaction (Pseudocercosporella geotrichum) (33). The ability to induce plant-microbe interactions also supports the ability of plant-microbe interactions to stimulate and develop mutualistic interactions (34).

The Rhizosphere soil community of tomato roots is comprised of different bacterial taxa. These bacterial taxa may be found in both the rhizosphere soil community and the rhizosphere soil communities of tomato plants.
