# CRISPR/Cas9-Mediated Gene Disruption in Rice Mycoplasma ruminantium Highlights Genes Involved in Fungal Virulence
Quaglee Dragontacos


## Abstract
Two species of the genus Fusarium (F. bipolarus and F. chrysogenum) are found in the Fusarium graminearum-dominated ectomyosphaera (F. bipolarus) population in the Netherlands. They are morphologically similar to those of F. graminearum, F. subglobosum, F. subglobosum-intervalum, and F. membranifacium. Although the morphological characters of F. subglobosum and F. membranifacium are distinct, the main molecular characters of F. subglobosum are congruent. Conidia of F. subglobosum are larger and have wider, higher basidia cut off by short filaments of fused hyphae. Both species also have anterior fossa running from the base of the basidiospores to the apical regions of the hyphae. F. subglobosum is characterized by a short, hyaline, pseudomonadous, and with pigmented, hypogeous ascomornate hyphae that generally have a longer, sinuous appendage that becomes longer when mature. F. membranifacium is characterized by a long, sinuous appendage that becomes shorter when injured and without a filament, while F. subglobosum is characterized by a long, sinuous appendage that becomes shorter when injured, and with pigmented, hypogeous ascomornate and hypogeous ascomornate hyphae that generally have a longer, wider basidiospores that run parallel to the base of the basidiospores. The genomes of the species were compared using alignments, and the results showed that the four major molecular characters (ATPases, Adhesion and Transcriptional regulation) were significantly different in the two F. subglobosum species. The comparison of these species revealed that F. subglobosum had the most extensive and elaborately designed conidial organelles and F. subglobosum had the least. F. subglobosum strains from the Netherlands displayed significantly reduced conidial sizes compared to the genomes of the two F. subglobosum strains.


## Introduction
The tree of phragmites (Zingiber officinale) is a highly destructive phoretic genus, and some species are associated with phoretic trees [1].


## Methods
Yeast strains and plasmids
A total of 8 of the 12 Candida spp. were used in this study.


## Results
Identification of fungal orthologues of the P. brasiliensis R. oryzae PbSOD1 gene
In previous phylogenetic studies, the R. oryzae PbSOD1 gene was only identified in a single clade of P. brasiliensis sequences [15], [18], [19].


## Discussion
The gene-deleted mutant TATA-LTR2 has been reported to be virulent in a rice blast fungus, but not in a human fungal pathogen [53. The analysis of TATA-LTR2 in the rice blast fungus T. reesei also revealed its high transcript levels in response to stress conditions, such as nitrogen and temperature [54].

The TATA-LTR2 in the rice blast fungus T. reesei is highly expressed in response to nutrient and stress conditions. The expression levels of TATA-LTR2 in rice mycelia and the rice blast fungus T. reesei were found to be comparable to those in the mycelial stage of the fungal pathogen, but to a higher level in the rice blast fungus, suggesting that TATA-LTR2 could be involved in the pathogenicity of T. reesei.

TATA-LTR2 was found to be highly expressed in the rice blast fungus T. reesei. The transcript levels of TATA-LTR2 in rice mycelia were found to be significantly higher than those in the rice blast fungus T. reesei. Moreover, the transcript levels of TATA-LTR2 in rice mycelia were higher than those in the rice blast fungus T. reesei, indicating that TATA-LTR2 could be involved in the pathogenicity of T. reesei. The results of this study suggest that TATA-LTR2 is required for the biosynthesis of TATA-LTR2 in rice.

The transcript levels of TATA-LTR2 in rice were higher than those in the rice blast fungus T. reesei. The transcript levels of TATA-LTR2 in rice were significantly higher than those in the rice blast fungus T. reesei, indicating that TATA-LTR2 could be involved in the biosynthesis of TATA-LTR2 in rice.

Genome sequencing of the 5â€™-tubulin-containing protein TATA-LTR1 encoding gene was performed to examine the genes involved in cell wall and cell membrane-degrading pathways.
