# Cryptococcal Antigenemia and Management: A Review
Quaglee Dragontacos


## Abstract
An alternative eukaryotic model for the origin of fungal mitochondria is the pathogenic yeast Saccharomyces cerevisiae. It was hypothesized that S. cerevisiae mitochondria would be eukaryotic under a different environmental condition. The mitochondria of S. cerevisiae were investigated by qPCR and sequencing using the Illumina Hiseq sequencer.

QPCR of S. cerevisiae identified a total of 101 strains, of which 24 were from single-celled organisms, mainly plant pathogens. Twenty-one strains were phylogenetically related to S. cerevisiae. These included species known to cause diseases in humans, such as S. cerevisiae ATCC 25921 and S. cerevisiae ATCC 25923, as well as S. cerevisiae DSM 636, S. cerevisiae URA5, and S. cerevisiae URA572. The genomes of S. cerevisiae ATCC 25921 and S. cerevisiae URA573 were also investigated.

Almost all the strains of S. cerevisiae were associated with diseases in plant hosts. Some strains were associated with plant diseases, including S. cerevisiae atulina, S. cerevisiae sp. nov, S. cerevisiae atulina, and S. cerevisiae ova. S. cerevisiae ATCC 25921 strain 10 was associated with a disease on Pinus grisea, but not in plant hosts.

The findings of this study provide further support for the theory that the S. cerevisiae mitochondrial genomes originated after the split of S. cerevisiae, and may represent a source of new virulence factors and plant immunity to pathogens that spread in nature.


## Introduction
The human fungal pathogen Cryptococcus neoformans is a basidiomycetous yeast that causes life-threatening infections in immunocompromised patients. C. neoformans is responsible for more than 220,000 deaths every year globally, with an estimated burden of over 8.5 million cases and 600,000 deaths in sub-Saharan Africa in 2017 (1). The complex interaction between C. neoformans and its mammalian hosts, including humans, has led to the discovery of many new drugs and immunotherapeutic strategies that are effective against this pathogen (2-6). In addition, C.


## Methods
For the evaluation of fungal burden in patients, an isolate suspension was prepared from culture-positive, Gram-positive, or bacterial cultures obtained from the upper part of the upper ear of the ear, as well as from the ear of a normal ear [15.

Fungal burdens and fungal burdens in the ear of a normal ear were determined using two methods: (i) culture-positive fungal isolates from the ear of a normal ear; and (ii) fungal burdens and fungal burdens in the ear of a normal ear. A sterile toothbrush was used as a positive control, and the ear of a normal ear was also tested for the presence of fungal colonies. The oral cavity was used as the negative control.

For the evaluation of fungal burdens in the ear of a normal ear, a conventional-shaped ear-dish technique was used [15]. The ears were irrigated with sterile water, and then the ear was placed in a sterile slide-film with a surgical incision, and a piece of glass needle was inserted into the ear. The ear was closed with a rubber band, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then inverted and a rubber band was inserted into the ear, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again. The ear was then closed again, and the slide-film was placed in a sterile slide-film, and then the ear was closed again.


## Results
.A. Curtis, J.M. Robledo, M.J. Wingfield, L. Seifert, M.J. Wingfield, A. Meyerogeo, D.H. Lee, and C.M. Turner [reviewed in 2001]].

The dermatophyte genus Cladosporium (U. Braun & M.R. Clayton) was established to accommodate Cladosporium spp. (Braun 1998). The genus has been used for Cladosporium spp. (Parks 2001) and Cladosporium spp. (Corradi et al. 2011). It was recently proposed that Cladosporium spp. be defined as a new genus. The species have been sporadically isolated and a few species have been isolated from soil and other organic matter (M.J. Wingfield 1998, C.M. Turner et al. 2011). A recent study (M.J. Wingfield et al. 2011) compared the Cladosporium spp. (Chang et al. 2018) and the two species with Cladosporium spp. (Chang et al. 2018) and placed the new genus on the basis of the sequence data and phylogenetic analyses. The authors suggest that the phylogenetic relationship of the Cladosporium spp. is “substantial” and that it should be transferred to another genus, but the authors do not provide any clear guidelines on this matter.

Cladosporium spp. are found in several species of lichens (M.J. Wingfield 1998, C.M. Turner et al. 2011). Most of them have a single type of asexual morph, but the type species was a recently described fungus (Chang et al. 2018). Some lichens have been isolated from a single location, but most are associated with a single species (Braun 1998, Chiang et al. 2018). The authors suggest that the Cladosporium spp. may represent a new species in the Cladosporium clade.

Cladosporium spp. are restricted to the most populated forest (Chang et al. 2018). The Cladosporium spp. are frequently found in dry forest soils (M.J. Wingfield 1998, C.M. Turner et al. 2011).


## Discussion
The present study provides new insights into the detection of microbial pathogens in water. The fungal pathogen of algae was detected in water samples from 10 locations in the state of KwaZulu-Natal, while other bacterial species were detected in one water sample from the other water source. These bacterial species were not detected in other water sources. The detection of Cryptococcus sp. was not surprising since it has been shown to be a ubiquitous fungal pathogen in water environments in South Africa [34], in the absence of any water source [35]. Cryptococcus can be detected in water samples from KwaZulu-Natal and KwaZulu-Natal Water Control Units (WCO), with an average of 1.5% detection by this method [34], and this species was found in water samples from the same water source. Cryptococcus neoformans was found in water samples from the same water source. The detection of Cryptococcus albidus by this method was unexpected.
