# Ctr2 Cross-Consanguineous Asian Lumbar Fungi: New and Old Friends
Quaglee Dragontacos


## Abstract
The current and most widely used genetic exchange platforms are mostly created by genetic exchange between individuals using known or potential species. While it is known that some species have evolved a few additional species exchange strategies. These include complex multicellular populations with a large number of exchange strategies. This review will discuss the current and most commonly used genetic exchange platforms, and discuss what role they play in genetic exchange. They also provide information about how to identify and manipulate these new exchange platforms, and how to implement them in genetic exchange studies. The importance of these new exchanges in genetic exchange studies will be discussed, and the effect that the new exchange platforms have on genetic exchange in the wild can be visualized.


## Introduction
Fungi belonging to the genera Trichosporon, Pseudallescheria, and Curvularia were reported as important groups of dermatophyte. In the last decade, a few species were reported to cause serious infections in patients with leishmaniasis, ranging from cutaneous (e.g. T. chrysosperma) and visceral disease, to deep-seated systemic infections (e.g. T. inkinii, T. krechigati, T. interdigitale, T. solstitialae, T. aurantiacum). Although a few dermatophyte species were reported as causative agents of visceral diseases (e.g. T. krechigati, T. interdigitale), most of them are now regarded as parasitic or saprobic. For example, the species of Talaromyces (T. solstitialae) and the dermatophyte Exophiala (E. delemar) that cause the skin lesions of leishmaniasis, are now known to be saprobic on both superficial and deep sites (Vandenkoornhiana and Skowy, 2007; Zordan et al., 2011). These species are found in the warm-blooded animals, especially in mammals, as the parasites live as free-living, biparental diploid cells (Khan, 2000; Ganz and Ganz, 2009). After the successful isolation of T. solstitialae from skin lesions, the first reports of the mycological importance of this parasite were published in the 1980s (van der Aa 1973). Most of the current studies have focused on the Trichosporon spp., although other species are known to infect skin. These include T. krechigati, T. interdigitale, T. candidum, and T. solstitialae (Pagani et al., 2016).

The genus Trichosporon includes four species: Trichosporon dermatis, T. candidum, and T. candidum. They are usually saprobic on the skin, although they can also infect other tissues such as the nails, hair, and nails (Khan, 2000; Ganz and Ganz, 2009).


## Methods

RNA sequencing and analysis
For RNA sequencing, RNA was isolated from a mycelium of . oleaginosus using the Qiagen RNeasy kit following the manufacturer’s protocol. First-strand cDNA was synthesized using SuperScript III reverse transcriptase (Invitrogen, Carlsbad, CA) according to the manufacturer’s instructions. Subsequently, synthesized cDNA was used to obtain cDNA samples using the SuperScript III RT High Fidelity cDNA synthesis kit (Invitrogen, Carlsbad, CA). For qRT-PCR, cDNA samples were amplified using a 7500 Real-Time PCR System (Applied Biosystems, Foster City, CA) using SYBR Premix Ex Taq (Tli RNaseH Plus; Invitrogen, Carlsbad, CA). The following primers were used: RT-qPCR, forward: 5'-CTCATCGGCTCAGCCACCTATC-3' and reverse: 5'-GATGGAGTTGAGTTCCTCTCCA-3'. Each cDNA sample was normalized to the expression of GAPDH in the control control (WT) cells. For all samples, the “Noise-Expert” labelling assay was performed in a BioRad CFX96 Touch Real-Time PCR Detection System using the BioRad CFX Manager software version 3.0 (BioRad, Hercules, CA). Each sample was normalized to the expression of GAPDH in the control (WT) cells. The data were acquired using an Agilent Bioanalyzer and the data were analyzed using the BioRad CFX Manager software version 3.0. The expression level of GAPDH was calculated by using the GAPDH-seq analysis tool of the C. neoformans var. grubii H99 reference genome [30]. For the qRT-PCR analysis, the genes with the highest level of expression were selected from the qRT-PCR results, as previously described [31]. The gene expression level of the gene in the control (WT) cells was also calculated.


## Results
Na. Acad. Sci. Philadelphia, No. 102, F. Liu, L. Cai & Crous, Stud. Mycol. 83: 40. 2015.

Culture characteristics: Colonies on OA, 60-70 mm diam after 7 d, with margins undulate, whitish to pale olivaceous-brown, whitish due to aerial mycelia, reverse olivaceous-black, velvety, margin colourless to pale greyish grey, glabrous, aerial mycelia abundant, loose to dense, loose to dense, loose to dense, growth effuse, growth flat with somewhat prominent raised, somewhat furrowed, whitish to pale olivaceous-brown, sporadically sporadically producing conidia; sporadically producing small, spherical, ellipsoid, apical conidia.

Material examined: China, Tibet, Lushan, on leaves of Camellia sinensis, 14 Sep. 2012, F.-Y. Bai, HMAS 246167 (holotype CGMCC 3.2724^T preserved in a metabolically inactive state, ex-type CBS 15522 = M16.2).

Notes: We have here designated C. boeremia Crous (as Ctr2) as lectotype here. This species is associated with Camellia sinensis (Table 2).

Coniella alpina Crous, sp. nov. MycoBank MB822820. Fig 20.

Etymology: Name derived from the host genus Alpina, from which it was collected.

Conidiomata pycnidial, globose, semi-immersed, scattered, semi-immersed, semi-subglobose, dark brown to black, semi-immersed to erumpent, subglobose to flask-shaped, 180-240 × 120-140 µm, (x¯ = 313 × 246 µm, n = 20), solitary, semi-immersed, ostiolate. Ostioles single or sometimes aggregated, slightly to distinctly protuberant, not or only slightly papillate, with distinct periclinal thickening. Pycnidial wall pseudoparenchymatous, composed of brown, flattened polygonal cells. Conidiogenous cells phialidic, hyaline, smooth, subcylindrical, 3-7 × 2-3 µm.


## Discussion
In the current study, it was discovered that the CP gene of . pullulans (Gnt1) encodes a gene required for asexual reproduction. This was also reported for other fungal species (Kwon-Chung and Chen, 2015). P. pinicola (CBS 240) encodes a gene encoding a gene encoding a putative endocytic component, which is also involved in sexual development. This is the first report of a sexual cycle for P. pinicola.

In addition to the genome of P. pinicola, we also found that another gene of P. pullulans (Gnt1) encodes a putative endocytic gene. The putative gene encodes a putative endocytic component, which was previously reported for other fungal species (Kwon-Chung, Kwon-Chung and Chen, 2015). This gene is also involved in the synthesis of a melanin pigment (Kwon-Chung et al., 2014).

In addition to the genome of P. pinicola, we also found that P. pullulans (CBS 240) encodes a gene encoding a putative endocytic gene. The putative gene encodes a putative endocytic component, which was previously reported for other fungal species (Kwon-Chung, Kwon-Chung and Chen, 2015). This gene is also involved in the synthesis of a melanin pigment (Kwon-Chung et al., 2014).

The genomic analysis revealed that the five genes that encode for putative endocytic genes are located in chromosome 2, which is closely related to the chromosome 6B and chromosome 3 (Fig 3). This chromosome 2 contains one additional gene (Cdc42) that encodes a putative endocytic gene. These genes are linked by a protein, called a Dicer. The Dicer protein encodes a Dicer-like protein. This Dicer protein has been reported previously for P. melanosporum, P. syringae, P. pustula, P. syringae and P. viticola. It was also found that P. syringae encodes a Dicer-like protein (Kwon-Chung et al., 2014). P.
