# Draft Genome Sequence of Plasmodium falciparum, a Brazilian Pathogen That Affected Fusarium subsp. falcilla Hosts
Quaglee Dragontacos


## Abstract
In this study, we verified asexual asciophiles as well as aneuploids in a large number of Candida parapsilosis isolates from various countries. This included a large number of non-Candida spp. from Korea, Vietnam, and Costa Rica. A number of these non-sexual non-susceptible Candida parapsilosis isolates were also detected in the same group as C. albicans. In addition, asexual asciophiles were also detected in the sexual crosses between Candida glabrata and Candida krusei. We also investigated whether sexual aneuploidy changes correlated with the sexual development of these isolates. These studies show that asexual asciophiles and non-ascomycetes may be closely related, but the sexual development of these asciophiles may be different. We also concluded that the in vitro sexual development of non-ascomycete Candida parapsilosis may be governed by the complement system, which may be a more efficient strategy for the management of Candida infections in humans.

In this study, we verified asexual asciophiles as well as non-ascomycete yeast species as a group of causative pathogens in: C. parapsilosis; Candida albicans; Cryptococcus neoformans; and Zygomycetes ileoascolarii.


## Introduction
Cryptococcosis is an infection caused by yeasts belonging to the genus Cryptococcus. The genus Cryptococcus includes about 1.5 million species (http://www.cryptococcus.org/). The incidence of cryptococcal meningitis is ~50% among immunocompromised patients (1). The most common opportunistic pathogenic agents are Cryptococcus neoformans (Cn) and C. gattii (2, 3). Cn is a leading cause of morbidity and mortality in AIDS patients, while C. gattii is increasingly recognized as a primary pathogen (4). Cryptococcus neoformans is the most common species in this genus and is responsible for 15% of all cryptococcosis cases in patients with AIDS (5). In contrast, Cryptococcus neoformans and C. gattii are more common etiologic agents of infection in immunocompromised patients. The infection is primarily initiated by inhalation of infectious propagules, which produce a disseminated disease that is fatal if not treated (6).

Cryptococcus neoformans and C. gattii are found worldwide, including Vancouver Island, British Columbia, and the islands of Micronesia, and are responsible for most disease in otherwise healthy individuals. The two species are most commonly isolated in patients with an intact immune system and immunosuppression, but infections due to C. gattii are increasingly recognized as opportunistic infections in immunocompromised patients (7).


## Methods
In order to validate the findings, the genome of the pathogenic fungus has been submitted to the National Centre for Biotechnology Information (NCBI) (accession number DN0669047) for annotatio. The genome sequences of the pathogenic fungus are available from NCBI under accession numbers MF653527-MF6527.

Genome annotation
The sequenced genomes of the pathogenic fungus species were aligned with Muscle v. 1.12.18 (Edgar 2004) using default parameters. The gap filling algorithm was used to correct the positions of ambiguous positions. The protein coding sequences of the pathogenic fungi were predicted with SignalP v. 1.6 (Petersen et al. 2012).

Phylogenetic analyses
For phylogenetic analyses, a neighbor-joining (NJ) phylogeny was constructed using the maximum-likelihood method (Lehnken et al. 2013). In this phylogeny, we used the program MEGA v. 7.0 (Kumar et al. 2016) to study the clade structure of the pathogenic fungi. For this analysis, a phylogeny was constructed based on the combined data of the protein coding sequences of the pathogenic fungi using the neighbor-joining method. The species were selected from a phylogeny that was composed of six genera, namely, Cladosporium, Leptosporium, Toxoplasma gondii, Trichoderma, Fusarium, Rhizopus, Penicillium, Toxoplasma, Tilletia, and Ustilago maydis. A total of 20,000 bootstrap replicates were performed. For the NJ analysis, a tree-bisection-reconnection method was used.

For phylogenetic analyses, we used the maximum-likelihood method (Kumar et al. 2016). In this analysis, a phylogeny was constructed based on the combined data of the protein coding sequences of the pathogenic fungi. Two trees were created based on the gene tree of the pathogenic fungus. A total of 10,000 bootstrap replicates were performed. A NJ tree was constructed based on the combined data of the protein coding sequences of the pathogenic fungi. Three phylogenies were constructed based on the gene tree of the pathogenic fungi. The phylogeny was derived from the single most frequent protein coding sequences of each species using the program MEGA v. 7.0 (Kumar et al. 2016).


## Results

Phylogenetic Analysis of Phylogenetic Plasticity
In order to analyze the relationships among the sampled loci, we compared the phylogenetic distribution of the individual loci of the . falciparum genome with the corresponding loci of the P. lutzii genome (Figure 2). For each locus, a signal-likelihood ratio (R^2) was calculated using the Bayesian neighbor-joining method (Park et al., 2011). The R^2 was set as 0.5, and the R^2 value of the Bayesian method was set as 0.3, with the minimum R^2 value of 0.1. The Bayesian posterior probability of the R^2 value of the Bayesian method was set at 0.65, and the maximum R^2 value of the Bayesian method was set at 0.69.

The results of the Bayesian analysis of the P. falciparum genome are shown in Figure 3. The phylogenetic analysis shows that P. falciparum is a typical Plasmodium species with an integrated basal lineage, and is therefore well placed to be treated as a Plasmodium species (Figure 3). The phylogenetic analyses of the P. falciparum genome also indicate that the fungus has a basal lineage that is distributed over the whole Plasmodium clade.

Phylogenetic Distribution of Phylogenetic Plasticity
The Phylogenetic Distribution of Phylogenetic Plasticity (PD) of the P. falciparum genome is shown in Figure 4. The results indicate that the P. falciparum genome has a basal lineage that is distributed over the whole Plasmodium clade. The phylogenetic analysis shows that P. falciparum is a typical Plasmodium species with an integrated basal lineage, and is therefore well placed to be treated as a Plasmodium species (Figure 4). The phylogenetic analysis also indicates that P. falciparum is a typical Plasmodium species with an integrated basal lineage. The phylogenetic analysis also shows that P. falciparum is a typical Plasmodium species with an integrated basal lineage, and is therefore well placed to be treated as a Plasmodium species (Figure 4).


## Discussion
A total of 16,051 contigs were assembled (Table S1. The genome of Plasmodium falciparum was mapped onto the S. mansoni GISIMS genomic contig collection, yielding a total of 31,377 contigs (Table S2). This assembly yielded two large contigs, covering 1,152 kb (Table S2). This resulted in a genome size of approximately 50 Mbp (Table S2). Genome sequencing of Plasmodium falciparum was conducted using Illumina MiSeq platforms (GenBank accession number JX67963, NCBI accession numbers EU246062 and EU246064) and successfully sequenced (GenBank accession number JX67962) the genome of the pathogen. The genomes of the two closely related species, Plasmodium falciparum and P. falciparum var. falciparum, were sequenced to determine the genomic contents and position of each contig in each species. The results from genome sequencing of Plasmodium falciparum were concordant with the Illumina annotations. The genome sizes of the closely related species, P. falciparum and P. falciparum var. falciparum were approximately 10-20 Mbp (Table S2). The genome of P. falciparum var. falciparum was estimated at approximately 5,600 Mbp with a size of approximately 11 Mbp (Table S3). The genome of P. falciparum var. falciparum was estimated to be approximately 9 Mbp (Table S3). Genome sequencing of P. falciparum was successful and resulted in a genome size of approximately 1,200 Mbp (Table S3). A total of 12,936 contigs were assembled (Table S3). The genome of P. falciparum var. falciparum was estimated to be approximately 6 Mbp (Table S3). Genome sequencing of P. falciparum was successful and resulted in a genome size of approximately 8 Mbp (Table S3).
