# Dual-thermalionic dioxygenase expression regulated by GATA receptor 8 and protein expression in activated microglia promotes echinocandin resistance
Quaglee Dragontacos


## Abstract
The Natural Resource could not be more appropriate requisitioned for oil and gas exploration. The country is currently in its third decade of geothermal energy development, and exploration for tarzan oil and ginsengtarzan gum are two highly desirable natural resource fields worth over US$3.3 billion dollars and US$750,000 dollars respectively. Despite the success of these two promising oilsands activities, there is a dearth of qualified, proven oil and gas resources in the respective resource areas. The tarzan oil field in southern China was selected because it produces both hydrocarbons and sulfur and because it can be effectively used as a raw material. To the best of our knowledge, this is the first study exploring the feasibility of extracting tarzan oil from tarzan gum using different techniques, including hydrocarbon assimilation, gel degradation and ion exchange chromatography.


## Introduction
Invasive fungal disease (IFD) is an important cause of morbidity and mortality in immunocompromised patients, particularly those with acquired immunodeficiency syndrome (AIDS). [1], [2] Invasive aspergillosis (IA) is caused by fungal pathogens including Candida albicans, Candida glabrata, and Candida tropicalis. The epidemiology of IA is complex and varies depending on the geographical region, patient population, and patient population. The current global estimation of the incidence of IID in Africa was estimated at 223,700 cases [3], [4] and by the World Health Organization (WHO) estimates of the incidence in Africa at 10,000 cases annually [5] (Table 1) there is a high prevalence of IID in Africa. [6], [7]

The African continent has an extremely resource-limited health system.


## Methods
The relative expression of GATA receptor was determined by the microarray experimen.

RNA-seq and RNA-seq library preparation
The RNA-seq library was prepared using the TruSeq RNA Sample Preparation Kit v2.1 (Illumina, San Diego, CA, USA) following the manufacturer’s recommendations. The RNA-seq libraries were sequenced on an Illumina HiSeq 2000 instrument and then quality checked using Agilent Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA). The RNA-seq libraries were used for the annotation of the dendritic cells (CMCs) in S. cerevisiae, and the RNA-seq libraries were used for the study of cell-to-cell interactions. The RNA-seq libraries were sequenced on an Illumina HiSeq 2500 instrument and the data were submitted to NCBI, the release of which was deposited in the Gene Expression Omnibus (GEO) with the accession number GSE111259.

RNA-seq data processing and data quality control
The RNA-seq data were processed with the RNA-seq-Qiagen mRNA-seq platform. In brief, each sample was sequenced in two lanes on an Illumina HiSeq 2500 instrument using a MiSeq platform (Illumina, San Diego, CA, USA). To identify and filter the reads, the quality of the reads was examined using the FastQC tool (http://www.bioinformatica.com/bbtool/) and the sequences were also filtered for a quality score below 20 using Trimmomatic (http://sourceforge.net/projects/trimmomatic/) [32]. Reads were then mapped to the NCBI GEO database using bowtie2 [33]. Reads with an average quality score below 20 were discarded. Differential expression analysis was performed with DESeq2 [34]. Differentially expressed genes were identified based on the Pearson correlation between the differentially expressed genes (DEGs) and the log_2 transformed relative expression value [35]. The quantitative gene set of interest was used as a reference of the whole set of genes in a biological process model.

ChIP-seq analysis
ChIP-seq data was processed with the ChIP-seq v1.5 software [36].


## Results
albicans
To test if activation of microglia with GATA receptor 8 promotes echinocandin resistance, we analyzed the expression of echinocandin in microglia stimulated with an echinocandin-based mediu. By quantifying the expression of echinocandin in microglia of the wild type, we found that 6.9% and 4.7% of echinocandin-responsive microglia, respectively, were able to elicit a response against an echinocandin-based medium, respectively (Fig 4A and B). After exposure to 10 µM echinocandin, these microglia showed a high level of echinocandin-responsive microglia (Fig 4C). These results suggested that activation of microglia by GATA receptor 8 promotes echinocandin resistance in C. albicans.

To further confirm this conclusion, we analyzed the expression of echinocandin-responsive microglia in wild type, gt<U+0394>, and gt<U+0394> gt<U+0394>/<U+0394> cells. We found that expression of echinocandin-responsive microglia in wild type cells was strongly reduced compared to that of gt<U+0394> cells (Fig 4D), indicating that activation of microglia by GATA receptor 8 promotes echinocandin resistance in C. albicans.

Microglia activated with GATA receptor 8 promote echinocandin resistance in C. albicans

We examined the response of echinocandin-responsive microglia to the GATA receptor 8 agonist gt<U+0394>, and found that activation of microglia by GATA receptor 8 promotes echinocandin resistance in C. albicans (Fig 5). We also analyzed the expression of echinocandin-responsive microglia in wild type, gt<U+0394>, and gt<U+0394> gt<U+0394>/<U+0394> cells, and found that activation of microglia by GATA receptor 8 promotes echinocandin resistance in C. albicans (Fig 5). These results indicated that activation of microglia by GATA receptor 8 promotes echinocandin resistance in C.


## Discussion
cerevisiae [12, 14. In C. albicans, a “one-way (either one-way)” search for GATA binding sites and de novo binding sites between the two is required for activation of GATA-binding complex genes and adaption to caspofungin [52]. In S. cerevisiae, a “one-way (either one-way)” search for GATA binding sites and de novo binding sites leads to “strong” activation of GATA-protein genes [53]. In Cryptococcus neoformans, GATA-binding sites are located in a domain-containing protein-protein interaction domain (DSF)-like subdomain of the de novo GATA-protein binding complex [54]. In addition to the previously characterized GATA-binding complex genes, GATA-binding sites are found in multiple other proteins including homologs of P-type ATPases, promoters of the transcription factor P-type ATPases, and GATA-binding proteins, including the pks1 gene and pks2 gene [55].

The GATA-binding complex gene repertoire is expanded in C. neoformans

A total of 45 GATA-binding genes have been identified in C. neoformans, compared to 46 in C. albicans (Figure 5 and Table 2). The largest category of GATA-binding genes, which includes the GATA-binding complex gene families, is the de novo GATA-protein-protein interactions (GATAF) gene family, which is composed of six gene families (Table 2). These GATAF genes are encoded in eukaryotes with only a single GATAF gene in C. albicans. To compare the GATA-binding complex gene repertoire in C. neoformans to C. albicans, we compared the repertoire of GATA-binding complex genes in C. neoformans to C. albicans (Figure 6). The C. neoformans GATA-binding complex genes are clustered into a single cluster. In the present study, only one de novo GATA-binding complex gene family was identified in C. neoformans, including the GATA-B box-binding complex and GATAF family members. Since de novo GATA-binding complex genes are generally found in the C. neoformans genome, they represent a good candidate for further characterization.
