# Dual DNA barcoding of Nesmithia bifida: a proof of concept built right at the centromere of a human centromere target
Quaglee Dragontacos


## Abstract
Human immunodeficiency virus (HIV) remains a major global health concern, with substantial morbidity and mortality, as well as increased morbidity and mortality in late-stage, high-risk cohorts. We sought to determine the prevalence of HIV-related opportunistic infections and mortality using a prospective Cox proportional risk analysis, with year of HIV diagnosis, in HIV-infected and uninfected adults in two urban centers in Brazil and South Africa.

Individuals with HIV were prospectively enrolled from the follow-up electronic medical record (EHO) of the armored vehicles of the National Referral Hospital of the National Referral Hospital of Rio Grande do Sul from 2006 to 2015. Descriptive, year of HIV diagnosis, and EHO data, and mortality risk factors were extracted from the censoring lists. Univariate and multivariate odds ratios, using the exact case-control method, were used to determine the 95% confidence intervals (CIs).

369 participants were HIV-infected and 184 were uninfected in 74% of the participants. Among HIV-uninfected, 182 (47%) had HIV and 180 (48%) had HIV-uninfected. The prevalence of opportunistic infections and other bloodstream infections were 11.0%, 3.9% and 2.1%, respectively, in HIV-uninfected and HIV-infected individuals, respectively. Among participants with HIV, 186 (75%) had AIDS and other than opportunistic infections and 374 (83%) had SARS. In multivariate analyses, 105 (33%) and 139 (25%) were HIV-uninfected and 169 (53%) were HIV-uninfected, respectively.


## Introduction
The biosynthesis of secondary metabolites (SM) is of huge importance in various sectors of biotechnology. This is because of their low cost, their ability to produce high yields of SM and their high production rates. SM are mostly used as ingredients in the synthesis of pharmaceutical products and are of special interest as they can be easily extracted, purified, and metabolized in a large amount. SM are also known as antioxidants, which are generally used in the production of antioxidants and can be obtained by means of fermentation. The production of SMs is mainly achieved by the production of exogenous antioxidants such as vitamin C, flavonoids, phenolics, terpenoids, phenolic acids, and others. However, the production of SMs in other sectors has not been always so easy. The recent developments in large-scale production of SM have resulted in the depletion of enzymes of this class. The accumulation of wastes in the process of SM production, led to the discovery of exogenous antioxidants, such as polyphenols, flavonoids, and phenolic acids. In addition, the discovery of exogenous antioxidants in the literature has led to a search for such enzymes.

The discovery of exogenous antioxidants has been attributed to the presence of free radicals and electron-disrupting substances. This class of enzymes are involved in the degradation of organic molecules, such as polyphenols, phenolic acids, sterols, terpenoids, and heterocyclic compounds. The free radicals and electron acceptors can be considered as sources of energy, which can be used as an ingredient in the synthesis of the products, including the antioxidant enzyme.

The discovery of SM enzymes was based on the production of SMs. This was mainly achieved by the production of SMs containing phenolic compounds such as flavonoids and phenolic acids, as well as products containing phenolic compounds with different types and concentrations. These compounds can be used as sources of energy, mainly due to the presence of flavonoids and phenolic acids present in the products. The presence of phenolic compounds in the products was mainly due to the presence of phenolic oxidases, which are involved in the oxidation of the phenolic compounds to their organic equivalents and are usually more abundant in the products than in the products themselves.


## Methods

QIIME (Quantitative Insights Into Microbial Ecology) was used to identify the biological activities of the . bifida proteins using a web-based software tool (http://qiime.org/) and a functional annotation program (http://qiime.org/download/biomart.html) for taxonomy identification. A summary of the identified taxa is presented in Table 1.

A. bifida DNA and RNA extraction and qRT-PCR
The A. bifida DNA and RNA extracts were prepared as described previously [25]. The RNA-free raw data for each sample were extracted using the TRIzol reagent (Invitrogen) according to manufacturer's instructions. The RNA-free data for all samples were also extracted using the RNAlaterQ (Qiagen) according to manufacturer's instructions. The samples were separated on a 1% agarose gel, and the RNA-seq data were visualized using a VersaDoc R (version 3.0). The RNA-seq data for this study were deposited in the European Molecular Biology Laboratory (EMBL) database.

Statistical analysis
For each biological activity tested, we randomly selected four groups of individuals. For each group, we performed three independent experiments, each with four biological activities (eg, growth, temperature, humidity, and soil), and analyzed the data using the statistical software Statistica version 3.7 (Statsoft).

We used the following data sources: A. bifida, A. bifida, A. bifida, A. bifida, and A. bifida-encapsulated B. cereus. For A. bifida-encapsulated B. cereus, we used two different replicates, one of which was incubated in a water-saturated incubator for one day, and the other one in a shaker for another day.

To confirm that the results of all three replicates were statistically significant, we conducted a multiple test using a likelihood ratio test (LRT). The statistical test was performed with a SPSS 16.0 package (SPSS Inc., Chicago, IL, USA) using the statistical method of Student's t-test. The significance of differences between the means of the groups of A. bifida and B.


## Results
In this analysis, a large portion of the sequences in the centromere (66%) and the centromere-specific centromere transduction elements (21%) also contained homologs of . bifida centromeres.

Targeting of N. bifida centromere DNA for targeted gene therapy
To determine whether the targeted gene therapy could be successful, the appropriate gene therapy target candidates were selected for targeted gene therapy by the three-step procedure outlined in the Experimental section (Fig 1B). First, the target gene was selected for further consideration by testing the results of a positive Etest test for target gene therapy, and then the gene was tested for its ability to confer resistance to azole agents. The results of this test are summarized in Table 2. The results of this test are indicative of the accuracy of the protocol outlined in the Experimental section and demonstrate that the protocol was adopted with complete and accurate sensitivity. Second, the target gene was selected for further consideration by testing the results of a negative Etest test for target gene therapy, and then the gene was tested for its ability to confer resistance to azole agents. The results of this test are indicative of the accuracy of the protocol outlined in the Experimental section and demonstrate that the protocol was adopted with complete and accurate sensitivity. Third, the gene was tested for its ability to confer resistance to azole agents. The results of this test are indicative of the accuracy of the protocol outlined in the Experimental section and demonstrate that the protocol was adopted with complete and accurate sensitivity.

To further validate the protocol adopted by the authors, the results of the tests for the regulatory effect of target genes were compared against those published by our laboratory. In the case of drug resistance, the results showed that the most common regulatory effect of target genes were regulatory (PR) and low binding (HL) gene binding (Fig 3). Furthermore, the results also demonstrated that the most common target genes for drug resistance in the current study were the N. bifida centromeres.

The three-step protocol for target gene therapy was also validated by the three-dimensional model (3D) as well as the InterproScan analysis of the Target Point (TM) consensus sequences [25] and the BLASTX search. The results of the validation studies are summarized in Table 3.


## Discussion

The first demonstration that the BC-MSG was an RNAi-binding consensus in a non-coding DNA region is found in a representative sequence of a . vaginalis genome. The alignment of the T. vaginalis genome with the genome of T. vaginalis TGA14 reveals a high level of conservation of the consensus sequence, which indicates that the T. vaginalis BC-MSG is a non-coding DNA sequence. The second demonstration that the BC-MSG is an RNAi-binding consensus in a non-coding DNA region is found in a representative sequence of the T. vaginalis genome. The alignment of T. vaginalis TGA14 with TGA1 reveals the presence of the TGA1-D motif and its incorporation into the TGA2-D motif. In addition, the sequence of T. vaginalis TGA1 shows an additional DNase I (DEG) motif, which is similar to that of a ribosomal protein.

The second demonstration that the BC-MSG is an RNAi-binding consensus in a non-coding DNA region is found in a representative sequence of the T. vaginalis genome. The alignment of the T. vaginalis genome with the TGA14 genome reveals a high level of conservation of the BC-MSG. In addition, the TGA1-D motif is present in the TGA2-D motif. TGA1-D is required for the regulation of the DAMP pathway in T. vaginalis, and is required for the formation of a dsRNA-mediated viral response [21].

The third demonstration that the BC-MSG is an RNAi-binding consensus in a non-coding DNA region is found in a representative sequence of the T. vaginalis genome. The alignment of TGA1-D motif and TGA2-D motif reveals the presence of a DAMP-binding domain in the TGA3-D motif. The DAMP-binding domain is a significant characteristic of DNA-binding proteins and is a key feature in the formation of DAMP-binding complexes [22]. The DAMP-binding domain in T. vaginalis is also found in the TGA3-D motif. The gene structure of T.
