# Fe-S Clusters and Their Identification by High-Throughput Sequencing in the SSC Functional Gene Network of SH2011-SH2011
Quaglee Dragontacos


## Abstract
We have previously reported that the genus Neocosmospora is largely non-veto, and that this is associated with a small number of phenotypic and virulence factors. This is the first study to examine the gene content of a full-length DNA-binding domain in a non-veto Neocosmospora.

We found that a single introns were found in all samples of N. areculoseum, N. humilis, N. schenckii and N. immangia, with some boundaries being difficult to distinguish. Many of the comparisons were made with the genomes of taxa that do not carry N. variabilis, such as Botrytis cinerea, Aspergillus fumigatus and Penicillium sp. We also identified a segment of N. humilis's DNA binding region that is common to all four strains, and that was the primary source for the segment found in the genomes of all four strains of N. humilis. A comparison of the introns in each sample revealed a similarity in their PROSITE motifs.

The results of our study indicate that N. variabilis is a single species with some intron gains, but not with complete loss of function. In addition, we provide evidence that additional genes, possibly attributable to a genetic process, might be involved. This putative neoplasia is a novel phenomenon that we describe here. We also describe the first known species of a typical seed-formed neocosmotic fungus. Our results indicate that seed-form is an attractive substrate for future investigation on the structure and function of fungi.


## Introduction
There has been a steady increase in the number of infections caused by fungi over the past few decades due to an increasing number of immunocompromised individuals. Fungal diseases affect more than 400 million people worldwide, with at least 7.6 million deaths annually [1]. The most common fungal infections are caused by Candida, Cryptococcus, and Pneumocystis species, which are responsible for ~40% of all pneumocystis-related deaths, and C. albicans accounts for ~15% of all invasive candidiasis [2]. Although these two species account for most fungal infections, they can be found in the same population, and are often related by blood, skin, or vaginal mucosal surfaces [3].

C. albicans is capable of two major phenotypes, morphogenetic and non-mature morphogenetic forms. Morphogenesis is a reversible process, in which cells of the opposite mating type are faced with the challenge of developing new attachment structures and expressing virulence factors. While cell-cell fusion is considered a prerequisite for cell-cell fusion, the key requirement for cell-cell fusion in C. albicans is the presence of a specific transcription factor, AP-1, in the mating type locus. AP-1 is a 14-kDa protein that is highly expressed in the majority of the yeast cells in vitro, and in the human lungs, where it is induced. AP-1 is a key regulator of morphogenesis and virulence factors [4].


## Methods
Using a high-throughput sequence library of SSC genes from SSC FASTA v.0 [14], a large sequence pool of SSC genes was obtained from the H99 (SOFT_LF), and the sequences were clustered in the sequence pool. All sequences were aligned and trimmed to include only the SSC genes that showed significant similarities with the SSC genes with the lowest phylogenetic conflict. The alignment of the sequences was done by ClustalW and Maximum Composite Likelihood (ML) was performed using RAxML v. 8.2.8 [15].

C. gattii identification
The genome sequences of C. gattii were deposited in the NCBI GenBank database under accession number MG018413. The sequence of the genome was deposited in the NCBI database under accession number MG018414. For identification of the sequences of Cg0579 and Cg0580, a BLAST search of the genome sequence was conducted using the BLAST program from GenBank with the same parameters as above.

Isolation of mycelium from SSCs
The SSCs were obtained from the mycelium of S. leo (S. leo, Iliacea syriaca, L. sympodialis, L. africana) by placing the mycelium on a plug of PDA, allowing the mycelium to grow at room temperature for 3 days, and then harvesting the mycelium with a fine grout. The SSCs were cultured on PDA and maintained at room temperature for 4 days. The mycelium was collected by filtration through a filter paper and then dried in a funnel for one minute. Then, the mycelium was vacuum-filtered through a filter paper and then freeze-dried in a gas chromatograph. The samples were ground to a fine powder in liquid nitrogen and then frozen at -20°C until DNA extraction. The DNA was extracted using the DNeasy Plant Mini Kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions.

Phylogenetic analysis
The genomic DNA from the SSCs was isolated by using the DNeasy Plant Mini Kit. The DNA was extracted using the DNeasy Plant Mini Kit according to the manufacturer’s instructions.


## Results
.


## Discussion
-SYN-C4L3C3L2-SYN-LPL1-SYN-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SYN2-SY
