# Fucosyltransferases: identification and characterization of functional ones in mammals
Quaglee Dragontacos


## Abstract
Sulfamethalloyl-CoA: synergistic antimicrobial, anti-inflammatory, antioxidant, and immunomodulator of S. aureus in a plant-based therapy.

Sulfamethalloyl-CoA: synergistic antimicrobial, anti-inflammatory, antioxidant and immunomodulator of S. aureus in a plant-based therapy.

The anti-infective activity of methylene blue used for the treatment of infectious skin infections by S. aureus could be synergistic against bacterial and fungal infections. This study shows that the in vitro activity of methylene blue combined with amphotericin B against S. aureus could be used as a potential antibiotic to treat infection.


## Introduction

The bacterial and fungal cellulases are the most widely used cellulases in the industrial secto. They form an integral part of the biodegradation process of various biomolecules in organic matter, including food, feed, and pharmaceuticals. Among the cellulases, cellulases with a high enzymatic activity can be classified into two main groups, and the group of cellulases with the highest enzymatic activity is the non-ribosomal peptide chain (NRPS) [12]. Some non-ribosomal peptides (NRPS) were initially isolated from bacteria and fungi, and they are categorized in two major groups, with the first category of compounds being poly- and pyrrolidines, which are biodegradable and biodegradable [13,14]. Among them, poly- and pyrrolidines were first reported in bacteria and fungi [15]. Novel cellulolytic enzymes, including ribosomal peptide synthetase (RPS), which was first isolated from N. polyporus, and pyrrolidine-mediated cellulase (Pmr), which was first isolated from N. polyporus, were classified in the second category [16]. Among them, Pmr was first reported as a cellulase which could degrade the lignin in N. polyporus [17]. Among the cellulolytic enzymes, Pmr is one of the most important enzymes, which is responsible for the degradation of a wide range of polymeric materials, such as textiles, textiles, plastics, paper, leather, and synthetic chemicals [18]. Pmr has a wide spectrum of activity, which can be classified into four main groups, namely, antimicrobial, cytotoxic, and biodegradable [19]. The enzymatic reaction of Pmr in vitro was shown to be effective in degrading a broad range of substrates including a wide range of pharmaceuticals, plastics, paper, and textiles, including plastics, textiles, plastics used in industry, textiles used in biodegradation, and textiles used in bioremediation [20,21].

In this study, the enzymatic reaction of Pmr in vitro was performed using the standard microfiche technique, denoted as “the Pmr YY method”.


## Methods
oculata strain, . oculata f. sp. tritici was sequenced using the Illumina technology (version 3.1.8, http://www.illumina.com/). All sequences of H. oculata f. sp. tritici strain H1667 were assembled and edited using the SeqMan pipeline [46]. The sequence data was deposited in NCBI (www.ncbi.nlm.nih.gov/tools/SeqMan) and the alignments and phylogenetic trees were generated using the program Mega 5.0.

The F. graminearum strain, FGM-065 was sequenced using the ABI PRJNA395866. The sequencing data were deposited in NCBI (www.ncbi.nlm.nih.gov/genome). The taxonomy of each of the four strains was verified using the online database of fungal sequences at NCBI (http://www.ncbi.nlm.nih.gov/genome/Fusarium_group/).

Phylogenetic tree of F. graminearum
F. graminearum was sequenced on the Illumina HiSeq 2000 platform at NCBI. The sequence data were deposited in NCBI (www.ncbi.nlm.nih.gov/genome/Phylogeny.html) and the alignments and phylogenetic trees were generated using the program Mega 5.0.

F. graminearum conidia
F. graminearum conidia were prepared from an infected leaf of F. graminearum in order to obtain conidia. The conidia were harvested from the leaf of a 10-week old F. graminearum cultivar F. graminearum in the USA, USA, in order to obtain conidia from the leaf of another F. graminearum cultivar, F. graminearum, in the USA, USA. The harvested conidia were kept at 25°C in the dark for 24 h before the conidia were obtained by inoculation of the mycelium.

Plant infection
In order to identify fungal infections in the leaf of F. graminearum, a total of five leaves were inoculated with 1 × 10^4 conidia from the same plant.


## Results
graminearum are presented in Table . The number of genes identified in the F. graminearum genome is shown in Table 5, with the following gene names: FGRH1, FGRH2, FGRH3, FGRH4, FGRH6, FGRH9, FGRH11, FGRH12, FGRH14, FGRH2, FGRH3, FGRH4, and FGRH5 (see Additional file 1 for the complete list of FGR genes).

The five most highly expressed F. graminearum genes, FGRH1, FGRH2, FGRH3, FGRH4, FGRH5, and FGRH6, are named as FGRH1, FGRH2, FGRH3, FGRH4, FGRH5, and FGRH6. As expected, these genes were found to be highly expressed in the wild type, with no statistical significance, and no significant differences were observed between wild type and mutant strains (p-value < 0.05). In addition, the expression levels of genes FGRH1 and FGRH2 were not significantly different between the wild type and mutant strains (p-value > 0.05). This finding implies that FGRH1, FGRH2, FGRH3, and FGRH4 were not directly regulated by F. graminearum.

In F. graminearum, the gene FGRH1 is the most highly expressed gene in the wild type, but not in the mutant strain. In contrast, the gene FGRH2, which was also expressed in the wild type but not in the mutant strain, has a moderate expression level in the mutant strain (Table 4). This gene has been shown to be important for pathogenicity [25], [26], [27], [28], [29], [30], [31], [32], [33], [34], [35], [36], [37], [38], [39], and has been found to be highly expressed in the wild type but not in the mutant strain [37], [39], [40], [40], [41]. In addition, FGRH4 has a low expression level in the mutant strain, with p-values of 0.


## Discussion
cb.nlm.nih.gov/projects/cps/cpsdb/) includes a set of 26 fungal homologs, and a phylogenetic tree of their relationships to fungal homologs (Figure 2). The NCBI taxonomy database (http://www.ncbi.nlm.nih.gov/projects/cps/cpsdb/), based on BLAST searches of the NCBI-NR database, retrieved the same fungal homologs in both the BLAST and the phylogenetic tree of the NCBI-NR database (Figure 2, Additional file 2). Two fungal homologs, F. culmorum and F. graminearum, were reported to be involved in the initial stages of cellulose degradation, with Fg_6-1 and Fg_6-2 being the most abundantly identified. The NCBI taxonomy database (http://www.ncbi.nlm.nih.gov/projects/cps/cpsdb/), on the other hand, retrieved Fg_2-1 as the most highly expressed, as well as Fg_2-2 and Fg_2-3, in the NCBI database (Figure 2, Additional file 2).

In the phylogenetic tree (Figure 2), the NCBI taxonomy database (http://www.ncbi.nlm.nih.gov/projects/cps/cpsdb/) retrieved Fg_1 as the most highly expressed homolog in F. graminearum, followed by Fg_2-1 (Fg_6-1) in F. culmorum and Fg_6-2 (Fg_6-2) in F. graminearum. This is in contrast to the phylogenetic tree (Figure 2), which recovered Fg_6-1 as the most highly expressed homolog in F. culmorum. Fg_6-1 was reported to be expressed in the first stages of cellulose degradation in Fusarium graminearum [51], and Fg_6-2 was found to be expressed in the second stages of cellulose degradation in Fusarium oxysporum [52]. The NCBI taxonomy database (http://www.ncbi.nlm.nih.
