# Fungal Gene Order Conservation and Evolution in the ITS1-Like Histone Acetyl Co-Finger Transcribed by the Exosome RNAGene Stranscript Gene
Quaglee Dragontacos


## Abstract
Ochroconis spp. are dimorphic fungal pathogenic fungi that are responsible for a wide spectrum of human and animal diseases. These fungi are known as drug producers andaders, with some species exhibiting resistance to multiple classes of antifungal drugs. This study aimed to perform a molecular genetic analysis of the yeast strains from the North American (2015) and South American (2016) basidiomycetes to determine their genomes and to determine the phylogenetic relationships amongst these strains. The genomes of three strains (Ochroconis nigericis CBS 2370, O. australis sp. nov.) were analyzed by Illumina technology. The two genomes of each strain were also analyzed by molecular sequence analysis (MOG) using the CLSI method. Multigene phylogenetic analysis showed the North American as the sister group to the European strains. A sister-group analysis was carried out among the three strains, which divided the North American into the South American into the European strains. These discordant results indicated the North American strains are distinct from the North American strains, and as such, they belong to O. nigericis rather than O. australis. This study also showed that the fungus has a strong tendency to form new strains and the expression of the gene encoding the galactomannan (GM) and the hemeprotein GM-7 is significantly higher in the genomes of the North American strains than in the South American strains. The two genomes of O. australis and O. nigericis, however, showed differences in gene content.


## Introduction
1987; Luttrell 1996.

Despite the potential importance of the Exosome in host defense against fungi, few studies have investigated the molecular mechanisms of the function of this organelle in the defense against fungi. In fungi, the cytoplasmic organelles constitute a major cell population and maintain many metabolic processes that are not normally controlled by the cytoplasmic organelles. Thus, in fungi, it is usually called the microglial cell network (MCC) (Gadd et al. 2000). This organelle is involved in the transport of nutrients and also plays a central role in the regulation of other cellular processes, such as the phagocytosis, transport of different kinds of biomolecules, signal transduction, and many others processes (Gadd et al. 2000). The genes that are involved in the production and secretion of these cell-wall proteins include those encoding the Protein-Active dismutase (PAD) and the secreted protein secretion protein (SDP) (Ditterel et al. 2001; Tuller et al. 2002). The PAD is a membrane protein that consists of an a-1,4-linked, four-stranded dipeptide loop and is mainly expressed in the Golgi and the nucleus (Gadd and Suha 2002; Suha and Gadd, 2003). The a-1,4-PAD is a short protein consisting of two, partially linked dipeptide repeats that form a dolipore (Gadd and Suha, 2003). The PAD is a highly conserved protein that is secreted in the cytoplasm. The PAD is present in the nucleus and other organelles, including the mitochondria (Gadd and Suha, 2003; Suha and Gadd, 2003; Lam et al. 2005; Suha and Gadd, 2005; Suha et al. 2005; Suha et al. 2007; Suha and Gadd, 2007; Suha et al. 2007; Suha et al. 2011; Suha and Gadd, 2011; Suha and Gadd, 2011). In fungi, the PAD and the SDP are differentially distributed, and they act as signaling proteins and co-factors (Suha and Gadd, 2007; Suha and Gadd, 2007).


## Methods
), and a search in the Fungal Genome Database (version .0) for sequences that have been deposited in the GenBank database.

For each of the six-fungus species studied, samples of the four different hosts were isolated and included in the analysis. The four host species were selected because they had been successfully differentiated in a previous study (Bengdahl et al., 2015) with an important difference: host species were classified as “polar” if the sample was located in a polar (north or south) or equatorial (east or west). Host species were classified as “medium” if the sample was located in a polar (north or south) or equatorial (east or west). Host species were classified as “fungal” if the sample was located in a fuchsia (north or south) or a green (east) form of the genus name of the studied host. Host species were classified as “vital” if the sample was located in the "vital" range.

Bacterial species were identified using BLAST analysis and then categorized according to their secretion secretion signal in the presence of different monoterpene (mC) compounds. The genera and species were used in the phylogenetic analysis.

The phylogenetic tree was constructed by using the online software PhyML v. 7.0 (Guindon and Gascuel, 2008). The phylogenetic tree was constructed by the neighbor-joining method using the MEGA v. 6.0 software (Kumar and Gascuel, 2016). The gene tree was inferred by using the neighbor-joining method. The alignment was done using the online software MUSCLE v. 4.8 (Edgar, 2004). The neighbor-joining method was used to evaluate the validity of the tree topology, and the branch-site reliability was evaluated using the bootstrap method with 1000 replications. The phylogenetic tree was analyzed using the neighbor-joining method, and the gene tree was analyzed with the bootstrap method with 1000 replications. The general time reversible model was used for the analysis.


## Results
velutipes
The results of our analysis are illustrated in Fig . The combined analyses of the three-dimensional topology of the backbone region revealed that the ancestral state of F. velutipes was different from that of S. cerevisiae and S. pombe. The reconstructed phylogenetic tree revealed that the two strains share a common ancestor in the last common ancestor (LD) of S. cerevisiae and S. pombe (Fig 4B). The recovered ML tree also shows that the three-dimensional topology of the backbone region revealed the same basal clades as the phylogenetic tree. However, the analysis of the combined analysis of the backbone region revealed that the ancestral state of F. velutipes was different from that of S. pombe and S. cerevisiae. The reconstructed phylogenetic tree revealed that the three-dimensional topology of the backbone region revealed the same basal clades as the phylogenetic tree. The divergence of the two strains was also different from the phylogenetic tree. The reconstructed phylogenetic tree revealed that the three-dimensional backbone region revealed the same basal clades as the phylogenetic tree. The divergence of the strains was also different from the phylogenetic tree. The three strains were separated from S. pombe and S. cerevisiae by a long branch. The reconstructed phylogenetic tree revealed that the ancestral state of S. pombe was different from that of S. cerevisiae and S. pombe. The reconstructed phylogenetic tree revealed that the ancestral state of S. pombe was different from that of S. cerevisiae and S. pombe. The divergence of the strains was also different from the phylogenetic tree. The three strains were separated from S. pombe and S. cerevisiae by a long branch. The reconstructed phylogenetic tree revealed that the ancestral state of S. pombe was different from that of S. cerevisiae and S. pombe. The divergence of the strains was also different from the phylogenetic tree. The two strains were separated from S. pombe and S. cerevisiae by a long branch. The reconstructed phylogenetic tree revealed that the ancestral state of S. pombe was different from that of S. cerevisiae and S. cerevisiae. The divergence of the strains was also different from the phylogenetic tree. The three strains were separated from S. pombe and S.


## Discussion
Gene sets for each of the five Exosome RNAGene Strans were designed in the following two way. First, we used the gene set for each of the five Exosome RNAGene Strans as shown in Fig 2. First, we used the set of exosome sequences for each of the five Exosome RNAGene Strans, which were designated as Exosome5 and Exosome7. Second, we added the gene sets for the exosome sequences for each of the five Exosome RNAGene Strands. Finally, we added exosome sequences for each of the five Exosome RNAGene Strans in the same way as for the exosome sequences. The result is shown in Fig 2A.

To determine whether each of the five Exosome RNAGene Strings had the same gene order conservation, we used the gene set for each of the five Exosome RNAGene Strings to determine if each of the five Exosome RNAGene Strings had the same gene order conservation (see Additional file 2). Then, we used the exosome sequences for each of the five Exosome RNAGene Strings to determine whether each of the five Exosome RNAGene Strings had the same gene order conservation (see Additional file 2).

To determine whether each of the five Exosome RNAGene Strings had the same gene order conservation, we added exosome sequences for each of the five Exosome RNAGene Strings to the Exosome5 gene set. Next, we used the gene set for each of the five Exosome RNAGene Strings to determine if each of the five Exosome RNAGene Strings had the same gene order conservation (see Additional file 2).

The results shown in Fig 2A show the gene order conservation of the five Exosome RNAGene Strings in the five Exosome RNAGene Strings in the EST data from the S. cerevisiae gene set for the five Exosome RNAGene Strings. The exosome sequences for each of the five Exosome RNAGene Strings were then added to the Exosome5 gene set for the five Exosome RNAGene Strings.
