# Fungal Biofilms: Where Do They Come
Quaglee Dragontacos


## Abstract
The use of antiretroviral therapy (HAART) has been associated with favorable outcomes, with some individuals experiencing subacute asymptomatic or paradoxical deterioration after initiating HAART therapy. This is a case report of a 32-year-old male patient with a Noncompliant HAART presenting as subacute asymptomatic Cryptococcus neoformans infection, after initiation of HAART. The clinical manifestations were characterized and the outcomes evaluated. The most common clinical manifestations were fever and headache. The results of biochemical testing and viral detection of HIV were positive. The patient was started on HAART as a cure option but a paradoxical deterioration developed.


## Introduction
.


## Methods
brasiliensis and . brasiliensis Bifida
For the P. brasiliensis Bifida biofilm assay, the biofilm produced was suspended in RPMI-1640 (Invitrogen) and then adjusted to 1 × 10^6 CFU/mL. For the P. brasiliensis Bifida biofilm assay, the biofilm produced was suspended in 100 mM citrate-phosphate buffer (pH 5.5) with 2 mM NaCl. After 24 h of incubation, the biofilm was washed with RPMI-1640 (Invitrogen) for 24 h. Subsequently, the biofilm was washed with PBS (pH 6.0) and the C. albicans cells were fixed with methanol (1:1) for 30 min at room temperature. After the addition of 2% SDS, the stained biofilm was washed with PBS and the C. albicans cells were stained with a peroxidase-conjugated secondary antibody (1:50, Sigma) in PBS. The biofilm was then washed with PBS and the stained C. albicans cells were analyzed with an LSRFortessa^TM flow cytometer (BD Biosciences) and FlowJo v.10.1.1 software.

P. brasiliensis Bifida Biofilm Assays
The P. brasiliensis Bifida biofilm assay was performed as previously described [20], using C. albicans SC5314 cells as a reference. Briefly, P. brasiliensis Bifida cells were grown in YPD broth at 30°C. Then, the cells were diluted to OD_600 = 0.1 in fresh YPD broth and cultured at 30°C for 24 h. Then, the OD_600 was measured using an EnSpire Multiskan microplate reader (Molecular Devices, Sunnyvale, CA, USA). The number of CFU per mL was calculated by counting the number of cells in the wells of the microplate.

P. brasiliensis Bifida Biofilm Assays
For the biofilm formation of P. brasiliensis Bifida, P. brasiliensis Bifida cells were cultured in YPD broth at 30°C for 24 h. Then, the cultures were diluted to OD_600 = 0.


## Results
. Hyde, Mycologia 32: 557. 2009.

Materials examined: Australia, New Zealand, on living leaves of Acacia sp., 5 Mar. 2012, P.W. Crous, JW 23014 (holotype CBS H-22153, culture ex-holotype CBS 101854 = CPC 23153).

Notes: The three strains in this clade cluster in the well-supported clade (BS = 100 %, PP = 1.0), and are therefore tentatively named as Septoria pulcherrima (for the smaller conidia, CBS 101854 = CPC 23153).

Clade 4: Pseudocercospora

Pseudocercospora Videira & Crous, Stud. Mycol. 83: 126. 2015.

Description: See Videira et al. (2015).

Materials examined: Brazil, Rio de Janeiro, Minas Gerais, on leaves of Camellia sinensis, Oct. 2013, H. Voglmayr (holotype CBS H-22097, culture ex-holotype CBS 123120 = PD 83/4584).

Notes: Pseudocercospora is introduced here as a new combination for the type species, Pseudocercospora crinicola (Crous et al. 2015a). The phylogenetic analyses show that the two strains in this clade cluster in a well-supported clade (BS = 100 %, PP = 1.0), and are therefore tentatively named Pseudocercospora crinicola (for the smaller conidia, CBS 123120 = PD 83/4584).

Clade 5: Pseudocercospora murphyi Videira & Crous, Stud. Mycol. 83: 126. 2015.

Description: See Videira et al. (2015b).

Materials examined: Brazil, Rio de Janeiro, Minas Gerais, on living leaves of Carex aculeatum, Oct. 2013, H. Voglmayr, CBS H-22097, culture CBS 123120 = PD 83/4584.

Notes: The phylogenetic analyses show that the strain CBS 123120 from the same host as the type species is here named Pseudocercospora murphyi.


## Discussion
The ITS1, ITS2, RPB2, RPB3, LSU, and SSU genes were clustered into ITS1 and ITS2 clusters (Fig 1) and the ITS2 gene was named as ITS1_LSU_3A4, ITS1_LSU_3B4, ITS1_LSU_3C5, and ITS1_SSU_2A1 (Fig 2. The two gene clusters were separated by a single gene (Fig 1). The two gene clusters were separated by seven genes (Fig 2). The ITS1 gene was divided into ITS2 and ITS3 and the ITS1_LSU_3A4 gene was named as ITS1_LSU_3A1 and ITS1_SSU_2A2. The phylogenetic tree of the ITS1 gene clusters from the present study (Fig 2) indicated that the two gene clusters were related to each other and were clustered together. The three genes were separated by six genes (Fig 2). The SSU gene (Fig 1) was named as SSU_4377 and SSU_4435 and was named as SSU_3886 and was named as SSU_3906. The LSU gene (Fig 1) was named as LSU_6894 and was named as LSU_6894. The amino acid sequence of the ITS1 gene (Fig 1) was identified by ITS3 and LSU. The ITS2 gene (Fig 2) was named as ITS1_LSU_3A4, ITS1_LSU_3C5, and ITS1_SSU_2A1. The phylogenetic analysis indicated that the three gene clusters were related to each other. The two gene clusters were separated by seven genes (Fig 1). The SSU gene (Fig 1) was named as SSU_7889 and named as SSU_7635 and was named as SSU_8451 and was named as SSU_8552. The LSU gene (Fig 2) was named as LSU_6894 and was named as LSU_6894. The three gene clusters were separated by seven genes (Fig 1).
