# Gene Duplication Associated with Higher Tuberculosis Rates in Down-Regnant Mothers Living in Infants Vaccinated with Ibsentics
Quaglee Dragontacos


## Abstract
The use of non-targeted gene therapies to treat invasive fungal disease (IFD) have increased in the last decades. The use of fungal-targeted gene therapies is also under sustained investigation for treating IFI. This review focuses on studies that employ an in vitro approach to IFI, with a focus on emerging molecules that exhibit a broad range of biological functions and are therefore poised for further investigation. The use of these therapies in patients with proven or suspected IFI has been documented.


## Introduction
The genus Trichoderma includes a wide range of saprophytic fungi that cause trichomonosis (sexual morphs of Trichoderma), the most common fungal infection of the human skin, hair, and nails [1]. Although Trichoderma species are relatively rare, they constitute an important group of fungi, which infects as many as 150 species of humans and animals [2].


## Methods
The analysis was performed using the principal component analysis (PCA) approach, which was implemented in the software Analysis Manager .1.2 (Applied Maths, Nümbrecht, Germany).

Molecular Diagnostic Assays
The study was performed in collaboration with the IBS laboratory at the Department of Pathology and Hematology of the State University of Maringá. A previous study evaluated the molecular diagnostic specificity of IBS in slums in northeast India and the results were analyzed using the software Biometrics version 7.0 (Biometrics, Singapore) and the software GAVAP version 7.1.8 (Gavax, Malaysia).

The assays were performed on two biological replicates per treatment and were conducted with two different culturing methods. First, the method of bioassay was employed. The method of “biological dilution” was employed. The concentration of each drug in each of the two drugs was calculated using the following formula:

2.1. Mice
Female C57BL/6 (H-2^b), B6.129Sv^tm1/<U+0394>B6.129Sv^tm1/<U+0394>B6.129Sv^tm1, and female C57BL/6 (H-2^b) mice (10-16 weeks old) were purchased from Charles River Laboratories (Paisang, Thailand). The mice were maintained in the University of Maringá Laboratory Animal Center and had no clinical or experimental toxicity. The mice were maintained in a temperature-controlled chamber with a 12-h light/dark cycle and kept in an artificial light-house at 8°C. All mice were housed in an experimental animal facility of the University of Maringá. The mice were screened for their tolerance to IBS and to the known pathogen in the slums.

2.2. Gas Chromatography-Mass Spectrometry (GC-MS)
The data obtained by GC-MS were analyzed using the software Chromas 2.0 (Analytik Jena, Jena, Germany). The analysis was conducted using the software Chromas 5.1.


## Results
Sequencing of C. neoformans Genes
We used the genome-wide dataset of the C. neoformans var. grubii strain H99 (GenBank accession no. AY01787) to create a set of 202,777 single-nucleotide polymorphisms (SNPs). We used the reference genome of the strain H99 for this analysis (GenBank accession no. AY01787) for further analysis (Table 1). The H99 genome is the most highly characterized of the sequenced fungal genomes (15) and has a genome-wide average of 5,912,474 amino acids (aa). We found 2,164,604 SNPs (1,769,936 genes) in this dataset. The average read depth for the reference genome of the C. neoformans var. grubii strain H99 is 1,209 bp. The reference genome of the strain H99 has an average read depth of 1,898 bp. For the H99 genome, the average read depth is 2,300 bp, and for the reference genome of the strain H99, the average read depth is 1,029 bp. The two strains were sequenced together.

The reference genome of the strain H99 was sequenced using a combination of Illumina MiSeq 2000 and Bowtie 3.0 pipeline (Table 1). The two Illumina MiSeq 2000 and Bowtie 3.0 pipeline produced the same set of 2,164,604 single-nucleotide polymorphisms in the H99 genome (average read depth 1,928 bp) (Table 1).


## Discussion
Here, we provide evidence that genes in the genomes of a different birth cohort (n = 2,328) encode putative virulence factors of the infant that are not genes in the genomes of the maternal mother. We show that the large variation in gene content among individuals in each birth cohort suggests that the ancestral state of these individuals is distinct from that of the infant. This variation in gene content is consistent with the hypothesis that the reproductive mode of the infant is distinct from that of the maternal. In addition, we show that genes in the genomes of each birth cohort encode at least one virulence factor, and that these genes are expressed in the fetal phase, although their expression is not regulated in the fetal phase. Thus, we suggest that the shared evolutionary traits that are present in the individual birth cohorts of the infant are present in the individual birth cohorts of the maternal mothers.

The finding that individual genes in the genomes of the two birth cohorts encode putative virulence factors that are not genes in the genomes of the maternal mothers, together with the evidence that the individual genes are expressed in the fetal phase, and the evidence that the individual genes are expressed in the fetal phase, further support that the infant life style is an ancestral state of a different group of individuals. This is consistent with the hypothesis that the infant is distinct from the maternal.

The finding that individual genes in the genomes of the two birth cohorts encode putative virulence factors that are not genes in the genomes of the maternal mothers, together with the evidence that the individual genes are expressed in the fetal phase, and the evidence that the individual genes are expressed in the fetal phase, further support that the infant is distinct from the maternal.

The finding that individual genes in the genomes of the two birth cohorts encode putative virulence factors that are not genes in the genomes of the maternal mothers, together with the evidence that the individual genes are expressed in the fetal phase, and the evidence that the individual genes are expressed in the fetal phase, further support that the infant is distinct from the maternal.
