# Generation and Characterization of Indoor Microbiome Using Shotgun Metric Mapping
Quaglee Dragontacos


## Abstract
Liposomes are the first nano drug delivery systems that have been successfully translated into real-time clinical applications. Delivery of clinically relevant liposomes in the body allows the drug delivery to be carried out by a third-generation drug delivery system (NGS) and the use of nanoparticles are becoming a standard therapy. The purpose of this review is to compile all relevant studies that have been published in the last 6 months, and to provide an overview of the latest developments, in the field of liposomal drug delivery. In addition, the role of nanoparticles in the delivery of different classes of liposomes, and different delivery techniques are presented. Recently, liposomes have also been an interesting source of new applications, such as as anti-infective and immunosuppressive agents. In this respect, novel applications are in store for the future technology transfer and drug delivery systems.


## Introduction
Biofilm formation is an important process in all domains of life, and is defined by the ability of an organism to adhere to, invade, and form biofilms. The formation of biofilms is the result of a combination of mechanisms, including the formation of extracellular matrix, the ability to adhere to a substrate, and the ability to generate extracellular traps (traps) through the formation of adhesive cells (vulvovigaments). ^1^,^2 The production of extracellular traps is important in a number of infectious diseases, including cancer, diabetes, and heart disease. ^3^,^4 These diseases are associated with poor prognosis, as several studies have shown that the frequency of such events is almost exclusively in cancer patients. ^5^,^6 In contrast, the prevalence of autoimmune conditions and inflammatory diseases, including inflammatory bowel disease (IBD), has been shown to be increasing in the recent years. ^7^,^8 A number of studies have shown that the occurrence of IBD is increasing in patients admitted to intensive care units. ^9^,^10 Most of the IBDs are caused by Streptococcus mutans (SP), a pathogen first identified in the 1950s. ^11 These infections are acquired by inhalation of contaminated aerosols.


## Methods
These reads were mapped to the human genome sequence (HGS) of the ribosomal RNA gene of our sample (S1 Fig), using the STAR software (http://ww.expasy.org/star/). The mapped reads were classified into orthologs using OrthoFinder (version 4.1.0) [69], and the orthologs were defined as the following groups: subgroups H1-H2, and H3, which are orthologs of H2. The use of the HGS data as the input to the STAR software was done using the default parameters. The HGS data are available at the Gene Expression Omnibus database (accession number GSE1723). The raw data from this study have been deposited in the Gene Expression Omnibus database (accession number GSE17923).

Detection of putative transcriptome changes
RNA was extracted using RNAiso plus (Invitrogen) and treated with DNase (Sigma-Aldrich). The RNA was precipitated with 4 M urea, and the precipitate was resuspended in 500 Âµl of TE buffer (TE buffer:20 mM Tris, 1 mM EDTA, pH 8.0) to eliminate the RNA from the sample. The RNA concentration was determined using a NanoDrop (Thermo Scientific, Wilmington, DE). The RNA quality was analyzed using a Bioanalyzer 2100 (Agilent Technologies, Santa Clara, CA) and a Bioanalyzer 2100-Agilent system (Agilent Technologies, Santa Clara, CA). The RNA content was normalized to that of the reference genome using a RNA-seq library preparation kit (Quorum Technologies, San Jose, CA). The number of reads mapped to the HGS of the HGS of the reference genome were calculated. The transcriptome data were obtained from the genome sequencing of the HGS. The RNA-seq data were aligned to the HGS data using the Bowtie2 software (http://www.bdse.org/bowtie2/) and the best-fit model was determined by mapping the reads to the HGS data. The transcriptome data from this study have been deposited in the Gene Expression Omnibus database (accession number GSE1723).

Sequence data analysis
The sequences were obtained from the Ensembl H.


## Results
For each gene, we compared the mRNA expression of a particular transcript in each microarray set (Figure 1. From this, we constructed homozygous deletion mutants and determined the relative transcript abundance in each of the mutant strains. We then tested the expression of the genes in the mutant strains by constructing homozygous deletion strains. As shown in Figure 2, homozygous deletions were consistently expressed in all the tested strains. For each gene, we found that the relative transcript abundance in the mutant strains was generally similar to the corresponding transcript in the wild-type strain. In contrast, homozygous deletion mutants displayed a dramatic reduction in the transcript abundance in comparison to the corresponding transcript in the wild-type strain. In particular, homozygous deletion strains showed a reduction in the transcript abundance in comparison to the corresponding transcript in the wild-type strain.

Identification of ChIP-seq Genes
We used the SignalP 2.0 algorithm to predict the sequence of the ChIP-seq locus in the mutant strains. The results from SignalP 2.0 can be used to identify potential transcription factor genes. The results of the ChIP-seq analysis indicated that the predicted loci were highly expressed in all the tested strains. We have previously shown that a single copy of the ChIP-seq gene from strain F2 is sufficient to identify the location of the ChIP locus in the mutant strains. This observation was further supported by the fact that the ChIP-seq locus in the F2 strain was found to contain a predicted short ORF containing a C-terminal sequence with an internal RING-like DNA binding domain (Figure 2). To examine if the predicted ChIP-seq locus would be a target of ChIP-seq, we performed a BLAST search for the predicted chitin synthase gene from F2. This gene was found to be in sequence with high similarity to the ChIP locus in the wild-type strain. In contrast, the ChIP-seq locus in the F2 strain was found to contain an ORF containing a C-terminal sequence with an internal RING-like DNA binding domain.


## Discussion
The current genome assembly was then constructed using the full genome datase. The annotation of the 5' and 3' ends of the genes was carried out using a mixture of consensus sequences from the gene families. The assembly of the assembled genome and transcriptome data was based on the previously published genome annotation of the C. neoformans var. neoformans strain DSM-2a [14], and the whole genome sequence of C. neoformans var. grubii [15] has been deposited at NCBI. The assembly of the genomic data was based on the gene families DSD, DSD-1, DSD-2, DSD-3, DSD-4, DSD-5, DSD-6, DSD-7, DSD-8, and DSD-9, respectively. The genome of C. neoformans var. grubii was constructed using the full genome sequence of the strain [15]. The assembly of the transcriptome data was based on the gene families UM, UM-1, UM-2, UM-3, UM-4, UM-5, UM-6, UM-7, UM-8, and UM-9, respectively. The genome of C. neoformans var. grubii was constructed using the full genome sequence of the strain [15]. The assembly of the assembled gene set was based on the gene families DMZ, DBZ, and EZ, and the gene set was based on the gene families GZ, ZM, and VZ, respectively. The genome of C. neoformans var. grubii was constructed using the full genome sequence of the strain [15]. The genome of C. neoformans var. grubii was built using the full genome sequence of the strain [15]. The assembly of the transcriptome data was based on the gene families SFA, TAP, TAP-1, TAP-2, TAP-3, TAP-4, TAP-5, TAP-6, TAP-7, TAP-8, TAP-9, TAP-10, TAP-11, and TAP-12. The genome of C. neoformans var. grubii was built using the full genome sequence of the strain [15]. The genome of C.
