# Gene Flow in Fungal Pathogens: Putative Mitochondrial Duplications Rules for Non-Coupled Generation of Specific Genes in the Wild
Quaglee Dragontacos


## Abstract
Cryptococcus neoformans is an important fungal pathogen of immunocompromised people. The fungus can switch between two types of differentiation: heterothallic and autosurulating. In contrast to autos, the two types of differentiation are regulated differently. Differentiation regulation involves cellular transport across the blood-brain barrier, where it interferes with transport of nutrients, and where it results in tissue invasion and infection. The heterothallic switch is regulated by many molecular pathways, including copper and protein transport. Copper is a very efficient copper exporter and is also the predominant copper transporter in the lungs of C. neoformans. In contrast, non-polarized switch is regulated by the transcription factor calcineurin, an important regulator of iron homeostasis. The two different cellular types of switch are related to a specific fungal pathogenicity. In fungi, calcineurin is critical for establishment of virulence, but it is also involved in regulating the host immune response. Transcriptomics and transcriptomic analysis identify many genes that are differentially regulated in the switch from heterothallic to autosurulating states. The analysis of the transcriptomes of the two switch strains, C. neoformans var. grubii and C. gattii, has led us to identify important regulatory genes that are differentially regulated. This analysis not only identifies putative roles of the differentially regulated genes in the switch, but also indicates that differentially regulated genes can serve as targets for therapeutic strategies. The roles of these differentially regulated genes in the switch from heterothallic to autosurulating states are interesting, and future studies should explore the roles of these genes in the development of disease.

A genetic mutation has occurred in a gene in the common ancestor of all fungi that adds to the number of genes that are regulated in fungi. This gene is known as Zap1. In the pathogenic fungi, it encodes an enzyme that converts zinc to copper, which in turn reacts with copper to produce cyclic or reactive oxygen species. This reaction results in the formation of copper-dependent plasma.


## Introduction
086/3462), and Ensembl Proteins, http://ensemb.fungi.org/pdf/fungi_solutions.pdf (FungiDB, doi:10.1086/3462).

Phylogenetic tree obtained from the draft genome of the fungal pathogen U. maydis (GenBank AJ629887, Identities = 377/418 (93 %), 27 gaps (2 %)), obtained from the draft genome of the model fungus Neurospora crassa (GenBank NR_160427, Identities = 495/520 (93 %), 29 gaps (5 %)) and from a draft genome of the plant pathogen Phytophthora infestans (GenBank NR_160427, Identities = 468/524 (93 %), 22 gaps (5 %)). The draft is four reads short of the nomenclature required for full quality genome sequences (GC-ENO; NCBI method). The newly generated sequence reads are deposited in the GenBank database under the ATR-HELO and ATR-DIGG/TAT and GC-ENO; GenBank accession numbers are indicated at the nodes.

Colour illustrations. U. maydis in natural habitat, Southern Africa. Colony on malt extract agar; conidiogenous cells and conidia. Scale bars = 10 µm.

Etymology. Name reflects the location of culture inoculated and isolation from the soil.

Classification - Gulphomycetaceae, Agaricales, Agaricomycetes.

Conidiomata unhispulated, setae ellipsoid, rarely immersed, consisting of coarse, brown, branched, septate, globose to coarse, brown, globose, 5-6(-8) × 5-7 µm, arising from the base of a subepidermal connection. Conidiophores hyaline, smooth, subcylindrical, subcylindrical with apical taper, cylindrical, 0-1-septate, 10-20 × 2.


## Methods
capsulatum deletion mutant strains was performed as described [40. Briefly, in order to select for a complementation effect of the deletion mutant strains, the genomic DNA of each strain was extracted from the indicated colonies. The DNA extract was purified with a QIAGEN QIAquick PCR Purification Kit and then quantified with a Qubit® 2.0 Fluorometer. The concentration of DNA in each sample was measured in a NanoDrop 1000 spectrophotometer (Thermo Fisher Scientific, Inc.).

Intron-anchored PCR was used to determine the homology of the deletion mutants to the H. capsulatum deletion mutant genome. The gene of interest was replaced with the gene of interest using the primers 5’-F-UTR-ctTCTAGAGTGGTTG-3’ and 5’-F-TCCTCCTGCGTTCT-3’ (Table 1) and a final 5’-end was inserted into the A. fumigatus genomic DNA using the NcoI restriction enzyme and the primers. The PCR mixture contained 10 µL of 2× SYBR Green Master Mix (NEB), 1 µL of each primer (10 µM), and 1 µL of template genomic DNA. The PCR program was as follows: 5 min at 95 °C, followed by 35 cycles of 30 s at 94 °C, 30 s at 60 °C, and 20 s at 72 °C. Each PCR product was purified with the QIAquick PCR Purification Kit. The PCR product was then sequenced by the University of California, San Francisco Genomics Laboratory and aligned with the H. capsulatum genome with the Broad Institute’s Gene Expression Workbench (version 8.0) (http://www.broadinstitute.org/software/geneexpression/) (Table 1). The H. capsulatum genome sequence was used as a reference sequence for the determination of the evolutionary relationships among the deletion mutants.

Bioinformatic Analysis
The H. capsulatum genome sequences were assembled using the CLC Genomics Workbench (version 8.0) program and the alignments were checked and refined using the CLC Genomics Workbench. An example of the H. capsulatum genome is shown in Table 1. The H.


## Results
In addition, we investigated the impact of selection on the growth of the wild-assigned strain on various physiological condition. We inoculated wild-assigned mice with 10^7 C. neoformans strains and observed that in comparison with the wild-assigned strain, the wild-assigned strain showed significant growth inhibition in the presence of 5 mM H_2O_2 (Fig 2). These findings are consistent with our previous study that found that C. neoformans cells do not grow better in the presence of 10 mM H_2O_2 compared with the wild-assigned strain [9]. We also examined the growth of the wild-assigned strain in the presence of 2 mM Methyl-dextran in the presence of 5 mM Methyl-sulfite (Fig 3). The wild-assigned strain showed significantly increased growth compared with the wild-assigned strain. Methyl-sulfite inhibits mitochondrial functions in a number of ways [2], and Methyl-sulfite treatment promotes the growth of C. neoformans in the presence of H_2O_2 [11], [12], [13], [14], [17]. We also observed that the wild-assigned strain had a growth defect in the presence of Methyl-sulfite. However, this growth defect did not change in the presence of 10 mM Methyl-sulfite (Fig 3).

We then examined the ability of the wild-assigned strain to grow in the presence of H_2O_2. The wild-assigned strain was able to grow in the presence of 10 mM H_2O_2, as measured by anaerobic growth (Fig 4). Although the wild-assigned strain exhibited significantly more growth in the presence of H_2O_2 than the wild-assigned strain, this difference did not reach statistical significance (P = 0.59). The wild-assigned strain also showed significant growth inhibition in the presence of 2 mM Methyl-sulfite (Fig 4). The results suggest that the wild-assigned strain has a strong ability to grow in the presence of H_2O_2.

We next tested the growth of the wild-assigned strain in the presence of 2 mM Methyl-sulfite.


## Discussion
marcescens (Table 1), . aureus (Table 2), and S. bovis (Table 3), respectively. The FPI of S. marcescens was equal to the FPI for S. aureus (Table 2) but higher than the FPI for S. bovis (Table 3). In addition, the FPI for S. bovis was also higher than that of S. aureus (Table 2). This result indicates that the S. marcescens genome is likely to be more genetically diverse than that of S. aureus.

In conclusion, the Fungal Pathogenicity Index (FPI) for S. marcescens has been found to be greater than that for S. aureus and that the S. marcescens genome is significantly enriched for mitochondria compared with S. aureus. This result is consistent with the results of previous studies that have used the same approach to identify genes involved in pathogenicity [15, 17, 22, 32-34]. In addition, the FPI of S. marcescens for S. aureus and S. bovis is higher than that for S. aureus [32]. This result indicates that S. marcescens is a good candidate for S. aureus as a potential pathogenicity target in the field. Additionally, the FPI of S. marcescens for S. aureus was higher than that for S. aureus [32].

The results of the Fungal Pathogenicity Index (FPI) for the seven disease pathogens used in our study indicate that the FPI of S. marcescens is greater than that of S. aureus and that the S. marcescens genome is significantly enriched for mitochondria compared with S. aureus. The FPI of S. marcescens for S. aureus was equal to or lower than that of S. aureus [32]. This result is consistent with the results of studies that used the same approach to identify genes involved in pathogenicity [15, 17, 22, 32, 34-36, 40, 41]. In addition, the FPI of S. marcescens for S.
