# Genome Characterization of Fermenting Yeast Species from Ethiopia Using An Extracellular Macroarray
Quaglee Dragontacos


## Abstract
To survive interactions with the host cells, phagocytic cells must be able to anticipate and adapt to the dynamic changes in their host cells and effectively combat the infection. The conserved cell surface recognition receptors Ser, Cha and Mmm are crucial for Chamoeba histolyticus and are required for the virulence of many pathogens. In this study, we identified and characterized the Chamoeba histolytica-derived transcription factor (ChHfT) as an independent transcription factor of the cell wall. To study its function and to identify its regulatory mechanism, ChHfT was grown in the presence of Gln and the poly-butyl-alcohol fatty acids, a substrate known to prevent their synthesis. The ChHfT mutant was more susceptible to stress than the wild-type strain. When exposed to the stress of low alcohol content, the ChHfT overexpression did not abate due to loss of its transcript levels. Instead, it developed new stress resistance and lost its ability to synthesis of the components of the stress response.


## Introduction
Fungi are ubiquitous in the earth and are responsible for invasive fungal disease (IFD) in immunocompromised individuals. The most common presentation is aspergillosis, with a prevalence of 4%-6% in Europe and ~50% in the United States [1]. The causative agents include Histoplasma capsulatum, Blastomyces dermatitides, Coccidioides immitis, Coccidioides posadasii, and C. posadasii species complex (the so-called M. posadasii complex) [2], [3].

The M. posadasii species complex is responsible for the majority of disease in AIDS patients and other immunocompromised individuals. The M. posadasii complex is most frequently isolated in tropical and subtropical climates, and causes a broad spectrum of diseases in immunocompetent patients [4]. The M. posadasii species complex is divided into four subtypes, namely, M. circinelloides, M. restrictum, and M. sympodialis. The M.


## Methods
.0 software (QIIME Inc., Madison, WI, USA). The resulting data was analyzed using R [37] and the results are reported as the number of “expressed” proteins in each of the yeast species.

The raw data of the yeast species were aligned with the raw RNA-seq data of R. oryzae using the online software R. oryzae-scan-tool [38]. The reads were aligned to the sequences of the yeast species using Tophat2 software [39]. The top-scoring similarity score was selected for each sequence and the positions with the highest number of nucleotides were considered to be “similar” sequences. The positions of the top 3 nucleotides in a sequence were considered to be “similarly” sequences and the top 100 nucleotides in a sequence were considered to be “unsimilar” sequences.

Gene Structure and Structure Prediction
The gene structure of the different fungal species was predicted by using SignalP v.1.0 software [40]. The P. brasiliensis RNA-seq dataset was used for the protein prediction by using SignalP v.1.0 software [40].

RNA-seq Analysis
The RNA-seq data of P. brasiliensis were mapped to the P. nordica genome using the program R package [41]. The RNA-seq data were mapped to the C. elegans genome using the program LAMP v.0.3.6. The transcripts of C. elegans were predicted by using the software P. brasiliensis InterPro v.1.3.1.

Gene Structure and Structure Prediction
The data of C. elegans were aligned to the C. neoformans genome using the program Clustal W v1.0 [42]. The alignments were analyzed using the software MEGA v. 6.0 [43]. The transcriptome of C. elegans was mapped to the C. neoformans genome using the program M-Coffee v. 7.0.3.

Statistical Analysis
The data of C.


## Results
, 24 and 48 hour. The time of cultivation was also compared with the other three strains, i.e., the culture of the strains was carried out for 6-8 days at the highest inoculum (Figure 2). The fungal biomass in the culture medium was found to be significantly higher than the other two strains (p<0.001) (Table 2). The highest biomass value (log_2FC) was observed at 72 h, when the yeast strain was cultivated at a constant inoculum concentration of 100 g L^-1, and was found to be highest at 72 h when the yeast strain was cultivated at a constant inoculum concentration of 100 g L^-1. These results indicate that in the present study, the fungal biomass was significantly higher in the culture medium than in the other two strains.

Effects of Growth Conditions on the Fermenting Yeast Species and the Fermenting Microbe
The results obtained in this study indicate that the fungal biomass and the growth rate were affected by the incubation of the yeast strain in the presence of 10 g L^-1 and 10 g L^-1 of glucose. The higher fermentation rate, however, was found to be beneficial for the yeast strain. Fermentation of the other three yeast strains, i.e., the strains of P. mirabilis, P. aeruginosa and P. mirabilis ATCC 9231, was found to be significantly faster than that of the other three strains, i.e., the culture of the strains was carried out for 6-8 days at the highest inoculum (Table 2). In order to confirm the results obtained by microarray analysis, the fungal biomass and the growth rate of all yeast strains in the present study was determined by using the quantitative PCR assay (Figure 3). Results showed that the yeast biomass and growth rate of the three strains were significantly higher than that of the other three strains. The results indicated that the growth rate of the three yeast strains was significantly different than the growth rate of the other three strains.

Effects of the Fermenting Microbe on the Growth Rate of the Fermenting Yeast Species and the Fermenting Microbe
As can be seen in Figure 4, the yeast strain P.


## Discussion
terricola is encoded by 26 genes, including 13 predicted genes from the yeast proteom. A. terricola encodes five secreted proteins (ATCC 28964 and ATCC 5837) and four introns (1 and 2) (Figure S4A). The introns are located in the 5' and 3' UTRs and are conserved in the proteins involved in the cellular response to environmental stresses. In the yeast, these introns are located in the 5' UTR and are conserved in all the genes involved in the cellular response to environmental stresses (Liu et al., 2012). In the genome of A. terricola, 22 genes are encoded by introns (Figure S4B). These genes encode a few proteins involved in the cell wall integrity. The introns are located in the 3' UTR and are conserved in all the genes involved in the cell wall integrity. The introns are located in the 5' UTR and are conserved in the genes involved in the cell wall integrity. The introns are located in the 3' UTR and are conserved in all the genes involved in the cell wall integrity. The genes involved in the cell wall integrity were identified as CgLRR12, CgLRR24, CgLRR26, and CgLRR28 (Figure S4B). The genes encoding the CgLRR24, CgLRR24, CgLRR24, CgLRR26, and CgLRR28 were encoded by three different genes: CgLRR24 (ATCC 28964), CgLRR26 (ATCC 5837), and CgLRR28 (ATCC 5837). The CgLRR24 gene encodes a serine/threonine protein with the conserved amino acid residue (GGGGG)GGG in the consensus sequence (Figure S4C). The CgLRR26 gene encodes a serine/threonine protein with the conserved amino acid residue (GGGG)GGG in the consensus sequence (Figure S4D). The CgLRR28 gene encodes a serine/threonine protein with the conserved amino acid residue (GGGGG)GGG in the consensus sequence (Figure S4E).
