# Genome-Wide Identification and Analysis of Fatty Acid Synthase Gene Family in Basidiomycete Diaporthe
Quaglee Dragontacos


## Abstract
Members of the Dikarya (Mycaceae) were common inhabitants of tropical forests from what is now South America and North America, forming a dense layer of macrofungi. The genus Mycadorn is composed of more than 15 species, of which one species is described as a new species, M.


## Introduction
Cryptococcus neoformans is an opportunistic fungal pathogen that can cause pneumonia, meningoencephalitis, and lethal meningitis in immunocompromised patients.1, 2 The pathogen is found in the soil, on trees, in water, in food, and even inside human body (with the help of the host's immune system). It is an environmental saprophyte and is able to thrive in a wide range of conditions, including, but not limited to, the presence of macrophages, neutrophils, and T cells.3

The innate immune system plays an important role in the control of cryptococcal infection. Although the innate immune system plays a limited role in the control of the pathogen, it is important in the treatment of cryptococcosis. The presence of macrophages is an important factor that limits the pathogen’s ability to evade or evade the host’s immune response. In fact, macrophages are known to be the first cells that respond to cryptococcal infection, and their ability to kill the fungus is critical for the pathogen survival.


## Methods
graminearum (S1 File), . circinelloides (S1 File), F. circinelloides ctenophilae (S2 File), F. sputum (S3 File) and F. circinelloides ctenophilae (S3 File) were retrieved from public databases (NCBI) using a combination of the BLAST algorithm (http://www.ncbi.nlm.nih.gov/BLAST/) and BLASTN (http://blast.ncbi.nlm.nih.gov/BlastN/). The genome sequences of F. circinelloides were retrieved from NCBI using a combination of BLASTN and BLASTP (http://blast.ncbi.nlm.nih.gov/BlastN/).

The phylogenetic tree of F. circinelloides was generated using the neighbor-joining method with bootstrap values of 100 (Mason-Gamer and Kellogg 1996). The protein domain structure of F. circinelloides was predicted using the online tool ExPASy (http://expasy.org/). The C-terminal region of F. circinelloides was predicted using the online tool ProteinPilot (http://www.expasy.org/).

The ab initio model of protein kinase function of F. circinelloides was used to estimate the homologous proteins of F. circinelloides. In brief, the sequence of F. circinelloides ctenophilae was downloaded from NCBI. The sequences of F. circinelloides ctenophilae were aligned with Clustal X and the maximum likelihood phylogenetic tree was constructed using MEGA 7.0 software (http://www.megasoftware.net/).

F. graminearum Isolate and Genetic Profile
The F. graminearum isolate was selected from a collection of clinical isolates in Svalbard, Norway. F. graminearum isolate was isolated from this fungus from the respiratory system in Svalbard. F. graminearum was isolated from a mixture of the respiratory and non-irradical isolates. F. graminearum and F. circinelloides ctenophilae were maintained in Sabouraud dextrose broth at 25°C.


## Results
oxysporum, . graminearum, F. verticillioides, F. oxysporum, F. oxysporum, F. verticillioides, F. oxysporum, F. verticillioides, F. graminearum, F. verticillioides, F. oxysporum, F. graminearum, and F. verticillioides (Fig 1).

The fungal and plant pathogenicity genes were represented by the phylotypes of F. oxysporum, F. graminearum, F. verticillioides, F. graminearum, F. verticillioides, F. oxysporum, F. verticillioides, F. verticillioides, F. graminearum, and F. verticillioides (Fig 2).

The fungal and plant pathogenicity genes were represented by the phylotypes of F. oxysporum, F. graminearum, F. verticillioides, F. graminearum, F. verticillioides, F. oxysporum, F. verticillioides, F. verticillioides, F. graminearum, F. verticillioides, F. verticillioides, F. graminearum, F. verticillioides, and F. verticillioides (Fig 3).

The fungal and plant pathogenicity genes were represented by the phylotypes of F. oxysporum, F. graminearum, F. verticillioides, F. graminearum, F. verticillioides, F. verticillioides, F. verticillioides, F. graminearum, and F. verticillioides (Fig 4).

The fungal and plant pathogenicity genes were represented by the phylotypes of F. oxysporum, F. graminearum, F. verticillioides, F. graminearum, F. verticillioides, F. verticillioides, F. graminearum, and F.


## Discussion
For example, the expression of the GH6_2 gene family in Basidiomycota is highly conserved (50% identity and 10% overlap in 40 gene families) and has been shown to be required for full virulence in Fusarium head blight [44. The GH4_3 gene family is highly conserved in Saccharomycetales species (35% identity and 30% overlap in 40 gene families). In contrast, the GH5 gene family is highly conserved in Saccharomycetales species (40% identity and 18% overlap in 40 gene families). The high levels of sequence conservation suggest that the genes in the gene family could have been introduced into a common ancestor that was an early diverging lineage. This implies that the gene family could have been introduced into a common ancestor that was an intermediate to a diverging lineage. This idea is supported by the phylogenetic analysis of the 40 individual genes in each gene family. However, the comparative analysis of the 28 individual genes in each gene family indicates that only the genes in each gene family could have been introduced by diverging genes within the gene family.

A similar arrangement of gene families has been reported in other basidiomycetous yeasts, including C. albicans [45], Schizosaccharomyces pombe [46], C. neoformans [47], and Cryptococcus gattii [48]. However, this does not explain the large number of different gene families in each gene family in basidiomycetous yeasts. Furthermore, the presence of different gene families within each gene family in basidiomycetous yeasts suggests that the gene families could have been introduced from different early diverging lineages.

Host-specific gene families have been shown to be involved in several functions in the pathogenicity of microorganisms, including pathogenicity of plant pathogens and infection of the human host [49].
