# Genome sequence and comparative analysis of clades of theropithecipariotrichum tree species of Iran
Quaglee Dragontacos


## Abstract
Cryptococcus gattii is the etiological agent of the Pacific Northwest outbreak. Recent studies have implied that this pathogen affects primarily immunocompetent hosts in the western North. Little is known about the genomic variation among Pacific Northwest isolates of C. gattii. To better understand the genomic variability of this pathogen, we characterized the genomes of two clinical isolates (a male and a female) and a cox10 strain and two environmental isolates (a female and a zygomycete). We also examined the genomes of four additional clinical strains and four additional environmental strains, and determined the presence of phenotypic markers. The clinical isolates contained 12 genomic markers that were similar to those found in Saccharomyces cerevisiae, but were enriched in transposable elements. The environmental isolates were molecular crosses between two clinical strains, each of which had a unique genetic background and was more closely related to the host isolates. Phylogenetic analyses were performed using nuc rDNA internal transcribed spacer (ITS) region sequences and the nuclear ribosomal DNA (nudI) domain region from the clinical isolates and the environmental isolates. The clinical isolates were genotypically highly related (all P = 0.0003) to one another and were nested within a strain cluster that included the clinical isolates and the environmental isolates. The clinical isolates were more closely related to each other than to the environmental isolate. The environmental isolates had a unique genetic background, and they were subjected to repeated sexual crosses and exchange of the clinical isolates. The environmental isolates also had a unique genetic background that was likely inherited from a third-partner. No significant difference in the genetic background or the nested crosses between the two clinical isolates was observed. The nested crosses observed in the environmental isolates are likely responsible for the observed genetic differences. These results are consistent with the hypothesis that environmental strains can be transmitted from mothers to transmit infection from their host(s), and further support the hypothesis that the presence of genetic differences among environmental isolates can influence the transmission of a novel genetic background from one host to another.


## Introduction
The cephalobacterium and grasshopper genera are related to the cephalobacterium and grasshopper (Scedosporium and Phaeosphaerella), whereas . aceris is closely related to P. aceris (Cephalobacterium), and P. aceris is related to the cephalothallus (Cephalothallus) [19]. The tsetse fly (Cephalothallus) and the grasshopper (Scedosporium) are widely distributed in nature and important plant pathogens in the grasshopper genera, e.g. Scedosporium and Phaeosphaerella [15,16,19,20]. In addition, the cephalothallus is highly pathogenic in animal and plant hosts, as it causes disease in humans (mostly in humans) [15,16,19].

Pseudomonads are classified in five different families, namely, pterodactyls, monodactyls, pteromonads, and eu- and di-pterodactyls [21,22]. Monodactyls are well-known to form the backbone of the tsetse fly and occur in all genera of insects, including cephalopods [23], and are mainly responsible for disease formation in humans [22,24]. Monodactyls are relatively simple in structure, and have a unique three-dimensional structure and are characterized by a cationic conformation [25]. The cationic conformation is usually found in lower order biological units such as: dactyls, dactyl-lysine, and lysine, but they have a more complex structure and are more abundant in the tsetse fly and other insects [26,27]. The dactyl-lysine structures are very similar and the dactyl-lysine-lysine (DLC) structures are very similar and are common to all the other insects.


## Methods
These two genomes were aligned using the CLUSTALW command-line tool [54] and the alignment was subjected to multiple sequence alignment. The alignments were manually adjusted and the intron positions were deleted using the E-INS-i option in BioEdit v7.0.9 [55]. The sequences of the two clades were concatenated and concatenated and concatenated with the three accession numbers of the concatenated genomes of clade 1 (clade 1: hgci_d-hgcc_c_c) and clade 2 (clade 2: hgci_d-hgcc_c_d). The remaining concatenated genomes were compared with the complete genomes of the other clades of the tree species, the clade containing the clade containing clade 4 (clade 4: hgci_d-hgcc_c_d) and the clade containing clade 5 (clade 5: hgci_d-hgcc_c_d). The data from concatenated genomes were compared with the concatenated genomes of the whole tree species of Iran.

The intron positions of the concatenated genomes were checked manually and the positions of the intron in the concatenated genomes were eliminated manually.

Gene expression of theropithecipariotrichum by qRT-PCR
The transcript of the rpoA gene was expressed in all clade 1 and clade 2. The expression level of rpoA was analyzed in all clades of the tree species of Iran. The qRT-PCR was performed with SYBR Green PCR Master Mix (Applied Biosystems, Foster City, CA, USA) using the primer pairs of rpoA_F/rpoA_R and the PCR primers of the rpoA gene (Table 2) and the primers of the two clades (Table 2). The gene expression of rpoA was analyzed in all clade 1 and clade 2.


## Results
flavigenum-like genus is not supported in the phylogenetic analyses (Fig 3. The branch length analysis indicated that the clade containing the E. flavigenum-like genus is in conflict with the clade containing the E. neoformans-like genus, which is strongly supported by bootstrap values of 95% and 95%, respectively. A phylogenetic tree of the E. flavigenum-like and E. neoformans-like clades (Fig 4) was found to be in conflict. The clade containing the E. flavigenum-like and E. neoformans-like clades (Fig 5) was located in a highly supported branch with bootstrap values of 94% and 95%, respectively. The branch length analysis indicated that the E. flavigenum-like and E. neoformans-like clades are in conflict.

The E.


## Discussion
The identified genes are highly similar to those in other phylogenetic species, including aeoliolipids and the halophilic Saccharomyces species (Fig 3.

Phenotypic analysis of the tetrachordin gene
The tetrachordin gene in the chloroplast genome was sequenced and the tetrachordin gene cluster was annotated in the NCBI-NR sequence database. Two independent studies (Nishida et al. 2007) and a single phylogenetic study (Mao et al. 2008) have annotated the chloroplast genome and tetrachordin genes, respectively. The latter study suggested that tetrachordin gene cluster may be a remnant of the chloroplast, which was lost and duplicated in the chloroplast (Nishida et al. 2007). The phylogenetic analysis of tetrachordin gene cluster confirmed the presence of two additional genes in the chloroplast. The tetrachordin genes in the chloroplast were clustered into two clusters, one of which is the one containing the tetrachordin gene cluster. The second cluster contains the tetrachordin gene cluster. The tetrachordin gene cluster was located in the chloroplast (Nishida et al. 2007). The results of the phylogenetic analysis of tetrachordin gene cluster are shown in Fig 4.

Phenotypic analysis of the tetrachordin gene
The tetrachordin gene cluster was identified in the chloroplast of the chloroplast (Nishida et al. 2007). The phylogenetic analysis of tetrachordin gene cluster is shown in Fig 5.

Phenotypic analysis of the tetrachordin gene
The tetrachordin gene cluster was identified in the chloroplast of the chloroplast (Nishida et al. 2007). The phylogenetic analysis of tetrachordin gene cluster is shown in Fig 6.

Phenotypic analysis of the tetrachordin gene
The tetrachordin gene cluster was identified in the chloroplast of the chloroplast (Nishida et al. 2007). The phylogenetic analysis of tetrachordin gene cluster is shown in Fig 7.
