# Hyphal Development in Candida Metastases Isolated from Mammals
Quaglee Dragontacos


## Abstract
There is a paucity of new genetic information released in the last decade concerning the evolution of the eukaryotic microbiome. To fully understand the role of microbial genes in shaping fungal communities and their adaptive traits, we analyzed fungal gene transcripts from a wide range of fungi.

We found that a broad range of fungal species contains abundant and at least one bacterial component in their communities. This abundance is related to their growth and/or virulence, and related to the phylogenetic composition of the species. Our analyses also uncovered a common aspect among the fungal kingdom with the taxa of which these samples belong and also to the relationship between their abundance and virulence.

We found abundant and unique microbial genes that are common to all taxa and are maintained over evolutionary time. These include a group of fungi with the ability to undergo vegetative growth in mexic acne and a group of plants with the ability to grow on and release spores. We identified genes important to the virulence of a broad spectrum of fungi and a few small molecules that are especially active in virulence of a few bacterial species. We also identified genes that are frequently absent from the genomes of bacterial species, and a group of genes that are especially important in the development of a new disease caused by a single bacterial species.

Fungal genes are common in all eukaryotic genomes and are maintained over evolutionary time. This implies that they evolved rapidly and independently. However, we identified abundant genes that were either absent or highly conserved in fungi. We found evidence for a common ancestral pathway for the gene expansion of the virulence factor phytoeba in fungi and/or in bacteria. This ancestral pathway appears to have enabled the rapid expansion of the virulence factor complex in bacteria and for the accumulation of the virulence gene in fungi. We found genes that are associated with virulence and/or that are likely to contribute to the expansion of phytoebic fungi in fungi.

The study contributes to the understanding of the functional roles of fungal gene repertoires by demonstrating that the evolution of fungal virulence and for the expansion of the phytoebic fungi in the fungi are intertwined.


## Introduction
The World Health Organization (WHO) has recognized the need for safe, rapid and effective vaccines against bacterial and fungal pathogens. Immunization has been a key tool in combating disease [1], [2] and the introduction of new antimicrobial vaccines has been largely associated with the development of recombinant vaccines [3], [4]. The current model of naturally-acquired vaccine candidate against bacterial and fungal diseases, the pneumococcal conjugate vaccine (PCV), is one such example. A natural PCV vaccine, with a molecular mechanism of action, is the key development model in vaccine design, development and commercialization [5].


## Methods
oryzae, . mackenziei, M. sydowii, M. sydowii, and M. trachomatis. The cultures were maintained on potato dextrose agar (PDA) plates at 37°C for 7 days. BHBs were transferred to PDA plates and transferred to 20% PDA at 37°C. The colonies were identified by colony morphology and by morphological features on the PDA plates. The cultures were further subcultured onto PDA and incubated at 20°C for 7 days. After 7 days of incubation, the colony morphology of the isolate was observed and described by Dr. T.J. Koch (Colleges of the University of São Paulo, São Paulo, Brazil).

DNA extraction, PCR amplification, and sequencing
The genomic DNA of the isolate M. oryzae was isolated from the blood of five BHBs and the total DNA was extracted using the DNeasy Plant Mini Kit (Qiagen, Hilden, Germany). The internal transcribed spacer (ITS) regions of the bacterial 16S rRNA gene were amplified using the universal bacterial primers 338F (5'-GTGCCAGCGTTTGAGCAG-3') and 606R (5'-TTCCGTGTTTCAAGACGAC-3') [26]. The bacterial 16S rRNA gene sequences were assembled and compared with the reference sequence of M. oryzae (GeneBank: L134597) using BLAST. The primers ITS5 and ITS4 [27], which have been described previously [28], were used for the amplification of the 16S rRNA gene sequences of the M. oryzae isolate M. oryzae. The PCR products were purified using the QIAquick PCR Purification Kit (Qiagen, Hilden, Germany) according to the manufacturer’s recommendations. The sequencing of the sequences was performed on a Genome Sequencer platform (GENESYS, Uppsala, Sweden).

Phylogenetic analyses
The deduced amino acid sequences of the sequences obtained in this study were deposited in GenBank (accession numbers are given in Table 1) and aligned in MEGA 6.0 (Tamura et al., 2011).


## Results
NRRL 34438 was isolated as the representative of a novel species, Pseudomonas s. NRRL 34438, from the leaves of two different dead rodents in a field (Figure 2).

The sequence of the a-tubulin (BnTub) sequence of the a-tubulin (BnTub) of Pseudomonas sp. NRRL 34438 was shown in BLASTp searches against the GenBank database and the S. cerevisiae genome databases (http://genbank.expasy.org/). The phylogenetic tree of the a-tubulin sequence of Pseudomonas sp. NRRL 34438 (Figure 2) is supported by a well-supported tree topology that is similar to that of the M. oryzae-pathogen (Figure 2).

The complete a-tubulin sequence of Pseudomonas sp. NRRL 34438 (Figure 2) was searched against the M. oryzae genome database, using the BLASTp search against the S. cerevisiae genome database, using the S. cerevisiae genome database (http://genbank.expasy.org/). The result of the S. cerevisiae and M. oryzae comparisons revealed that the genome of Pseudomonas sp. NRRL 34438 is very similar to that of the M. oryzae comparison (Figure 2).

To further validate the P. chrysogenum and M. oryzae phylogenetic tree of Pseudomonas sp. NRRL 34438 and the M. oryzae comparison, we performed a phylogenetic analysis of the a-tubulin sequence of Pseudomonas sp. NRRL 34438 (Figure 3). The tree topology of the a-tubulin phylogeny of Pseudomonas sp. NRRL 34438 (Figure 3) is similar to that of the M. oryzae phylogeny (Figure 2). The tree topology of the a-tubulin phylogeny of M. oryzae (Figure 3) is similar to that of the M. oryzae phylogeny (Figure 2).

Pseudomonas sp. NRRL 34438 Is a Hyphalologous Plant Pathogen
To investigate the relationship between the a-tubulin and M. oryzae phylogenetic tree of Pseudomonas sp. NRRL 34438 and the M.


## Discussion
Metagenomic reads can be useful in identifying fungal species and fungi with a wide range of lifestyle. However, this approach is not feasible for eukaryotic species because of the limited number of fungi available, and the high abundance of fungi and bacteria in the phylogenic range of metagenomic reads. To investigate this further, we constructed and compared the metagenomic libraries from fungal species and the bacteria, using Illumina MiSeq sequencing and the TALIBITS sequencing. We compared the metagenomic libraries from metagenomic reads from fungi, plants, and animals, and compared the metagenomic libraries from fungi and bacteria. Results showed that the fungi and bacteria with a large number of fungi and bacteria were the most frequent fungal species, followed by the bacterial species. The fungi were found in all studied communities. In contrast, bacteria were found in only two of the communities. The bacterial species were found in only two of the communities. These results indicate that the fungi and bacteria with a large number of fungi and bacteria are the most frequent fungal species, followed by the bacteria.

The Metagenomic Sequences of Fungi and Plants were Expanded to 107,000, 234,000, and 45,000 bp, respectively, and the Metagenomic Reads from Fungi and Plants were Expanded to 25,000, 230,000, and 77,000 bp, respectively. The Expanded Metagenomic Reads for Fungi and Plants were Expanded to 35,000, 73,000, and 87,000 bp, respectively. The Expanded Metagenomic Reads for Fungi and Plants were Expanded to 45,000, 59,000, and 164,000 bp, respectively.

We have shown that the genome size of C. neoformans increases with the increase in the number of copies of the gene. This is consistent with the idea that genes can be expanded by gene duplications [20]. In addition to gene duplication and gene loss, gene loss and duplications can occur, and the gene remains within the genome. Therefore, the gene may be subject to recombination and gene transfer [21].
