# Identification and Characterization of Two Candida glabrata Species Identification Chromosomes in Water Samples from the Sopco State, South Vietnam
Quaglee Dragontacos


## Abstract
The wheat stripe rust fungus Puccinia graminis f. sp. tritici (Pgt) is a causative agent of stripe rust, which is threatening the establishment of Europe’s wheat belt. The aim of this study was to generate genetic markers of the Pgt-causing agents of stripe rust, Puccinia graminis f. sp. tritici (Pgt), by using gene disruption and functional assays. The resulting genome was approximately 2.5 Mb and encodes a poly-nucleotide array containing 15,571 predicted proteins. A total of 112,740 genes were identified including genes involved in secondary metabolite biosynthesis, secondary DNA and RNA metabolism, and predicted proteins related to the fungal organization and pathogenesis. In silico prediction of the Pgt protein-encoding genes was performed by comparison with expressed proteins in the Pgt-causing Pgt clonal. The results revealed that the Pgt genetic markers had the potential to be used in a practical way to generate genetic markers of Pgt species.


## Introduction
The detection limits for bacterial DNA detection in the Sopco State were reported as 10^6 to 10^16 (9.4 %) to 10^15 (93.5 %), whereas in Guangdong Province, the detection limits were 10^1 to 10^2 (86.4 %) and 10^3 to 10^5 (85.2 %). The bacterial DNA detection techniques were developed using different methods including amplicon pyrosequencing (PCR, sequencing, and qPCR) and universal primers (Candida Genome Project Consortium, Candida Genome Project). The PCR amplification of bacterial DNA using the primer CAAX1 and CAAX2 was used to amplify and sequence the sequences of the bacterial pathogen Enterococcus faecium from water samples collected from Sopco State. The internal transcribed spacer (ITS) and intervening small subunit (28S) sequences were used for PCR amplification of the fungal pathogen Bacillus cereus using the primer ITS1 and ITS2. The ITS2 and ITS3 sequences were used to amplify the fungal pathogen Bacillus lentus ammonium and Bacillus anthracis from water samples collected from Sopco State. PCR-RFLPs were developed and tested on a variety of substrates including surface-sterility substrates, glycerol, glycerol-containing medium, and organic extracts from plant material, organic solvent, and agricultural residue. The PCR primers were designed to amplify the fungal pathogen and the host proteins, including those found on fungal plant material (Agroscope, Agrio, Agroscope) and organic solvent. The ITS2 and ITS3 sequences were used to amplify the fungal pathogen and the fungal host proteins. The ITS sequences were used to develop a model to investigate the PCR primers. The nucleic acid amplification method was developed by using a conventional primer and an ITS1 primer. The nucleic acid amplification method was developed using a combination of two-step primer formation and amplification of the fragments by PCR. The primers were designed to amplify the fragments by endogenously, followed by PCR to detect the internal transcribed spacer (ITS) and intervening spacer (SS) sequences.


## Methods
glabrata species (. glabrata P7 and P7-P7) were identified by sequencing the fungal genomic DNA (gDNA) as previously described [5]. Genomic DNA from the two C. glabrata strains was isolated from a dried rhizome of a rice cultivar and stored at -80°C until use. PCR amplification was performed using the universal fungal primers, ITS1F and ITS4 [20], and the internal transcribed spacer (ITS) region of the rDNA, as previously described [22]. The PCR amplification was performed using a C1000 Thermal cycler with the following conditions: initial denaturation for 5 min at 94°C, followed by 35 cycles of 30 s at 94°C, 30 s at 55°C, and 30 s at 72°C. The products were visualized on 1% agarose gel. Each PCR product was purified with an enzymatic PCR cleanup [23] and the sequence was checked for homology with the C. glabrata P7 and P7-P7 strains. The resulting sequence was deposited in GenBank (accession number MF8801).

To identify the fungal species, an ITS region was amplified using the fungal ITS4 region as described previously [24]. In brief, 5 µL of a 10× PCR reaction mixture contained 5 µM of each fungal gene, 1.5 µM of each fungal ITS gene, 1.25 µM of each primer (5 µM), 0.25 µM of each deoxynucleoside triphosphate, 0.2 µM of each of the four primers, and 0.25 µM of DNA template [25]. PCR reactions were performed in a C1000 Thermal cycler with the following conditions: initial denaturation at 95°C, followed by 35 cycles of 30 s at 94°C, 30 s at 54°C, and 30 s at 72°C. The product was visualized on 1% agarose gel. Each PCR product was purified with an enzymatic PCR cleanup [25] and the sequence was checked for homology with the C. glabrata P7 and P7-P7 strains. The resulting sequence was deposited in GenBank (accession number MF8801).

The ITS region was amplified using the fungal ITS2 region [26], as previously described [27].


## Results
The yeast isolates were identified as yeast strains of the Ascomycota, Basidiomycota, Glomeromycota, Glomeromycete, Mucoromycotina, and Zygomycot. The most common yeast species isolated from water samples were Aspergillus fumigatus (56.3%) and Penicillum canescens (31.6%). Several isolates of Aspergillus spp. were also found in water samples, and these isolates were identified as Aspergillus spp. s.str. 18 (Table 2).

The yeast isolates were grouped into three different families (Table 1) based on the ITS sequence and poly-allelic sequence analysis (Fig 1). Ascomycetes were the most commonly isolated yeasts from the water sample (37.5%), followed by Basidiomycetes (18.5%), and Mucoromycotina (10.3%). Aspergillus spp. was the most commonly isolated yeast from the water sample (53.7%), followed by Glomus spp. (20.5%) and Mucoromycotina (9.2%). Mucoromycotina was the most commonly isolated yeast species from the water sample (40.8%). Interestingly, the Dothideomycete yeasts were more commonly isolated from the water sample (43.4%) than Basidiomycetes (22.7%). Aspergillus spp. was most commonly isolated from the water sample (46.4%) followed by Glomus spp. (10%) and Mucoromycotina (7.3%). The most common Aspergillus spp. isolates were Aspergillus fumigatus (44.5%) and Penicillum canescens (13%) (Fig 1).

Among the ascomycetes, the most frequently isolated yeasts were Penicillum canescens (15.2%) (Table 2). In addition, the most frequently isolated yeasts were Alternaria alternata (19.6%), Aspergillus fumigatus (19.4%) and Alternaria bovis (7.5%).


## Discussion
glabrata, . krusei, C. parapsilosis, and C. tropicalis) and one species (C. glabrata) in each of the water samples (Figures 1 and 2). A total of 18 species (C. parapsilosis, C. tropicalis, C. parapsilosis, C. lusitaniae, C. lusitaniae, C. nectarius, C. niger, and C. harringtonii) were identified based on the sequences from the six-plex chromosome of the water samples (Figures 1 and 2).

The analysis of the six-plex chromosome of the water samples from the Sopco State, South Vietnam revealed five species (C. glabrata, C. krusei, C. tropicalis, C. lusitaniae, C. nectarius, C. niger, and C. harringtonii) and one species (C. glabrata) in each of the water samples (Figures 1 and 2).

The analysis of the six-plex chromosome of the water samples from the Sopco State, South Vietnam revealed two species (C. glabrata and C. krusei) in each of the water samples (Figures 1 and 2).

The analysis of the six-plex chromosome of the water samples from the Sopco State, South Vietnam revealed five species (C. glabrata, C. krusei, C. tropicalis, C. lusitaniae, C. nectarius, C. niger, and C. harringtonii) in each of the water samples (Figures 1 and 2).

The analysis of the six-plex chromosome of the water samples from the Sopco State, South Vietnam revealed two species (C. glabrata and C. krusei) in each of the water samples (Figures 1 and 2).

The analysis of the six-plex chromosome of the water samples from the Sopco State, South Vietnam revealed one species (C. glabrata) in each of the water samples (Figures 1 and 2).
