# Identification of viral RNA as a target for drug development by Matrix NAST-PCR
Quaglee Dragontacos


## Abstract
Insecticides are widely employed in agriculture to control pests and pathogens in crops. However, the toxicity and the consequent environmental impact of insecticide use is poorly defined. In this study, we determined the in vitro effects of the chemical class Bd of the essential oil of Eryngin (Eryngin) against two strains of Galleria mellonella, a model of pathogenic insects, were determined using minimal inhibitory concentration (MIC) and minimal fungicidal concentration (MFC). In addition, the effects of the essential oil on G. mellonella were determined using a minimal fungicidal concentration (MIC) and minimal in vivo effect (MICI) on Galleria larvae. The minimal inhibitory concentration (MIC) and minimal fungicidal concentration (MFC) of the essential oil were 0.0625 mg/mL and 0.0625 mg/mL, respectively. The MFC concentration and MIC of Eryngin ranged from 0.0625 mg/mL to 0.0625 mg/mL for Eryngin-containing Eryngin-containing chemicals (e.g., eicosapentaenoic acid) and from 0.0625 mg/mL to 0.0625 mg/mL for non-containing chemicals (e.g., naphthalene). The in vitro MFCI concentration was 0.


## Introduction
Invasive fungal infections (IFIs) continue to be an increasing problem in the clinical setting, with over the last two decades an estimated 1.7 million cases and >450,000 deaths per year worldwide (1, 2). The rising number of immunocompromised patients, particularly patients undergoing chemotherapy, radiotherapy, and organ transplantation, has also led to an increase in the incidence of IFIs (3, 4). The most common opportunistic pathogen is Candida albicans, accounting for >95% of IFIs in the United States, and >85% of all invasive fungal infections (IFIs). The most common non-albicans species, C. glabrata and C. tropicalis, account for most of the infections (4-7). IFIs are often initially diagnosed by fungal susceptibility testing (AFS) and by using conventional microbiological methods, and IFIs are still an important cause of morbidity and mortality in immunocompromised patients (8-10).

Given the growing incidence of IFIs, several strategies have been developed to reduce the incidence of fungal infections.


## Methods
RNA was extracted with the Trizol reagent (Invitrogen) according to the manufacturer's recommendation. RNA integrity was verified by agarose gel electrophoresis. RNA was stored at -20Â°C until use.

Preparation of cDNA and cDNA libraries
Total RNA was isolated from five million cDNA reads using the RNAeasy kit (Qiagen) according to the manufacturer's instructions. A random number generator was used to identify the reads with a quality score of at least 10. Differentially expressed genes (DEGs) in each sample were determined using the Gene Expression Index (GSE) (http://www.ncbi.nlm.nih.gov/geo/) and the relative expression of a gene was calculated using the method of Ellman (1999) and normalized using the read count. The normalized expression was considered significant if the P value was less than 0.01.

Generation of adapter sequences
The primer sequences of the C. albicans genome-wide Sanger sequencing (Sanger, Inc.) were designed and the sequence reads were assembled with the Geneious v. 7.1.9 software. The primer sequences were then combined with the VCFs to generate the assembly file for the forward primer of C. albicans. The primer sequences were then used as input for the Blast2GO v. 4.6.0 program to calculate the gene model. The Gene Ontology (GO) and Kyoto pathway enrichment analysis (KOG) were conducted using the Blast2GO v. 4.6.0 program to determine functional functional groups of the different genes. The KOG was assigned to the protein class by the KOG annotation of the Protein Data Bank. The pathway analysis of C. albicans was conducted using the KOBAS2 software (http://www.macrocognition.org/index.html) to determine the effect of different oomycete fungal pathologies on C. albicans gene models. The KEGG gene network was assigned to the protein class by the KOG annotation of the Protein Data Bank. The KEGG gene network was then used to predict the ability of C. albicans to host the pathogen in a balanced manner.


## Results
We performed a Bio-Rad VITEK MS analysis using a 2-step analysis of the noncovalently bound RNA with Target Finder (TMAC) software, which uses the same viral DNA as the virus RNA librar. RNA isolated from the 5' and 3' ends of the viral RNA were pulsed with a serial dilution of 10 ng/ml of virus RNA in YPD broth prior to determining the percentage of viral RNA that could be detected using the RNA-Seq assay (Figure 1A). The number of viral RNA detected was proportional to the amount of viral RNA in the library, indicating that the viral RNA was efficiently picked up by the VSAM assay. The VSAM assay is a technique for detecting the presence of viruses in the body during clinical infection [21]. The viral RNA sample was prepared in 2-step sequence analysis by placing it in the absence of tryptophan, followed by serial dilution with 10 ng/ml of viral RNA in YPD broth and then pulsed with a serial dilution of 10 ng/ml of virus RNA in YPD broth for a total of 4 h. The results of the VSAM assay were shown in Figure 1B. The results indicated that the viral RNA was detected in the 2-step sequence analysis of the VSAM assay.

In order to determine if the viral RNA was detected in the HIV-1 macrophage assay, we repeated the detection and characterization of the viral RNA in the HIV-1 macrophage assay using the viral RNA-detection kit (Figure 1C). The results indicated that the viral RNA was detected in the 2-step sequence analysis of the HIV-1 macrophage assay, but not in the HIV-1 macrophage assay (Figure 1C).

Viral RNA is a target for antifungal agents
We next tested the susceptibility of the VSAM assay to the antifungal drug amphotericin B. We also performed a MIC test using the bacterial cell line J774. A. to macrophage. We used a phagocytic assay to evaluate the susceptibility of the macrophage-inoculated J774. A. to macrophage mixture to fluconazole, and observed that the MIC was significantly lower for the virus RNA than for the virus-infected J774. A. to macrophage mixture (Figure 2A).


## Discussion
In order to understand the gene expression of a specific subset of genes, a BLASTX search was performed against the human genom. The identified gene expression targets, in order to demonstrate their utility in the development of novel drugs, were compared with those of the corresponding human homologues (Figure 1A). The hit genes from the Biosystems database were also compared with the human homologues. The Biosystems database was used to identify novel drug targets. The number of hit genes from the Biosystems database was used to determine the number of gene families (total of 6). The number of hit genes in the Biosystems database was used to determine the number of genes that are present in the Biosystems database.

For each gene, a total of 4,906 genes were identified, and the hit gene families were assigned to the following categories: genes involved in amino acid metabolism, sugar metabolism, DNA repair, cell cycle, transcription, and apoptosis, as well as genes involved in the biosynthesis of the other components of the cell membrane, such as cell wall, extracellular matrix, melanin, and fatty acid metabolism. The results show that the total number of genes for each hit gene was higher than that for the corresponding gene set, indicating that genes in the Biosystems database are more likely to be involved in the biosynthesis of amino acids and lipids. These results suggest that genes in the Biosystems database are more likely to be involved in the biosynthesis of lipids and other components of the cell membrane and extracellular matrix. The highest number of genes in the Biosystems database were involved in the biosynthesis of sterols and catalases. The Biosystems database was used to find genes in the biosynthesis of sterols and catalases. The number of genes in the Biosystems database was used to assign the hit genes to the corresponding genes. The number of genes in the Biosystems database was used to assign the hit genes to the corresponding genes. The number of genes in the Biosystems database was used to assign the hit genes to the corresponding genes.
