# Importance of Resolving Fungal Nomenclature: The Role of Morphological and Genetic Analyses of Fungal Nomenclature across Domain Kingdoms
Quaglee Dragontacos


## Abstract
Extracellular proteases are critical for the virulence of many bacterial pathogens including methicillin-resistant Staphylococcus aureus (MRSA) and Salmonella enterica serovar Typhi. To identify extracellular secretory proteins in MRSA, we generated mutant strains for all five proteins in the secretory machinery of MRSA. We found that the MRSA secretory machinery utilizes a specific and conserved region called MYM, which is involved in coordinating secretory events with other protein components. Mutant MRSA strains with MYM deletion displayed reduced virulence on the animal infection model of Escherichia coliosis. Myrm MYM-P6A abolished the virulence of MRSA in the Galleria mellonella model of MRSA. MRSA strains with MYM deletion also induced reduced neutrophil invasion and innate immunity in the murine infection model. MRSA strains with MYM deletion also induced higher levels of IL-1ß production in the brain than the WT strain. MRSA strains with MYM deletion also induced lower levels of IL-6 and IL-10 in the brain than the WT strain. Furthermore, MRSA strains with MYM deletion lost secretory protein MUMM1, which is involved in coordinating secretory events with other protein components. In summary, MRSA is a major cause of acquired immunodeficiency syndrome-associated cerebral infarction in S. aureus and MRSA is involved in the immune response against Salmonella enterica serovar Typhi.


## Introduction
Condwell,

We are pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it complies with all outstanding technical requirement.

The following external editor has agreed:

The following external referees have agreed:

The following external editors have agreed:

The following external peer reviewers have agreed:

The following external peer reviewers have provided the following comments:

The manuscript will proceed to peer review, and the external review process will be notified to the authors. Please review the revised manuscript carefully, and let us know what you think of the work. If the paper is rejected, the next round of peer review will be scheduled for you.

Within one week, an external editor has selected a final version of the manuscript and sent it to the peer review system. The final version will be formally published online.

A version of the manuscript with figures showing the proportion of gene copies of different major fungal phyla is generated and deposited in a public database called Index Fungorum. The Index Fungorum data set contains information on publicly available fungal information from inception to date. The data set is publicly available, and all identified genes or species are deposited in the database. The database is designed to provide a platform for rapid and efficient gene identification and, therefore, an efficient resource for the study of fungal Nomenclature.

The data generated from Index Fungorum is deposited in PubMed Central, and the most common primers used for gene prediction are Aspergillus nannfii, Candida glabrata, Cryptococcus neoformans, and Pichia kluyveri. For a complete overview of the available fungal Nomenclature, refer to the website of Index Fungorum.

The following external peer reviewers have agreed:

The authors of this study have made excellent suggestions and we feel that they have improved the manuscript. However, we are aware of the risks associated with resubmitting, and there is a possibility of legal repercussions for doing so. Therefore, we have opted to not resubmit.

The manuscript will proceed to peer review but you will be notified of the decision upon the publication date.

The manuscript will proceed to peer review, and you will be notified of the decision regarding publication once it has met all of the above criteria.


## Methods
cb.nlm.nih.gov/pubmed/FungiNomenclature/) for the identification of Fungal Nomenclature-Topical identities. To prepare the final dataset, we performed a mixed-methodology search of the PubMed database and searched for references to Fungal Nomenclature. Only articles from “Fungal Nomenclature” were selected and the remaining articles were not considered relevant for the final dataset. We excluded all articles from “Fungal Nomenclature” that could not be assigned to a domain kingdom. We considered a domain to be “topical” if its name contained more than one domain name, as opposed to “domain”, and considered this as a topological ranking when the number of topical names in the final dataset was greater than 10^4, or less than 10^4, respectively. We used this topological ranking to group organisms into two groups: “domain” and “classical”.

Bacterial and fungal Nomenclature
We searched for species that were closely related to known fungi in the Fungal Nomenclature Database. For each genus, we searched the database against the Fungal Nomenclature Database (http://fungidb.org/) for the classification of genus. The genus name, genus name, or genus name with the highest genus rank in Fungal Nomenclature was considered as the major clade.

We used the “Fungal Nomenclature” search results to assign species to the “Domain Kingdom”. For each genus, we searched the database against the Fungal Nomenclature Database (http://fungidb.org/) for the classification of genus. The domain name and classical name of the major clade were not considered for the final dataset.

We used the “Classical Nomenclature” search results to assign species to the “Domain Kingdom”. For each genus, we searched the database against the Fungal Nomenclature Database (http://fungidb.org/) for the classification of genus. The domain name and classical name of the major clade were not considered for the final dataset.


## Results
. Snyder, F.Y. Li & J.F. Lee) [37] and the possibility to identify a general pattern or set of traits that might be common to fungal and bacterial Nomenclature.

The unifying pattern of fungal Nomenclature across Domain Kingdoms was revealed by analyzing the multilocus sequence data. The UniProt3 package (Version 3.4.0) identifies a wide range of fungal Nomenclature across Domain Kingdoms. The main pattern of fungal Nomenclature across Domain Kingdoms was the hierarchical cluster of domain-level domain (DLD) structure. The clusters were most frequently identified in fungi, with the fungi being the most predominant genus. The fungal Nomenclature of the DLDs in the fungal kingdom was largely dominated by highly variable genes. The most frequent genes were those involved in DNA replication, RNA metabolism, protein biosynthesis, RNA transport, RNA helicase function, protein synthesis and cell division. In most fungi, the most frequent genes were involved in the synthesis of amino acids, vitamins, lipids, carbohydrates, and many other metabolic reactions. The most frequent genes in fungi were those involved in metabolism, transport and metabolism of amino acids, vitamins, lipids and sugars. Most of the fungal Nomenclature of the DLDs in the fungi was composed of genes involved in amino acid metabolism, protein synthesis and cell division.

The Nomenclature of Domain Kingdoms in the three fungal species was dominated by highly variable genes (Figure 2). The most frequent genes were those involved in protein synthesis, nucleic acid metabolism, protein synthesis, RNA helicase function, protein synthesis and cell division. The most frequent genes in fungi were those involved in nucleic acid metabolism, protein synthesis and cell division.

The most frequent genes in fungi were those involved in protein synthesis, nucleic acid metabolism, protein synthesis, RNA helicase function, protein synthesis and cell division. The most frequent genes in fungi were those involved in protein synthesis, nucleic acid metabolism, protein synthesis, RNA helicase function, protein synthesis, cell division, and translation. The most frequent genes in fungi were those involved in translation and protein synthesis.

The most frequent genes in fungi were those involved in protein synthesis, translation and protein synthesis. The most frequent genes in fungi were those involved in translation, protein synthesis, translation, cell division, protein synthesis and cell division.


## Discussion
A total of eight species-level domain-specific gene families were identified in fungi and one set was present in the basidiomycetous fungi (Table S1. The top eight species-level gene families were related to the high-affinity Fungal Nomenclature (Figure 1A). Two pairs of gene families, F_1 and F_2, were found in the Ascomycota and Basidiomycota phyla, respectively. The genes F_1 and F_2 were shared by species-level and basidiomycetous fungi. F_2 was the most divergent gene family, although it was only present in the basidiomycetous fungi. This may suggest that F_2 and F_1 are functionally redundant. The gene families F_1 and F_2 were similar across domains. However, gene families F_1 and F_2 were divergent across domains. Therefore, we postulate that the gene families F_1 and F_2 are different in domain evolution, gene loss, and functional redundancy.

Conclusions and Future directions
In this study, we have studied the evolution of gene families in fungi. In addition, we have examined the evolutionary evolution of fungal Nomenclature. We have identified highly divergent gene families in fungal domains. We have identified highly conserved gene families and gene clusters in fungi, such as F_1 and F_2. We have identified functional genes involved in the metabolism of plant metabolites and fungi, such as F_1 and F_2. We have identified genes involved in the biosynthesis of cell wall metabolites and fungi, such as F_1 and F_2. We have identified genes involved in the transport of small molecules, such as F_1 and F_2. We have identified genes involved in the transport of intracellular proteins, such as F_1 and F_2. We have identified genes involved in the transport of secreted proteins, such as F_1 and F_2.

The results indicate that the domain-specific gene families F_1 and F_2 play an important role in domain evolution and functional redundancy.
