# Inteins are no longer alone; introns differ in fungal interiors with species
Quaglee Dragontacos


## Abstract
We previously reported that the filamentous fungus Botrytis cinerea is able to undergo sexual development via a gene-mediated interaction with a specific physical position of the cell surface-associated transcription factor BcYps1. In the present study, we show that BcYps1 is a key player in regulating the expression of genes involved in sexual development and sporulation. BcYps1 is induced during the sexual development of B. cinerea when cells are attached to walls and walls-dependent processes are induced by environmental stress and drugs that induce expression of genes involved in stress resistance. Furthermore, stress-induced expression of BcYps1 results in sporulation and development of meiosis. These results demonstrate that BcYps1 is a crucial factor that mediates regulation of sexual development and sporulation in B. cinerea and other filamentous fungi.


## Introduction
In addition, introns are present in all eukaryotic genomes, including the genomes of animals, plants, fungi and protist.

In fungi, the introns are very often located in their substrate genes. For example, the introns in the C. neoformans mating type locus (MAT a) contain a 15-nt introns and a 20-nt introns, respectively, and some types of fungi have even reported 20-nt introns as introns in their substrates. This leads to the hypothesis that the fungal mating type has acquired the ability to ‘steal’ the non-matating type and thus to retain the tetrapolar configuration [5, 6]. In addition, the occurrence of introns in the genomes of eukaryotes is thought to be related to the function of these introns as they are involved in many important processes such as cell cycle control, mating type determination, vegetative growth and expression of virulence factors and secondary metabolite biosynthesis [7, 8].

There are several factors involved in the regulation of the mating type. These factors are diverse and all occur in the cell, including the interactions among individual genes and other proteins, cell wall components, cellular processes, transcription factors, and other protein substrates [9, 10]. Some of these factors interact with the mating type locus and are involved in the regulation of the mating type [11]. In contrast, other factors, such as DNA replication factors, are not essential for mating type regulation, as they are present in the cell but are associated with other processes, such as the maturation process and the expression of sexual or aherbore specification genes [12].

The mechanisms of the dimorphic transition from yeast to filamentous growth in the dimorphic fungi are largely controlled by two types of transcription factors, the p38 and the p65 proteins [13]. The p38-p65 transcription factors are present in the nucleus and regulate the expression of genes involved in the switch from yeast to filamentous growth [14]. The p65-dependent switch to filamentation is controlled by two transcription factors, the SWI and SWI-d. SWI is a member of the p38-interacting protein family [15].


## Methods
The remaining interiors have been replaced with genomic DN. Each genome is composed of four different introns. The first two introns correspond to gene-specific motifs, while the third and fourth introns are the structural motifs used to generate the in-frame sequence for the in-frame motif. Intergenic motifs are defined as those motifs that contain a homology to the target gene. The in-frame motifs that contain introns are unique to the fungus. Intergenic motifs are defined as those motifs that contain both a homology to the target gene and a homology to the in-frame motif.

The 20 most common introns present in the yeast genomes were taken from previous studies [12, 13]. The yeast genomic DNA sequences were aligned and used as queries to determine their homology and their similarity to the conserved domains in the yeast genome. The longest homology and the longest similarity to the conserved domains were chosen for alignment and also for comparison with the other yeast species. A total of 200 contigs of the conserved domains were retrieved from the yeast genome. The same analysis was carried out in order to validate the gene-specific motifs.

The genes are listed in Table 1. The C. albicans genome was downloaded from the NCBI (https://www.ncbi.nlm.nih.gov/genome/) and used as query to search the whole genome for conserved motifs. The C. glabrata genome was downloaded from the Sanger Institute (https://www.sanger.ac.uk/index.html). The gene sequences of C. glabrata were aligned with those of the yeast genome and the C. glabrata gene models were retrieved from the PAML (https://paml.org/). The genes were then aligned with the genomes of C. glabrata and C. glabrata wild type. The whole C. glabrata genome was used as a query to search the conserved motifs. The genes of C. glabrata were identified and compared with other yeast species.

Microscopic observation of yeast strains
The yeast strains were observed on a Nikon Eclipse TS100 microscope equipped with a DS-Ri5 camera and a DIC camera. Thirty-five representative yeast strains from the yeast species of C.


## Results
This is consistent with the apparent emergence of more diverse eukaryotic lineages [21.

Astromathermitis fusiformis, which has a highly variable fungal-specific intron, has been shown to be genetically more divergent than other fungi [8], but is not genetically more divergent than other fungi [7,9,10,14]. In our analysis, our results are consistent with the hypothesis that intron gain and loss have evolved independently.

Class-specific expression patterns are divergent in fungal eukaryotes
As eukaryotes are diverging from the common ancestor of eukaryotes and fungi, it is also likely that the intron-intron frame of eukaryotes evolved independently in a common ancestor. To test this hypothesis, we studied the expression of different eukaryotic genes using RNA-Seq. Using two sets of genes (ODiY-3.0 and SpiG-11.0) for both eukaryotes and fungi, we compared the transcriptome of eukaryotic and fungal eukaryotes using a quantitative RT-PCR approach [19]. We compared the transcriptome of eukaryotic and fungal eukaryotes using the RNA-Seq data set and found that the transcriptome of eukaryotic eukaryotes is highly similar to that of fungi. We also found that the transcriptome of fungi is highly similar to that of eukaryotes. These results indicate that the intron-intron frame of eukaryotes has evolved independently in the common ancestor of eukaryotes and fungi.

The absence of intron gain in fungal eukaryotes is consistent with the hypothesis that intron gain and loss have occurred independently in eukaryotes and fungi. Intron gain and loss occurred independently in eukaryotes and fungi. As noted by our analysis, this is the first report of an intron-intron frame in a fungal species.

Class-specific expression patterns are divergent in fungal eukaryotes
We also examined the expression patterns of different eukaryotic genes using RNA-Seq and observed the same pattern of gene expression for eukaryotic and fungal genes as reported previously [19].


## Discussion
These differences result from the gene duplication of gene loci that occurred in paralle. In a parallel evolution, the two loci have undergone recombination, resulting in the loss of gene functions (over- or under-expression). In this scenario, the “genome duplication” mechanism would produce highly repetitive sequences, such as those that are present in the <U+0394>NosA mutant and other mutants of A. fumigatus, that are over- or under-expression in strains of the <U+0394>NosA mutation. The loss of gene function in the <U+0394>NosA mutant would result in increased sensitivity to fluconazole, a well-established azole drug (32, 33). In the A. fumigatus <U+0394>NosA mutant, this would result in the loss of an extra copy of the CYP51A gene. In a parallel evolution, the second gene in the CYP51A gene would be lost in the <U+0394>NosA mutant. The loss of this gene would result in the loss of the CYP51A gene, resulting in a <U+0394>NosA mutant. This process would result in the loss of the ability to break down cell walls, resulting in a <U+0394>NosA mutant (33). The second gene in the CYP51A gene would be lost in the <U+0394>NosA mutant, resulting in the loss of fluconazole resistance genes (e.g. PEXA_3G2400 and PEXA_3G9200) (34). The loss of the ability to break down cell walls would result in the loss of fluconazole resistance genes (e.g. PEXA_3G9200 and PEXA_3G9200). This process would result in the loss of the ability to cause lethal sepsis (35). The loss of the ability to cause sepsis would result in the loss of the ability to metabolize ketoconazole (36). The loss of CYP51A gene would result in the loss of the ability to metabolize ketoconazole (D-fructose).
