# Intron Structure and Evolution
Quaglee Dragontacos


## Abstract

The Hox/Arf1-O-Flu GalAFO network comprises a series of extracellular ATPase proteins which function as mediators of the action of the enzyme. The Hox/Arf1-O-FluGalAFO network is shaped by the presence of two domains, the mid-piece domain and the short interlink domain. Using a combination of structural studies, a survey of the Hox/Arf1-O-FluGalAFO network reveals a network of nucleoplast-like structures which includes ATPase proteins, as well as ATPase-encoding genes which are unusual to our knowledge in the Hox/Arf1 families.

The Hox/Arf1-O-FluGalAFO network offers an opportunity for the Hox/Arf1-family to discover new Hox/Arf proteins, new mechanisms of action and novel evolutionary trajectories of their type. Our group has now uncovered the Hox/Arf1-subfamily of nucleoplast Hox/Aflox-like proteins which have been found in all three Hox/Aflox subfamilies.

The Hox/Arf1-subfamily of nucleoplast Hox/Aflox-like proteins has been found in all three Hox/Aflox subfamilies. The Hox/Arf1-subfamily is present in all three HoxAflox subfamilies and is present in all three HoxAflox subfamilies with an unclear origin. Possible explanations for the origin of the Hox/Arf1-subfamily are hypothesized to rest with their phylogenetic relationships amongst the eukaryotes studied, the molecular mechanisms underlying their nucleoplast formation, and the known cellular functions of these nucleoplast Hox/Aflox nucleoplast (NUP) proteins. We propose that the existence of the Hox/Arf1-subfamily in eukaryotes may have originated with the divergence of nucleoplast Hox/Aflox-like proteins that emerged after the divergence of most nucleoplast proteins.


## Introduction

The authors use the term "complexity" in a straw-cankered contex. They also use the term "demyelinating" in a straw-cankered context.

The sentence is confusing. It is confusing and needs addressing. It should be rewritten.

Minor comment:

Line 661: Correct the numbers at the end of the subsection “Figure 1. The structure of fungal genomes shows that approximately 90% of the fungal genomes have a single, non-homologous end and that other fungal genomes have two or more copies of the gene coding for these copies.” I don't recall which genes were homologous between the two graphs. Correct me if I'm wrong.

Line 684: A sentence that needs some discussion. The authors should include a table that lists the N-terminal region they used for their BLAST analysis.

Line 694: The sentence is confusing and needs to be corrected. The author should split the manuscript into two paragraphs.

The following lines need revision:

Line 692: The N-terminal region is different from other fungal regions. The authors should split the manuscript into two paragraphs.

Line 695: In Figure 1A, the authors show that the protein with the most similarity to the host gene is the fungal protein with the highest similarity to Candida albicans. Is this protein the fungal protein with the most similarity to C. glabrata? The authors should discuss this in more details in the Discussion section.

Response: In our experiments, we have chosen the fungus as the host for the present study. The yeast form was selected because it is a better substrate for BLAST. Moreover, we did not have any fungal isolate to isolate the C. glabrata-Candida albicans interaction. Therefore, in our experiments, we chose the fungus as the host because it is the most efficient host for BLAST. Moreover, we did not have any fungal isolate to isolate the C. glabrata-Candida albicans interaction. Therefore, in our experiments, we chose the fungus as the host because it is the most efficient host for BLAST. Moreover, we did not have any fungal isolate to isolate the C. glabrata-C. albicans interaction. Therefore, in our experiments, we chose the fungus as the host.


## Methods
anserina species, including the four best-performing species, were downloaded from the NCBI (SRA) database (version SRP134420) and the SwissProt database (version 2001-08-01) and used for the sequence alignmen. The alignments were conducted using MAFFT (Katoh and Standley 2013) and were edited in Geneious (Biomatters, Auckland, New Zealand) to generate a concatenated alignment. The phylogenetic tree was generated using the neighbor-joining method (Proctor and Stamatakis 2008) and the neighbor-joining tree was used to construct a 50% majority rule consensus tree with bootstrap values of 100 and a substitution model of the number of substitutions per site. The tree was visualized in FigTree v. 1.4.2 (http://tree.bio.ed.ac.uk/software/figtree/) and a version 3.0 of the software was used (http://tree.bio.ed.ac.uk/software/treeview/index.html).

Phylogenetic Analysis
The phylogenetic tree was constructed using two methods: Maximum Parsimony (MP) and Bayesian Inference (BI). MP analyses were performed using PAUP* 4.0b10 (Swofford 2002) and MrBayes 3.1.2 (Ronquist and Huelsenbeck 2003). The parameters used for MP analyses were set as follows: Markov Chain Monte Carlo (MCMC), Maximum Parsimony (MP) generations, and a sampling frequency of 10,000. The phylogenetic tree was generated with the program MEGA 5.0 (Kumar et al. 2016). The model for the substitution model was estimated by MrModeltest (Nylander 2004) with a rapid bootstrap algorithm (Rambaut and Drummond 2007). The parameters used for BI analyses were set as follows: Gamma distribution and invariant sites. The bootstrap values obtained from MrModeltest and MrBayes were set as 100 and 0.01, respectively, and the gaps in the model were filled by a Markov chain Monte Carlo (MCMC) method. The results were displayed in a tree topology and a branch-swelling rate of 0.5 for the first 10% of trees. The trees were viewed using FigTree v. 1.4.2 (http://tree.bio.ed.ac.


## Results
One of the novel retrotransposons that is most often associated with in plants (. Weinberg 2002) is the fungal retrotransposon L56A (Y.L. Zhang et al. 2002). The L56A-like retrotransposon L56A was shown to have a strongly conserved LTR retrotransposon domain (Zhang et al. 2002) and the two L56A-like retrotransposon sequences have been shown to be identical by comparative structural analysis (Matsushima et al. 1997). The latter two sequences are located on scaffold 20 (see Fig S3 in the supplemental material) and all three of the three L56A-like retrotransposons are highly divergent (Fig 1, Fig S3). Thus, the L56A-like retrotransposons are most likely derived from the same source as the L56A-like retrotransposons. However, the two sequences that are most often associated with in plants and animals are also highly divergent (Fig 1, Fig S3).

The loci with the highest number of L56A-like retrotransposons are on scaffold 1 (Fig 1, Fig S3). The four L56A-like retrotransposons that are most frequently associated with in plants (O. Salgado et al. 2011) are located on scaffold 1 and scaffold 3. The L56A-like retrotransposons that are most frequently associated with in animals (H. Croll et al. 2015) are located on scaffold 4. These L56A-like retrotransposons have been shown to be located on scaffold 5 (see Fig S3 in the supplemental material) and all four are located on scaffolds 1, 3, and 4 (see Fig S3).

A previous study suggested that a recent phylogenetic tree might reflect the diversification of the L56A-like retrotransposons (Korn et al. 1999; Mantel et al. 1999) and that the L56A-like retrotransposons that are most frequently associated with in plants (N. Weinberg 2002) may have been lost independently. Based on our phylogenetic tree, the L56A-like retrotransposons that are most frequently associated with in plants may have been lost independently.


## Discussion
NTU-1, the sexual state of which is not know. However, this species is very similar to the recently described species Spizellomycetes sp. NAT-1, a species closely related to the same as shown above (GenBank accession number CC027475) (Fig 4B). The sexual state of Spizellomycetes sp. NTU-1 has been previously described by Young and colleagues [28] and was confirmed in the present study by the recombination of two loci (Fig 4C). The recombination of the locus AT_1 and AT_2 and the tandem duplication of the two loci AT_1 and AT_2, respectively, and the duplication of the locus AT_2 and AT_3, has been reported in Spizellomycetes [29], but the recombination of the locus AT_1 and AT_2 is still not seen (Fig 4C). The molecular analysis of Spizellomycetes shows that the recombination events that occur in the two loci (and the locus AT_1 and AT_2) are identical to those that occur in the wild-type strain (Fig 4C). We hypothesize that the recombination of the locus AT_1 and AT_2 is a result of the recombination between the two loci (and the locus AT_3) as well as the recombination of the locus AT_1 and AT_2.

The molecular analysis of the recombination event in the two loci revealed that the two loci are functionally equivalent and that the recombination between them is due to the expression of the genes that are involved in the mating type switching (MAT1-2) in C. neoformans (Fig 5). This result suggests that the two loci are functionally interchangeable and that the recombination between them is a result of recombination between the two loci. In contrast to the other two species of the Spizellomycetes, which have a single mating type locus, S. reilianum has two loci (at the same locus, which is MAT1-1 and MAT2-1).
