# Leishmania amazonensis: The Mating Type Locus of the Capsular Polysaccharide CapsRX14
Quaglee Dragontacos


## Abstract
The foliar mycobiome in the Dannarchaeota (D.Lgs. 152/2012) remains a major topic of research in this area. In this study, we compared the fungal and bacterial community structures of the root of two different dannaceous dannoan species, D. levadasii and D. phyllodiorum. We also reconstructed the correlation between mycobiome and root mycobiome using fungal-secreted mycobiome as a proxy. Among the 75 fungi detected, 58 genera were identified, and the relative abundances of the tested genera varied from 0.4 to 1.1%, depending on the species. Fungal communities were dominated by the genera D. levadasii and D. phyllodiorum, which were more closely related than D. rischio-like genera such as D. suttoniae and D. chamaesus. Furthermore, all the strains of D. levadasii and D. phyllodiorum were identified as belonging to genera such as D. argillosum, D. acqua, D. astrakis, D. tiliae, D. labyrinthorum, D. ovina, D. reimeri, D. verrucosa, D.


## Introduction
The difference in morphology between the two types of mating projections has been previously described (Cui et a., 2012). Therefore, the pheromone-mediated development is not only a function of the morphology but also a function of the type locus. However, a number of studies have demonstrated that the two mating types differ by as much as 50%, a significant number for any other single loci (Cui et al., 2012). Moreover, the pheromone-binding proteins of the Mating Type Locus (MtBPs) are highly expressed in both mating types, suggesting that they may share a similar function (Cui et al., 2012).

The purpose of this study was to determine the population structure of the Mating Type Locus of Cryptococcus. The C. neoformans and C. gattii mating type loci are closely related e.g., Cryptococcus neoformans and Cryptococcus gattii have a single mating type, VNI (or VNIV) (Epp et al., 2012). The VNI and the VNIV loci are found in both clinical isolates of C. neoformans and environmental isolates of C. gattii (Bento et al., 2006; Bhatt et al., 2012). The C. gattii mating type loci are commonly distributed in nature, including tropical and subtropical regions, and are highly dynamic and host-specific (Cui et al., 2012). Several studies have demonstrated that the number of MtBPs and the presence of the mating type locus in their genomes are highly related (Cui et al., 2012). MtBPs are thought to play an important role in virulence of C. gattii by promoting mating and promoting the spread of the pathogen (Cui et al., 2012). In contrast, MtBPs are thought to be essential for the virulence of C. neoformans by promoting mating and promoting the dissemination of the pathogen from the lungs (Cui et al., 2012). MtBPs are more virulent than the corresponding environmental strains, as indicated by greater survival time in the mouse lungs (Cui et al., 2012). The three MtBPs are found in the genomes of both C. neoformans and C.


## Methods
Joerg Knudsen (Universidade Federal de São Paulo, São Paulo, Brazil.

The mating type of CapsRX14 was determined using the PCR-based method. Briefly, the mating type of CapsRX14 was amplified from genomic DNA of C. capsulatum strain YJM-S (P_scRX14) using a forward primer pair of CapsRX14-1 and CapsRX14-2 (the mating type of CapsRX14 is unknown), and the reverse primer pair CapsRX14-3 and CapsRX14-4 (the mating type of CapsRX14 is unknown). The PCR was performed with the following thermal profile: 95 °C for 1 min, followed by 35 cycles of 95 °C for 30 s, annealing temperature of 55 °C for 30 s, extension time of 60 s and a final extension of 10 min at 72 °C. The PCR product was purified by using an AxyPrep PCR Purification Kit (Axygen, Jiangsu, China). The purified PCR product was sequenced in both directions by the BGI (Beijing Institute of Bioscience and Biology, Shanghai, China).

The protein profile of CapsRX14 was determined by the Bioneer MALDI-TOF-TOF MS/MS system (Bioneer, Shanghai, China). Briefly, the MALDI-TOF-TOF MS/MS method is used to identify the molecular size and structure of the a-type of CapsRX14, which is a sensitive mass spectral detection and analysis tool for C. albicans. The m/z data were obtained using the MALDI-TOF MS/MS (Bioneer) method, in which the differential fragmentation (DID) was performed using the Proteal MS/MS (Bioneer). The DID spectrum obtained was converted into a matrix-file format, using the Molecular Dimensions tool (MDS) module, to which the MDS model and data types of the MALDI-TOF-TOF MS/MS were used. The molecular mass of CapsRX14 was obtained by the Molecular Dynamics Analysis module, and the MDS matrix file was converted to a table format using Mascot search engine (Matrix Science, London, UK).

The sequences of CapsRX14, which were aligned with the sequences of C.


## Results
The molecular formula of the ITS and the LSU rRNA gene sequences of the Capsular Polysaccharide CapsRX14-D are given in Table . The molecular weights of CapsRX14-D were found to be 18.6 kDa (from the nrLSU sequence) and 12.4 kDa (from the rpb2 sequence). The molecular weights of CapsRX14-D were found to be 22.9 kDa (from the nrSSU sequence) and 19.5 kDa (from the rpb2 sequence). The molecular weights of CapsRX14-D were found to be 22.6 kDa (from the nrSSU sequence) and 21.3 kDa (from the rpb2 sequence). The identity of the sequence of the R. mucilaginosa CapsRX14-D was confirmed by using the maximum parsimony method (FISH-MLP) and the maximum parsimony tree analysis (TOL-MLP).

The Maximum Parsimony Parsimony (MP) and Maximum MP trees were constructed for CapsRX14-D and CapsRX14-D and CapsRX14-D, respectively. The MP tree analyses revealed a clear separation of CapsRX14-D from CapsRX14-D. The MP tree analyses revealed a separation of CapsRX14-D from CapsRX14-D. The Bayesian analysis of the sequences revealed that CapsRX14-D was most closely related to the Capsular Polysaccharide CapsRX14-D (Fig 1), whereas the ITS and LSU rRNA gene sequences of CapsRX14-D were most closely related to the Capsular Polysaccharide CapsRX14-D. The analysis of the combined dataset revealed that the Capsular Polysaccharide CapsRX14-D and CapsRX14-D had very similar topologies (Fig 1). The ML analysis of the MLP and MP tree revealed that CapsRX14-D and CapsRX14-D had similar topologies (Fig 1).


## Discussion
A1 and CapsRX14A2 (GenBank: AABF2009.1) [49]. Both strains share the mating type with the Mating type of the mating type gene in Cryptococcus neoformans (CNAG_03323) and that of the mating type gene in Cryptococcus gattii (CNAG_03329) [50]. CapsRX14A1 is unique among the CapsRX14A1 strains in that it is the only strain with all mating types and also has a mating type that is genetically distinct from the mating type of the type gene in Cryptococcus neoformans (CNAG_03323) [50]. The mating type of CapsRX14A1 has not yet been determined because this strain does not produce conidia. The mating type of Cryptococcus gattii (CNAG_03323) has not yet been determined because this strain produces no conidia [51]. The mating type of Cryptococcus neoformans (CNAG_03323) has not yet been determined because this strain does not produce conidia [51].

The present study provided further evidence that the mating type of the mating type gene of Cryptococcus neoformans (CNAG_03323) is unique and may be a member of the mating type locus in C. neoformans. Further studies are required to understand the mating type locus in Cryptococcus.

The mating type locus of C. neoformans (CNAG_03323) is similar to the mating type gene of the mating type locus in the closely related species C. gattii (GenBank: XP_0071925.1). The mating type locus of Cryptococcus neoformans (CNAG_03323) encodes a C-terminal gene with a consensus sequence with the conserved Gly/Gly motif and contains a large number of genes that encode proteins with high similarity to homologs in other basidiomycetes. Cryptococcus neoformans has three mating type loci in addition to the mating type locus. The C-terminal gene is unique and appears to be linked to the mating type locus in two distinct mechanisms. First, it encodes a serine/threonine protein-coding gene, which is encoded by a tandem duplicated gene arrangement in a serine/threonine pair.
