# Long transposon-rich centromeres in an oomycete reveal divergence of centromere features in the evolution of centromere domains
Quaglee Dragontacos


## Abstract
The ability of pathogens to develop novel virulence factors and avoid vaccination is an important virulence determinant of the human fungal pathogen Cryptococcus neoformans, which is capable of evolving resistance to multiple host-imposed stresses. One of the unique virulence factors of this microorganism is its ability to survive the immune response of the host. This ability results in the attenuation of the immune response and its subsequent loss of virulence. This immune suppression is most critical for its pathogenesis. The innate immune system has evolved an intricate set of strategies to combat infection. This review will focus on the strategies employed by fungal pathogens to evade or avoid immune responses and the strategies employed by human fungal pathogens to infect and disseminate throughout the world. The strategies employed by fungal pathogens to evade or avoid immune recognition are at the core of complex multidrug-drug interactions with mammalian hosts.


## Introduction
The genus of the Trichocomaceae was established by O’Donnell (1995) to accommodate the genera Tachydera, T.chalybdis, T.cereinus, T.dioleus, T.dioleus, T.dioleus, T.pais, T.pyrena, T.sakravithii and T.sakravithii (van Niekerk et al. 2016). The family of Trichocomaceae was further expanded by O’Donnell (1994), who placed T.chalybdis in Trichocomaceae (Hawksworth et al. 2009).

O’Donnell (1995) placed T.corymbalis, T.paracasei, T.sakravithii and T.montagnees in Trichocomaceae, but provided no detailed description of Trichocomaceae. The family was further expanded by O’Donnell (1994) who placed T.montagnees in Trichocomaceae. The family was further expanded by O’Donnell (1994) who placed T.montagnea, T.corymbis, T.chalybdis, T.montagnea, T.sakravithii and T.montagnees in Trichocomaceae. The family was further expanded by O’Donnell (1994) who placed T.cypria, T.sakravithii and T.montagnees in Trichocomaceae.

The genus Trichocomus is typified by T.stilbomycetes and T.muttii in the family Trichocomaceae.


## Methods
Primer sequences were modified by cutting out the exon and exon-intron sequences in the forward primer and the reverse primer, respectivel. For each centromere (a-tubulin, ß-tubulin, and ß-tubulin), an eukaryotic centromere was included in the comparison. The generated sequence comparisons were performed using the software program program MEGA version 7.0 (Tamura et al. 2011).

DNA extraction, PCR, and DNA sequencing
Total DNA was extracted from freeze-dried mycelia grown on PDA, using the Power Soil DNA Isolation Kit (Mo Bio Laboratories, Carlsbad, CA, USA) following the manufacturer's instructions. The genomic DNA was extracted using the QIAprep™ Spin Miniprep Kit (Qiagen, Hilden, Germany), according to the manufacturer's recommendations. The primers used in this study are listed in Table 1.

PCR was performed using the same set of primers as in the previous analysis. The primer sequences are listed in Table 2. PCR products were purified using the QIAquick PCR Purification Kit (Qiagen, Hilden, Germany) according to the manufacturer's recommendations. The purified PCR product was sent to the DNA platform at the Centre for Genomics and Bioinformatics (CGCB, São Paulo, Brazil). The products were sequenced using the BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems, Foster City, CA, USA) and the ABI Prism BigDye terminator v. 3.1. ABI v. 6.0 (Applied Biosystems, Foster City, CA, USA).

The extracted DNA was stored in -80°C, and the amplification products were separated on 1% (w/v) agarose gel using 1X TAE buffer (Sigma-Aldrich, St. Louis, MO, USA). The gel was run at 80 V for 30 min, and the banding pattern was observed under ultraviolet light.


## Results
In addition, we found that centromere domains in the centromere of the fungal Dictyostelium discoideum were similar to those found in the centromeres of the Saccharomyces cerevisiae . fumigatus (Fig 1).

Centromere domains in the centromere of the two distantly related fungi F. graminearum and U. maydis (Fig 1) were homologous to those in the centromeres of the Saccharomyces cerevisiae A. fumigatus and U. maydis (Fig 1), but different from those in the centromere of the fungal Dictyostelium discoideum. In F. graminearum and U. maydis centromeres, the centromere domains were homologous to those in the centromere of the two distantly related fungi (Fig 1). In contrast, in the centromeres of the two distantly related fungi, centromeres in the centromere of the two distantly related fungi were homologous to those in the centromere of the U. maydis species. These results suggest that centromeres in the fungal genome may be homologous to the centromeres in the centromere of the two distantly related fungi, F. graminearum and U. maydis.

In the centromeres of the two fungi, the centromeres of the centromeres of the two distantly related fungi were homologous to those in the centromeres of the U. maydis species, F. graminearum and U. maydis. These results suggest that the centromeres in the centromere of the two fungi are homologous to those in the centromeres of the U. maydis species. In addition, the centromere of the two fungi also contained a long endosome (Fig 1).

Centromeres in the centromere of the two fungi have been shown to be involved in binding of various extracellular matrix components [6-8]. In addition, the centromere of the U. maydis species has a long endosome [35].


## Discussion
The oomycete genomes from dually dividing organisms, such as yeasts and filamentous fungi, are richer in transposable elements than the genomes from eukaryotes and plants [34,35. The genomic DNA of oomycetes is rich in telomerase and telomerase-like elements. This DNA diversity is likely to be the result of a co-evolutionary radiation of centromere-like elements between the eukaryotic and plant genomes. A possible model of this co-linearity would be that a protein-coding gene system is more complex in oomycetes than in yeasts. In oomycetes, the genes encoding telomerase and telomerase-like proteins are clustered together in the genome, which in oomycetes, is a condition that may be different from that in yeasts. The silencing of a gene encoding a TIL retrotransposon in a human recipient strain has no effect on the centromere-like elements and telomerase-like elements. This is because the genes are deleted in the recipient strain by the siRNA-like protein. The recipient strain produces mature telomerase-like elements and telomerase-like elements. However, the presence of telomerase-like elements in the recipient strain requires that the silencing factor is silenced in the recipient strain. This may explain why the recipient strain does not produce telomerase-like elements and telomerase-like elements in the recipient strain. The silencing factor was silent in the recipient strain, and the silencing factor was also silent in the recipient strain. It is possible that the silencing factor in the recipient strain is silenced by silencing factors in the recipient strain. The silencing factor was silenced in the recipient strain, and the silencing factor was silenced in the recipient strain. It is possible that the silencing factor in the recipient strain is silenced by the silencing factor in the recipient strain. The silencing factor was silenced in the recipient strain, and the silencing factor was silenced in the recipient strain. In the Oomycetes, centromeres are conserved in the genomes of many species. They are localized in the genome, and are also found in many plant genomes [36-38]. These centromere-like elements appear to be involved in gene expression regulation in Oomycetes.

Gene content analysis
The genome of O.
