# Modulating the Fungal Growth Responsive Properties of Porphyromonas gingivalis Strain 230 Preventable Diseases in Graphididae (Annonaceae)
Quaglee Dragontacos


## Abstract
Due to the wide usage of ftp and ftp2 as the conduits of fungal eukaryotes, knowledge about their mechanisms of eukaryotic adaptation and regulation is being lost. The availability of the vast majority of genomes of fungi and the vast majority of eukaryotic genomes are in EST databases, making them an logical prerequisite for the analysis of their genome-wide functions. Here we present an analysis of EST database evolution, gene loss, and eukaryotic genome evolution. We infer that the number of genes required for maintaining viability in the fungi increased with time as revealed in genome architecture and protein diversity analyses. We also provide a new estimate of the ancestral dosage of cell wall degrading enzymes (CWD), which is the most common eukaryotic enzymes identified in fungi. We find that the number of copies of genes encoding mobile elements (like RNA helicases and trehalose degrading enzymes) and genes encoding non-homologous end joining (eukaryotic short conserved motifs) were all downregulated in the non-homologous end joining group. Furthermore, we find that the number of genes required for maintenance of viability in the non-homologous end joining group was also increased, and the number of genes required for pathogenicity were all increased. Our results suggest that the evolution of the eukaryotic genome is largely due to increased eukaryotic gene activity and, therefore, point to fungal origins.


## Introduction
Invasive fungal infections (IFIs) are a significant cause of morbidity and mortality in immunocompromised patients. Many of these infections are caused by Candida species, Cryptococcus neoformans, and Aspergillus species [1]. However, despite a considerable increase in the incidence of IFIs, the incidence remains low.


## Methods
DNA extraction and PCR
DNA was isolated from the fungal mycelium grown on PDA (PDA containing 0.9% wt/vol yeast extract, 0.5% wt/vol peptone, and 2% wt/vol glucose) at 30°C. DNA was extracted by using the Wizard® Genomic DNA Purification Kit (Promega, Madison, USA) according to the manufacturer's instructions. For PCR, the DNA extractions were performed as described previously (Merritt et al., 2014) with minor modifications. Briefly, DNA from mycelium and mycelium-exposed mycelium was recovered from the surface of the PDA plate, and then extracted as described above. DNA was eluted with sterile distilled water, and then purified using a QIAquick PCR purification kit (Qiagen, Hilden, Germany). Purified DNA was stored at -20°C until use.

PCR was performed using the GeneAmp^® PCR Master Mix (Applied Biosystems, Foster City, USA) according to the manufacturer's instructions.


## Results
gingivalis strain that was identified as a potential pathogen of Graphididae by the same group of pathogenic fungi as . gingivalis, were examined. As a consequence, we did not observe any significant differences in the virulence of the N. gingivalis strain (Table 1).

The virulence of the N. gingivalis strain was shown to be dependent on the type of host, as shown in Fig 1. The pathogenic factors of N. gingivalis were determined by evaluating the growth of the isolate on solid media (Fig 2). The growth of the N. gingivalis strain was fully inhibited by the combination of the three factors. The growth of the N. gingivalis strain was inhibited by the combination of the three factors, although this is not obvious by the color of the agar. In contrast, the growth of the N. gingivalis strain was completely inhibited by the combination of the three factors. The combination of the three factors inhibited the growth of N. gingivalis strain in a concentration-dependent manner. In contrast, the growth of the N. gingivalis strain was fully inhibited by the combination of the three factors.

In our previous work, the growth of a Graphididae strain in solid media was inhibited by the combination of the three factors. The growth of the N. gingivalis strain was inhibited by the combination of the three factors, but not by the combination of the three factors. The growth of the N. gingivalis strain was also inhibited by the combination of the three factors, but not by the combination of the three factors. In this study, the growth of N. gingivalis strain was inhibited by the combination of the three factors, but not by the combination of the three factors.

The virulence of the N. gingivalis strain was tested in the test animal. The N. gingivalis strain was found to be less virulent than the S. gingivalis strain and the S. gingivalis strain was less virulent than the N. gingivalis strain.

The survival rate of the N. gingivalis strain in the test animal was examined. The survival rate of the N. gingivalis strain in the test animal was analyzed. The N.


## Discussion
gingivalis Strain 230, which was originally identified as a genus of ericoid fungi, shares many properties with . siamensis and P. jirovecii, which is the type species of ericoid fungi in Graphididae (Annonaceae). We also found that P. gingivalis Strain 230 could inhibit the growth of ericoid fungi, which is also a feature of P. siamensis. In addition, we found that P. gingivalis Strain 230 could also inhibit the growth of non-ribosomal peptide synthase (NRPS)-negative bacteria. These features may be attributed to the cell wall component Asn1, which is involved in cell wall stability, but also the ability to form cellulose, which plays an important role in the cell wall of fungi [48]. Moreover, the virulence of S. aureus and P. aeruginosa is attributed to the cellulose content [49].

Based on these observations, we also obtained a further assessment of the ability of P. gingivalis Strain 230 to inhibit the growth of ericoid fungi. In our study, P. gingivalis Strain 230 was found to be able to inhibit the growth of P. aeruginosa, P. guilliermondii, P. tubingensis, and P. curcasum in vitro, which is in line with previous studies that also observed that ericoid fungi were inhibited by P. gingivalis strains [30, 32, 33]. Furthermore, we found that P. gingivalis Strain 230 could inhibit the growth of P. aeruginosa, P. guilliermondii, P. tubingensis, and P. curcasum in vivo, which is in line with previous studies that also observed that ericoid fungi were inhibited by P. gingivalis strains [30, 32, 33, 39].

We also found that P. gingivalis Strain 230 could inhibit the growth of P. aeruginosa, P. guilliermondii, P. tubingensis, and P. curcasum in vitro, which is in line with previous studies that also observed that ericoid fungi were inhibited by P. gingivalis strains [30, 32, 33, 39]. In addition, we also found that P.
