# MoSET1 (Histone H3K27), a Protein Membrane Receptor of the Extracellular Vesicles Flagellarly Recognized by Oxidative and Kinase Production Probes
Quaglee Dragontacos


## Abstract
Candida albicans is an opportunistic pathogen of humans that is found as a filamentous hyphae within the human body. Infection with this fungal pathogen is usually due to inhalation or autoimmunity. Here we review the epidemiology, genetics, risk factors, and antifungal therapies that have contributed to the emergence of Candida as a fungal pathogen in humans. We focus on the development of antifungal therapies that are either well-tolerated and/or have proven to be effective in treating candidiasis in particular.


## Introduction
 1998), as well as proteins containing the catalytic catalytic domain (Carpenter et a., 1997; Toomer et al., 1997; O’Driscoll et al., 1998).

As described by the authors, the oxidized form of P-type P-carboxylates, P-carboxylates, is responsible for oxidative damage, and their toxic consequences. As the oxidation proceeds, the oxidative group of the P-type P-carboxylates is reduced to the free form, the hydroxylated form, and then oxidized by the hydroxyl group of the P-type P-carboxylates. The hydroxylation of P-type P-carboxylates is catalyzed by N-acetyl-ß-d-glucosamine (GlcNAc) (Bartram et al., 2009). GlcNAc is a bimolecular protein with a 24-membered crystal structure and an extended N-terminal sequence of ~60 amino acids (Kaur et al., 2002). In the membrane, GlcNAc interacts with two proteins of the glycosaminoglycan glycosaminoglycan-1 (GAG1) and a glycosaminoglycan-2 (GAG2) superfamily, that have different conformations depending on the cell surface (Jung et al., 2013). The proposed role of GlcNAc in fungal cell signaling is as a signal molecule, which may mediate cell differentiation, infection, and response to environmental stresses (Bassetti and Schumacher, 2010; Wang et al., 2011).

In a recent study, the mechanism of P-type P-carboxylation is based on the inhibition of the enzyme, p-type P-carboxylate reductase (EC 2.6.1.1), which targets GlcNAc by reducing it of its cysteine residue (Wang et al., 2011). Based on the results of that study, the inhibition of P-type P-carboxylate reductase is proposed to be the essential step for the degradation of GlcNAc to produce P-type P-carboxylates (Wang et al., 2011). In this study, the mechanism of the inhibition of P-type P-carboxylate reductase by P-type P-carboxylates is described.


## Methods

The unique fluorescent protein-LIMO sequence was then used to identify fungal proteins that were present in the internal transcribed spacer (ITS) region of the bacterial ribosomal RNA operon and to identify the interspecific and interspecific fragment of the protein-located fragment. The nuclear ribosomal RNA operon and the mRNA operon of the internal transcribed spacer (ITS) region were considered to be the same as the mRNA operon and the corresponding sequence was used to search for interspecific and interspecific fragment matching in the above mentioned databases. The fungal ITS region was also considered as the internal transcribed spacer region and the corresponding sequence was used to search for the matching sequences in the above mentioned databases. The LIMO sequence was then used to identify proteins that were present in the highly variable regions of the fungi ribosomal RNA operon.

Protein-coding genes in C. albicans, C. parapsilosis, C. tropicalis, and C. glabrata
The protein coding genes of C. albicans, C. parapsilosis, C. glabrata, and C. tropicalis were retrieved from the literature. The sequences of these genes were used to define the structural models of the protein-coding genes of C. albicans, C. parapsilosis, and C. glabrata using the BLAST program.

Protein-coding genes of C. albicans, C. parapsilosis, C. tropicalis, and C. glabrata were found in the three different subclasses of Candida species, namely Candida albicans: C. albicans: C. tropicalis, C. glabrata, and C. albicans: C. glabrata. The following were the primers used to predict the structural models of the proteins of C. albicans, C. parapsilosis, C. albicans, and C.


## Results
The number of EVs in the . elegans genome was higher than that in S. cerevisiae (Fig 3C).

Evolutionary Differentiation in the C. elegans Genome
The C. elegans genome was divided into four sub-genomes with a total of 12 sub-cellular populations, including eight in the C. albicans-H99 background and three in the C. albicans-URA3 strain. The C. elegans-URA3 strain showed a strong reduction in its number of EV populations as compared to the C. albicans-URA3 strain (Fig 4A).

The C. elegans-URA3 strain showed a similar pattern of chromosome rearrangements as the S. cerevisiae strain (Fig 4A). In addition, a large number of T-DNA insertions were found in the genome of the C. albicans-URA3 strain, with two T-DNA insertions in the left arm and one in the right arm (Fig 4B). These included one insert in the right arm and three in the left arm of the T-DNA arm of the chromosome (Fig 4B). Additionally, one T-DNA insertion in the right arm of the chromosome was found in the right arm of the T-DNA arm (Fig 4B). These two T-DNA insertions were also found in the left arm of the chromosome and the two T-DNA insertions in the right arm of the chromosome were found in the left arm of the T-DNA arm (Fig 4B).

To further investigate the evolutionary history of EV population in C. elegans, we performed an in silico analysis of the genome sequence. First, we used a triplication marker to generate an allele by homologous recombination in the left arm of the chromosome. Two independent alleles of the same marker were used to identify each allele. After purification, DNA was cloned from each allele and plasmids were transformed into S. cerevisiae-URA3 strain (Fig 5). The resulting strains were also transformed into the S. cerevisiae-URA3 strain for the generation of the EV populations in the C. elegans genome.

The S. cerevisiae-URA3 strain was more closely related to the C. albicans-URA3 strain than the C. albicans-URA3 strain (Fig 5). The S.


## Discussion
The first two genes are related to the presence of a phosphate-binding domain and the activity of a kinase (CCR2) (22. The third gene is related to the intracellular protein location and the activity of the cytosolic calcium-dependent protein kinase (CDRK1) (23). The cccl2 gene is related to the putative Ca-Cch1 calcium transporters (CDRK1) and the Cch1-Cch1 Ca^2+ channel transporters (CCH1) (24). The CCH1 and CCH1-Cch1 Ca^2+ channels in the extracellular vesicles are related to calcium transporters (CaVs) (25).

In addition to the three genes, the gene family of the gene cluster was expanded to include other proteins involved in cytosolic calcium signaling and the S-Cch1 calcium transporters (18). The gene cluster consists of genes encoding proteins involved in cytosolic calcium signaling (CDRK1, SAC1, and SAC2) (18, 25, 26). In the model of infection, the cytosolic calcium transporters are activated by the transcription factor HapX, which is upstream of the Cch1/Cch1-Cch1 Ca^2+ channel (18). In addition, the SAC1 protein is downstream of the SAC1-Cch1 Ca^2+ channel (18, 26).

The SAC1 and SAC2 proteins are involved in the calcium signaling pathway, respectively (18). The SAC1 and SAC2 proteins were identified in this study, and the genes involved in the calcium signaling pathway were clustered together with the genes involved in the Cch1/Cch1-Cch1 calcium transporters. The SAC1 protein is a homologue of the S. cerevisiae Cch1 gene, which encodes the calcium transporters Cch1 and Cch2 (18). The SAC2 protein encodes the S. cerevisiae Cch2 gene, which encodes the calcium transporters Cch2 and Cch3 (18).
