# Multilocus DNA sequencing of the fungus Fusarium graminearum provides abundant examples of geographic populations of F. graminearum
Quaglee Dragontacos


## Abstract
Aspergillus fumigatus is a saprophytic fungus that can cause life-threatening infections in immunocompromised individuals. The lack of efficient antifungals in the era of the modern antifungals makes this pathogen a serious threat to human health. Additionally, the development of new therapeutic strategies to treat invasive fungal infections is an increasingly important clinical challenge. Development of new antifungals to treat Aspergillus infections is currently in its infancy. Thus, the development of an effective and safe new antifungal strategy to treat invasive fungal infections is in its infancy. This review discusses the development of novel antifungals to treat Aspergillus infections and the challenges of developing an effective and safe new antifungal strategy to treat these infections.


## Introduction
Due to the spread of fungal diseases, the incidence of fungal infections has increased significantly in the past 20 years. Cryptococcus neoformans (C. neoformans), an opportunistic fungal pathogen, is the leading cause of fungal meningoencephalitis worldwide [1], [2]. Although C. neoformans causes pulmonary disease, cryptococcosis is a systemic fungal infection in immunocompromised patients. In the context of immune compromise, the yeast cells of the respiratory tract and brain can develop into cryptococcal pneumonia, which is often fatal. Cryptococcosis has been reported to cause up to 25% of deaths from fungal infections worldwide, including the United States, and causes approximately one-third of AIDS-related deaths [3], [4].

Despite the recognition of C. neoformans as an emerging opportunistic fungal pathogen, the majority of patients with cryptococcosis are never diagnosed and treated. Therefore, new treatments are urgently needed. The conventional antifungal agents, such as the azoles, polyenes, and echinocandins, are effective only in a few cases and are ineffective in many cases [5], [6]. The development of new antifungals is mainly based on their significant drug-drug interactions, toxicity, and side effects. However, the emergence of new drug targets that are not relevant to the problem of fungal infections are important and need to be optimized.


## Methods
graminearu.

Genomic DNA was extracted from fungal cells by using a CTAB-based method. Genomic DNA was extracted using a modified CTAB protocol for fungi [7]. The method was developed by Prof. Akio Shimogawa, University of Tokyo. The quantity and purity of the extracted DNA were determined using the NanoDrop 1000 spectrophotometer (Thermo Scientific, Waltham, MA, USA). The extracted DNA samples were analyzed on a Bio-Rad CFX 3000 Touch (Bio-Rad, Hercules, CA, USA) system. A standard curve was developed by adding 1 µM (w/v) and 2 µM (d/w/v) dNTPs to the extracted DNA samples. The standard curve for quantification of DNA abundance was obtained by adding 1 µM and 2 µM dNTPs to the DNA samples. The DNA samples were analyzed on a Bio-Rad CFX 3000 Touch (Bio-Rad, Hercules, CA, USA) system. The standard curve for quantification of DNA amount was obtained by adding 1 µM and 2 µM dNTPs to the DNA samples.

Fungal strains were grown on potato dextrose agar (PDA, Oxoid, UK) for 7 days. Then, the mycelium was transferred to 1 mL Erlenmeyer flasks and mixed with 1 mL of 0.5% (w/v) sodium carbonate in phosphate buffer (pH 5.0) at a ratio of 1:10 (v/v) to 1:100 (v/v) for 24 h. The mycelium was then transferred to a 1 mL Erlenmeyer flasks and was then thoroughly mixed with 1 mL of 0.5% (w/v) sodium carbonate in phosphate buffer at a ratio of 1:10 (v/v) to 1:100 (v/v) for 24 h. The mycelium was then transferred to a 1 mL Erlenmeyer flasks and was then thoroughly mixed with 1 mL of 0.5% (w/v) sodium carbonate in phosphate buffer at a ratio of 1:10 (v/v) to 1:100 (v/v) for 24 h.


## Results

. graminearum is Fusarium graminearum-like
To determine the origin of F. graminearum, the Fusarium graminearum-like, a fungal mycobiome, was isolated from F. graminearum in the Netherlands. The four sequences of the ITS region of the GenBank sequence of the F. graminearum-like were deposited in GenBank under the accession numbers KJ512777, KJ512777, KJ512777, KJ512777 and KJ512777. The four sequences of the LSU region of the GenBank sequence of the F. graminearum-like were deposited in GenBank under the accession numbers KJ512777, KJ512777, KJ512777, KJ512777, KJ512777, KJ512777 and KJ512777. The four sequences of the ITS region of the GenBank sequence of the F.


## Discussion
A recent survey of the United States identified 27 geographic populations of Fusarium species, but they were not reported to be present in the area [26. The presence of Fusarium species in the environment has been observed in the state of California, but the frequency of this species has not been reported to date in the United States. Fusarium graminearum was reported to be the most common disease in Arizona, New Mexico, and Alabama [27]. In recent years, environmental studies have revealed that the number of species in Fusarium species has increased [28], and the results of this study were consistent with these studies.

The following studies are included in the discussion of the currently recognized Fusarium species.

Genome-wide association analysis
The identification of fusariae as a new species in Fusarium species complex revealed that it has been isolated from a wide range of environmental samples in the USA, Mexico, and Canada. The fungus was isolated from two sites in the USA, both in Mexico City. It is known as a synonym of Fusarium species, Fusarium solani, and has been described as a new species in South America [29].

The isolation of Fusarium solani from the same sampling sites, in the same location, in the same location, and the subsequent identification of Fusarium spp. in this study suggests that this fungus is now a distinct species from those of Fusarium species that are more abundant in other locations. The results of this study indicate that the isolates of Fusarium spp. are phylogenetically similar to those from Fusarium spp. from other geographic locations.

The studies of the Fusarium species are summarized in Table 2. The majority of the isolates in the current study were obtained from a single location, although the numbers of isolates that were obtained from two locations were statistically significant. The number of isolates in this study was not statistically significant because the samples from two locations were not collected twice. The isolates from the other two locations were obtained from different sites, and the number of isolates obtained from the two locations was statistically significant. However, the number of isolates obtained from the second location was statistically significant and these were obtained from two locations in two weeks. The isolates obtained from the third location were obtained from the same location twice.
