# Natural Killer T Cells and Abiotic Factors of Pathogenic and Non-Pathogenic Yeasts
Quaglee Dragontacos


## Abstract
The combination of systemic (colony) disease, fungal infections, and immune reconstitution inflammatory syndrome (IRIS) can lead to major complications in the case of patients on immunosuppressants, such as solid tumors and infectious agents. As such, it is necessary to develop strategies to increase the survival of these pathogens and to develop immunomodulatory drugs against the colonizers of this complication. However, the use of adjunctive antifungal therapy in the treatment of fungal colonizers has been rarely reported. The aim of this study was to investigate the possibility of the adjunctive use of terbinafine in the treatment of severe fungal colonizers, particularly Cryptococcus neoformans, in a non-HIV-bearing patient.

This was a single center study, conducted in our department. Fifty-two patients with HIV and 58 HIV-negative controls were recruited. Nine patients were categorized as severe and one as non-severe fungal colonizers. The indications for adjunctive therapy of terbinafine in these patients have not been described previously.

We report a case of a 36-year-old female with a severe fungal colonizer treated with antifungal drug regimens. She recovered very well and was later diagnosed with a pulmonary cryptococcosis with a rapid progression to new-onset manifestation of fungal infection.

To our knowledge, this is the first report of a pulmonary cryptococcosis treated with adjunctive antifungal therapy in a non-HIV-bearing patient. The results emphasize the possibility of the use of terbinafine in the treatment of these severe and potentially fatal fungal colonizers.


## Introduction
The main fungal pathogens of the human body, including Candida albicans, Cryptococcus neoformans, and Aspergillus fumigatus, are the main etiologic agents of both superficial and invasive fungal infection. Candida albicans is a common opportunistic pathogen of the central nervous system (CNS) in the developing world (5). Infections due to Candida albicans are generally acquired by inhalation of infectious conidia or conidia-laden droplets from the environment. These infectious organisms typically invade the respiratory tract by the respiratory tract, either as part of the respiratory chain or as the spread from the skin, sinuses, or mucocutaneous region (6). The initial infection usually begins with the ingestion of contaminated food or water and is often asymptomatic. It can progress to a disseminated disease with varied clinical manifestations and correlates with the immune status of the host (7).

The first fungus to cause human disease was described by Gallagena in 1883 and was named C. neoformans (8). It was initially described as a dematiaceous fungal pathogen causing a disease known as gasterosis (9). The disease was not discovered until later that same year (10). Since then, C. neoformans has been reported more commonly from diverse sources, including Brazil, China, Japan, Thailand, and Korea (11). The outbreak of C. neoformans in the United States in 1999 was caused by this pathogen that is capable of causing pulmonary disease in immunosuppressed individuals, such as those with AIDS, patients with surgical and neoplastic complications, and those infected with HIV/AIDS (12).

C. neoformans was initially described by Eriksson in 1894 as an endophyte from the genera Epizootyperma and Exophiala (13). E. neoformans is found worldwide in the environment and is found in bird droppings, decaying wood, and plant debris (14). The fungus can also be found as airborne or aquatic propagules, which are usually enclosed in a microflora (15). Due to the low oxygen availability and the heat stress, C. neoformans is more susceptible to oxidative stress, oxidative stress, and other oxidative and non-oxidative stress conditions (16).


## Methods
In brief, cells were cultured overnight in M9 medium and were harvested at 0, 6, and 24 h post-inoculatio. The culture was centrifuged and washed with PBS. A volume of 100 Âµl of culture was transferred to 96-well microtiter plate (Nunc, Inc., Corning, NY). The supernatant was removed and replaced with fresh M9 medium. The assay was performed as described (26) and the optical density at 600 nm (OD600) was measured in a spectrophotometer (Spectramax epiological Optima II; BioTek Instruments, Inc., Winooski, VT).

For the non-pathogenic yeasts, plates were placed in sealed, polycarbonate, polycarbonate plastic tissue-culture treated (PC) bagels and allowed to air-dry. The slides were washed with PBS and counted by a hemacytometer. The number of yeast-associated protoplasts was determined by a hemacytometer, and the percentage of yeast protoplasts was calculated using the formula:

For the non-pathogenic yeasts, plates were placed in sealed, polycarbonate, polycarbonate, and non-EMB-treated (EMB-E) bags (BD Biosciences, San Jose, CA) and allowed to air-dry. The slides were then washed with PBS and counted by a hemacytometer. The number of yeast-associated protoplasts was determined by a hemacytometer. The percentage of yeast-associated protoplasts was calculated using the formula:

For the pathogenic yeasts, plates were placed in sealed, polycarbonate, polycarbonate, and non-EMB-E-treated (EMB-E) bags (BD Biosciences, San Jose, CA) and allowed to air-dry. The slides were washed with PBS and counted by a hemacytometer. The number of yeasts-associated protoplasts was determined by a hemacytometer. The percentage of protoplasts-associated protoplasts was calculated using the formula:

For the pathogenic yeasts, plates were placed in sealed, polycarbonate, polycarbonate, and non-EMB-E-treated (EMB-E) bags (BD Biosciences, San Jose, CA) and allowed to air-dry.


## Results
cerevisiae strains with altered genotypes were significantly more sensitive to . aureus infection than the wild type and the p-p strain, indicating that they were more susceptible to S. aureus infection than the other sordariomycetes. Moreover, the pathogenic yeast strains in our study were not significantly different from the wild type or the p-p strain in terms of virulence (Fig 4A). Taken together, our data indicate that sordariomycetes are more tolerant to S. aureus infection, and these results suggest that sordariomycetes are more likely to be virulent than the other sordariomycetes.

S. aureus colonization and pathogenicity of S. aureus
The most prevalent pathogen of fungal infections is bacterial pathogenicity. Microorganisms can replicate in the host, invade the immune system, and cause disease [25]. S. aureus colonization and pathogenicity are characterized by a predominance of pathogenic strains. We next compared the pathogenic and non-pathogenic strains in the collection of the S. aureus pathogenic strains. S. aureus strains were confirmed by PCR to be non-pathogenic by qPCR. The results indicated that S. aureus was not present in the PDB and MAL, but was present in the PDB and MAL of the S. aureus-susceptible isolate. The S. aureus PDB and MAL were confirmed by sequencing to be non-pathogenic by qPCR. The S. aureus PDB and MAL were confirmed by sequencing to be non-pathogenic by qPCR. In the S. aureus MAL, the bacterial pathogenicity was the most prevalent pathogen, followed by the other pathogenic species (Fig 5A). S. aureus PDB and MAL were confirmed by sequencing to be non-pathogenic by qPCR.

S. aureus colonization and pathogenicity were also assessed using a non-randomized approach. A non-randomized PCR was used to select the S. aureus PDB and MAL strains by sequencing. The results indicated that the S. aureus PDB and MAL were not present in the PDB or MAL of the S. aureus-susceptible isolate.


## Discussion
dermatitidis

Proteomic studies were performed in this study to identify the genes involved in the biosynthesis of non-fermentable carbon sources (NFS) in . dermatitidis. The genome of B. dermatitidis contained four gene clusters (Figure 2A). The clusters involved in the biosynthesis of NFS were clustered into clusters 1, 3, 5, 7, 10 and 11. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 4, 5, 6 and 7. The genes that are involved in NFS biosynthesis were clustered into clusters 5, 8, 9 and 12. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 6, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 4, 5 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 6 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 6 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8. The genes that are involved in NFS biosynthesis were clustered into clusters 1, 3, 5, 7 and 8.
