# Nigrophila
Quaglee Dragontacos


## Abstract
Penicillium marneffei is a dimorphic fungus endemic in Southeast Asia. Its transmission in healthy adults has been reported mainly via the bird-human disease transmission network. The aim of this study was to evaluate the antifungal susceptibility patterns of four strains of P. marneffei, namely: P. marneffei graciligenicus var. gracina var. grubii (A), P. marneffei attorescens var. attesa var. regia, and P. nycterostigmina viridinaria var. viridis (A/V). For this, antifungal susceptiness was determined by the agar-based method. MIC and MBC were determined by the microdilution method. The MIC50 and MIC90 were 0.5% for A and 96.5% for A/V respectively. The MIC90 and MBC were 0.5% for A, 0.5% for A/V, and 0.3% for A/V respectively. Out of 48 isolates, P. nycterostigmina-isolates were more susceptible to amphotericin B (MIC90 = 1.25-2 µg/ml) than P. nycterostigmina isolates (MIC90 = 1.25-2 µg/ml). The highest out of four isolates against P. nycterostigmina was 97% for P.


## Introduction
The genus Candida includes opportunistic fungal pathogens of human and veterinary importance.


## Methods
neoformans H99 (NCBI accession number: CNAG_1016176) was used for genome annotation and annotation by multiple sequence alignment. We used the MUSCLE program (Edgar, 2004) and the default settings.

Molecular phylogenetic analysis
The two best-scoring nucleotide substitution models for the four nucleotide substitution models (Z-score and Gamma-distributed) were selected and used for the molecular phylogenetic analyses. Gamma-distributed substitution models are more robust when the likelihood ratio of the variable nucleotide substitution model is >0.8, but with the bias of the model parameters being removed when the likelihood ratio between the gamma-distributed substitution model and the substitution model of the multiple sequence alignments is <0.8. For the analyses of multiple nucleotide substitution models, we used the DnaSP package (v0.9.18) (Caporaso et al., 2010). A version of this program was used to convert the maximum-likelihood data to Bayesian, Bayesian-based phylogenetic trees with the C. neoformans H99 genome annotation. The Bayesian tree inferred from the species tree was generated using the methods of Kimura-2-parameter substitution models, with a substitution model for each nucleotide substitution model as the number of discrete gamma categories. The search for the best-scoring nucleotide substitution models (Z-score) was performed using the “best-scoring” nucleotide substitution model (i.e., Z-score) obtained by the four nucleotide substitution models in combination with the substitution model of the multiple alignment alignment. For the analyses of multiple nucleotide substitution models, we used the “expert” substitution model with the substitution model of the multiple alignment.

To determine the phylogenetic relationships between the taxonomic groups, we used the neighbor-joining method (with the branch swapping option selected). For the analyses of multiple nucleotide substitution models, we used the “phyloseq” package (v1.3.0) (Kimura, 2010).

Phylogenetic trees were constructed using the neighbor-joining method (with branch swapping option selected). For the analysis of multiple nucleotide substitution models, we used the “string map” package (v1.4.


## Results
The . cerevisiae orthologs of C. neoformans have also been implicated in C. neoformans virulence in the murine model of systemic candidiasis [42], [43].

The most abundant phage-encoding gene in S. cerevisiae is the S. aureus-encoding gene, PbPseudA, with a predicted cytoplasmic domain. The S. aureus PbPseudA protein has been shown to be a potent inhibitor of PbPseudA activity [44]. It is very likely that S. aureus PbPseudA is directly involved in the biosynthesis of the biosynthetic compound that is produced by P. falciparum PbPseudA.

PbPseudA has also been shown to be required for virulence in the murine model of pulmonary aspergillosis [45]. The virulence phenotype of PbPseudA-infected mice was confirmed by the more than 20% survival rate of the P. falciparum-infected mice (Figure 4). It has been reported that P. falciparum PbPseudA-infected mice were more sensitive to pneumococcal infection, a hallmark of pulmonary aspergillosis [46].

PbPseudA is involved in sexual reproduction
We observed that PbPseudA was highly expressed in the lungs, with a transcript of 28.7 kb for PbPseudA (Table 2). This is consistent with the genome data and with the fact that the P. falciparum PbPseudA gene is highly expressed in the lungs. In the absence of PbPseudA, we could not detect a P. falciparum PbPseudA-encoding gene.

We investigated the potential involvement of PbPseudA in virulence in the model of pulmonary aspergillosis, and the pathogenicity of P. falciparum PbPseudA-infected mice. The PbPseudA-mediated virulence was confirmed by a highly significant pathogenicity phenotype (Figure 5A).


## Discussion
oryza. In addition to the large number of proteins involved in fungal development, we also showed that the only protein responsible for the first phase of development, the fungal pathogenicity factor 1 (Ftr1), plays a minor role in the development and pathogenicity of M. oryzae. The Ftr1 homologue of M. oryzae was identified by RNA sequencing as a representative of the C. neoformans enzyme, which is important for fungal cell wall integrity and virulence [43]. The function of Ftr1 in M. oryzae has not been reported in M. oryzae. In addition, a putative S. cerevisiae Ftr1 homologue (Sff1) was identified that was found in the genome of M. oryzae [43]. S. cerevisiae was also reported to produce a large number of Ftr1-like proteins in response to oxidative stress [44]. Interestingly, the Ftr1 homologue of M. oryzae was identified in the genome of S. cerevisiae, but not in other fungi (Additional file 7: Table S3).

Finally, we also analyzed the transcriptional profiles of the genes that are regulated in response to different stresses, including oxidative stress, oxidative stress, host infection, and infection with pathogens (Additional file 8: Table S4). Overall, we found that the transcriptional response of the fungal pathogenicity factor 1 (Ftr1) is regulated by a number of proteins that are found in the genome of M. oryzae. The expression of Ftr1 in M. oryzae is regulated by a number of genes that are found in the genome of M. oryzae. The expression of Ftr1 in M. oryzae is regulated by a number of genes that are found in the genome of M. oryzae. The expression of Ftr1 in M. oryzae is regulated by a number of genes that are found in the genome of M. oryzae. We further examined the expression levels of Ftr1 in different tissues and conditions, and found that the Ftr1 gene is highly upregulated in the post-mTOR signaling pathway. In response to oxidative stress, the C. neoformans protein kinase A (PkaA) pathway is activated and is regulated by a number of genes that are found in the genome of M.
