# Nitrilases are required for stability of threonine aminoacyls under alkaline conditions and their regulation by Cdc3+ azide hydrolase
Quaglee Dragontacos


## Abstract
Meningitis and encephalopathy are common clinical features in adults with advanced HIV infection.


## Introduction
Intron number is an extremely variable factor and is controlled by a variety of processes including genome size, gene order complexity, gene transcription, gene regulation, protein translation and transcription levels of mRNA and proteins (Chang et al., 2014; Matuo et al., 2016; Hamel et al., 2017; Chen et al., 2018). The most common introns are among the small noncoding RNA (sRNA) genes (25-54 nucleotides) and are present in a range of eukaryotic genes including genes involved in ribosomal RNA metabolism and metabolism of exosomes and exoproteome recycling (Walsh et al., 2013; Haddar et al., 2017). The most common introns include exon-intron, enhancer-intron and dion-intron. These genes encode small RNA (sRNA) proteins that are highly expressed in eukaryotes. They are found in eukaryotic genes, especially in the somatic cells and exosomes, and can serve as endogenous elements in the genome. Some examples of sRNA genes can be found in animals, plants, fungi and protists (Matuo et al., 2016).

The eukaryotic cell is a huge source of sRNA and has about 200,000 described genes (Ma et al., 2015).


## Methods

Morphology and culture conditions
Morphological observations of vegetative cultures were performed using a Nikon Eclipse 80i microscope with a Nikon DS-U2 digital camera and a Nikon DS-U2 digital camera with NIS elements and software NIS-element. The samples were dried at room temperature, mounted on microscope slides, and examined with a Zeiss LSM 510 D camera. Colony morphology and pigmentation were examined in 10% KOH buffered gelatin.

DNA extraction, PCR, and sequencing
Total genomic DNA was isolated using the DNeasy Plant Mini Kit (Qiagen) following the manufacturer’s instructions. The DNA was purified using a QIAquick PCR Purification Kit (Qiagen) following the manufacturer’s instructions. The purified DNA products were sequenced by Macrogen (Seoul, Korea) using the BigDye v. 3.1 Cycle Sequencing kit (Applied Biosystems, Foster City, CA, USA) and an ABI Prism BigDye v. 3.1 Cycle Sequencing kit (Applied Biosystems).

The bacterial 16S rRNA gene was amplified using the primers 515f and 806r (Bergman et al., 2011), and the internal transcribed spacer (ITS) region of the ribosomal RNA gene was amplified using the primers ITS1 and ITS4 (White et al., 1990; Crous et al., 1990; Gardes and Bruns, 1993). The fungal ITS region was amplified using the primers ITS4 and ITS5 (Vilgalys and Hester, 1990; Wiens and Borkhout, 2000). The PCR products were purified using the DNeasy Plant Mini Kit (Qiagen). The amplicons were sequenced using the BigDye v. 3.1 Cycle Sequencing Kit (Applied Biosystems, Applied Biosystems). The sequences were edited using BioEdit 7.0.5 (www.mbio.ncsu.edu/BioEdit/BioEdit.html). Sequences were deposited in the Sequence Read Archive of the National Center for Biotechnology Information (NCBI) under accession numbers LC02522-LC02524 and LC02524.


## Results
However, the possibility exists that these threonine aminoacyls are in fact mislocalized during in vitro culture in the absence of azide stress [41, 42. Thus, it is important to monitor the structures of the threonine residues in the active site of the Cdc3-Arg4-Cys3-Cys7 complexes for any changes in their stability.

As a first step, we tested the stability of the Arg1-Arg4-Cys7 complex at pH 4.5 by incubating the reaction products with alkaline pH media (Fig 5). In the absence of alkaline pH, the Arg1-Arg4-Cys7 complex exhibits the expected structure with the corresponding characteristic stearic acid residues at the His-Asp/Thr5-Asp-Asp and Arg2-Arg4-Cys5-Asp-Asp positions (Fig 5). This is confirmed by the crystal structure of the complex, which shows a typical, inverted form with the His-Asp/Thr5-Asp-Asp and His-Asp/Thr5-Asp-Asp residues in the active site (Fig 5). The three-dimensional (3D) crystal structure of the complex is shown in Fig 5 and S1 Table. In the case of the Arg1-Arg4-Cys7 complex, the Arg1-Arg4-Cys7 structure is similar to the crystal structure of the S. cerevisiae Arg2-Arg4-Cys5 complex, with the Asp at the His-Asp and Arg2-Arg4-Cys5-Asp residues in the active site (Fig 5). This is consistent with the crystal structure of the S. cerevisiae complex Arg2-Arg4-Cys5 complex, which shows the characteristic inverted form with the Asp/Thr5-Asp residues in the active site (Fig 5). The Arg1-Arg4-Cys7 complex exhibits the same type of conformational change as the S. cerevisiae complex and shows no change in the structural features. The Arg1-Arg4-Cys7 complex is stable at pH 4.5 at room temperature, but at pH 5.5 and above, it exhibits a conformational change (Fig 5) that is similar to the S. cerevisiae complex and shows no change in the structure features.


## Discussion
fumigatus fungus, . crassa, and Fusarium graminearum, respectively [24-26].

During the first step of the acidification process, Fusarium graminearum and Fusarium graminearum can synthesize and store their contents, but the presence of sulfide in the culture medium leads to the inability of these two fungi to grow at a high temperature [6]. In the present study, the growth of Fusarium graminearum was restricted to a high temperature, and the sporulation and sporulation of Fusarium graminearum was not inhibited under a high temperature. However, the sporulation of Fusarium graminearum was inhibited by the salt, but the sporulation of Fusarium graminearum was not inhibited by the salt. The growth of Fusarium graminearum was not inhibited by the high salt.

Besides the effect of pH on the growth of these two fungi, different pH values are known to affect the growth of these two fungi. The pH values of the pH plates for the two fungi were found to be acidic, which is compatible with pH-dependent acidification of the water. The pH values of the plates in the same pH range showed that both fungi were grown at a pH of 4.0. This was also consistent with the pH values of the cultures. A significant difference between the pH values of the two pH plates of Fusarium graminearum and Fusarium graminearum was observed, which could be due to the different growth conditions. This could be due to different growth conditions, but pH values were found to be similar. The pH values of the two cultures were found to be alkaline, which was compatible with pH-dependent acidification of the water. The pH values of the two cultures were found to be alkaline, which was compatible with pH-dependent acidification of the water. The pH values of the two cultures were found to be alkaline, which was compatible with pH-dependent acidification of the water. These results indicate that the pH values of the two cultures were found to be alkaline, which was compatible with pH-dependent acidification of the water.

The pH values of the two cultures were found to be alkaline, which was compatible with pH-dependent acidification of the water.
