# Novel QTLs to Drug Catalytically Arrest the Antifungal Phenotype: A Bacterial Paradox explained by Cyclic CTR-3
Quaglee Dragontacos


## Abstract
Ocoteâ€™s disease, caused by the fungus Podospora blakesleeanus, is caused by a single dimorphic fungus, P. bakersfieldii. Pbs. blakesleeanus is distributed worldwide and its spores are frequently found in soil. The ability of P. blakesleeanus to produce spores is a major virulence factor for the pathogen. In this study, we used comparative genomics to identify genes and genetic material related to the ability of Pbs. blakesleeanus to produce conidia. We identified 148 genes from the conidia genome that were either highly conserved (ST298), highly divergent (ST145), or divergent (ST309). Excluding three group comparisons (ST313), we identified ST129 as the most conserved gene. Genomic data further confirmed that these genes were derived from Pbs. blakesleeanus. The distribution of these genes under standard growth conditions (normal or hypoxia) and in a Pbs. blakesleeanus-induced superinfection model suggest that they will undergo horizontal transfer during infection. We also analyzed the migration of the conidia to form hyphal filaments and tested the antifungal action of the migration agent, amphotericin B. We observed that Pbs. blakesleeanus hyphae did not form conidia at a dosage similar to other fungal morphotypes.


## Introduction
Histoplasmosis is a systemic fungal disease, which is endemic in Southeast Asia, including Malaysia, Thailand, and Indonesia.1,2 The disease is caused by the dimorphic fungus Histoplasma, which is found in the soil and on plants. It is believed that exposure to the fungus results in the yeast-to-hyphal transition, which is followed by the formation of mold, yeast, and mycelia. In many tropical countries, it is considered to be endemic to tropical and subtropical areas. In Indonesia, the disease is endemic to Malay, Pampanal, and Bali islands.3,4 It is estimated that only 6.3% of patients have been diagnosed with histoplasmosis, while in Malay, it is estimated that over 180 cases of histoplasmosis occur annually.5,6

Malaysia is a subtropical country with a long history of dealing with Histoplasmosis. Indonesia is the largest exporter of Histoplasma in the world. Indonesia is a tropical country with high humidity, high temperature, and humid tropical conditions. In the past few decades, the incidence of Histoplasmosis has increased dramatically in Malay, in part because of increasing numbers of immunocompromised patients and immunocompetent individuals.7,8

The disease has a severe clinical course, and the overall mortality rate has reached over 30% in immunocompromised patients.8,9 In the past few decades, the incidence of histoplasmosis has been decreasing, and this is mainly due to the widespread use of antifungal drugs, especially azoles and allylamines, for the treatment of infections caused by fungi like Histoplasma and Cryptococcus. The treatment of Histoplasma has been challenging due to the high toxicity, the lack of good bioavailability, and the fact that there are no effective antifungal drugs. The antifungal therapy for Histoplasma is based on the combination of several antifungals, mainly in the azoles, with different mechanisms of action. The azoles inhibit ergosterol biosynthesis, which is an important component of fungal cell membrane. The allylamines inhibit the biosynthesis of lanosterol from E.


## Methods
Study population
We conducted a retrospective study from January 2011 to December 2016 in a tertiary hospital in Tehran University Hospital, Iran. The primary endpoint was the diagnosis of IA, followed by hospitalization for a second consecutive month. The secondary endpoint was the outcome of follow-up in the following year: the detection of Cryptococcus neoformans, and the relapse of ICU. In the overall study population, we identified 42 patients with HIV/AIDS, 5 with AIDS, and 6 with chronic kidney disease. Among these, we also analyzed HIV/AIDS patients with ICU admission and all patients who had positive results were included in the study. In the time period since ICU admission, 38 patients were initiated in our study. We excluded patients who were known to be HIV-infected and had a history of CNS infections prior to ICU admission, and were therefore excluded from the study.

Study setting and study population
This study was approved by the Ethics Committee of the Institute of Medical Research, Tehran University Hospital. The study was carried out on a routine basis. All participants were screened for medical, clinical, and laboratory tests, and were referred to the study hospital if they were still alive and were eligible for enrollment into the study. The study was approved by the Ethics Committee of the Institute of Medical Research, Tehran University Hospital.

The study was conducted in accordance with the guidelines of the Declaration of Helsinki and was conducted according to the ethical principles of the Declaration of 1975. The study was conducted in accordance with the principles of the Helsinki Declaration and the principles of the International Conference on Harmonization on Humanized Care and Use of Human Resources in Respect of the Human right of the individual to Life and Health.


## Results
The QTL was .4 kb in length and involved CTR-3-RFLP (Figure 2), FKBP120, and IOG1. FK506 treatment did not cause an obvious phenotype, and the CTR-3-RFLP-induced phenotypes of FKBP120 and IOG1 were similar to those caused by FKBP120 and FK506 treatment (Figure 2). A BLASTp analysis showed that CTR-3-RFLP is the most conserved drug-catalytically expressed by FKBP120, whereas FKBP120 and IOG1 showed different QTLs, which can be attributed to CTR-3-RFLP, FKBP120 and IOG1. We next investigated whether CTR-3-RFLP was also expressed by FKBP120, FKBP120, and IOG1. FK506 treatment did not cause an obvious phenotype, and the FKBP120- and IOG1-induced phenotypes were similar to those caused by FKBP120 and FK506 treatment (Figure 2). Collectively, these data strongly support the QTLs identified in the qTLs as the QTL to drug-catalytically arrest the antifungal phenotype of FKBP120, FKBP120, and IOG1.

RNA-Seq Analysis of FKBP120 and IOG1 Genome Genome
To gain insight into the biological roles of FKBP120 and IOG1, we used a forward mutation approach to search for gene products that are essential for FKBP120 function. This approach was applied in the transcriptomic studies of FKBP120 and IOG1. A forward mutation was inserted into the FKBP120 gene, resulting in the loss of a predicted protein of approximately 110 amino acids. The loss of the predicted protein was confirmed using the in silico model (http://www.broadinstitute.org/annotation/genome/kim/view/1.00). The deletion mutant was complemented by introduction of a constitutively active copy of FKBP120 (Figure 3A).


## Discussion
toruloides (Rt CTR-3) at 24 h after treatment (Fig 3A. The CTR-3 mutants were also treated with the drug in the presence of CRT-3 (Fig 3B). Both the CTR-3 and the CRT-3 mutants were also treated with CRT-3 in the presence of CCL-4 (Fig 3B). In addition, CTR-3 was treated with the drug (CRT-3) at both 24 h and 48 h after treatment (Fig 3C).

In addition to the previous results, we found that CTR-3 genes in R. toruloides and R. toruloides were significantly upregulated by CRT-3 treatment (Fig 3D). The upregulation of CTR-3 genes was not observed in the CRT-3 mutant, indicating that the CRT-3 treatment did not inhibit CTR-3 expression. Interestingly, the CRT-3 treatment treatment did not completely abolish CTR-3 expression, suggesting that CRT-3 treatment has some effect on CTR-3 expression in R. toruloides. However, we found that CTR-3 expression was significantly decreased by CTR-3 treatment in the absence of CRT-3 (Fig 3E). Taken together, these results suggest that CRT-3 treatment has some effect on CTR-3 expression.

In summary, we found that R. toruloides CTR-3 is a novel CTR-3 gene which encodes a CTR-3 protein which is involved in CTR-3 induction in the absence of CTR-3. It is worth noting that CTR-3 is expressed in the presence of CRT-3 and that CTR-3 is expressed in the absence of CRT-3 in the presence of CRT-3. It is also possible that CTR-3 and CTR-3 regulate CTR-3 expression through their respective genes. It is also important to note that CTR-3 was not expressed in the present study and therefore that further studies are needed to characterize CTR-3 expression. It is possible that CTR-3 expression in the present study is not regulated by the CTR-3 gene.
