# Oligo-DNA Custom Macroarray for Monitoring Major Pathogenic Fungi and Bacteria in the Phyllosphere of Apple Trees
Quaglee Dragontacos


## Abstract
Neurosyphilis is a rare infectious disease, which is usually caused by syphilis infection with syphilis chelon-irreversible syndrome (SISTS), and usually presents in late-stage, untreated syphilis infection in a rural setting. At presentation, patients present with a wide variety of symptoms and signs of neurosyphilis. This is the first reported case of neurosyphilis as a single entity in a rural setting. Diagnosis of neurosyphilis in Rwanda is sporadic, and diagnostic work-up is frequently non-productive because of the Rwandan population residing in rural areas, where the disease has not been reported in many other parts of the world.


## Introduction
Conforto & . Chevalier)

PONE-D-19-20402R1

Dear Dr. C. Conforto,

We are pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it complies with all outstanding technical requirements.

==============================

I received the following response from the PLOS Genetics team:

==============================

We are delighted to inform you that your manuscript has been deemed suitable for publication and will be formally accepted for publication once it complies with all outstanding requirements.

The following additional details are included with the acceptance:

The manuscript will be formally described in a suitable response to the corresponding journal requirements; please refer the accompanying Figure 2-figure summarising the standard procedures for submissions to, and including discussions with funding agencies.

The PLOS Genetics system requirements for Figures 2 and 3 have been included in the corresponding PDF files.

Reviewer #1: The manuscript looks remarkably similar to the model published by van Dijk et al. in their paper. The authors point out that the authors used an invertebrate model (Galleria mellonella) for the phylogenetic analyses. The authors then go on to list several potential problems with this model, including the possibility of ectomycorrhizal (ECM) relationships, taxonomic bias, and the fact that the authors only tested ECE populations, not sponges. The authors should point out these problems before revising the manuscript, as these organisms are generally considered non-model organisms.

The authors should also provide additional data that support their model and its phylogenetic conclusions. For example, their model should include three phylogenetic species, Daphnia, Lophophyton, and Phloeospora.

Reviewer #2: The manuscript looks very similar to the model published by van Dijk et al. In their study, the authors used a sponges model. Unfortunately, the authors did not provide enough information for the readership and these authors could not provide enough background for the manuscript. The authors should provide more information about their reasoning behind the use of the sponges model. For example, they should state that the ECM community does not associate fungal taxa with each other. In other words, the ECM community does not associate taxa with each other.


## Methods
A fungal pathogen (Microsporum canis) was used as the outgroup, whereas non-pathogenic bacteria (Zymosarpsa ss. oryzae and Escherichia coli) was used as the outgroup. The results of PCR amplification were analyzed by Sanger sequencing the first PCR products. The operational taxonomic units (OTU) were defined as the number of bacteria identified by quantitative PCR with fungal species. In total, 41,272 OTUs were detected and the database was searched using the Basic Local Alignment Search Tool (BLAST) and the Basic Local Alignment Search Tool (BLAST) to identify fungal species in the apple tree. All the bacterial and fungal OTUs were annotated according to the deduced amino acid sequences of the known fungal sequences. The taxonomy of the fungal OTUs was determined based on the following criteria: (i) BLAST hits with high similarity and (ii) BLAST hits with an expected phylogenetic distance of more than 50% with a maximum length of 10,000 bp.

The phylogenetic analysis was performed using MEGA7 software (Kumar et al., 2016). The number of OTUs was divided into four groups according to their phylogenetic position in the tree: (i) bacteria (p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a.), fungi (p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a.), fungi (p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a.), and (ii) fungi (p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.f. if p.a. and b.


## Results
We first examined the comparison of the obtained plant-plant interaction profile. Using the PCA, the apple tree was used to reconstruct the tree topology and the plant-plant interactions. The tree topology reconstructed by the PCA was most similar to the PCA based on the plant interaction profiles of soil-flowering Fungi, which was the only plant interaction profile that could be found to be highly similar to the PCA based on the plant interaction profiles of soil-flowering Bacteria.

In order to further characterize the fungal-plant interactions, we compared the plant interaction profiles of plant-plant interactions to determine if the fungal-plant interactions were highly similar to each other. We found that the plant-plant interactions were highly similar to each other, as expected for plant interactions. We found that the plant-plant interactions were highly similar to each other, as expected for plant interactions. For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions. For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions.

For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions. For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions. For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions. For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions.

The combined approach of the two plant-plant interactions and the plant-plant interactions, with the addition of soil-flowering Fungi, was able to reconstruct the plant-plant interactions. The plant-plant interactions were highly similar to each other, as expected for plant interactions. The two plant-plant interactions were highly similar to each other, as expected for plant interactions. For the two plant-plant interactions, we found that the plant-plant interactions were highly similar to each other, as expected for plant interactions.


## Discussion
We used a fungal consortium consisting of a total of nine fungal species and one microorganism, and selected the two main fungal species (Candida albicans and Cryptococcus neoformans) as the target species, and the three main bacterial species (including the two main pathogens, namely . aureus and P. aeruginosa) as the target species, and the three main bacterial species (including P. aeruginosa) as the target species. The results are summarized in Table 3.

The results showed that the whole genome sequence analysis was used to identify the fungal community associated with apple trees in the phyllosphere of apple trees, with the aim of identifying fungal species. The results also revealed that the total fungal community was significantly increased by the addition of 10 µg/ml of the fungal peptide-related peptide (MLSP) to the apple tree peptide-based metagenomics analysis. Moreover, the addition of 10 µg/ml of the peptide-related peptide (MLSP) to apple tree metagenomics analysis resulted in a significant increase in the total fungal community associated with apple trees. The results also revealed that the fungal community associated with apple trees were significantly increased by the addition of 10 µg/ml of the fungal peptide-related peptide (MLSP) to apple tree metagenomics analysis. Furthermore, the addition of 10 µg/ml of the fungal peptide-related peptide (MLSP) to apple tree metagenomics analysis resulted in a significant increase in the total fungal community associated with apple trees.

The phyllosphere fungal community associated with apple trees was significantly increased by the addition of 10 µg/ml of the fungal peptide-related peptide (MLSP) to apple tree metagenomics analysis. In addition, the addition of 10 µg/ml of the fungal peptide-related peptide (MLSP) to apple tree metagenomics analysis resulted in a significant increase in the total phyllosphere fungal community associated with apple trees.
