# Oral microbiome analysis of out-patent adults sensitized to touch and to harmful bacteria (MixrLab)
Quaglee Dragontacos


## Abstract
The pathogenic yeast Candida albicans can occupy a wide range of metabolic niches and is thus capable of adapting to environmental stresses. Under these unfavorable conditions, this yeast can become a formidable resistance. To identify the molecular pathways that underlie this resistance, we analyzed a large genome copy number (GWK) in the model strain C. albicans. This copy number showed strong support among qPCR (quantitative PCR) and MALDI-TOF-MS analyses, and showed that the genes covered by the copy number are closely related to the metabolic pathways of this yeast. In particular, the metabolism of the methanol extract from the clone D235-5700 generated a strong preference for the three major metabolic pathways of this yeast. In vivo assays revealed that this extract has potent antifungal activity against the tested Candida species. Moreover, the gene expression levels of the metabolite 1-octyl-5-one (oct-IFN-a) in C. albicans were up-regulated in a dose-dependent manner.


## Introduction
Cryptococcosis is a major public health problem in patients with human immunodeficiency virus (HIV). Although early diagnosis and treatment are poor, they are associated with mortality rates as high as 80% in HIV patients and as high as 60% in non-HIV patients [1-3]. The burden of cryptococcosis in HIV-infected patients is even greater in the era of antiretroviral therapy (ART), where early ART initiation in HIV-infected patients can reduce the risk of developing cryptococcosis [4].

The Clinical Problem
Cryptococcosis is a severe and life-threatening disease with a high morbidity and mortality, especially in resource-limited settings [5-8]. The most common risk factor for cryptococcosis is HIV infection and is manifested by opportunistic infections such as cryptococcal meningitis (CM), cryptococcemia, cryptococcal pneumonia, and cryptococcal pneumonia [9-12]. CM is an AIDS defining illness, which is usually self-limiting. However, the mortality of HIV-infected patients with CM is reported to be over 40% [13].

Cryptococcosis can be diagnosed in HIV patients by two main methods: direct immunological or quantitative detection of the fungus in serum and cerebrospinal fluid (CSF) [14]. In HIV-infected patients, quantitative serological methods are now commonly used to identify cryptococcal meningitis cases, including PCR-fingerprinting and culture of cryptococcal cells [15]. Cryptococcal culture is usually obtained within 2 days of antifungal treatment, and the results can be positive or negative depending on the characteristics of the cryptococcal cell count and cytology. Antifungal susceptibility testing for cryptococcal cells has been performed using the point mutation assay, and cryptococcal isolates resistant to fluconazole have been detected using the non-moleptic technique, i.e., the culture of cryptococcal cells in CSF [16].

The Microbiological Diagnosis

The ability of Cryptococcus to cause disease has been described as ‘cellular’, that is, the cells that persist in the cerebrospinal fluid.


## Methods
 2016. In brief, adult cadavers were removed from the tray and placed into a sterile and pre-weighed tube containing sterile distilled water. Cadavers were then placed into a sterile and pre-weighed tube containing 1% agarose and then washed with sterile distilled water. Bacteria suspension was then removed by scraping, and then the tubes were placed into sterile and pre-weighed tube containing 1% agarose and then the tubes were stored at 4 °C until the next use. For bacterial sampling, the tubes were placed into a sterile and pre-weighed tube containing 1% agarose, and then the tubes were washed with sterile distilled water. Bacterial samples were then removed by scraping, and then the tubes were placed into a sterile and pre-weighed tube containing 1% agarose and then washed with sterile distilled water. This procedure was repeated until the bacterial pellets were collected. Bacterial pellets were then removed by scraping, and then the tubes were placed into a sterile and pre-weighed tube containing 1% agarose and then the tubes were stored at 4 °C until the next use.

A semi-quantitative method was used to determine the relative abundance of bacteria in each sample. After that, the samples were mixed for 1 min, followed by centrifugation at 3000 × g for 10 min at 4 °C, and then the supernatant was used for a count of bacterial counts.

Microbiological analysis
For the qualitative analysis, a fresh fecal sample was taken from the bottom of the tube and then stored at -80 °C for bacteria analysis. For the negative control, only the fecal sample was taken. The microbiological analysis was carried out at the Department of Microbiology, Department of Veterinary Tropical Diseases, University of KwaZulu-Natal (UNESP).

Statistical analysis
All the experimental procedures were conducted in accordance with the guidelines of the relevant ethics committees (CEUA-IRS), Government of South Africa. All the statistical procedures were performed with GraphPad Prism 6 software.

Ethics statement
This study was approved by the Ethics Committees of University of KwaZulu-Natal, University of KwaZulu-Natal, Government of South Africa (No: 2017/03/14) and the Netherlands National Institute of Health Research (No: 2019/09/17).


## Results
A total of 64,204 continuous sequences were obtained from the Illumina HiSeq Genome Analyzer (JGI) library. The sequencing depth and coverage of the assembled genome were 100% (Table S1). The resulting total of 242,918,904 bp sequences were classified into 5,898 OTUs, corresponding to 7,633,717,947 genes (Table 1). The genome coverage of fungal genome was 0.15, and the sequencing depth and coverage of fungal gene were 94%, which was similar to that of fungi (0.16 to 0.23).


## Discussion
In contrast, no differences were observed between out-patent adults and controls (Table 3. However, a difference in fungal community structures was observed between out-patent adults and controls when compared to controls. This difference might be due to differences in patient populations.

In this study, different fungal communities were found in both adult and pediatric patients. Most of the fungal genera were found to be present in all patients but less than 30% of pediatric patients were present in this study (Table 3). The predominant genera of these patients were Malassezia, Alternaria, Candida, Cryptococcus, and Aspergillus. Other genera were found to be less prevalent than the genera of out-patent adults and in-patent children. Malassezia was present in the majority of patients and Alternaria and Candida were present in only half of patients. Malassezia species were present in only half of patients. However, Malassezia genus was present in only 1 of the patients. The presence of other genera of the genera Malassezia, Alternaria, Cryptococcus, and Aspergillus was not found in this study. Other genera of the genera Malassezia were not detected in this study but were found in other studies. Some of these genera are found in other patients, such as Aspergillus, Blastomyces, and Candida. However, this study did not detect these genera in all patients.

In this study, a significant proportion of pediatric patients were affected by chronic fungal infections. A significant proportion of pediatric patients had HIV infection. These patients were mainly immunocompromised and those with immunodeficiency symptoms were more likely to have chronic fungal infections. HIV infection was the predominant risk factor for fungal infection in this study. However, HIV infection was not found in this study.

In the pediatric population, there were more patients with severe infections. The majority of pediatric patients were infected by C. albicans. The majority of patients were infected with C. albicans, but only C. albicans was present in all patients. The presence of C. albicans in pediatric patients is not uncommon.

The presence of C. albicans in pediatric patients is not uncommon. The presence of C.
