# Paralogues between environmental and genetic factors affecting the laccase-binding specificity of the Saccharomyces cerevisiae protein
Quaglee Dragontacos


## Abstract
Here we report the genome sequences of three laccase-binding-type (AD) genes from the Northern China Mountains where wild-type laccase populations are located and phylogenetically characterize. Homologous sequences for IleF (samsF) and laccase-binding-type (LccA/LccF) genes from a single wild-type laccase isolate (c. 15877T bp) yielded two putative laccases, that is, the type LccA/LccF and type LccA/LccP (LccA/LccP) in laccA and LccA/LccF, respectively. Sequence analysis of the laccase-binding-type (LBA) of the type LccA/LccF (samsF) showed that the type LccA/LccP (lacA) contains two LccA domains, the type LccA-1 and the type LccA-2. A comparison of the genomes of laccase-deficient (LBD) and laccase-null (lacA/lacA) strains revealed that the genes encoding the type LccA/LccP and type LccA-1 were expressed differently in laccase-deficient (LBD) strains compared to wild type strains. The laccase genes in the laccase-deficient (LBD) strains were also highly expressed throughout the fungal growth. These results indicate that laccase has a unique genetic environment and is acquired by both the laccase-dependent and laccase-elicited pathways of the fungus. The laccase-dependent and laccase-elicited pathways may represent promising targets for biotechnological and life sciences.

Laccase is a glycolytic enzyme that is critical for lysis of lignin and lactic acid in the ascomycete yeast Saccharomyces cerevisiae. In the pathogenic yeast Rhodotorula mucilaginosa, laccase catalyzes conversion of l-lysine to lactic acid in order to produce chitosan (a polymer of D-lysine).


## Introduction
A recent study suggested that the loss of the laccase function in . cerevisiae could result in an increase in substrate specificity [17]. The laccase enzyme is a ÃŸ-1,4-linked N-acetyl-glucosamine monohexosamine polymer, which is secreted from yeast cells in response to environmental stimuli [18]. The two protein types are very similar in structure and are classified as two different enzymes: laccase and laccase-like proteins (Lhe). Lhe is a pro-membrane protein with a short cytoplasmic tail, while laccase-like proteins are a cysteine-rich protein group [19]. Although the two proteins share a common feature, the laccase enzyme possesses distinct properties that distinguish them from the other enzymes. Lhe proteins can be synthesized in two forms: a heme cysteine-containing endopeptidase and a pro-membrane protein cleavage site. Lhe proteins cleave a peptide, which may or may not be a ligand-binding site and are consequently referred to as cleavage products [20]. Lhe proteins are a class of membrane proteins that form a typical substrate complex for laccase [21]. Lhe proteins were first identified in S. cerevisiae and are involved in several cellular processes, such as replication and cellular differentiation [22-24]. Lignin in the budding yeast cell is an extracellular matrix material that is attached to the cell wall, where it is stabilized by chitin and deacetylation [25-27]. Deacetylation is catalyzed by chitin deacetylase (CDA), which is a heterodimer consisting of a catalytic subunit and a regulatory subunit. Deacetylation is a common feature of the laccase catalytic domain, and chitin deacetylase is responsible for the deacetylation of the elongated polysaccharide chains, with a length of 20-30 amino acids [28]. Lignin is a thin layer of extracellular polysaccharides [29], and chitin is a thick layer of chitin. Lignin is not incorporated into the cell wall by the catalytic domain, but is instead bound to an abundant matrix of chitosan.


## Methods
The data of the genetic factors were only available for the environmental variable. The PTT index (P = 0.05) was used to evaluate the specificity of the Saccharomyces cerevisiae protein prediction. A PTT is a score that expresses the confidence of the prediction. The PTT index is defined as the following value:

2.2. Prediction of the Transcription Factor of Laccase
The transcription factor of Laccase was used to predict the expression of the transcription factor of the Laccase genes in the genome of the Saccharomyces cerevisiae strain 805. The transcription factor of the gene of interest was predicted by the PTT method. The PTT index (P = 0.05) was used to evaluate the specificity of the Saccharomyces cerevisiae protein prediction. A PTT is defined as the following value:

2.3. Intron Length
The intron length of the gene of interest was predicted by the online program BLASTp, version 2.2.32 [34]. The gene of interest was predicted by the KEGG database (http://www.genome.jp/kegg/) and was found to be the most conserved among the total genes predicted. The gene of interest was chosen based on its introns and its introns plus the conserved sequences of other genes in its genome. The gene of interest was also chosen based on its introns and its introns plus the conserved sequences of other genes in its genome. The intron length of the gene of interest was also calculated by the online program BLASTp, version 2.2.32 [34].

2.4. Antifungal Activity Assays
The antifungal activity of the Laccase was evaluated by the MIC test according to the method described by the Clinical and Laboratory Standards Institute (CLSI) [35]. The microdilution method was used to determine the minimum inhibitory concentration (MIC) of the Laccase against Candida albicans (Candida parapsilosis ATCC 22019, Candida krusei ATCC 6258, and Candida guilliermondii ATCC 22019).


## Results
In particular, we determined that laccase activity in the wild type, <U+0394>mkr1, <U+0394>mkr2, and <U+0394>mkr3 strains was significantly increased by 4.3% and 14.8%, respectively, in the <U+0394>mkr1 strain compared with the wild type strain (Figure 1A). The <U+0394>mkr1 strain had a significantly higher laccase activity than the wild type strain (Figure 1B). The increased laccase activity in the <U+0394>mkr1 strain was also observed in the <U+0394>mkr3 strain, which had a significantly higher laccase activity than the wild type strain (Figure 1C). These data suggest that the laccase activity of the <U+0394>mkr1 strain is at least partially related to the laccase activity of the <U+0394>mkr2 strain.

The laccase activity of the <U+0394>mkr1 strain was not statistically different from the wild type strain. To confirm the laccase activity of the <U+0394>mkr1 strain, the growth of the <U+0394>mkr1 strain was further confirmed by culture filtration and the colony count in the presence of galactose. These data suggest that the laccase activity of the <U+0394>mkr1 strain is not related to the growth or growth rate of the <U+0394>mkr1 strain.

Laccase activity of the <U+0394>mkr1 strain was found to be significantly lower than the wild type strain in the presence of galactose. The lower laccase activity in the <U+0394>mkr1 strain compared with the wild type strain was also observed in the presence of galactose. These data suggest that the laccase activity of the <U+0394>mkr1 strain is related to the growth or growth rate of the <U+0394>mkr1 strain.


## Discussion
In this context, the results of this study demonstrate that the two fungal species, . cerevisiae and S. paradoxus, are similar in their ability to bind to laccases and that LAP represents a promising target for the design of a laccase inhibitor.

The above results are in agreement with previous reports on the antagonism between the laccase and the fungal virulence factors in S. cerevisiae [21, 25, 29, 50]. However, we did not find the opposite effects of the two laccases in either the fungal species. A study reported that both Hsp90 and DsRed are required for the production of pyrimidine, but not pyrimidine-like compounds [51]. Furthermore, the laccases of S. cerevisiae, S. paradoxus, and C. albicans were inhibited by PLCs, while other studies demonstrated that PLCs can bind to the laccase and cause laccase inhibition [22, 43, 51, 52]. These data suggest that the two fungal species are different in their ability to bind to laccases.

The studies on the antagonism of Hsp90 and DsRed with LAP were inconclusive. In the presence of PLCs, the LAP activity was moderate, whereas the Hsp90 activity was high. In contrast, in the presence of PLCs the LAP activity was high and the activity of LAP was low. The low LAP activity of the Hsp90-binding protein kinase was attributed to the lower binding affinity of Hsp90 to the LAP [13]. However, the PLCs did not inhibit the activity of Hsp90 by PLCs. PLCs are considered to be the least harmful inhibitors of Hsp90, because they are the most competitive inhibitors of Hsp90 and prevent Hsp90 from binding to Hsp90, which is not possible in a laccase inhibitor. Furthermore, the Hsp90-binding protein kinase is a low-affinity inhibitor of Hsp90, but not of the LAP [13]. The Hsp90-binding protein kinase is a competitive inhibitor of Hsp90 and has a lower affinity than LAP [13].
