# Partial purification and evaluation of aspartic peptidase and phosphoinositideasas as potential antimicrobial agents
Quaglee Dragontacos


## Abstract
Nosocomial infections caused by fungi have increased significantly over the past few decades. This scenario is most likely due to the advent of the medical treatment of immunocompromised patients, including invasive aspergillosis. Immunocompromised patients, especially those with organ or hematopoietic transplant, have an increased risk of complications. Furthermore, the number of immunocompromised patients increases, as well as the number of opportunistic pathogens, leading to increased morbidity and mortality. Therefore, proper clinical, endoscopic, and microbiological examinations are mandatory for the diagnosis of aspergillosis caused by these harmful human fungal pathogens. These examinations should cover both hematologic, leukocyte, and hematopoietic cells, and should also include biopsy findings, such as evaluation of a thermograph, smear, and culture, of the lesions. An early diagnosis of aspergillosis due to Aspergillus species, should be established with proper clinical, histologic, and microbiological examinations.


## Introduction
The Aspartic peptidase (ATP) system was developed by using aspartic peptidases (ASPs) which can be found in humans and in non-human primates and are responsible for the production of the peptidase active products and so provide adequate protection against oral pathogens and oral and oropharyngeal pathogen. To date, more than 20 ATP enzymes have been identified and characterized from human and non-human primates, mostly from the gastrointestinal tract, including the polypolis (Perlrothopsis poae) and guinea pigs (Perlrothopsis graminis), and from saliva (C. elegans) (Li et al. 2003; Limon et al. 2005; Samuelson et al. 2007; Pusch et al. 2007; Wijayawardene et al. 2010; van der Linde et al. 2011; Kumar et al. 2011; Agustin et al. 2013; Wang et al. 2013; Zhang et al. 2013; Du et al. 2013; Wang et al. 2015). However, the molecular mechanism of ASP production has not been fully characterized in humans.

The A. poae ASP and its peptidase enzymes have been characterized using a kinetic kinetic analysis protocol, where the ASP and peptidase enzymes are separated by a chromatographic separation. The ASP and peptidase enzymes are purified from the medium and subjected to purification and purification by physical method. The ASP and peptidase enzymes are separated by a porous surface and subjected to strong adsorption by an optical filter. The ASP and peptidase enzymes are separated by a unique polymer matrix and then purified by a circular dichroism method. The optical filter is connected to a primary dyeing solution and irradiated. Secondary Filtration Techniques (CDT) or optical tweezers (OT) are used to sterilize the ASP and peptidase enzymes and the optical filter is connected to a primary dyeing system.


## Methods
Asp-1 strains were grown on SCA at 37°C and transferred to MM at 5% CO_. Yeast cells were harvested by centrifugation at 3000 × g for 20 min at 4°C and washed twice with sterile PBS. The cell pellets were resuspended in 1 ml of sterile PBS and diluted to an OD_600 = 0.2 in sterile PBS. For each test, 10 µl of each dilution was plated on SCA and the plates were incubated at 37°C for 48 h. Percent growth inhibition was determined by taking serial two-fold dilutions of the cell suspension and incubating at 37°C for 24 h. In each experiment, the most active compounds were used in the final concentrations tested.

Antifungal assays of Asp-1 and Asp-2
The Asp-1 and Asp-2 gene deletion mutants were constructed by PCR with the primers PstA/PstB and PstA/PstB, respectively, and as previously described [18]. Asp-1 and Asp-2 mutant strains were used as a control for growth inhibition and antifungal activity. PstA and PstA genes were replaced with either PstA/PstA or PstA/PstA/PstA/PstA in A. fumigatus (Fumigatus ATCC 22019) and A. flavus (Fumigatus ATCC 22019). The pyrG gene was replaced with pyrG in A. fumigatus (AF293) and A. terreus (AF293). Asp-2 and Asp-3 genes were deleted in the Asp-1 mutant and A. fumigatus (AF293). The PstA gene was replaced with pyrG in A. fumigatus (AF293). Asp-3 was deleted in the Asp-2 mutant and A. fumigatus (AF293). Asp-4 and Asp-5 genes were deleted in the Asp-4 mutant and A. fumigatus (AF293). All mutants were tested for growth and antifungal activity.


## Results

The . fumigatus gene was found to be expressed in the chlamydial stage of the phagocytic cell cycle, which was most likely the case for the early aspartate aminopeptidase (FAS) that was expressed at least once in A. fumigatus during phagocytic process [43]. A. fumigatus and A. nidulans were found to produce FAS and aspartate aminopeptidase respectively at different times after cell induction [44]. As shown in Fig 4, both of the A. fumigatus FAS and aspartate aminopeptidase genes were expressed in both the chlamydial and phagocytic cell cycle phases of the phagocytic cell cycle in A. fumigatus.

FAS and aspartate aminopeptidase are expressed in A. fumigatus in vitro

The in vitro expression of FAS and aspartate aminopeptidase genes was evaluated using real-time qPCR. The expression of FAS and aspartate aminopeptidase was evaluated using the In-cell Image Studio Exosome Profiler (Illumina) v.4.2 platform (http://imagingstudio.mit.edu/) (Fig 5). The expression of FAS and aspartate aminopeptidase in the Aspergillus mutant strain was detected at a concentration of 10 µg/ml and 1 µg/ml, respectively.

To investigate the expression of the three genes in A. fumigatus, RNA-Seq was performed using the Illumina Genome Analyzer version 2.0 and the qPCR results showed that the expression of FAS and aspartate aminopeptidase was detected at both the 5' and 3' ends of the A. fumigatus genome [15].

As shown in Fig 6, the expression of FAS and aspartate aminopeptidase genes was increased in the Aspergillus mutant strain compared to the A. niger and A. fumigatus strains, which suggested that the genes were expressed at high levels.


## Discussion
In the present study, we isolated an additional 7,977 species, which represent 62.7% of the total reads in the NCBI nrLSU Genome Database (NCBI, http://www.ncbi.nlm.nih.gov/nrg? task=search& report=NCBI); of which, 10,873 were found to be monophyletic, representing 2.5% of the reads in the NCBI database and 2.8% of the total reads in the NCBI nrLSU Genome Database (NCBI, http://www.ncbi.nlm.nih.gov/nrg? task=search& report=NCBI); 2,865 were represented by unknown species, representing the largest number of reads in the NCBI database (5.2% of the reads), and 2.5% in the NCBI nrLSU Genome Database (NCBI, http://www.ncbi.nlm.nih.gov/nrg? task=search& report=NCBI). Our analysis showed that the number of novel species represented by the new species in the NCBI database is close to the number reported in GenBank, which represents 95.2% of the reads in the NCBI database (NCBI, http://www.ncbi.nlm.nih.gov/nrg? task=search& report=NCBI). We found that most of the new species in NCBI databases are pathogenic (69.6% in C. elegans and 45.1% in C.
