# Pectinase Required for Fabric Acid Production in the Rice Blast Fungus, Magnaporthe oryzae
Quaglee Dragontacos


## Abstract
In this study we isolated the fungal species Microdochium sp. from soil and plant debris in the Baotia Terrestre forest in Portugal, using a combination of both culture and molecular methods. The fungus was isolated using both culture and molecular methods and was identified based on the multilocus sequence typing (MLST) analysis of the nuclear ribosomal DNA (LSU) and the translation elongation factor 1-a (EF-1a), the largest and second largest subunits in ß-tubulin were found. Culture-independent identification of the fungus resulted in isolation of several species from different soil habitats, including typical fruiting bodies (SBM) and pilea bread (CPB). Microdochium sp. was identified by the high-performance liquid chromatography mass spectrometry (HMSI) assay, and two new species were isolated: Microdochiapilus sp. from the same site (NSK) is shown in Fig 4a, and Microdochium sp. from the same site (SPM) is shown in Fig 4b. The new species Microdochium sp. were identified by the LARSIA and CLSI assays.


## Introduction
Invasive fungal infections (IFIs) are an important cause of morbidity and mortality. The incidence of IFIs is increasing due to immunosuppressive therapies such as chemotherapy, organ transplantation, immunosuppressive drugs, and infections such as tuberculosis (1). Treatment of these infections has expanded the number of patients with weakened immune systems and ability to perform many activities of the immune system (2, 3). The immune system is critical for the host defense against many pathogens, including bacteria, viruses, fungi, and protozoa. In this review, we will focus on how the immune system is involved in the management of IFIs.

The opportunistic fungal pathogen Candida albicans is an opportunistic pathogen that causes mucosal and/or systemic candidiasis. It is a member of the human microbiome and can cause life-threatening systemic candidiasis. The incidence of candidiasis is increasing because of the rise in the number of immunocompromised patients and the use of immunosuppressive drugs (4, 5). The worldwide prevalence of C.


## Methods
albicans, . parapsilosis, C. tropicalis, and C. krusei) were used as the commercial standards. Amino acid sequences of Pectinase C_D-pectinase (Table 1) were prepared by PCR using a 27S rRNA gene fragment as the template. The DNA was amplified from yeast genomic DNA with the D1/D2 region of the 16S rRNA gene, using primers pectin1 (5'-CCTTCAGAGTCCGTGAACCTTC-3') and pectin2 (5'-CCGGCACCCAGCTCCACCTC-3') [63] in combination with a D1/D2 region of the ITS rRNA gene as the template. The PCR program consisted of a denaturation at 95 °C for 2 min, followed by 35 cycles of denaturation at 95 °C for 1 min, annealing at 56 °C for 1 min, and extension at 72 °C for 1 min, followed by a final extension at 72 °C for 10 min. The PCR products were purified and then sequenced. The sequences of the synthesized Pectinase C_D-pectinase were obtained from the C. albicans, C. parapsilosis, and C. krusei Pectinase C_D-pectinase databases (accession number: CBP_ID:2297).

Identification of Exon-Associated Pectinases
The sequences of exons from Pectinase C_D-pectinase (Table 1) were obtained from the Saccharomyces Genome Database (SGD), which is a fasta resource for annotated proteins of many fungal species (http://www.yeastgenome.org/). The sequences of the exons from Pectinase C_D-pectinase were obtained from the C. albicans and C. parapsilosis genomes using the online version of the P. chrysosporium genome browser [64]. The P. chrysosporium genome browser is a web-based tool that integrates multiple databases such as the NCBI protein database, HMM, and several other databases, such as the NCBI non-redundant (nr) database. The P.


## Results
It has been shown that . sojae does not produce any enzyme during the late stages of the phytoalexin stage and thus may be considered as a potential food source for P. sojae [13]. Therefore, the complete P. sojae fermentation process is recommended.

3.3. Antifungal Activity of P. sojae Fermentations
P. sojae fermentation of 2.5 g/L of sugar cane molasses, sugarcane bagasse, and 1.5 g/L of sorbitol showed a growth inhibitory effect on C. neoformans and C. albicans. The fungicidal effect of P. sojae on C. neoformans was observed by the C. neoformans growth inhibition zone area and the radial growth inhibition zones of the black yeast C. neoformans. In contrast, the P. sojae fermentation of sorbitol showed a more pronounced fungicidal effect on the black yeast C. neoformans and C. albicans. The zone of inhibition of the black yeast C. neoformans was approximately 6-8 mm, the zone of inhibition of the black yeast C. neoformans was approximately 1.8-2.0 mm, and the zone of inhibition of the black yeast C. neoformans was approximately 2.0-2.5 mm. In addition, the zones of inhibition of the black yeast C. neoformans were approximately 1.8-2.0 mm, the zones of inhibition of the black yeast C. neoformans were approximately 2.0-2.5 mm, and the zones of inhibition of the black yeast C. neoformans were approximately 1.8-2.5 mm.

3.4. Antifungal Activity of P. sojae Fermentations
P. sojae fermentation of 2.5 g/L of sugar cane molasses, sugarcane bagasse, and 1.5 g/L of sorbitol showed a growth inhibitory effect on C. neoformans and C. albicans. The growth inhibition zone area of the black yeast C. neoformans was approximately 6-8 mm, the zone of inhibition of the black yeast C. neoformans was approximately 1.8-2.0 mm, and the zone of inhibition of the black yeast C. neoformans was approximately 2.0-2.


## Discussion
Although not detected in any other fungal species analyzed in this study, we cannot exclude the possibility that the enzyme was not present in the fungal cell wal. The isolated enzyme in the rice blast fungus Magnaporthe oryzae is a product of two distinct enzymes, the ERG6 and the ERG12, which are essential for fungal growth and conidiation (24). The expression of ERG6 is regulated by the ERG12 complex (25). Interestingly, the mRNA level of ERG6 was detected in the rice blast fungus Magnaporthe oryzae, which is an ortholog of the yeast ERG6, and the expression of ERG12 was induced by Hwang et al. (26). The expression of ERG12 was up-regulated in the rice blast fungus Magnaporthe oryzae, which is a homolog of the yeast ERG6, and the expression of ERG12 was up-regulated in the rice blast fungus A. fumigatus, which is a homolog of the yeast ERG12 (27). Therefore, it is possible that the gene encoding the ERG6 gene is involved in the secretion of UDP-Glucose, which is a constituent of rice blast fungus. UDP-Glucose is synthesized from glucose-6-phosphate by a UDP-glucose-1-phosphate dehydrogenase (GlcNAc) in fungi (28). UDP-glucose is then converted to UDP-glucose-5-phosphate by a UDP-glucose-4-phosphate dehydrogenase (UDP-GlcNAc) (29). UDP-glucose is then converted to UDP-GlcNAc by the UDP-glucose-5-phosphate dehydrogenase (UDP-GMP) (30). UDP-Glucose is then converted to UDP-UDP-5-phosphate by the UDP-5-glucose-3-phosphate dehydrogenase (UDP-GMP) (31). UDP-Glucose is then converted to UDP-5-glucose by the UDP-5-glucose-3-phosphate dehydrogenase (UDP-GMP) (32).
