# Phosphate compensation explains spliceosomal recombination in the pathogenic fungi Ustilaginina and UstilagininaULT
Quaglee Dragontacos


## Abstract
.


## Introduction
Hemophagocytic lymphohistiocytosis (HLH) is a rare but potentially life-threatening disease characterized by a clonal depletion of lymphocytes and granulomas in the brain. The incidence of HLH is 1.4% to 3.8% in the general population, and is associated with a high rate of morbidity and mortality (1). The most common manifestations are lymphopenia, subcutaneous or visceral disease, and cerebral infarction, lymphadenopathy, and, rarely, cryptococcal meningitis (2). The most common pathological sites of HLH are lymph nodes, spleen, lymph nodes with an abnormal platelet density, and basal ganglia and granulomas (3). These sites are often multinucleated in nature. They are usually located on the basal ganglia and spleen (4, 5). The basal ganglia is composed of four main segments: the microvilli, which contain granules, and the mesenteric bone marrow, which contains cells of the lymphatic circulatory system (4, 5).

Subcutaneous HLH is rare and represents an uncommon form of inflammatory demyelinating polyneuropathy. It is characterized by chronic skin lesions that are not controlled by a functional immune system, resulting in permanent skin lesions in the CNS, accompanied by a progressive neurologic deficit and the formation of peripheral lymphoid progenies. The clinical features are similar to those of HLH in that the lesions can be controlled with an active immune system, but there are clinical manifestations that are nonspecific and nonspecific in clinical presentations. We present a case of subcutaneous HLH with a diffuse spinal infection and diffuse nodular lymphohistiocytosis in a patient with diffuse low-grade spinal cord disease.


## Methods
IMATE-1, a monokaryotic fungus originating from U. maydis [28]. To determine spliceosomal and parasite gene expression levels, we used the Gene Expression Omnibus (http://www.ncbi.nlm.nih.gov/geo/) database and the software program Gene Expression Omnibus (http://www.ncbi.nlm.nih.gov/geo/). In addition, we used a previously described method to measure the expression of two genes: the ß-actin gene and the interleukin-4 (IL-4).

Transcriptional data
For all of the experiments, RNA was extracted from spore samples (10^6 spores/mL) in a total volume of 0.2 µL using the RNeasy Plant Mini Kit (Qiagen). The RNA was treated with DNase I (Qiagen). To detect RNA polymerase activity, the RNA was reverse transcribed using the SuperScript III First-Strand Synthesis SuperMix (Invitrogen) according to the manufacturer's protocol. The resulting cDNA was quantified with the Quant-iT PicoGreen dsDNA Assay Kit (Invitrogen). The qPCR experiments were performed in a StepOnePlus Real-Time PCR System (Applied Biosystems, Foster City, CA, USA). All reactions were performed in triplicate in a total volume of 20 µL. For all experiments, we used an iCycler iQ 7300 Real-Time PCR system (Bio-Rad, Hercules, CA, USA). The primers for each gene were designed using Primer3 software (http://www.r-project.org/). The values for the primers were set to 0.3 (positive control) and 0.8 (negative control). The results were confirmed with a StepOnePlus iQ 7300 system (Applied Biosystems, Foster City, CA, USA) and an ABI Prism 7300 Fast Real-Time PCR system (Applied Biosystems, Foster City, CA, USA).

Transcriptomic data
For the quantification of RNA polymerase activities, RNA was isolated from spore samples using the RNeasy Plant Mini Kit (Qiagen). The RNA was treated with DNase I (Qiagen). The cDNA was quantified with the Quant-iT PicoGreen dsDNA Assay Kit (Invitrogen).


## Results
It was shown that a functional phosphoproteomics strategy is required for successful protein phosphatase gene conversion, and phosphoproteomics analysis indicates that Ustilaginina and UstilagininaULT1 and UstilagininaULT2 encode recombinant protein isoforms with no splicing product. However, our results indicate that the U. maydis gene in this set of genes does not have any splicing products. The recombinant protein isoforms contain one or more phosphatase genes. The phosphatase gene is encoded in the U. maydis genome with a predicted GPI-anchored protein sequence. As a result of the analysis, U. maydis protein isoforms with no splicing products are inactivated (Figure 1).

Functional characterization of U. maydis gene transcripts
To further explore the function of U. maydis gene transcripts, transcriptome analysis was performed with the U. maydis-RFP/RFP marker. As shown in Figure 2A, the majority of the transcript expression in U. maydis was observed in the cytoplasm, but there was also a large proportion of transcript in the nucleus. In addition, the expression levels of some genes were not detected at all. The transcripts of genes involved in DNA replication, spindle assembly, translation and oxidative stress were also not detected.

The transcript abundance of the U. maydis gene encoding protein phosphatase (S3 Table), is shown in Figure 2B. A transcript of protein phosphatase (S3 Table) was also identified in the U. maydis genome, but it was not detected in the cytoplasm. The transcription levels of the protein phosphatase (S3 Table) were also not detected in the U. maydis genome. Finally, the U. maydis gene encoding RNA polymerase (RFP) was also identified in the U. maydis genome. It was not detected in the cytoplasm. However, the transcription levels of the protein phosphatase (S3 Table) were also not detected in the U. maydis genome.

A comparative analysis of U. maydis transcriptome
The transcript levels of the U. maydis gene encoding protein phosphatase (S3 Table) were detected in the U. maydis genome, and they were not detected in the cytoplasm. However, the transcript abundance of the U.


## Discussion
Our phylogenetic analysis shows that UstilagininaULT1 is polyphyletic, and that the Ustilaginaceae is not monophyletic, but rather represents two separate clade.

The Ustilaginaceae complex is distinct from Ustilaginaceae by having a sister group relationship to the Ustilaginaceae, and by having a higher degree of phylogenetic concordance to the other families. The Ustilaginaceae is monophyletic, and is separated from the other families by the number of members, whereas the Ustilaginaceae is monophyletic, and forms a well-supported clade with the other families. The two groups are separated from the other families by the presence of a few members in the Ustilaginaceae and by the presence of the other families.

The Ustilaginaceae is a well-supported monophyletic group, and contains members from a number of families, such as Ustilaginaceae, Fusarium, Sporobolomyces, and Ustilaginaceae, and the other families are not supported in this study. Ustilaginaceae, although represented by two species, is distinct from other Ustilaginaceae species in its polyphyletic nature, and contains a few members, such as the members of the Ustilaginaceae, which are often used to distinguish the two families in fungi. The polyphyly of Ustilaginaceae has not been observed in other fungi, and therefore, the phylogenetic relationships of the two families have been inferred using the species trees obtained from the two phylogenetic analyses.

The Ustilaginaceae has four species in the Ustilaginaceae complex, but the species relationships of the Ustilaginaceae and the Ustilaginaceae complex have not been investigated. The Ustilaginaceae is characterized by the presence of four species, and members of the Ustilaginaceae are polyphyletic. The Ustilaginaceae is polyphyletic, and is separated from the other families by the number of members. The Ustilaginaceae is not monophyletic, and has an opposite relationship to the other families, which are represented by the Ustilaginaceae.
