# Phylogeny of a remarkable new order in oleaginous yeasts: Nelumbo (Arecaceae, Ascomycota)
Quaglee Dragontacos


## Abstract
Cryptococcus gattii is the leading cause of fungal meningitis in immunocompromised patients worldwide. This human pathogen infects healthy mammalian hosts, primarily in the lungs and meninges. Despite the current therapeutic approaches for cryptococcosis in the central nervous system, recent advances in the treatment and understanding of this central nervous system infection have made available a platform for discovery of cryptococcosis-specific compounds with a potential antifungal activity. In this review, we discuss the current strategies for treating cryptococcosis, discuss the challenges of understanding the innate immune mechanisms that control the host response to cryptococcosis, and consider the future perspectives of developing novel therapies targeting fungal virulence factors and pathogenicity in the central nervous system.


## Introduction
2017. It is widely distributed in the tropics, subtropics, and temperate regions and is commonly found in soil, plants, and detritus of the bark, leaf, and fruits of angiosperm trees, which are primarily grown in tropical regions. The genus contains more than 50 species, and among them only few are commercially cultivated and used for food or for other purposes.

The genus Eumelcoridea was initially proposed by Horsfall & Priest (H. & Priest 1973), a pioneer group in the oleaginous yeasts and diatoms of the family, and later extended to other eukaryotic taxa, including oleaginous yeasts (E. bakanae, E. gympoixana, E. nomographus, E. minutispora, E. oleatum, E. portilloides, E. violaceum, E. sorbitosa, E. tremula, E. sinensis, E. lusitanica, E. resta, E. sinensis, E. tannophilus, E. vietnamensis, E. washburnii, and E. vermis). However, H. & Priest (1973) was unable to find valid taxonomic criteria for Eumelcoridea. Since then, a number of other genera in Leucotrophoda have been described from the genus Eumelcoridea, which have been either associated with or synonymized with the genus in other families (E. bakanae, E. gympoixana, E. nomographus, E. portilloides, E. violaceum, E. sinensis, E. lusitanica, E. vietnamensis, E. washburnii, E. sinensis, E. portilloides, E. vietnamensis, E. sinensis, E. portilloides, E. vietnamensis, E. sinensis, E. vietnamensis, and E. vermis, have been reported (Pereira et al. 2015, Liu et al. 2016, Zeng et al.


## Methods

Isolation of oleaginous yeast cells from the culture media
The genomic DNA of each isolate was extracted using a standard bacterial DNA isolation protocol (Qiagen) following the manufacturer's protoco. The genomic DNA was also extracted using a standard DNA extraction protocol (Qiagen).

Library preparation
The genomic DNA was extracted using a standard DNA extraction protocol (Qiagen) following the manufacturer's protocol. The genomic DNA was also extracted using a standard DNA extraction protocol (Qiagen). The resulting DNA was precipitated with methanol and the genomic DNA was purified by PCR. The resulting DNA was dried and stored at -20Â°C.

Library preparation
The sequencing of the NELBOV sequence was performed on an Illumina HiSeq 2000 (Illumina, San Diego, CA, USA). The reads were mapped to the O. sativum genome using the Mapping Tool in MOTHUR (v0.9.0). The aligned reads were mapped to the genome of O. sativum using the Mapping Tool in MOTHUR. The reads were then filtered to remove adaptor reads and to remove low-quality reads. The filtered reads were then mapped to the genome using the Mapping Tool in MOTHUR. The mapped reads were then filtered to remove low-quality reads and to remove high-quality reads. The filtered reads were then mapped to the genome of O. sativum using the Mapping Tool in MOTHUR. The mapped reads were then filtered to remove low-quality reads and to remove high-quality reads. The mapping quality was then verified using the mapping quality check tool in MOTHUR.

Library preparation
The sequencing of the ITS region of the yeast isolates was performed on an Illumina HiSeq 2000 (Illumina, San Diego, CA, USA). The sequencing was performed on an Illumina HiSeq 2000 (Illumina, San Diego, CA, USA). The reads were aligned to the O. sativum genome using the Mapping Tool in MOTHUR. The mapped reads were then mapped to the genome of O. sativum using the Mapping Tool in MOTHUR. The mapped reads were then filtered to remove low-quality reads and to remove high-quality reads. The filtered reads were then mapped to the genome of O. sativum using the Mapping Tool in MOTHUR.


## Results
tetraspora (subge. Metschnikowia) was performed using a well-defined backbone node (Fig 4) and a well-defined non-synonymous topology (Figs 3 and 4). A preliminary analysis of the four most similar and well-defined groups (including the Nellorectum clade) revealed the same topology (Fig 4) as the earlier phylogenetic analysis based on the Nellorectum clade (Fig 3) but the Nellorectum clade was subjected to much more extensive branch-recipitations, which largely resulted in a very different topology (Fig 4). The Nellorectum clade is sister to the Nellorectum spp. (Fig 3), and it was also subjected to the most extensive split of all the Ascomycota spp. in the phylogenetic tree (Fig 4). A similar topology was observed for N. oleaginosum (subgen. Oleaginosum) and N. dothidea (subgen. Dothidea). However, the Nellorectum clade had a much narrower backbone node than the Nellorectum spp. and the Nellorectum spp. were also subjected to extensive split, whereas the Nellorectum clade was subjected to the most extensive split of all the Ascomycota spp. (Fig 4). This is probably the reason why the Nellorectum clade was more highly supported than the Nellorectum clade in the phylogenetic tree (Fig 3).

In contrast to the two Nellorectum clades, the Nellorectum clade is more compact and well-defined in the ITS region (Fig 3). In the species of the Nellorectum clade, all the backbone nodes were well-defined in the ITS region (Fig 4), although the backbone node of the Nellorectum clade was poorly defined in the LSU region (Fig 3). In the ITS region, the backbone node of the Nellorectum clade was well-defined in both the ITS and LSU regions (Fig 3).


## Discussion
sinensis (Cyphellophila. In C. sinensis, there is a remarkable conserved, multi-locus phylogeny (Fig 6), which reflects a major expansion of the subphylum Oomycetes and an expansion of the subphylum Ascomycota. Most of the newly sequenced strains have also been observed to be distinguished from C. sinensis (i.e., >10% identical); however, the C. sinensis genome is much larger (38.5 Mb vs. 7.2 Mb, respectively). This size is of great importance as C. sinensis is one of the smallest species in the Oomycetes (fig 6). The expanded subphylum and the expansion of the subphylum Ascomycota as well as the expansion of the subphylum Basidiomycota are all well documented in Oomycetes (fig 6). A. fumigatus is the only species identified in the C. sinensis genome (fig 6). Thus, these two species are distinct and represent two major lines of evidence for the lineage-specific expansion of this order in oomycetes.

In addition to the subphylum Oomycetes, two other orders have been expanded in the C. sinensis genome (fig 6). A. bieneusi is the only species with an expanded subphylum (fig 6). C. heterostrophus is the only species in the C. sinensis genome with an expanded subphylum (fig 6). Therefore, C. sinensis represents a new order in the Oomycetes, and the expansion of this order in oomycetes has been well documented.

C. sinensis and C. heterostrophus are most closely related to C. heterostrophus. However, in C. heterostrophus, the two species share many of the same phenotypic traits and are associated with the same ecological niches and host niches (fig 6). These three species may represent two distinct clades in the Oomycetes, or may represent two distinct lineages in the C. sinensis genome. These three species also differ in their genomes, with more than two-thirds of the total genome sequence shared with C. heterostrophus and a large number of conserved genes (fig 6).
