# Phylloporialonicerae (Hymenochaetales, Basidiomycota), a new species on Lonicerajaponica from Japan and an identification key to worldwide species of Pseudomonas reinhardtii
Quaglee Dragontacos


## Abstract
The genus Neocosmospora (Taphrinomycotina) is characterised by polygonal structures, many of them shared with fungi. The genus includes many genera and their host-specific lifestyles, and are important for plant pathogens such as Aspergillus niger, for which nival mycorrhizal (Mycorrhizaceae) is the only host-specific name for this group of fungi. The genus is mainly found in subtropical southern Europe with North America as the climatic region with the highest prevalence of N. reinhardtii. The genus is distributed in the United States, with some exceptions, especially to California, where it has been described as a tropical fungus. We studied the morphology and life cycle of two North American isolates of N. reinhardtii and hypothesized to what extent the fungus may have become a pathogen of N. mycorrhizae, thereby altering the natural microbial succession of this species in the American continent. The two isolates were indistinguishable in morphology and life cycle, but grew at higher rates and were fertile at the two ends of the pathogen distribution spectrum compared with the other arms. The two isolates showed the same sporulation patterns, and the phenotypes were clearly more similar to those of N. reinhardtii than to those of A. baumannii. The three isolates were fertile in vitro, and also produced fertile colonies in mycelium when grown in the presence of N. reinhardtii. Conventional spore staining showed that A. baumannii and A. baumannii were in a much stronger sexual phase, a prerequisite for the formation of fertile colonies. However, the three isolates did not form fertile colonies in mycelium when grown in the presence of N. reinhardtii, and the patterns were different from those of A. baumannii. The two isolates did not have a strong sexual phase, and hence, it is possible that the sexual phase was not sufficiently primed to form fertile colonies.


## Introduction
The genus Laccaria includes approximately 150 species, of which at least 50 species have been described as plant pathogen of temperate and tropical trees. The genus Laccaria, as distinguished from Laccaria angustis, is characterized by the colourless to black conidiophores and the presence of thick-walled, aseptate conidia. Laccaria angustis is commonly found in pine, oak and evergreen forests, and is mainly associated with leaf spots, powdery mildew, dirty white spots on leaves and bark, and leaf blotch (Santoyano et al. 2013; Sung et al. 2014; Cossart et al. 2014; Maza et al. 2014). In contrast to the laccariids, Laccaria angustis is reported to be rare but endemic in many areas of Brazil, especially in the southeast and northeast of the country (Sung et al. 2014). The genus Laccaria angustis was traditionally described as a saprophytic fungus that produces conidia that are forcibly discharged from conidiophores. These conidia are sterile and can be seen directly on the surface of the conidiophores (Sung et al. 2014). Recently, several studies have shown that Laccaria angustis was the most common fungus in Brazilian forests (Kronstad et al. 2015; Mantoy et al. 2017; Almeida et al. 2017), and that there was a correlation between the number of described forest species with the number of known cases (Mantoy et al. 2017). However, little is known about the biology, pathogenicity, and evolutionary relationships of the genus Laccaria angustis.

The genus Laccaria angustis is widely distributed and is usually associated with saprophytic, fruit-brile fungi (Porcel et al. 2012; Maza et al. 2014). However, in recent years, several reports have described the occurrence of new species of Laccaria angustis on dead organic material (Beswers et al.


## Methods
The type of phyla involved in this study are shown in Table .

DNA extraction, PCR and sequencing
Total genomic DNA was extracted from fungi using a protocol previously described [31]. The sample was used for the PCR amplification using the primers ITS5 and ITS4 [32] and the DNA was amplified using the E.Z.N.A. PCR Kit (Omega Bio-teck, USA). The PCR was performed in a Veriti 96-Well Thermal Cycler (Applied Biosystems, USA) using the following thermal cycling program: 4 min at 95 °C followed by 30 cycles of 20 s at 95 °C, 1 min at 60 °C and 20 s at 72 °C. The DNA was cleaned up with a MasterPure™ Yeast DNA Purification Kit (Machery-Nagel, Germany) and the purified DNA was used for the sequencing.

Fungal sequences were aligned with MAFFT v. 7 [33] and the ITS rDNA sequences were extracted from the aligned sequences with the BLAST algorithm [34]. The phylogenetic trees were constructed with MEGA v. 6.06 [35] and were visualized with TreeView v. 4.0.8. [36]. The sequences were deposited in GenBank and the accession numbers for the species named in the phylogenetic tree are listed in Table 1. The sequences are deposited in the GenBank and their accession numbers are listed in Table 2.

Molecular identification
DNA was extracted from fungal mycelia grown on PDA and in liquid mycelium of a single colony of each species. The mycelium samples were spread on PDA, which was incubated at room temperature for 7 days. Mycelium samples were then transferred to a sterile PDA plate, and the isolates were labelled using the Multi-Symbion System II (MDS) (Micro Biosciences, UK).

Molecular identification
DNA from each isolate was extracted by using the QIAamp^TM DNA minikit (Qiagen, Germany). The DNA extraction was performed by using the standard QIAamp^TM DNA minikit protocol as described by the manufacturer (Qiagen, Germany).


## Results

Phylloporia Sidac & .P. Syd., Mycol. Progr. 7: 481. 1951.

Ascomatal state unknown. Conidiophores micronematous, septate, branched, cylindrical-oblong, straight, geniculate-sinuous, pale to medium brown or brownish, verrucose, 100-200 × 3-4 µm. Conidiogenous cells terminal, polyblastic, proliferating sympodially, geniculate, subhyaline to pale brown, verrucose, or subhyaline at the apex, containing up to four annellations; collarette not flared. Conidia solitary, hyaline to pale brown, smooth, cylindrical-oblong, straight to slightly curved, apex subobtuse, base truncate, base truncate, straight to slightly curved, base tapering to a bluntly rounded or flattened apex, hila not protuberant, (0.5-)1-2 µm diam, dome cylindrical, slightly to distinctly thickened, slightly darkened-refractive; microcyclic conidiogenesis not observed.

Type species: Phylloporia genera (Pers.) Sidac & G.P. Syd., Mycol. Progr. 7: 481. 1951.

Notes: The genus Phylloporia (Pers.) is based on a genus of plant pathogenic fungi (Pers.) Pers., who is also known as Phylloporia genera (Pers.) S.F. Ellis [1]. However, in the present study the name Phylloporia was used as the type of the genus, and the generic name Phylloporia is no longer valid. The generic name Phylloporia is therefore proposed. The genus Phylloporia (Pers.) is now only accommodated in the family Phylloporia.

Phylloporia berewardii (Pers.) S.F. Ellis & Co., New York, Mycopath. nov. 992. 1879. Fig 36.

Basionym: Sphaerophora berewardii Pers., Syn. Meth. Fung. (Abellini) 2: 367. 1882.

= Phoma berewardii (Pers.) S.F. Ellis & Co., New York, Mycopath. nov.


## Discussion
In this study we have characterized the genomes of a variety of eukaryotes and found that a large number of genes encode proteins with putative functions in the cell. We found that over half of the genes analyzed are up-regulated under nutrient stress conditions, which is in good agreement with the results from the previous studies conducted in laboratory conditions [32, 37, 39]. We found that the transcription factors involved in stress response, such as SreA and Ste12, are up-regulated at the start of gene expression in the wild-type strain. We have also observed that the expression of SreA is up-regulated during the early stages of the stress response, when the transcription factor Ste12 is down-regulated. These results indicate that a novel strategy of coping with starvation is possible for a strain with extremely low fitness and a high genetic redundancy.

In the present study, we identified a number of genes that were up-regulated at the start of the stress response, and characterized the roles of these genes in the pathogenicity of A. fumigatus. We also found that many of the genes involved in energy metabolism, such as NfsA, NfsB, and NfsC, are up-regulated at the start of the stress response. This indicates that the stress response in A. fumigatus is regulated by several genes involved in energy metabolism and that these genes play a major role in the pathogenicity of A. fumigatus. Our results indicate that the expression of the genes involved in energy metabolism, such as NfsA and NfsB, are up-regulated at the start of the stress response, and that these genes play a major role in the pathogenicity of A.
