# Phylogenetic Analysis of Lassa: An Application of the SpeciesSynNex
Quaglee Dragontacos


## Abstract
This review discusses the biology, pharmacology, and biological activities of essential oils obtained from the aerial parts of Piper race (Aphol) and Piper bark (Brassica) (Aphol) plants. It also discusses the safety and efficacy of essential oils obtained from the aerial parts of Piper race (Aphol) and Piper bark (Brassica) plants. The crude extracts were evaluated for their cytotoxic, anti-proinflammatory, antioxidant, and antiproliferative activities.


## Introduction

Biomass based phylogenetics of the Dendryphion genus has been used to study the phylogeny of genera within the Mycosphaerellacea. Several genera have been considered as important plant pathogens, including Phaeosphaeria, which causes foliar diseases on many important plant hosts, as well as Phaeodidyphila, which causes fruit rots on many citrus plants, and Phaeosphaeria, which causes anthracnose on many important citrus hosts [38]. In contrast, some Phylogenetic SpeciesSynNexus species, including Phaeosphaeria, Phaeosphaeria pseudopilosa, Phylosphaeria stellatae and Phylosphaeria osmophila, were considered to be of ecological interest and are presently accepted as biocontrol agents for citrus diseases [37, 38]. Species in the genus Phylosphaeria are mainly saprobic and live mainly on dead leaves and rhizomes and on living plants. They can also cause diseases on other plant hosts, including trees, herbs, and/or shrubs, in addition to some other economically important hosts, such as grapevines, tobacco, and oat melon. Many Phylosphaeria species are of agricultural importance, which produce significant economic losses in the United States and other agricultural markets [39]. Species in Phylosphaeria s. str. have been shown to be pathogenic to several citrus hosts, including citrus fruits and tobacco [40]. Several species of Phylosphaeria have been shown to be of potential economic importance, including Phylosphaeria osmophila, Phylosphaeria stellatae, Phylosphaeria stellatae, Phylosphaeria pseudopilosa, Phylosphaeria osmophila, Phylosphaeria osmophila, Phylosphaeria stellatae and Phylosphaeria stellatae [41, 42].

Phylosphaeria osmophila is the major osmotrophic pathogen of citrus fruit. This fungus infects young citrus fruits and develops a penetration peg that eventually rot the orange fruit and cause a rot that can last for months [43, 44]. Phylosphaeria osmophila infects and kills citrus fruits before they ripens.


## Methods
Two species of Lassa were analysed in different order: Myxobacterium tuberculosis (. tuberculosis) and Dictyostelium discoideum (D. discoideum). The concatenated phylogenetic tree was compared with the tree with the lowest number of parsimony hits, which are provided in the next section.

For the second analysis, phylogenetic trees were analysed by a maximum likelihood (ML) method with Bayesian inference (ML) and maximum likelihood analysis (ML) methods. The initial trees were generated from a sequence alignment of the protein sequences of Lassa and its closest relatives with the sequences available in the NEXUS database, which are available in the online version of the open-source software, (http://www.plantlife.org/). The alignment was done with the same parameters as the first tree, except that a tree length of 1000 bp was used. A maximum likelihood analysis was performed using the MEGA 6 software. The posterior probability of the phylogenetic trees was estimated by the bootstrap method with 1000 replications.

Phylogenetic Tree of Lassa: An Application of the SpeciesSynNexus software, (Lassa, Perugia)
The phylogenetic tree of Lassa was reconstructed in two steps: the first was a nested analysis of the non-synonymous (K-means) and synonymous (K-means) tree, and the second was a concatenated phylogenetic tree. Two species of Lassa were analysed in different order: M. tuberculosis (M. tuberculosis) and D. discoideum. Two phylogenetic trees were analysed in different order: Myxobacterium tuberculosis and D. discoideum. The concatenated phylogenetic tree was compared with the tree with the lowest number of parsimony hits, which are provided in the next section.


## Results
amazonensis gene was identical to the one of . cerevisiae (GenBank accession number: JQ790074) and S. sclerotiorum (GenBank accession number: JQ790079). The ITS region of L. amazonensis gene was identical to that of S. cerevisiae (GenBank accession number: JQ790074). L. amazonensis was originally identified as S. sclerotiorum (Crous et al. 2004, Persoon et al. 2014) based on the phylogenetic analysis of the alignments of the ITS sequences and a nrLSU gene, respectively. However, the phylogenetic analysis of the nrLSU gene and the nrLSU gene from S. sclerotiorum suggested that this species is a species of Sclerotinia sclerotiorum. In addition, the nrLSU gene from S. sclerotiorum was identical to the one of S. cerevisiae (GenBank accession number: JQ790074).

Taxonomy
L. amazonensis is considered a possible pathogen of Sporobolomyces in South Africa (Domsch et al. 2008). L. amazonensis is classified in the L. amazonensis clade in the Nectriaceae, which includes the genus Rhodotorula (Schubert 2010). L. amazonensis can be distinguished from other L. amazonensis species by the phylogenetic analyses of ITS and nrLSU sequences. L. amazonensis is phylogenetically related to S. cerevisiae and S. sclerotiorum in the phylogenetic analysis of the nrLSU and nrITS regions (Schubert 2010). S. sclerotiorum is classified in the L. amazonensis clade in the Nectriaceae, which includes the genus Rhodotorula (Schubert 2010). Therefore, this species can be considered as a potential pathogen of Sporobolomyces in South Africa.

Taxonomy
Kellermann & Voglmayr, Mycotaxon 61: 527. 2003.

Basionym: Sphaeria arachneae Kellermann & Voglmayr, Mycotaxon 61: 527. 2003.

= L.


## Discussion
It is a common tool used by researchers for phylogenetic analysis of Lassa, and is available for the following species: . mosseae, L. pseudoobscura, L. subg. subg. subg., L. pseudobscura, L. subg. subg., L. musae, L. subg. subg., L. subg. subg., L. subg. subg., and L. subg. musae. The system consists of two main components: (1) a binary tree with the HMM method, (2) a branch-by-branch analysis using a concatenated phylogenetic tree and (3) a classification using the combined phylogenetic trees. The Lassa server is used to perform phylogenetic analyses in L. subg. subg. subg., L. subg., L. subg. subg., and L. musae. The generated trees are presented in Fig 1. The phylogenetic trees generated in this study are also used for the classification of Lassa.

Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg. subg. Lassa subg. subg. subg. subg. subg.
