# Planktonic fungal community in Tibetan Plateau sediments and distinct bacterial communities among the pulping rock articelds
Quaglee Dragontacos


## Abstract
C-type lectin receptors (CLRs) are innate immune receptors that recognize a variety of proteins, carbohydrates, and secondary metabolites and regulate their recognition and secretion. In some immune cells, CLRs are essential to control such processes as cytokinesis, complement activation, and inflammation. However, recent studies suggest that some human fungal pathogens, such as Candida glabrata, are also pathogenic fungi. In this review, we discuss the current understanding of CLR signaling in C. glabrata and C. glabrata, and we consider the mechanisms by which CLRs function as important virulence factors. We also explore the interaction between the human fungal pathogen and several fungal mutants, and we suggest strategies for the control of fungal infections by C. glabrata.


## Introduction
Nitrogen (N), oxygen (O), and nitrogen (N) are the most abundant organic solvents, with up to 1.5 Gy of precipitation per liter of water (Yang et al., 2008; Jardetzky, 2009). The concentration of N in water and the amount of O in soil are also influenced by several biological processes, such as exothermic growth, alveolar apoptosis, nitrocellulose synthesis, nitrocellular growth, and morphological changes (Jardetzky, 2009; Bou et al., 2010; Jardetzky, 2010; Wang et al., 2012). The physiological state of N has been shown to be regulated by the formation of nitric oxide (NO) from N and its subsequent oxidation to NO.


## Methods

DNA extraction, PCR amplification, and sequencing
For DNA extraction, the mycelium of mycelium of the taxonomically defined taxa of the D1/D2 (Tables 1 and 2), D2/D3 (Tables 1 and 2), D3/D4 (Tables 1 and 2), and D4/D5 (Tables 1 and 2) subgroups was subcultured onto toitol-glucose-D-glucose-G-agarose (G-GAG-A-GA) agar, washed with distilled water, and dried at 40 °C until a sufficient amount of DNA was isolate. DNA from the mycelium was purified using a DNeasy Plant Mini Kit (Qiagen, Hilden, Germany) and used for PCR amplification. PCR was carried out in a total volume of 25 µL, including 1 µL of DNA extract, 1 µL of each primer (10 µM), 1 µL of genomic DNA, and 1 µL of PCR product. PCR products were visualized on 1% agarose gels stained with GelRed-Biotwin (Sigma-Aldrich, St. Louis, MO, USA). PCR products were purified using an AxyPrep DNA Gel Recovery Kit (Axygen Bioscience, Shanghai, China) following the manufacturer's instructions. Purified PCR products were stored at -20 °C until use.

Phylogenetic analysis
The fungal ITS and D1/D2 sequences of the D1/D3 subgroups of the subgroup of fungi were obtained from the European Molecular Biology Laboratory (EMBL, Strasbourg, France) and were compared with sequences of the taxonomic groups of Dothideomycetes and Diceromycetes. The sequences of the D1/D3 subgroup were compared with those of Dothideomycetes, Diceromycetes, and Duchyroidomycetes from GenBank (http://www.ncbi.nlm.nih.gov) and the sequences of Dothideomycetes were compared with those of Dothideomycetes from GenBank. The phylogenetic tree was constructed with MEGA 5.0 software (http://www.megasun.net/mafft/).


## Results
The bacterial communities of the outcrops were dominated by Bacillus, Ktedonobacter, Rhodomonas, Veillonella, and Methanosarcinoma, whereas the fungal communities of the outcrops were dominated by Capnodiales, Klebsiella, and Staphylococcu. In the outcrops, Bacillus, Ktedonobacter, Veillonella, and Methanosarcinoma were the predominant genera, whereas in the in silice sediments, Ktedonobacter, Veillonella, and Methanosarcinoma were the predominant genera. The bacterial communities of the in silice sediments were dominated by Bacillus, Ktedonobacter, Rhodomonas, Veillonella, and Methanosarcinoma, whereas in the in silice sediments, Ktedonobacter, Veillonella, and Methanosarcinoma were the predominant genera. The fungal communities of the in silice sediments were dominated by Burkholderiales, Capnodiales, Klebsiella, and Staphylococcus, whereas the fungal communities of the in silice sediments were dominated by Capnodiales, Ktedonobacter, Veillonella, and Methanosarcinoma.

The fungal communities of the in silice sediments were dominated by Flavobacterium, Nitrospira, and Phoma, whereas the fungal communities of the in silice sediments were dominated by Salmonella, Bacillus, Sclerotinia, and Bacillus, respectively. The fungal communities of the in silice sediments were dominated by Pneumocystis, Pseudomonas, and Salmonella, whereas the fungal communities of the in silice sediments were dominated by Staphylococcus, Sphingomonas, and Salmonella. The bacterial communities of the in silice sediments were dominated by the Salmonella species, whereas the fungal communities of the in silice sediments were dominated by Salmonella.


## Discussion

.3. Nuclear Magnetic Resonance Studies (NMR)
The NMR spectra of the samples analyzed in this study were mapped to the ONIOM (Onnit-a, Onnit-b, Onnit-d, and Onnit-e) magnetic resonance spectra (MS spectra) of the samples in this study (Figure 3). The results showed that the different ion channels were present in the rock articelds.

The NMR spectrum of the samples was generally higher than that of the ONIOM (Figure 3). The signals were assigned to the nucleophiles (1.95-3.97 ppm) and the non-nucleophiles (2.95-4.35 ppm). The signals were assigned to the C-type (C-O) and the C-S (C-S and C-S) classes. The signals of the C-type and the C-S classes of the samples were assigned to the C-O class (Figure 3).

The NMR spectrum of the samples was more consistent with those of ONIOM (Figure 3). The signals of the C-type (C-O) and the C-S (C-S and C-S) classes were assigned to the C-O class. The signals of the C-S and C-S classes were assigned to the C-O class. The signals of the C-O class were assigned to the C-O class and the C-S class, respectively. The signals of the C-O class were assigned to the C-O class. The signals of the C-O class were assigned to the C-O class and the C-S class, respectively. The signals of the C-O class were assigned to the C-O class and the C-S class. The signals of the C-O class were assigned to the C-O class and the C-S class, respectively. The NMR spectra of the samples in this study were in agreement with those of ONIOM (Figure 3). The signals of the C-type (C-O) and the C-S (C-S and C-S) classes of the samples were assigned to the C-O class. The signals of the C-O class were assigned to the C-O class.
