# Proteogenomics in southern China reveals eleven informative genes
Quaglee Dragontacos


## Abstract
Candida albicans is a commonly encountered fungal pathogen that causes candidiasis. Most C.


## Introduction
The transition from living to death asexual reproduction is a highly dynamic process, and can lead to many unanticipated consequences, such as low fertility and ecological degradation. In particular, the transition from asexual to sexual reproduction can impact the survival of microbial communities and the host environment [1].

The dynamic transition from asexual to a sexual life cycle in eukaryotes is a complex process involving the formation of reproductive structures called asexual spores [2]. The most common asexual spores are spores of bacteria, yeast and filamentous fungi [3]. Upon germination of the asexual spores, they form hyphal or conidia, which are able to pass through and enter the host [4]. Fungi play a key role in the transition from asexual to a sexual life cycle, as well as in controlling nutrient uptake and storage conditions [5].

As the spores germinate and form conidia, they germinate into hyphae and infect the host [6]. Fungal spores can survive and proliferate in the soil, which in turn changes into host infection [7]. As the infection process is controlled by the host immune system, spores germinate and invade the host cell. The host response depends on the fungal pathogen, leading to an ecological dependence of fungal populations, with high host specificity and reduced virulence [8].


## Methods
There are nine main phylogenetic species from different orders (species from Colletotrichum, Poeta, Fusarium, Microsporum, Oomycetes, Stramenopilum, Trichophyton and Tetrasphaeropilum. The average length of each species tree is 7,600 bp and the maximum number of residues is 20,500 bp. The phylogenetic species tree is maintained in FigTree v1.4.2 (http://tree.bio.ed.ac.uk/software/figtree/) with new taxonomic characters added every week. The phylogenetic species tree is drawn in FigTree v1.4.2 (http://tree.bio.ed.ac.uk/software/figtree/) and all new characters and new tables were added after the installation of FigTree v1.4.2.

2.3. Gene Classification
The gene prediction tool for fungi was developed and validated by GeneMark-ES (Boehm et al. 2009). The predicted protein sequences were classified into two subclasses: small and super-Small and large. The classification was performed by combining the ITS and TUB with a BLAST search with sequences from fungal strains from several species. The species with the highest number of reads in the dataset were selected for further analysis. The results of this analysis were summarized in Table 2. The prediction of the second subclasses was also performed by combined BLAST and TEF- search with a combined dataset of ITS and TUB sequences. The results of this analysis were summarized in Table 3. The second class of classification was named as the “Chequerboard.” This classification was used to assess the similarity of the obtained sequences to the reference sequences of fungi from the Fungal Genomics Centre, University of Valencia (U. Valencia) and related to our previous study (Pontecorvo et al. 2009). The results of this analysis were summarized in Table 4.

2.4. Genomic DNA Extraction and Screening for Fungal Mutants
The genomic DNA of the fungus was extracted from mycelia of fresh mycelium using the UltraClean^TM Microbial DNA Isolation Kit (MoBio Laboratories, Inc., Solana Beach, CA, USA) according to the manufacturer’s protocol.


## Results
pachyrhizi genome (Table S7. In addition, four genes were identified in the P. pachyrhizi genome, including one annotated as an endoglucanase (L8), and four genes identified as a flavoprotein (L12).

Genes with clear homology to fungal pathogenicity
The P. pachyrhizi genome possesses a number of genes with putative functions for pathogenicity, including the putative genes involved in the virulence of fungi such as the putative P. viris genes (Table S8). This list also includes genes involved in fungal virulence, including the putative P. viris genes and the putative P. pachyrhizi virulence genes. We also identified genes involved in pathogenicity and pathogenesis in P. viris (Table S8). In addition, we identified genes involved in pathogenicity in P. pachyrhizi.

P. viris P_SCC9101 encodes a putative putative putative carbohydrate transporter (Table S8). P. viris P_SCC9101 has seven putative putative transporters, including the putative P. viris putative extracellular transporter (Table S8). A putative P. viris P_SCC9101 gene was identified in the P. viris genome (Table S8). The P. viris P_SCC9101 gene was also identified in the P. pachyrhizi genome. We identified a putative P. viris P_SCC9101 gene in the P. viris genome (Table S8).

P. viris P_SCC9101 is closely related to the P. viris P_TIP9424, a putative S. pombe spp. putative extracellular transporter (Table S8). The P. viris P_TIP9424 gene was identified in the P. viris genome (Table S8).

P. viris P_TIP9424 is closely related to the P. viris P_DCL928, a putative extracellular sorter (Table S8). The P. viris P_DCL928 gene was identified in the P. viris genome (Table S8).


## Discussion
castellii (Figure 4. These include putative sexual and asexual loci, transcription factors, ATPases, sterol-related proteins, nitrogen-related proteins, nucleoporins, and RNA-binding proteins (Table S7, Additional file 1).

The genome of S. castellii encodes a single gene for the mitochondrial respiratory chain (ITS1a, BSSU, and MALDI-TOF MS), a gene for the cytosolic NADP-dependent dehydrogenase C-like gene (MdDHA), a gene for the mitochondrial respiratory chain ATPase (CAT1) and a gene encoding a chitin-binding protein (GPD1) (Table 1). Genes encoding a mitochondrial respiratory chain regulator, mitochondrial polyketide synthase, and mitochondrial sulphur reductase were also found in the genome. Genes encoding the ATPase, cytosolic respiratory chain regulator, and mitochondrial sulphur reductase were also found in S. castellii. The genes encoding putative asexual locus, transcription factors, nucleoporins, and RNA-binding proteins were also found in S. castellii. The gene encoding putative nitrogen-related protein was also found in S. castellii.

The genomic analysis also identified two genes encoding a putative iron regulator, a putative glucose transporter, and a putative protein of unknown function. Gene deletion was performed in S. castellii. The putative genes encoding putative genes for the putative iron regulator, a putative glucose transporter, and a putative protein of unknown function were also found in S. castellii. Gene disruption in S.
