# RAB-Like 2 Is Involved in Ustilago maydis Survival and Protect Lepista Prevalence from Invasive Antifungals
Quaglee Dragontacos


## Abstract

Sequencing of all isolates of the opportunistic fungal pathogen Candida albicans has provided evidence for the evolution of virulence mechanisms in this deadly fungu. The lack of efficient means to isolate and identify the species in the varied samples (egg-laying units, whole cells, masses) along with the difficulty in identifying all the strains with correct identification of the causal agent, makes the study of the evolution of virulence mechanisms in this fungus a daunting task. We generated Fungal Genome Database (GenDB) protein-centric and functional databases for the analysis of genome sequence variation among fungi by integrating forward-flow electrophoretic sequence (FUT) analyses of thousands of high-quality genomes to mine on the gene families of the identified fungi to identify the genes whose genomes are associated with virulence in this organism. The database incorporates 73, including 12 small genes, mostly involved in carbohydrate metabolism, for analysis. We also generated the genome of one isolate which is highly virulent to a human epithelial cell line and provided the molecular data necessary to understand the genomic changes in this organism which resulted in the emergence of virulence traits.

Sequencing of the in vitro cultures of two Cryptococcus species has identified the virulence genes as: CgYps1-1, which encodes a protein with the FUT domain; CgYps5, which encodes a protein with the conserved ATPase; CgYps9, which encodes a protein with the conserved DAMP/AMP-binding protein; CgYps7, which encodes a protein with the conserved CAMP/AMP-binding protein; and CgYps11, which encodes a protein with the conserved ZRT domain. We also identified the two genes as: CgYps1-1 and CgYps5-1 in the C. albicans <U+0394>cagewasfilm-1 isolate obtained from the murine model, as well as the CgYps9 strain in the C. parapsilosis strain. In vitro, the C. albicans <U+0394>cagewasfilm-1 isolate was virulent to C. parapsilosis and similar to C. albicans <U+0394>cagewasfilm-1, but virulent to C. orthopsilosis, C.


## Introduction
Fungi are eukaryotic organisms that can produce extracellular enzymes that are responsible for the degradation of various biomolecules in a variety of biological systems, including the cell, organelle and structure of the cell [1,2,3]. These enzymes, which are often encoded in the extracellular gene set, can cross pathways and transcellularly and can change the structure and function of the cell membrane, resulting in a number of important cell death processes [4,5].

The human fungal pathogen Candida albicans is a common cause of candidiasis and is also known to be associated with invasive infections, including aspergillosis, mucormycosis and mucormycosis. However, only a small number of clinical cases has been reported in the literature [6,7,8,9,10].

Candida albicans is an opportunistic fungal pathogen that can cause a life-threatening infection in immunocompromised patients, particularly those with hematologic diseases or bone marrow transplantations. Candida albicans is a major fungal pathogen that causes life-threatening invasive infections in susceptible patients, including those with hematologic diseases and bone marrow transplants [11,12]. Candida albicans has a worldwide distribution and can cause infections in healthy individuals, especially in those with hematologic diseases, such as those with chronic granulomatous disease (GMD). In these individuals, the disease progression mimics that of a GMD, whereby C. albicans becomes a normal part of the body microbiota, but when the host immune response is impaired, the fungus can proliferate and cause disseminated candidiasis [13,14].

In healthy individuals, the gut microbiota is largely comprised of microorganisms, including bacteria, fungi, and viruses.


## Methods
rb-institut.org/index.html. These fungi were grown on PDA for 3 days at 30 °C. For each fungus, four leaves were dissected from each leaf. The total number of leaves in each treatment was divided into two parts and the leaf fragments were directly submitted to the fungus for pathogen isolation. Ustilago maydis from leaves of each treatment was also subjected to sequencing.

In order to analyze the ustilaginous fungi, the pathogen was cultured on PDA for 7 days. Fungal mycelia were harvested from the tissue in PDA and washed with H_2O and then snap frozen in liquid nitrogen. The frozen mycelia were ground using a mortar and pestle and snap-frozen in liquid nitrogen. The mycelia were ground in liquid nitrogen with liquid nitrogen slurry and then extracted with 70% MeOH. The extraction was carried out with vortexing for 30 s and centrifuging at 10,000×g for 10 min. The supernatant was used for DNA extraction.

A high-throughput sequencing was carried out using the HiSeq2000 (Illumina, San Diego, CA, USA) as per the manufacturer's instructions. The sequences were assembled with Newbler (v. 2.3.1) [57] and compared with the National Center for Biotechnology Information (NCBI) reference database using the BioEdit software (https://www.ncbi.nlm.nih.gov/biocloud/bioedit/bioedit.html) for all fungal species.

Phylogenetic Analysis
Sequences of U. maydis were aligned with the MUSCLE algorithm [58] and phylogenetic relationships were constructed using the neighbor-joining method and bootstrapping was performed with 1,000 replicates.

To analyze the phylogenetic relationship between the different fungal species, the neighbor-joining method was used. First, the evolutionary distances were estimated using the “distance-based” method and the bootstrap values were estimated by the “survival” method. The bootstrap values of the “survival” method were the same as the “distance-based” method.


## Results
maydis UBE2 in . maydis survival and protection from antifungals, we measured the survival rate of Lepista larvae after 2 h of exposure to a selection of antifungal agents. U. maydis larvae were used to inoculate the larvae and the larvae were incubated at 30°C and 37°C in the dark for 10 days. Larvae were then tested for their ability to survive antifungal treatment by determining the survival rate. As shown in Figure 4, larval survival was significantly higher in larvae exposed to 4 µM of UBE2 compared with those exposed to 5 µM of the drug, whereas larval survival was significantly higher in larvae exposed to 5 µM UBE2 than those exposed to 1 µM of the drug. The survival rate of larvae exposed to 4 µM of UBE2 and those exposed to 5 µM of the drug was comparable or similar to that of larvae exposed to 5 µM of the drug.

The ability of larvae to survive in the dark was also measured for the following 24 h of infection. Larval survival was significantly higher for the larvae exposed to 5 µM of UBE2 than those exposed to the drug, whereas larval survival was significantly higher in larvae exposed to 5 µM of the drug compared with those exposed to 1 µM of the drug. The survival rate of larvae exposed to 5 µM of UBE2 and those exposed to 5 µM of the drug was significantly higher than that of larvae exposed to 1 µM of the drug. The survival rate of larvae exposed to 5 µM of UBE2 and those exposed to 5 µM of the drug was significantly higher than that of larvae exposed to 5 µM of the drug. The survival rate of larvae exposed to 5 µM of the drug was significantly higher than that of larvae exposed to 1 µM of the drug.

3.4. Unexpectedly, Larval Survival in Response to a Low Temperature
To further examine the impact of low temperature on larval survival, we measured larval survival following exposure to a low temperature. Larvae were incubated at 30°C and 37°C and treated with 1.5 M NaCl, 1.5 M KCl, and 0.1 M sorbitol. Larval survival was measured by determining the survival rate of larvae exposed to 0.5 M NaCl, 1.5 M KCl, and 0.


## Discussion
The LTR-like peptide-derived peptide derived from the C-terminal domain of the CAZymes A, B, and C of the . histolytica A/B family of glycoproteins (EHCB) was found to be involved in the ability to form biofilm in E. histolytica and the interaction with EHCB, thus acting as a barrier against the pathogen invasion (Fig 2 and Additional File 1). The same mechanism of action has been observed for the major glycoprotein EHCB of the E. histolytica and E. histolytica family, in which the peptide was found to be required for biofilm formation and also for the ability to form biofilm in S. cerevisiae (Hemileia) [47].

The peptide derived from the C-terminal domain of the CAZymes A, B, and C of the E. histolytica A/B family of glycoproteins (EHCB) was also found to be involved in the ability to form biofilm (Fig 2 and Additional File 1). In E. histolytica, the peptide-derived peptide was found to be involved in the ability to form biofilm (Fig 2 and Additional File 1). In the S. cerevisiae and C. neoformans, the peptide-derived peptide was found to be involved in the ability to form biofilm (Fig 2 and Additional File 1) [48].

The peptide derived from the C-terminal domain of the CAZymes A, B, and C of the E. histolytica A/B family of glycoproteins (EHCB) was found to be involved in the ability to form biofilm (Fig 2 and Additional File 1). The same mechanism of action has been observed for the major glycoprotein EHCB of the E. histolytica and E. histolytica family, in which the peptide was found to be required for biofilm formation and also for the ability to form biofilm in S. cerevisiae (Hemileia) [48].
