# Reassessing Putative Transcription Factor Homologues in Fungi
Quaglee Dragontacos


## Abstract
The entomopathogenic fungus Metarhizium acridum is a polymorphic, Gram-positive, saprophytic bacterium that causes the most lethal infections of the liver in humans and animals worldwide. There are approximately 600 described species and 228 described genotypes, which have been assigned to each of the main pathogenicity loci, M. acridum-virga, M.


## Introduction
The fungal cell wall is a complex structure composed of numerous polysaccharides, including glucans, chitin, mannan, and chitosan. The structural features and functions of the cell wall are diverse, including cell wall remodelling, wall integrity, cell wall remodelling, and cell wall remodelling [1,2]. The cell wall is a complex structure and contains a vast number of biologically active molecules. Among these, chitin is the most abundant component and the most abundant chitin-degrading enzyme [3,4]. Chitin is a linear polysaccharide consisting of a C-6-8-6-4-1- N-acetyl-glucosamine core with a C-5-5-5-7-glucose-binding sequence. In fungi, chitin is recognized by the human epidermal growth factor (EGF) receptor, the <U+03B3>-secretase [5] and expressed on a variety of cells, including keratinocytes, fibroblasts, endothelial cells, and macrophages [6]. In C. albicans, the chitin binding receptor LaeA is expressed on human keratinocytes [7]. In the pathogenic yeast Cryptococcus neoformans, the cell wall is known to influence virulence [8]. In the yeast pathogen Candida albicans, the cell wall is thought to contribute to virulence. In C.


## Methods
graminearum using the BLAST method, and selected the top five best BLAST hits for further phylogenetic analyse. The BLAST hits were selected for further phylogenetic analysis, and the results were displayed in the Gene Ontology (GO) terms as follows: FgCTGGCCAGAATCACAG, FgCTGGGAGGTCATC, FgCTGGCATAGCAGTCG, FgCTGGCATGGCATGTC, FgCTGGGTGGAGCAGT, FgCTGGACATCATTCCGCC, FgCTGGGACATCCGCCGCC, FgCTGGGACATCCCCGTGCC, FgCTGGGACATCCAAGCCTCA, FgCTGGGACATCCATCCAGCC, FgCTGGGACATCCCCGCCCCGCC, FgCTGGGACATCCAGCCTCA, FgCTGGGACATCCATCCTCGCC, FgCTGGTCAACTCACGATGAT, FgCTGGCATCCTCAAACCTGCC, FgCTGGTTCAACTCACGGATC, FgCTGGCATGGCATCTTCAGC, FgCTGGCCATCCCCCCGCC, FgCTGGTCAGACCCAGC, FgCTGGCCTCAACCTGGTC, and FgCTGGCCAGCCAAACTCGCC. These analyses were performed using the program BLASTn (http://www.ncbi.nlm.nih.gov/BLAST/) and the results were reported as the mean ± standard deviation (SD) of the two-tailed t-test for both the groups. The relative expression levels were calculated using the 2^-<U+0394><U+0394>CT method [38].

RNA Extraction
Total RNA was extracted from fungi using the DNeasy Plant Mini Kit (Qiagen, Valencia, CA) according to the manufacturer’s protocol. A total of five different concentrations of RNA were used for the first-strand cDNA synthesis: 1.5 µg of total RNA, 1.25 µg of first-strand cDNA, 0.


## Results
The genotypic diversity of the draft genome of H. jirovecii
We sequenced the draft genome of H. jirovecii, a representative of the H. jirovecii phylum (Figure 1). The genome of H. jirovecii was assembled using an Illumina HiSeq 2000 pipeline, with an average of 1.5×10^-3 scaffolds per scaffold, with an average of 3.5×10^-3 scaffolds per scaffold. This genome was generated using the H. jirovecii genome assembly project, as a member of the H. jirovecii subclade (Figure 1). The H. jirovecii genome assembly was confirmed by the annotation of predicted protein-coding genes using the well-characterized Ascomycota (Ascomycete) and Basidiomycota (Basidiomycete) yeasts (Table 1). The assembly of H. jirovecii has been previously reported [16].

Genomic DNA sequencing and assembly
The draft genome of H.


## Discussion
cerevisiae genome encodes for 4013 putative transposons (23447769. In comparison to the other fungi, the S. cerevisiae genome has an average of 12.13 transposable elements (TN) per genome (Table S1). There are 7.37TN per genome (Table S1). The average number of TN per genome is 6.25 (Table S1).

The average number of putative transposable elements per genome is 4.62 (Table S1).

The S. cerevisiae genome has an average of 47.97TN per genome (Table S1). The average number of TN per genome is 5.20 (Table S1).

The Fungal genome encodes for 32.35TN per genome (Table S1). The average number of FTR per genome is 6.62 (Table S1).

The C. neoformans genome encodes for 16.69TN per genome (Table S1). The average number of FTR per genome is 3.31 (Table S1).

The average number of FTR per genome is 12.62 (Table S1).

The fungal genome encodes for 8.38TN per genome (Table S1). The average number of FTR per genome is 6.09 (Table S1).

The C. albicans genome encodes for 8.84 TN per genome (Table S1). The average number of FTR per genome is 3.10 (Table S1).

The genome of Saccharomyces cerevisiae has a total of 6,329,483 bp of genome (Table S1). The average number of genome is 1,923.

The genome of Aspergillus fumigatus encodes for 5,158,075 bp (Table S1). The average number of genome is 1,155.

The S. pombe genome encodes for 4,772,982 bp (Table S1). The average number of genome is 1,185.

The C. albicans genome encodes for 3,245,944 bp (Table S1). The average number of genome is 1,041.

The C. glabrata genome encodes for 2,291,769 bp (Table S1). The average number of genome is 1,022.
