# Recent Advances in the Study of Fungal Cytochrome xylases
Quaglee Dragontacos


## Abstract
Background: The Cryptococcus neoformans var. grubii species complex is a global fungal pathogen that causes ~800,000 deaths per year worldwide. The opportunistic fungal pathogen, that infects ~10% of the world's population, employs two distinct strategies to survive: invasion and escape. We previously discovered that the dimorphic fungal cell wall contains a sugar cluster that isovaleric between ~5 and ~6 different amino acids. In this study, we examined the cell wall structure of C. neoformans var. grubii strain Y271.1. To analyze the cell wall structure of the strains, the polysaccharide matrix containing the fatty acid, glycerol, and d-amino acid was used as an in vitro model. In addition, we studied the cell wall structure of the strains under different stress conditions. We found that the C. neoformans Y271.1 polysaccharide wasovaleric between glucose and glycerol, and the cell wall could be protected from oxidative stress. In addition, the strain showed a decrease in the mitochondrial number of its ÃŸ-glucanase, NfixA, and an increase in the intracellular number of its cytochrome P450 enzyme (CYP51). Furthermore, the strain showed a decrease in the activity of the electron-targeting enzyme (ET1-ERF) on the cytochrome P450 enzyme (CYP51). Our results indicate that the cell wall and glycerol are the same structure, and that the cell wall is targeted by two different genes responsible for the survival strategies of C. neoformans var. grubii.

IMPORTANCE The Cryptococcus neoformans var. grubii species complex is a global fungal pathogen that causes ~800,000 deaths per year worldwide. The opportunistic fungal pathogen, that infects ~10% of the world's population, employs two distinct strategies to survive: invasion and escape. We previously discovered that the dimorphic fungal cell wall structure of C. neoformans var. grubii strains isovaleric between ~5 and ~6 different amino acids. In this study, we examined the cell wall structure of the strains under different stress conditions. We found that the C. neoformans Y271.


## Introduction
The CYPs were generally believed to be a remnant of CYP-containing substrates, as there was no evidence that cytochrome c oxidases could be used as substrates for the CYPs, which could be found in the cytoplas. However, in Pucciniomycotina, the CYP domain is not essential for the functioning of the cellular organ, and the enzyme can be taken up and degraded in the absence of the cytoplasmic substrate (Evans et al. 2005). In addition, in Pucciniomycotina, CYPs are used as substrates for other CYPs, such as GTPases, which are involved in the degradation of complex mixtures of sugars and other substrates, which in turn affect the composition of the fungal cytochrome c oxidase protein (Chatterjee and Dixit 2010). In addition, CYPs are involved in the detoxification of other CYPs, such as GTPase, which was first identified in the Ustilago maydis (Hughes et al. 2005). CYPs are an important group of enzymes involved in many different biological processes, including respiration, cell differentiation, cell signaling, and the metabolic process of photosynthesis (Oren and Huhtinen 2003). CYPs are highly conserved among different fungi, which makes the discovery of new CYPs very challenging. Moreover, the C. albicans CYP gene duplicated several times, which may indicate that the duplication occurred in different fungal species (Gomez-Garcia et al. 2008). In C. albicans, there are three CYPs, CYPs 1-7, which are grouped into two subgroups, CYPs 1-6. In this study, we have searched the literature for new CYPs from a number of fungal species. We have found several CYPs from the Penicillium deneoformans, which is a basidiomycetous fungus from the genus Pseudallescheria and has been used as an ecological model to study the fungal CYP system (Gomez-Garcia et al. 2008). CYPs can be recognized by the yeast.


## Methods
Reagents
All the chemicals used in this study were obtained from Sigma-Aldrich (Saint Louis, MO, USA). The cell walls were prepared as described by Eddy et al. (2005). The antifungal compounds were obtained from the following sources: C. albicans ATCC 90028, C. krusei ATCC 6258, C. parapsilosis ATCC 22019, C. tropicalis ATCC 32068, and C. albicans ATCC 90028. The peptides were obtained from peptide library collection of the Molecular Probes Laboratory, National Institute of Allergy and Infectious Diseases (NIAID, Bethesda, MD, USA). The chemical solvents used in the experiments were AspRed (Sigma-Aldrich), NBT (NBT; Thermo Fisher Scientific), DMSO (Sigma-Aldrich) and distilled water (Sigma-Aldrich).

Molecular typing
The sequences of the predicted proteins used in this study have been deposited in the National Center for Biotechnology Information (NCBI) database. They were first aligned with Clustal X [20] using the default parameters, and then manually edited in Bioedit (version 7.0) and edited manually in Bioedit 3.0. The final alignment of the predicted proteins was composed of 644 amino acid residues (3.98% of positions) with a total of 55 amino acid residues. The mutation rates were detected using the MOTHUR software (Version 5.1.2, 2016, http://mothur.org/wiki/MOTHUR_software).


## Results
nidulans, other fungal CYP genes have been shown to be involved in the biosynthesis of fungal metabolites and metabolites related to virulence [9, 27-29. Many of the CYP genes are involved in virulence factors and cellular processes, including defense against fungal pathogens [30]. In the present study, the CYP genes were identified in a gene cluster that was clearly enriched in the A. nidulans wild-type strain. It is worth mentioning that the CYP genes were located in the same cluster as genes that are involved in virulence factors. Moreover, the CYP genes were located in a cluster with the A. nidulans wild-type strain.

The presence of CYP genes in other fungi, such as Arabidopsis and other plant pathogens, is often associated with the functions of CYP enzymes that are involved in the defense against fungal pathogens. Therefore, it is important to study the effect of different CYPs on the growth and virulence of A. nidulans. The results presented in Table 2 indicate that the abundance of CYP genes in A. nidulans is highly conserved with A. nidulans A. niger A. nidulans A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A. niger A.


## Discussion
cerevisia. Therefore, as a result of the sub-group analysis, we did not define the overall specific amino acid sequences of fungal fungal CYPs. However, we have provided a comprehensive list of CYP genes that are not known to be involved in secondary metabolism and drug resistance. As a result, the results of this study will be used to provide a comprehensive list of CYP genes that are not known to be involved in secondary metabolism.

The S. cerevisiae CYP2A6 gene was found to be involved in secondary metabolism in S. cerevisiae. The CYP2A6 gene from S. pombe was found to be involved in secondary metabolism in S. cerevisiae. The CYP2A6 gene from S. cerevisiae was found to be involved in the secondary metabolism in S. pombe. The CYP2A6 gene from S. cerevisiae was found to be involved in the secondary metabolism in S. cerevisiae. The CYP2A6 gene from S. pombe was found to be involved in the secondary metabolism in S. cerevisiae. The CYP2A6 gene from S. cerevisiae was found to be involved in the secondary metabolism in S. cerevisiae. The CYP2A6 gene from S. cerevisiae was found to be involved in the secondary metabolism in S. pombe. The CYP2A6 gene from S. cerevisiae was found to be involved in the secondary metabolism in S. cerevisiae. This study provides a comprehensive list of CYP genes that are not known to be involved in secondary metabolism in fungi.

The CYP2B7 gene was found to be involved in secondary metabolism in C. neoformans. The CYP2B7 gene from C. neoformans was found to be involved in the secondary metabolism in C. neoformans. The CYP2B7 gene from C. neoformans was found to be involved in the secondary metabolism in C. neoformans. This study provides a comprehensive list of CYP genes that are not known to be involved in secondary metabolism in fungi.

The CYP2C5 gene was found to be involved in the secondary metabolism in C. albicans. The CYP2C5 gene from C.
