# Sequencer's Toolkit of RNA Sequences Analyzing Microfabricated Proteomes
Quaglee Dragontacos


## Abstract
Tuberculosis (TB) is the leading cause of death in people living with human immunodeficiency virus (HIV) in sub-Saharan Africa. TB treatment in HIV-infected adults is very effective, with a good response rate and prompt treatment. However, initiation of TB treatment can be complicated and patients must be monitored closely for signs of progression of the disease. This study evaluated the features of the first-line TB drugs in HIV-infected adults in sub-Saharan Africa.

This was a retrospective study conducted from December 2016 to March 2017, in 1,350 HIV-infected persons with CD4 < 100 cells/ul, in 35 HIV-infected patients with CD4 100-200 cells/ul, and in 64 HIV-infected patients with CD4 200-350 cells/ul, who were treated for TB. Participants were divided into HIV-related (34 patients; 79%), non-HIV related (16 patients; 19 non-HIV), and HIV-associated (16 patients; 25 patients; 25 non-HIV) TB episodes.

Among the 34 HIV-infected patients, 15 episodes of non-HIV-associated TB were diagnosed, of which four were excluded because of patients' travel, work, and self-care purposes. Median CD4 count was 61 cells/ul (IQR 40-87). Thirty-one episodes of non-HIV-associated TB were diagnosed (21%). Four episodes of non-HIV-associated TB occurred in non-HIV-associated patients in our study and three were excluded because of work restrictions.


## Introduction
These sequencing techniques have been used to analyze the RNA content of pathogenic fungi such as Aspergillus, Candida, Cryptococcus, Histoplasma, Coccidioides, and Blastomyces, and to probe the genetic basis for their pathogenic capabilities [15. Recently, several methods were used to investigate the content of RNA viruses, including RNAi methods [16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31], Non-Coding RNAs (ncRNAs) [32], RNA-seq [33,34,35,36,37], and UniProt DNA Shotgun Sequencing (uRSA) [38,39] methods [40,41]. Recently, it has been shown that the eukaryotic uRSA (uRSA-seq) method has considerable potential for the study of RNA viruses [42]. In this study, we utilized the rRNA-seq approach to identify the source of virulence in a set of Burkholderia pseudomallei strains. We found that Burkholderia pseudomallei has a large genome, composed of many contigs including the genome of the distantly related species Burkholderia pseudomallei and the genome of the closely related species Candida albicans and Cryptococcus neoformans. The Burkholderia genome is the most diverse eukaryotic genome, and it has undergone major changes in its size and structure and genes. The majority of the coding sequences in Burkholderia pseudomallei have been decribed and sequenced. In this study, we also investigated the sequence similarity and structure of the Burkholderia genome and found that there are large amounts of sequences with sequence similarity of only 0.05-1%.

Burkholderia pseudomallei is an ascomycete that was first discovered in the late 19^th century [43]. The pathogenic species of the genus Burkholderia are distributed in the Americas and are widely distributed in the Mediterranean, Africa, and Asia.


## Methods
Sequence data
The sequences of C. neoformans var. grubii strains used in this study (Table 1) were obtained from the Broad Institute of Harvard and MIT (http://www.broadinstitute.org/).


## Results
The first exception was a large clade representing the clade representing the latter two species (Fig 4B. The two Bayesian trees based on the topology of the Bayesian and ML trees showed that the clade representing the two Ascomycetes species (Fig 4C) was supported by the bootstrap values and the Bayesian and ML trees. In the Bayesian tree based on the topology of the Bayesian and ML trees (Fig 4C), the Ascomycetes was placed in the clade representing the clade representing the Ascomycota (Fig 4D). In the ML tree based on the topology of the Bayesian and ML trees (Fig 4D), the Ascomycota was placed in the clade representing the clade representing the Ascomycota (Fig 4E).

The alignment of the Proteome_P sequences with the phylogenetic tree of the Ascomycota was inferred by the Neighbor-Joining method. The topology of the Bayesian and ML trees indicated that the clade representing the clade representing the Ascomycota was supported by the bootstrap values (Fig 4D). In the ML tree based on the topology of the Bayesian and ML trees (Fig 4D), the Ascomycota was placed in the clade representing the clade representing the Ascomycota (Fig 4E).

The results of the Bayesian and ML trees (Fig 4E) showed that the Ascomycetes was placed in a separate clade representing the Ascomycota. In the Bayesian and ML trees (Fig 4E), the Ascomycetes was placed in the clade representing the Ascomycota (Fig 4F).

The tree based on the Bayesian and ML trees (Fig 4F) showed that the Ascomycetes was placed in a separate clade representing the Ascomycota species.


## Discussion
RNA-seq data is less useful for detecting bioinformatics errors because the amount of information is limited by the small read. In this work we have used the public gene prediction program (http://www.ncbi.nlm.nih.gov/) to predict gene expression in Mucorales. We found that the Mucorales are a good choice for deep sequencing analysis because of their large genome size and high gene expression levels. However, we also found that the Mucorales were a poor choice for RNA-seq analysis because of their high gene expression levels. In addition, the presence of multiple genes could make it difficult to predict gene expression changes in Mucorales. This could be overcome by the use of multiple genes in the RNA-seq data, which could be further improved by using RNA-seq data.

In order to identify the genes involved in metabolism, we generated a prophylactic strain of Mucorales using a 2-gene library that contained genes that were expressed at least 2-fold higher in the presence of glucose as the carbon source. We found that the Mucorales are also a good choice for in silico RNA-seq analysis because of their large genome size, high gene expression levels, and gene deletion and EST length. These results suggest that the Mucorales could be an effective deep-sequencing tool for the deep sequencing of fungi.

Fungal Proteome Sequences Are More Expensive than Other Proteins in Mucorales
In this study, the Mucorales genome was sequenced using the Illumina platform and the proteome of the protein-coding genes were compared with the sequences from the Mucorales. As shown in Figure 3A, the proteome of Mucorales harbors proteins with 22% sequence similarity and 9% identity. The Mucorales have the highest number of proteins with protein sequences that are similar to proteins from other fungi and the highest number of proteins with protein sequences that are similar to proteins from other fungal species (Figure 3B).
