# Sequencing and molecular characterization
Quaglee Dragontacos


## Abstract
Cryptococcus gattii is an emerging pathogen that is responsible for a life-threatening meningitis in patients with acquired immunodeficiency syndrome (AIDS). We have previously shown that C. gattii can invade the host lung in a time course and that mice infected with C. gattii were highly sensitive to the pneumococcal polysaccharide capsule. In this study, we examined the impact of the different infection phases on the inflammatory cytokines expressed in lungs and found that mice infected with C. gattii had significantly increased lung inflammatory cytokine levels and significantly attenuated the disease severity. In addition, we observed that mice infected with C. gattii had significantly increased lung fungal burden, which was associated with a high interferon-<U+03B3> production. These findings suggest that the host lung inflammatory response to C. gattii is highly sensitive to the pneumococcal polysaccharide capsule. Furthermore, we demonstrated that C. gattii can activate a local immune response against pneumococci and this effect was dependent on the capsular polysaccharide. Our results suggest that the pneumococcal immune response is highly sensitive to the cryptococcal polysaccharide and that this response may be crucial for the control of cryptococcosis.


## Introduction

Several gene sets and pathways are involved in the sugar metabolism, including LBA1, CGR1, IGR1, RAD1 and PAB1, all of which are required for the synthesis of lignocellulosic biomass [23-27. Lignocellulosic biomass is rich in sugars and proteins, including glucose, mannitol, trehalose and fructose, that are important for the degradation of various polysaccharides [27]. The lignocellulosic biomass comprises less than 10% sugar units and 10% protein and carbohydrate units, although the remaining carbohydrate (mannitol) contributes to a higher carbon content than glucose. The vast majority of the sugars and proteins are glycosaminoglycans and lack N- and O-glycans [27], [28]. Additionally, the cell wall is also important for the survival of fungi, and therefore is the first line of defence of the host against fungal infections [29]. The cell wall is important for the growth and survival of fungal cells, and a complex network of chitin-binding proteins, chitosans and chitinase is essential for the survival of fungal cells in the environment [29].

The cell wall is a complex structure that consists of a polysaccharide matrix, the cell wall, and a membrane of chitin and chitosans. Chitin is a polymer of N- and O-linked fatty acids and is essential for the cell wall, and chitin is a polymer of O-acetyl groups and O-mannulose. The cell wall is a major component of the fungal cell, as it comprises about 90% of the cell wall dry weight [30]. The cell wall is composed of three layers: a lipid-rich layer, including a lipid-binding pocket, which contains chitin and chitinase proteins, and a non-lipidic layer, including chitinase and chitinase-like proteins (ChiLPs) and chitinase-like proteins (ChlRPs). Chitin is the major component of the cell wall, and its synthesis occurs in a lipid-dependent manner.


## Methods
Ethics statement
All animal experiments were conducted in strict accordance with the German animal protection law (Law for the protection of animals in all its subdivisions) and were approved by the responsible Federal State authority (ÖCDU) under the supervision of the Federal Ministry of Environment (ZBB-Ustellamiersamiersam).

T. rubrum strain and culture
T.


## Results
prolificans from the Netherlands was isolated and sequenced (Table 1. It was isolated from a garden of a young woman with a small stature. DNA sequencing revealed that the isolate was L. prolificans var. prolificans and L. prolificans var. pseudoae (data not shown).

Identification of molecular markers
The identification of the isolate was based on PCR amplification of the ITS1-2 region of the rDNA of the isolate. The PCR products of the ITS1-2 region were positive for L. prolificans var. prolificans. The ITS1-2 amplicon was positive for L. prolificans var. pseudoae. PCR products of the rDNA of the isolate were negative.

The PCR products of the rDNA of the isolate were positive for L. prolificans var. prolificans. PCR products of the amplicon of the ITS1-2 region were negative. The amplicon of the rDNA of the isolate was negative for L. prolificans var. prolificans. PCR products of the amplicon of the ITS1-2 region were positive for L. prolificans var. pseudoae.

Fungal strains were isolated from various habitats and from different parts of the plant. Two strains of L. prolificans var. prolificans were isolated from the leaves of the apple tree, the dead stem of the vineyard and from a woody host plant. One of the isolates was isolated from a tree trunk and the other one from a woody host plant. All the isolates were sterile. One isolate was identified as L. prolificans var. prolificans by PCR amplification of the ITS1-2 region of the rDNA and PCR amplification of the ITS1-2 region of the 28S rDNA.

Identification of the molecular markers
The DNA sequence of the rDNA of L. prolificans var. prolificans was 100% identical to that of L. prolificans var. pseudoae. The ITS1-2 region of the rDNA of L. prolificans var. prolificans was 100% identical to that of L. prolificans var. pseudoae. PCR products of the ITS1-2 region of the rDNA were negative for L. prolificans var. prolificans. The ITS1-2 amplicon was negative for L. prolificans var. pseudoae. PCR products of the ITS1-2 amplicon of the ITS1-2 region were negative.


## Discussion
circinelloides
The sequences of . circinelloides GenBank number AG044983. This is the first report of a M. circinelloides genome sequence. The genome sequence of M. circinelloides is distributed in GenBank with additional information on the species identification. The genome sequences of M. circinelloides, C. parapsilosis and C. albicans are available in GenBank.

Biosynthesis of cell wall-degrading enzymes
The majority of the hydrolytic enzymes were only detected in the pathogenic fungus B. dermatitidis (Table 1). This species has been identified in the environment and is found in tropical, temperate, tropical and sub-tropical regions [10], [11], [13]. It is also known as “Dioszegia” or “Moliobolus” or “Hercules” [15], [16], [17]. B. dermatitidis has been isolated from the environment in temperate regions [18] and in the Northern Hemisphere [19], which has been reported to be a human pathogenic fungus [20], [21]. In contrast to B. dermatitidis, M. circinelloides appears to be a terrestrial fungus, which is widely distributed in the environment [20], [21].

A total of 19 hydrolytic enzymes were detected in the genome sequences of M. circinelloides (Table 1), including a predicted homologue from Trichoderma reesei (DG004641) [22], an acetyltransferase (DG042362) [23] and a large family of non-coding RNAs (DDRs) (DDR63873, DDR63874, DDR63880, DDR63881) [24]. These enzymes are also present in the genome sequences of B. dermatitidis and are known to be involved in the degradation of plant cell walls [25], [26].

Transcription factors
The transcription factor Rho GTPases were detected in the M. circinelloides genome sequence. These proteins were also detected in the genome sequences of B. dermatitidis and C. parapsilosis. The Rho GTPases were first identified in B. dermatitidis and C. albicans and their orthologues in B. dermatitidis and B.
