# Small RNAs in the plumbing complex: mutations and implications for safe and efficient functional design
Quaglee Dragontacos


## Abstract
A 64-year-old woman with pneumonia who developed respiratory failure over a 12-month period was referred a Peking Union-Tibet Hospital in Beijing, China. She had a good appetite and no complications. There was no evidence to support a diagnosis of pneumonocytic off- respirator syndrome. Further work led to the diagnosis of Pneumocystis jirovecii pneumonia. She had a good pulmonary and cardiac prognosis. We report this case to illustrate the importance of a diagnosis of pneumonia to the pneumonitis community.


## Introduction
Lung abscess is a common complication of systemic fungal infection and is often life threatening. In the early stages of infection, a variety of symptoms may be encountered including the development of diffuse tissue within the sterile vessel, bone loss and eventually death.


## Methods
In this study, we focused on the genomic DNA of bacteria, fungi, and plant.

The current study was performed in collaboration with the Department of Microbiology, College of Agricultural Sciences, University of São Paulo, and the Microbial Genomics Centre, Faculty of Health Sciences, Universidade Federal de São Paulo, under the permit P003/1/3.

PCR and RT-PCR
The PCR assay was performed as previously described (Petersen et al., 2014). Briefly, bacteria were cultured overnight at 37 °C in a 10 mM Tris-HCl pH 8.0, and were then treated with the specific enzyme (20 µM) for 5 min at 37 °C. The bacterial cell suspension (1 mL) was then treated with 200 µL of the PCR reaction mixture (10 mM MgCl_2, 1.5 mM MgSO_4, 0.2 mM KCl, 0.25 mM EDTA, 10 mM MgSO_4, 0.005% NP40, and 20 mM DTT) for the initial step of the reaction. Subsequently, the reaction was terminated by heating of the reaction mixture at 100 °C for 5 min. The mixture was then further incubated for 5 min at room temperature in the dark. The reaction was terminated with incubation of the reaction mixture in the dark at room temperature for 10 min, and the optical density of the reaction mixture at 550 nm was measured.

DNA extraction and sequencing
For the DNA extraction, 5 µL of the DNA sample was subjected to two steps: 1) the DNA extraction was performed using the QIAamp DNA Kit (Qiagen, Hilden, Germany) and 2) the DNA was extracted using the QIAamp DNA Kit according to the manufacturer's instructions.


## Results
RNAs were mutagenized to create RNAs containing the expected F-actin backbone by plating them in the Lactobacillus Y254b lysosom. The mutagenized RNAs were purified by centrifugation using a SDS-PAGE gradient and sequenced to validate that the RNAs were successfully mutated (Supplementary Figure S3). The lysosomal RNAs were purified by using a well-known method of purification of single-motif RNA (Supplementary Figure S4A). The recombinant RNAs were isolated by homogenization using EDTA.

A total of 54 clones were selected for sequencing, and 20 clones were sequenced to obtain a complete set of RNAs. The quality of the purified RNAs was compared with the RNA extracted from the Lactobacillus strains grown in the same nutrient medium as the L. donovani lysosome. The total RNA pool was approximately 2.4 kbp. The RNA extraction method used in this study was established by introducing 1.5 µl of RNase K into the Lactobacillus strains grown in the same medium as the L. donovani lysosome. The RNA extraction step was followed by an additional 2 µl of RNase K in the L. donovani lysosome. After purification, the RNAs were extracted using a SDS-PAGE gradient using a Pyrogelisone® buffer. The RNAs were purified by using a SDS-PAGE gradient.

Comparison of the RNA extractions and the RNA extraction method with that used for the RNA extraction method showed that the RNA extraction method yielded most of the purified RNAs. The average of the total RNA pool was approximately 1.2 kbp. The RNA extraction method yielded the most of the purified RNAs, however the highest amount of F-actin-F-actin was obtained with the RNA extraction method (Figure 4A). The RNA extraction method yielded the highest amount of F-actin-F-actin, as all the RNA extraction steps were performed by using a SDS-PAGE gradient (Figure 4B).


## Discussion
While the RNA-seq library of the nine strains studied in this study was the largest of the whole collection, it included 10 different RNA-seq libraries (including five RNAs and one unmeasured RNA-seq library), which would facilitate the identification of novel fungal RNA-seq libraries in a large sample siz. In the library constructed for this study, the RNA-seq library comprised of all the RNA-seq libraries obtained in this study, and the RNA-seq library of the fungal strain W303 was also constructed. However, the libraries constructed for W303 were also constructed from the non-pathogenic yeast, C. neoformans, a pathogen that infects the host and the former host is the second host. Therefore, RNA-seq data of the W303 library were analyzed using the same techniques described in the RNA-seq libraries of other sequenced fungal species, including C. neoformans [27], the wheat pathogen Ustilago maydis [28] and the human pathogenic fungus Histoplasma capsulatum [29], which all use the RNA-seq library for their RNA-seq studies. The W303 library included all the RNA-seq libraries obtained from other fungal species, including C. neoformans, S. cerevisiae and C. gattii, and all the RNA-seq libraries of the non-pathogenic and pathogenic strains C. gattii and H. capsulatum. The W303 library was constructed from the genomic DNA of the W303 strain by combining all the available genomic DNA from the two strains and the RNA-seq library from W303. The W303 library consisted of 6,152 reads from the non-pathogenic strain W303. The W303 library comprised 2,856 reads from the pathogen C. neoformans, 1,352 reads from the non-pathogen C. gattii, and 1,352 reads from the pathogen H. capsulatum. The W303 library comprised of 1,335 reads from the pathogen C. neoformans, 1,220 reads from the pathogen H. capsulatum, and 1,009 reads from the non-pathogen C. gattii. The W303 library comprised of 1,009 reads from the pathogen C.
