# <|startoftext|>Dicer-Dependent Biocontrol of Aliothrix brasiliensis, Imediothrix brasiliensis-formatI and Fuentesomersaturae (Tetrebriguidings): From Polyphenolic to Fuentesomers of Artesalogium spp.
Quaglee Dragontacos


## Abstract

De novo genome assembly: 382,571 genes were assembled and annotate. The assembled genome data were deposited in GenBank.

De novo assembly: 393,379 genes were assembled and annotated. The assembled genome data were deposited in Transmoleptiche Research Library.

De novo genome assembly: 280 genes were identified as new transcribed groups. The assembled genome data were deposited in Transmoleptiche Research Library.

Comparative genomics shows that Althiothrix spp. and Althiota brasiliensis are two closely related taxa. However, to our knowledge, the two new species are only distantly related to Althiota species and Althiota species of the Dactyloscelium clade (Sordariomycetes, Dactylales). Furthermore, the genome sequence of two new cultivars, Tetracladium spp. and Tetracladium-form I (Sordariales, Pleosporales) and Tetracladium-form II (Sordariales, Pleosporales), showed highly similar genome content and was highly similar to other Leotiostomi taxa (e.g. Trichosporonales and Leotiostomi) with the exception of occasional transitions from coding to non-coding nucleotides in Tetracladium.

The genome sequence of Leotiostomi taxa has been largely unaffected by mutations in the cytosolic Ca-type DNA polymerase (CTRP), and only one genomic element was found. The CTRP and Tsetilon radicans were found to be physically related, whereas the cytosolic Ca-type DNA polymerase (Caspase) and the non-coding RN (ncRNA) components were found to be highly conserved. The genomes of the two new cultivars were only slightly larger than the other genomes from the same clade.

These results indicated that Althiothrix brasiliensis is a sister taxon to Tetracladium, and their differences may indicate that the CTRP and the non-coding RN play complementary roles in regulating gene expression. The genome sequence of the Tetracladium-form I (Sordariales, Pleosporales) and the two new cultivars are available at http://www.itgo.


## Introduction
and . brasiliensis are recognized by name. The genus includes about 500 species classified in the order Pleosporales. The genus name has been used in several names, such as Artesomyces, P. brasiliensis, P. brasiliensis, P. lutzii, P. lignosa, P. schenckii, and P. soudanense [1,2]. The genus Artesomyces was initially recognized by Crous et al. (2014), who re-defined it as “non-albofungous fungi”. Crous et al. (2014) provided an overview of the taxonomy of the genus, including the genera and species, which are currently accepted in the genus. However, the concepts of “non-albofungous fungi” have been discussed recently [3].

Artesomyces is a well-known genus, which includes more than 1000 species classified in the order Mucorales. In the past, only four species have been reported from the genus: Artesophora, A. sphaerica, A. lignosa, and P. brasiliensis. These authors mainly studied Mucor circinelloides. In the past two decades, several studies have investigated the taxonomy of the genus, including the phylogenetic analysis of several groups of fungi, including the genus Artesomyces, and have reported new species, new genera, and new combinations of species and combinations. Many combinations were reported as “hymenomycetes”, for example Mucor, Arthrobotrys, and Rhinotyphidiomycota [4,5,6,7,8,9,10].

The genus Artesomyces was originally defined as a monotypic genus based on the analysis of the morphological characteristics, which were not found in other genera. Since the 1970s, the genus was reported from tropical and subtropical countries, mostly in Brazil, Mexico, and Colombia. In the late 1990s, an agent of P. lignosa (Artesophora) Bachelet (Moraceae), which belongs to the group of black yeasts, was reported from some Brazilian states and Venezuela [11,12]. Since the first description of this agent by Crous et al.


## Methods
For this purpose, we have included several species of species from different environments, from which we have identified different isolates of the same genu. In addition, we have also included the species of Aspergillus niger and Fusarium solani. In this study, the taxonomic and molecular identification of the species involved in the study of these species was performed as the well-established methods of the literature [4]. The species included in the study of the species of Aspergillus niger is Aspergillus niger. The species of Fusarium solani was Fusarium solani.

Dicer-Dependent Biocontrol of C. albicans
C. albicans was cultured in potato dextrose agar (PDA) medium at 37°C for 16 h. The fungal cells were collected by centrifugation at 3,000 × g for 5 min. The supernatant was transferred to a 1 ml tube, and the solution was centrifuged at 3,000 × g for 5 min. The pellet was washed twice with 200 µl of PBS, and the supernatant was transferred to a 1 ml tube. The tubes were centrifuged at 3,000 × g for 10 min, and the supernatant was transferred to a 1 ml tube. The solution was transferred to a 1 ml tube, and the solution was centrifuged at 3,000 × g for 10 min. The pellet was washed twice with 200 µl of PBS, and the supernatant was transferred to a 1 ml tube. The tube was centrifuged at 3,000 × g for 10 min. The supernatant was transferred to a 1 ml tube, and the solution was transferred to a 1 ml tube. The tubes were incubated for 24 h at 37°C with agitation (150 rpm). After incubation, the fungal cells were centrifuged at 3,000 × g for 10 min, and the supernatant was transferred to a 1 ml tube. The tubes were centrifuged at 3,000 × g for 10 min, and the supernatant was transferred to a 1 ml tube. The tubes were centrifuged at 3,000 × g for 10 min, and the supernatant was transferred to a 1 ml tube.


## Results
The strain CP 2764 was isolated from a contaminated sample of olive oil with a recent European application and was thus named as C. neoformans. The strain was previously used for the Southern blot analysis of C. neoformans strain CBS 277.18 and C. neoformans strain CBS 143.51. The Southern blot analysis indicated that strain CP 2764 had the same colony morphology as the strain CBS 277.18 (Fig 1). The similarity of the strain CP 2764 with other strains including CBS 277.18, CBS 143.51, CBS 143.51, C. neoformans and A. niger was also confirmed by both Western blot and quantitative PCR (Fig 1).

C. neoformans strains isolated from olive oil
To identify the strains of C. neoformans strain CBS 277.18 and C. neoformans strain CBS 143.51, we used primers to amplify a fragment of yeast DNA (genotype CNAG_01249) with a Southern blotting analysis (Fig 2) and plasmid pOTR2-RFPL. PCR products were amplified from the two strains CBS 277.18 and C. neoformans strain CBS 143.51. PCR products were negative for the genus-specific RFPL, and PCR products with an IC_50 of 10 ng/µl were positive for strains CBS 277.18 and C. neoformans. The combined ITS2 sequences of C. neoformans strain CBS 277.18 and CBS 143.51 were submitted to GenBank (accession number: KJ201002). The strains were named C. neoformans strain CNAG_00675 and C. neoformans strain CBS 143.51.

The ITS region of the C. neoformans strain CBS 277.


## Discussion
Compounds of this class have been isolated from different plants (for review see [27]), and in many cases, are derived from the same species of the same plant species (.e., seeds, fruits, leaves, stems, and shoots). The genus Imediothrix, which includes plants with large polyphenolic contents, has been used as an antifungal and immunotherapeutic agent [28]. Many studies have reported the occurrence of Imediothrix species in clinical plants [29,30]. It has been speculated that the isolation of Imediothrix species from the plant material is a useful technique for the diagnosis and treatment of plant diseases [31]. The species of Imediothrix have also been reported from various plant species, including I. longisporus, I. elaealba, I. squalens, I. rubescens, I. queensbergii, I. fusigata, and I. subleucadia [32,33]. However, the exact mechanisms underlying the synthesis of these compounds remain unclear. It is known that compounds derived from Imediothrix species in the seeds are involved in the synthesis of plant-derived compounds. This is consistent with the reported isolation of several compounds from the seeds that have not been reported previously (Table 1).

The data presented here provide the first description of a new class of Imediothrix compounds, which is an interesting class of chemical compounds that have been isolated from plants. The activity of these compounds against species of I. orientalis and I. oryzicola has been reported in vitro [34,35], and is also considered as a promising new avenue of antifungal therapy [36].

The isolation of Imediothrix strains from seeds confirmed that they are viable plant-derived compounds, and that they were able to inhibit I. orientalis and I. oryzicola. However, the isolation of these strains from the seeds revealed that they were not able to produce plant-derived compounds. This suggests that these strains were not truly the species of I. orientalis or I. oryzicola, but rather the plant-derived compounds. This indicated that the Imediothrix strains from seeds contained a novel species of I. orientalis or I. oryzicola, but they were also not able to produce plant-derived compounds.
