# Stereotrophy of a Core Repressor with Insertion into the Whole Genome of Shiraia aralpinii
Quaglee Dragontacos


## Abstract
The detection of environmental bacteria using conventional assays has led to a significant increase in the number of applications of detergents and other detergents in detergency. In the last decade, the use of detergent as a detection method has increased in recent years. Therefore, environmental isolates and environmental bacteria have improved their detergents and/or improved their bactericidal capacities. Moreover, there has been a significant increase of the use of bacteriostatic detergents, such as Klebsiella aerogenes, Pseudomonas aeruginosa and Pantoea sp. In general, Klebsiella aerogenes is a bacteriostatic detergent usually found in household products, whereas P.


## Introduction
This work was presented in support of PLOS Genetic.

Response: The authors point out the question raised. “Correct statement. The authors mention a model organism, but their analysis was based on a model organism alone.”

4. At para. 37, the authors do not mention the “data” portion of the data and “what”s the available data at this stage for further study? The available data should be presented as a figure legend or figure supplement. The authors should state clearly that the data is available.

Response: We agree. Our analysis was based on a phylogenetic model. The data presented in this paper is available in Figure 5-figure supplement 2. Figure 5-figure supplement 2 has been replaced with a figure legend.

5. At para. 19, the authors state that they used RNAseq to study the bacterial orthologues and not to study Shiga. The authors should state clearly that the RNAseq data are available. This statement should be unambiguously resolved.

Response: RNAseq data are available in Figure 5-figure supplement 1. RNAseq data are available in Figure 5-figure supplement 2. In the subsection “Shiga orthologues”, the term Shiga is added.

6. At para. 42, the authors state that they used “examples to show that RNAseq can detect” and this statement should be unambiguously resolved.

Response: The literature contains over 70,000 examples to demonstrate that RNAseq can detect different bacterial orthologues and species. The analysis was performed using different samples of the genome and strains tested and the results obtained were compared. We have now clarified the language in the subsection “Results”.

7. At para. 72, the authors state that they used RT-PCR for the majority of the species investigated, but they used microarray data to show that only Shiga and other Shiga types can consistently detect the major molecular types. This statement should be unambiguously resolved.

Response: The microarray data is available in Figure 4-figure supplement 1. It has been re-described as “Results”.

8. At para. 56, the authors state that the dataset is “representative of onlyShiga type andShiga-Shiga virus”. The authors should state this more clearly and unambiguously.


## Methods
A random region was selected and a random region containing a 1 kb region was added to the core regio. A gene was assigned to a specific gene by the BLAST search. The original core region was not aligned, as the probability of being assigned to a gene was <0.9% for all samples.

The Rfam database was used for all the calculations and visualization.

Protein model selection and BLAST search
To perform BLAST searches of the genome of Shiraia aralpinii, the genome of R. aralpinii was randomly selected from the collection of the Genome Institute of China. The genome of R. aralpinii was aligned using MUSCLE [49], which was used to search the protein families with the default parameters. The Rfam family was manually curated, and the best-scoring BLAST hit was assigned to a different family.

The protein model selection and BLAST search was performed using a Bayesian approach based on a Bayesian phylogenetic method with a gamma model for the evolutionary rate (GTR+G) and a proportion of invariable sites (I+G). The sequence alignment was performed using MUSCLE, and the best scoring BLAST hit was assigned to a different branch of the tree. The alignment was done using the ClustalW algorithm [50].

The protein model selection and BLAST search was performed using a non-redundant protein database (NrDB) [51]. The sequence alignment was done using MUSCLE, and the best scoring BLAST hit was assigned to a different branch of the tree. The alignment was done using the ClustalW algorithm, and the best scoring BLAST hit was assigned to a different branch of the tree. The alignment was done using the ClustalW algorithm, and the best scoring BLAST hit was assigned to a different branch of the tree.

The protein model selection and BLAST search was performed using a multiple alignment search method with a maximum likelihood method (ML) approach [52]. The sequence alignment was done using the ClustalW algorithm, and the best scoring BLAST hit was assigned to a different branch of the tree. The alignment was done using the ClustalW algorithm, and the best scoring BLAST hit was assigned to a different branch of the tree. The tree was then visualized using FigTree.


## Results
cerevisiae results in an increase in the levels of the . cerevisiae core repressor [27]. Thus, we investigated the impact of the endogenous locus of S. cerevisiae on the expression of the core repressor genes, which was identified as the target of the putative regulator, by Northern blot analysis.

The Northern blot analysis showed that the single nucleotide polymorphisms (SNPs) in all genes of the core repressor (compare Figure 3B) were associated with a genomic DNA fragment (5,735 bp) containing the endogenous locus of S. cerevisiae (Figure 3C). The Northern blot analysis revealed that the SNP in the 4,054 bp genomic fragment was highly abundant and contained a high level of genetic diversity. The gene expression of the genes of the core repressor was analyzed by qRT-PCR using a set of 5,735 putative SNPs. In contrast to the Northern blot analysis, a significantly higher level of diversity was found in the S. cerevisiae core repressor gene, which was also highly abundant (Figure 3D). This result suggests that the putative regulator is involved in the regulation of the S. cerevisiae growth and the expression of genes associated with this core repressor in S. cerevisiae.

The Northern blot analysis revealed that the gene expression of the genes of the core repressor was significantly increased in the sre1<U+0394> strain. The expression of the genes of the core repressor in the sre1<U+0394> strain was also significantly increased in comparison to the wild type (Figure 3E). The expression of the genes of the core repressor in the sre1<U+0394> strain was also significantly increased in comparison to the wild type (Figure 3F). This result suggests that the sre1<U+0394> strain may be more sensitive to the stress imposed by the deletion of the endogenous locus of S. cerevisiae.

Genome resequencing of the sre1<U+0394> strain revealed that sre1<U+0394> was highly sensitive to oxidant stress (Figure 4A).


## Discussion
In contrast to other species, such as Cryptococcus neoformans (10), . gattii (10) and M. oryzae (10) have a single cystereotrophy.

Shiraia aralpinii is a member of the Toxoplasma gattii complex. It shares the same cystereotrophy with Cystereopsis sp. (10) and is a member of the Sordariomycetes. Both Cystereotrophy and Cystereomyces sp. are well-known for their specialization in the Toxoplasma genus and the ability to cause systemic infection (31, 32). In addition to Cystereotrophy, the Cystereotrophy of Cystereopsis sp. is in contrast to other species of Toxoplasma and Toxoplasma gattii (10, 11, 43-45). In contrast to Toxoplasma gattii, Toxoplasma sp. is a member of the Toxoplasma clade. Cystereotrophy is usually observed in the Cystereopsis clade, whereas Cystereopsis sp. is not included in this clade (1). In contrast to Toxoplasma, Toxoplasma sp. is also a member of the Toxoplasma clade.

In the case of Cystereopsis sp. and Toxoplasma sp. they share the same cystereotrophy and Cystereopsis clade, whereas Cystereopsis sp. is a member of the Cystereopsis clade. They have a similar gene content with similar sizes.

A high degree of sequence conservation was found between Toxoplasma and Toxoplasma sp. The data suggests that the Toxoplasma cystereotrophy could be a result of gene duplication. The gene content was higher in Toxoplasma sp. than in Toxoplasma sp. The differences between Toxoplasma and Toxoplasma sp. are likely to be due to gene duplication and gene loss. It is possible that gene content is lost during the evolution of Toxoplasma and Toxoplasma species.
