# The Putative Regulator of Carbon Utilization in Artemisia biennis glutinis
Quaglee Dragontacos


## Abstract
Urokinase-type plasminogen activator receptor (uPA receptor) is an essential stimulator of cell proliferation and differentiation. It is also associated with immune system disorders such as multiple sclerosis, Alzheimer’s disease, and hypersensitivity pneumonitis. To understand the role of uPAR in plasminogen activator receptors (uPA receptors), we conducted a genome-wide scan of uPA receptor orthologues from the Aspergillus fumigatus genome. The roles of uPA and uPAR in immune system regulation were characterized and compared using a 2.5 kb window by analysis of the genes. The expression levels of uPA and uPAR were significantly regulated in response to various stress conditions and the expression patterns of uPA receptor subtypes were differentially regulated in response to axenic concentrations of uPAR-rP3, uPA-rP3-rP6, and uPA-rP4, all tested in a heterologous system. To investigate the role of the uPA and uPAR ligands in antigen presentation, we employed a mouse model of aspergillosis and found that the inflammatory response to Aspergillus fumigatus is dependent on the uPA ligands. In addition, the depletion of uPA ligands in the serum of mice results in a blockade of antigen presentation by uPA receptors.


## Introduction
(2014), reported the presence of five putative regulators of carbon utilization in . biennis. The Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsC, a SreA, a CsaA and a CsbA-encoding gene, in C. biennis.

The Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsC, a CsaA, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsC, a CsaA, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsD, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsF, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsM, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsM, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al. (2014) described the CsB, a CsbA and a CsbA-encoding gene, in C. biennis. In the Putative Regulator of Carbon Utilization in Artemisia biennis glutinis, Masaaki et al.


## Methods
.N.A. website. The genes in this genome were predicted by Blast2GO (http://blast2go.sourceforge.net/).

Identification of putative regulator genes and matching up- and down-regulation of them
The HMM profile for the putative regulatory genes was performed using HMMer v. 1.34 (http://hmmer.janelia.org/) [34] with default parameters.

PCR-RFLP assay
The PCR-RFLP assay was performed using HMMer (version 2.34) using the following primers: hph (5'-ATATAGATGCCCTCTCGGCTGTG-3') and hphr (5'-CTAGCCCTCCCCGAGCGGAG-3') for H. heterostrophus S-10, hphr (5'-CAGGGCGTGCCATC-3') and hphr (5'-TCCTCTTGCCGTGGCTTG-3') for H. heterostrophus S-10. The HMMer parameters were as follows: an initial denaturation step of 10 min at 94 °C, followed by 30 cycles of 30 s at 94 °C, 30 s at 60 °C and 30 s at 72 °C, and a final extension of 10 min at 72 °C. The PCR products were purified using Qiaquick PCR Purification Kit (Qiagen). The purified PCR product was sequenced using the same primers used for PCR-RFLP assay and a BigDye Terminator v. 3.1 Cycle Sequencing Kit (Applied Biosystems). The sequences were edited and assembled using the program SeqMan v. 6.0 (DNASTAR). The alignments and the resulting sequences were deposited in GenBank (Table 1).

Gene identification and annotation
GenBank accession numbers of all genes with an e-value lower than 0.1 and an alignment gap in between are shown in Table 1. The gene sequence of H. heterostrophus S-10 was used as the reference sequence for comparison with known H. heterostrophus S-10 gene sequences. H. heterostrophus S-10 gene was classified according to previous studies [35]. The gene annotation for H.


## Results
biennis va. arctii [31]. The putative carbon utilization pathway is significantly different in plants and fungi. The putative carbon utilization pathway is significantly different in both the plants and fungi. The putative carbon utilization pathway is enriched in the root fungal pathogen Botrytis cinerea and in the melanin-degrading fungi Fusarium graminearum and Penicillium marneffei [32,33]. However, the putative carbon utilization pathway in S. biennis var. arctii is largely unknown.

The putative carbon utilization pathway in S. biennis var. arctii is significantly enriched in the stem fungal pathogen Fusarium graminearum and in the melanin-degrading fungi F. graminearum and Penicillium marneffei [33]. However, the putative carbon utilization pathway in S. biennis var. arctii is largely unknown.

S. biennis var. arctii is a common pathogen of fruits and vegetables [34,35]. The pathogenicity of S. biennis var. arctii is not well described in plants and is considered a rare pathogen of food. However, recent studies have indicated that the pathogenicity of S. biennis var. arctii is comparable to that of other fungi [34].

The putative carbon utilization pathway is significantly different in the leaves and fruit of S. biennis var. arctii. The putative carbon utilization pathway is enriched in the root fungal pathogen F. graminearum and in the melanin-degrading fungi F. graminearum and Penicillium marneffei [34,35]. However, the putative carbon utilization pathway in S. biennis var. arctii is largely unknown.

The putative carbon utilization pathway in S. biennis var. arctii is enriched in the stem fungal pathogen F. graminearum and in the melanin-degrading fungi F. graminearum and Penicillium marneffei [34,35]. However, the putative carbon utilization pathway in S. biennis var. arctii is largely unknown.

The putative carbon utilization pathway in S. biennis var.


## Discussion
seavensis (Mansfield et a., 2015)

The synthetic expression of genes encoding putative carbon utilization genes of the main carbon utilization systems of Artemisia was confirmed. The complementation of Artemisia putative carbon utilization genes by the putative carbon utilization genes in the engineered Artemisia strain was conducted using the available metabolic pathway database. The results of the complementation of the gene expression in Artemisia strain by the putative carbon utilization genes in the engineered Artemisia strain are shown in Fig 7.

Concerning the chemical utilization of carbon, the gene expression results showed that the gene expression for the putative carbon utilization genes of Artemisia strain was stable in the presence of cation, sorbitol, and phenylalanine (Fig 7). The expression of the putative carbon utilization genes of Artemisia strain was stable in the presence of cation, sorbitol, and phenylalanine (Fig 7). In the presence of cation, the gene expression was stable for 3-5 h, but the gene expression was stable in the presence of sorbitol (Fig 7). These results indicated that the putative carbon utilization genes of Artemisia strain were expressed and expressed at higher levels in the presence of cation, sorbitol, and phenylalanine (Fig 7). The gene expression results for the putative carbon utilization genes of Artemisia strain were stable in the presence of cation, sorbitol, and phenylalanine (Fig 7). These results indicated that the putative carbon utilization genes of Artemisia strain were expressed and expressed at higher levels in the presence of cation, sorbitol, and phenylalanine (Fig 7).

2.3. The Gene Induction by Oxidative Stress
The transcriptional analysis of the putative carbon utilization genes of Artemisia strain indicated that the putative carbon utilization genes of the Artemisia strain were induced by oxidative stress. The transcriptional analysis of the putative carbon utilization genes of Artemisia strain indicated that the putative carbon utilization genes of the Artemisia strain were induced by oxidative stress. The transcriptional analysis of the putative carbon utilization genes of Artemisia strain indicated that the putative carbon utilization genes of the Artemisia strain were induced by oxidative stress. The transcriptional analysis of the putative carbon utilization genes of Artemisia strain indicated that the putative carbon utilization genes of the Artemisia strain were induced by oxidative stress.
