# The Significance of Whole-Genome Sequencing to Host Phenotypes in Cryptococcus neoformans
Quaglee Dragontacos


## Abstract
In this study, we have investigated the fungal community structure in the surface waters of polar lakes from points of deposit to points of refractory infectious eruption and from points of deposition. We found that members of the fungal community were stable with at least one eruption. The fungal community was dominated by Ascomycota, with the largest concentration of Trichosporon spp. and Aspergillus spp. in the cryosphere. The majority of the sampled sites were within freshwater bodies, including Lake Geneva and its main freshwater body, Lake Geneva. We identified the Aspergillus species and the Cryptococcus species as secondary metabolites of this group of organisms. The Aspergillus species was most common in lakeside sites and the highest number of isolated and identified conidia (9.5%) was found in 3/21 samples. The most frequent symptoms of Aspergillus disease in these sites were fever (74.5%) followed by diarrhea (32.5%) and vomiting (7.9%).


## Introduction
The polycationic lipid, lipid-inositol regulates the cell cycle, morphogenesis, and lipid homeostasis in eukaryotes. In eukaryotic cells, it has been associated with diverse functions, including phosphoinositide signaling, protein kinases, energy production, and cell cycle control [1-3]. In mammalian cells, it plays a role in the regulation of transcriptional and post-transcriptional events, cell cycle regulation, apoptosis, and cell cycle control [4-7].


## Methods
[32. WGST is a standard technique for the quantification of SNP signals in genome assemblies. In brief, the SNP signal is expressed in base pairs per kb per million (bp/bp) and is defined as the number of SNPs in a nucleotide sequence. The SNP signal value is calculated as follows: (i) SNP signal value = (bp/bp) × 100. (ii) SNP signal value = (bp/bp) × 100. (iii) SNP signal value = (bp/bp) × 100.

The WGST SNP analysis was performed in MEGA 6.0. The SNP detection tool (http://www.megasun.com/wiki/Profilerot_MEGA_6.0.html) was used to obtain a reference genome with the genome size set to 8 kb. The SNP calling tool (http://www.megasun.com/wiki/Profilerot_MEGA_6.0.html) was used to obtain a reference genome with the size set to 2 kb.

Transcriptome
The genome assemblies of C. neoformans strain Af293 and A. fumigatus strain were analyzed in this study. The genome assemblies were deposited at the National Center for Biotechnology Information (NCBI) (http://www.ncbi.nlm.nih.gov/genome/). The genome assembly results were verified by gene annotation and transcriptome analysis in NCBI BLAST (http://www.ncbi.nlm.nih.gov/blast/). The genome assembly results were also verified by gene annotation and transcriptome analysis in NCBI BLAST (http://www.ncbi.nlm.nih.gov/blast/).

The assembled genome of A. fumigatus strain Af293 was used to generate a transcriptome of A. fumigatus strain A1158. The genome assembly results were confirmed by transcriptome analysis in NCBI BLAST (http://www.ncbi.nlm.nih.gov/blast/).

Identification of highly expressed genes
The list of highly expressed genes in A. fumigatus strain Af293 was retrieved from the National Center for Biotechnology Information (NCBI) and the annotation was performed using the BLASTX software (http://www.


## Results
neoforman. FTR1 and FTR2 are important for pathogenicity in mammals and birds [26, 27, 28]. In addition, the Drosophila strain B3501[9] is the most virulent model organism for the study of virulence in Cryptococcus. The virulence of the Drosophila strain B3501 was confirmed by the comparison of its virulence phenotype with those of other strains of C. neoformans (Table 2). These findings are in agreement with the importance of FTR1 and FTR2 for pathogenicity of C. neoformans, and the importance of the FTR1, FTR2, and FTR3 domains in C. neoformans pathogenesis.

Transcriptomic Profiles of FTR1, FTR2, and FTR3 in Cryptococcus
To gain insights into the transcriptional changes in C. neoformans following infection, we identified the transcripts from the FTR1, FTR2, and FTR3 domains in C. neoformans. In order to identify the genes that are differentially regulated in response to infection with C. neoformans, we performed BLASTx analysis on the GenBank sequences for the DEGs that were differentially expressed in response to infection. For this, we used the transcriptomic profiles of C. neoformans infected with C. neoformans and then analyzed the transcriptomes of these strains by qRT-PCR. The results of the qRT-PCR analyses are shown in Table 3. We found that transcript levels of all three domains in the C. neoformans genome were differentially regulated following infection (Table 3). The majority of these genes were up-regulated in response to infection with C. neoformans (Table 3). Interestingly, the C. neoformans transcript levels of the genes involved in the TCA cycle, including the TCA cycle-associated subunits of the ATP-binding cassette (ABC) and mitochondrial ATPase (M), and the transcription factors Rho1 and Rho2, were differentially regulated. The gene expression of the genes encoding the ATP-binding cassette (ABC) subunits of the TCA cycle were down-regulated, whereas the gene encoding the mitochondrial ATPase (M) was up-regulated.


## Discussion
neoformans have shown that the genome is smaller than the average genome size of . neoformans (2.6 Mb vs. 5 Mb, respectively) [16]. In contrast, our genome size estimate of C. neoformans was similar to that of S. cerevisiae (2.7 Mb vs. 1 Mb, respectively). This difference is probably due to the fact that C. neoformans has two additional copies of the Saccharomyces genome. However, the genome of S. cerevisiae is not larger than that of C. neoformans [16]. Thus, C. neoformans and S. cerevisiae share many genes and are genetically distinct. However, we cannot rule out the possibility that other species in the human fungal species complex may be genetically distinct and we cannot rule out the possibility that the genomic size of C. neoformans is larger than that of S. cerevisiae.

The genome size of C. neoformans was estimated to be 2.6 Mb in our estimate and C. neoformans genome is only 1.2 Mb smaller than the S. cerevisiae genome (Table 3). Furthermore, our estimate of the genome size of C. neoformans was lower than that of S. cerevisiae (2.3 Mb vs. 4 Mb, respectively) and S. paradoxus (2.6 Mb vs. 2 Mb, respectively). Overall, the size of the C. neoformans genome is larger than that of S. cerevisiae (2.6 Mb vs. 1 Mb, respectively). The difference in size of C. neoformans and S. cerevisiae may be due to the difference in the chromosome size. In S. cerevisiae, the chromosomal size of S. cerevisiae is 7.2 Mb [40]. The same is true in C. neoformans, which has 7.8 Mb chromosome size [16]. The size of C. neoformans and S. cerevisiae is larger than that of C. albicans (2.2 Mb vs. 4 Mb, respectively). The differences between these two species may be due to the fact that C. albicans has a larger chromosome size compared to C. neoformans. Therefore, the sizes of C. neoformans and C. albicans are similar in terms of size and genome size.

The difference in the size of the C.
