# The ITS of Cryptococcus neoformans Mediator Protein SOD1 Controls Development and Pathogenicity in Cryptococcus neoformans
Quaglee Dragontacos


## Abstract
The aim of this study was to evaluate the anti-TNF-a inbred RY3/J33HB mice as compared to the inbred RY2/J83HB mice.


## Introduction
 2016. Moreover, Mediator is required for survival of the yeast-to-hyphal transition from the stationary phase to the stationary phase, which involves morphogenesis and virulence. The role of Mediator in pathogenesis is well established in animal and plant pathogenic fungi, including Candida albicans, Histoplasma capsulatum, Cryptococcus neoformans, and A. parasiticus (Rao et al., 2009; Rifaat et al., 2015). In addition, it is a co-inoculator of the mitochondrial electron transport chain (TDX) and participates in the regulation of the Fe-S cluster biosynthesis (Maurer et al., 1999). The role of Mediator in pathogenesis is also well established in plant pathogenic fungi, including M. oryzae and F. graminearum, which are members of the phytopathogenicity complex that causes anthracnose disease (Kaur et al., 2012; Romano et al., 2016).

The role of Mediator in plant-parasitic interactions has been extensively investigated in plant pathogenic fungi, including Magnaporthe oryzae, which is part of the phytopathogenicity complex that controls disease in the rice blast fungus (Liu et al., 2016). The role of Mediator in the interaction of M. oryzae with host plants has been established, and the association of Mediator with the regulation of fungal virulence has been demonstrated in this interaction (Diniz et al., 2008). Moreover, the roles of Mediator and Mediator-like proteins in pathogenesis have been shown in M. oryzae. In M. oryzae, the role of Mediator was related to the induction of the host-parasite immune response (Huang et al., 2011). In addition, the role of Mediator in the interaction of M. oryzae with host plants has been demonstrated in M. oryzae. In addition, the role of Mediator in the interaction of M. oryzae with human skin and the role of Mediator-like proteins in the interaction of M. oryzae with human skin have been demonstrated in M. oryzae.

The role of Mediator in the interaction of M. oryzae with human skin and the role of Mediator-like proteins in the interaction of M. oryzae with human skin have been demonstrated in M.


## Methods
The purity of the purified DNA was determined by NanoDrop 1000 spectrophotometer (Thermo Scientific, Wilmington, DE, USA) using a NanoDrop ND-1000 spectrophotometer (NanoDrop Technologies, Wilmington, DE, USA.

The purified DNA was subjected to a PCR amplification step with specific primers designed to amplify the ITS region (Table 1) to detect the presence of ribosomal DNA, using the universal primers ITS3 (5'-TCCGTAGGTGAACCTT-3') and ITS4 (5'-TCCTCCGCTTATTGATATGC-3') [51] and the primer pair ITS3 (5'-TCCTCCGCTTATTGATCTAACTT-3') and ITS4 (5'-TCCTCCGCTTATTGATCTAACTT-3') [52], using a PCR product size of 2.0 µg/µl. The PCR products were purified with the Qiagen PCR Purification Kit (Qiagen, Hilden, Germany) following the manufacturer’s instructions. The purified PCR products were sequenced using the BigDye v. 3.1.1 Cycle Sequencing kit (Life Technologies, Carlsbad, CA, USA) according to the manufacturer’s instructions.

Quantification of Microbial Communities
The pooled DNA was diluted to 10 ng/µl and sequenced using the Roche 454 GS FLX+ chemistry (Roche Diagnostics, Mannheim, Germany) according to the manufacturer’s instructions. The sequencing was performed on a Roche 515 GS FLX+ chemistry (Roche Diagnostics, Mannheim, Germany) using the same protocol as described above. The sequencing data was deposited in the Sequence Read Archive (SRA) of the European Molecular Biology Laboratory (EMBL, EMBASE) and the sequences deposited in the Sequence Read Archive (SRA) of the European Bioinformatics Institute (EMBL, EMBASE) were used in the study.


## Results
To determine if these genes are also involved in the pathogenicity of . neoformans in C. neoformans, we created a C. neoformans Mediator-expressing strain, C. neoformans Mediator-expressing strain, and C. neoformans Mediator-expressing strain and analyzed these two strains in the course of infection. As shown in Figure 3A, the C. neoformans Mediator-expressing strain showed a significant increase in virulence in the course of infection.

A significant reduction in virulence in C. neoformans infected by C. neoformans Mediator-expressing strain is not due to a reduction in virulence in C. neoformans Mediator-expressing strain
In order to examine the possible involvement of a specific protease in virulence, we also performed in vitro assays to assess the effect of this specific protease on the virulence of C. neoformans. A typical cationic protease was isolated from the C. neoformans Mediator-expressing strain (Table 3), which included a core region that contained the putative iron-sulfur-binding site of the SOD1 (CS02_0914) protein (Figure 3B), a region that was mutated to contain the specific ferroxidase domain (CS02_0712) (Table 3), and the domain of the ferroxidase domain (CS02_0712) (Figure 3C). As shown in Figure 3D, the C. neoformans Mediator-expressing strain did not show any virulence in the course of infection.

To determine if the C. neoformans Mediator-expressing strain is able to produce proteases that are able to degrade the iron in the host, we first examined the presence of this protein in the C. neoformans Mediator-expressing strain. As shown in Figure 3E, the C. neoformans Mediator-expressing strain produced multiple proteases with an extracellular size ranging from 1 to 10 µm. To further investigate the possibility that C.


## Discussion
In this study, we investigated whether DHGR is essential for the virulence of . neoformans and demonstrated that it is essential for the pathogenicity of this fungal pathogen. We demonstrated that Cryptococcus neoformans DHGR is essential for the pathogenicity of this fungus, but that it is not essential for virulence. We showed that the fungus was not resistant to the common antifungal agents used in this study, and that the fungus was avirulent in mice infected with C. neoformans. This result suggests that Cryptococcus neoformans DHGR is essential for virulence.

DHGR was required for the expression of the genes in both the urease-like genes and the ribosomal protein SOD1 in C. neoformans. Although the strains from the Ure2/SOD2 complex were highly virulent in mice, we failed to control the infection. The strains from the Ure2/Sod2 complex were virulent in mice infected with C. neoformans, but these strains were avirulent in mice infected with C. gattii. Therefore, we conclude that the Ure2/Sod2 complex is required for the virulence of C. neoformans.

We demonstrated that the Ure2/Sod2 complex is essential for the development of the yeast-to-hyphal transition in C. neoformans. However, the hyphal transition in C. neoformans requires the formation of the non-coding RNA-editing, RNA-binding protein (RNAB) subunit Rgt1. We have shown that RNAB is required for the hyphal formation of C. neoformans (23), and our previous study demonstrated that C. neoformans cells lacking the RNAB gene have no hyphal development (18). The Ure2/Sod2 complex is required for the hyphal formation of C. neoformans. We showed that the Ure2/Sod2 complex is required for the development of the yeast-to-hyphal transition in C. neoformans.

In summary, we demonstrated that the Ure2/Sod2 complex is essential for the development of the yeast-to-hyphal transition in C. neoformans.
