# The GeneACC consortium: analyses and insights
Quaglee Dragontacos


## Abstract
The transposable element chromatin (TE) plays a significant role in fungal resistance to host immune recognition and has a key role in regulating the expression of fungal virulence factors. The polysaccharide capsule and melanin pigmentation protect this cell type from host immune recognition, but it is also the target of antifungal drugs. Here, we show that the GATA factor Skp1 is a major component of the KEG+ terminal fragment of Skp1, an ancient gene that encoded an intron that inserted at a site previously intron-rich. We find that the Sskp1 homolog is not a repeat-rich core protein, since it is not exposed to short-varying environmental signals. Instead, it is highly trans-preferred to interacting with the known virulence factors, including its GATA group.


## Introduction
The study of the asexual genus Ascochyta constitutes a large group of fungi, which are mostly saprophytic and are often intracellular and sub-sporophore producing asexual spores (Sipiczki, 1999; LeBlanc et al., 2011). Apart from the ascomycetes, species of this genus are commonly found in various environmental niches such as soil, rotting vegetables, and plant debris (Dawson, 2002). It was first described by Hippocrates (lex dictyopus) and it has been accepted as the concept of the Greek and Latin literature for a wide variety of taxa (Hippocrates, 2011; Lamont, 2002; LeBlanc et al., 2011; Tullio and Droby, 2011). Recently, the genus was described from a phylogenetic perspective and it is currently divided into 15 sections (Wu et al., 2011). In general, species of the asexual genus Ascochyta are found in the phyllosphere of plants and insects (Wu et al., 2011). Phyllosphere fungi are saprophytic and live in the phylloplane (Dawson, 2002). In this sense, members of the genus are associated with decaying vegetables, plants and other microorganisms (Dawson, 2002; Wutana and Ngamskulipatana, 2010).

The genus Ascochyta was first described by Hippocrates (Phys.


## Methods
(http://ww.ncbi.nlm.nih.gov/genome/).

The gene annotations of the genomes of the Cryptococcus strains were obtained from the GenBank database (http://www.ncbi.nlm.nih.gov/genome/Cryptococcus_neoformans/).

For a genetic analyzer to determine the genetic variation of the strains, an analysis was performed on the strain data and the gene annotation of the strains.

The analysis was carried out in GeneACC: Genome Analysis v.2.0 (http://www.ncbi.nlm.nih.gov/genome/).

The phylogenetic analyses were carried out using the neighbor-joining method and neighbor-joining distances were calculated.

For the phylogenetic analyses, the genome sequencing of the Cryptococcus strains was performed at the Macrogen Europe (Amsterdam, The Netherlands) using the MiSeq platform (http://miprez.sourceforge.net/index.html).

The genetic variation between the strains was determined by a neighbor-joining (NJ) method. The bootstrap test of 1000 replications was carried out.

The phylogenetic analysis of the strains was performed on the genome sequencing of the Cryptococcus strains.

Genome Sequencing of the Cryptococcus strains
The genome sequences of the Cryptococcus strains were retrieved from the NCBI database using the BLAST search algorithm.

The genome sequences of the Cryptococcus strains were downloaded from the NCBI website (http://www.ncbi.nlm.nih.gov/).

The genome sequences of the Cryptococcus strains were obtained from the JGI (http://genome.jgi.doe.gov/TripleGap/TripleGap.html) and NCBI databases (http://www.ncbi.nlm.nih.gov/).

The genome sequences of the Cryptococcus strains were obtained from the NCBI website (http://www.ncbi.nlm.nih.gov/genome/Cryptococcus_neoformans/).

The genome sequences of the Cryptococcus strains were obtained from the NCBI website (http://genome.jgi.doe.gov/TripleGap/TripleGap.html).


## Results
Analysis of the gene sets was also performed to assess gene sets of genes that were differentially expressed (Additional file 10: Table S9. Among the 21,355 genes that were differentially expressed in the three strains, 813 (67.6%) were down-regulated, 1214 (40.4%) were up-regulated, and 2636 (45.5%) were up-regulated (Fig 7 and Additional file 11: Table S10).

The DEGs that were differentially expressed in the three strains are presented in Table 4. The numbers of down-regulated genes in the three strains are represented in Fig 7 and Additional file 11: Table S10. Down-regulated genes include those that are relevant to the pathogenicity of S. mansoni, such as factors related to pathogenicity, such as phosphatidylinositol 3-kinase (PI3K), phosphatidylinositol kinase (PI3K), and phosphatidylinositol kinase (PIK).

The DEGs that are differentially expressed in the three strains are also presented in Table 4. For example, in the P. graminis strain, 552 genes were differentially expressed, including genes involved in cell wall and fungal cell wall remodeling (Additional file 12: Table S11), and genes involved in the pathogenicity of C. neoformans, such as DnaK (an efflux pump), Kss (endocytosis pump), and Icl (mitochondrial respiration). Genes involved in cell wall remodeling were down-regulated in the three strains (Fig 7 and Additional file 12: Table S11). In contrast, the Kss strain had the largest gene set of up-regulated genes in all three strains. The most commonly expressed genes in the three strains are shown in Fig 7 and Additional file 13: Table S12.

The GO categories that were differentially expressed in the three strains are presented in Table 5. In the fungal pathogenicity classification, the most commonly expressed genes in the three strains are shown in Fig 7 and Additional file 14: Table S13.

The most commonly expressed genes in the three strains are shown in Fig 7 and Additional file 15: Table S14.


## Discussion
The majority of the reads in the library consisted of genes with an expression profile that included genes coding for eukaryotes' first-line activators (.e., histone H3 and Ile), as well as genes for a broad range of other types of histone deacetylases (e.g., Ile2, Ile3, Ile4, Ile5, and Ile6) (Figure 4). A total of 23,267 of the 23,267 genes were mapped to the T. cruzi genome (20,924 genes) (Figure 5). The total number of genes coding for the three T. cruzi gene clusters was 60 (50%) (Figure 5). The majority of the T. cruzi genes (58%) were associated with stress-related pathways, such as gene expression regulation, stress-associated molecular pattern (SAPM), and multi-stress response (MSR) pathways (Figure 5). The second largest cluster (36%) involved a key regulator of genes involved in stress response (i.e., mitogen-activated protein kinase 1 (MAPK1) and the transcription factor CAMP-1 (CRAMP-1) pathway; this gene cluster was the first gene cluster for T. cruzi (Figure 5). The T. cruzi gene cluster is comprised of genes coding for multiple genes involved in cell cycle regulation (i.e., cell cycle, meiosis, mitosis, and transcription factor activator) (Figure 5). The genes encoding the two T. cruzi genes were mapped to the TcL4 gene cluster (Figure 5). TcL4 is a gene encoding a regulatory subunit of the TcL4 transcription factor, which is involved in cell cycle progression, meiosis, and mitosis (Figure 5).

A total of 2,438 genes were mapped to the genome (14,595 genes). The TcL4 gene cluster contained 4,941 genes with a total gene density of 2.45 Mbp, which was 1.14 times higher than the genome of the T. cruzi strain H99 (5,471 genes) (Figure 5). The gene density of TcL4 is 1.04 times higher than the genome of the T. cruzi strain H99 (6,699 genes), and 1.
