# Transcriptomic Analysis of Grape (Vitis vinifera L.) Leaves Reveals Many Novel Substrate Specificities
Quaglee Dragontacos


## Abstract
In the present study, the expression of the catabolic enzymes (ECs) in a quail model (Pterocladium ostacolense) was investigated using high-throughput sequencing. A total of 598 strains were collected, and 219 genes were identified. The most abundant transcripts were: catabolism of galactofuranose by Cudonia fuckelii DSM 1019 (15.4%), carbon metabolism by Rhodotorula mucilaginosa DSM 20266 (23.4%), the feline immunodeficiency virus genome (6.9%), and the aspergillus fuckelii genome (1.1%). A total of 31 and 3 genes were identified, which were expressed differently in the model. Among these differentially expressed genes, the protein kinase pathway and the phosphorylase system encoded by the most abundant F1-a-like transcription factor were the most expressed in catabolism in the cat. In addition, in vitro aspergillosis transcriptome data revealed that the genes for the catabolic pathway and the phosphorylase system were highly expressed in the model. We also observed two genes that were highly expressed in the cat in vivo, but they were silenced by protease inhibitors. In conclusion, our findings suggested that a high level of expression of these genes in a cat-infected model may lead to the development of new therapeutic strategies against fungal infections.


## Introduction
losgenetic.org/static/s/s2161-vd2/

COPD Vol 10 No. 2, pages 18-23, 2018

Dear Dr. Rao,

We are pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it complies with all outstanding technical requirements.

The corresponding author will soon be receiving a typeset proof for review, to ensure errors have not been introduced during production. Please review the PDF proof of your manuscript carefully, as this is the last chance to correct any errors. Please note that major changes, or those which affect the scientific understanding of the work, will likely cause delays to the publication date of your manuscript. Please review the PDF proof of your manuscript carefully, as this is the last chance to correct any errors. Please note that major changes, or those affecting the scientific understanding of the work, will likely cause delays to the publication date of your manuscript.

Soon after your final files are uploaded, unless you have opted out or your manuscript is a front-matter piece, the early version of your manuscript will be published online. The date of the early version will be your article's publication date. The final version of your manuscript will be your final version of paper.

Thank you again for supporting open-access publishing; our mission is entirely tied to delivering high quality, open-access publishing. PLOS defines open-access publishing as “the systematic publication of the peer-reviewed results of research in a scientific community” (www.journals.plos.org/plosntv/s/publications). Open access publishing can be defined as “a publication program that promotes, directly or indirectly, the publication of the peer-reviewed results of a scientific diversity research program” (PLOS PLOS #PLOSProtos) (www.journals.plos.org/plosntv/s/publications).

Our online submission platform is designed to facilitate the exchange of knowledge and data, collaboration, and knowledge sharing. All PLOS submissions will be reviewed by a senior editor, and a decision will be rendered by an external body such as a committee or advisory panel. The online submission process is streamlined, and there is no need to log into any different systems.


## Methods
 Vitis vinifera ., Vitis vinifera C, Vitis vinifera C, Vitis vinifera D, and Vitis vinifera F) collected from grape varieties in Brazil (Table 1).

Using sequencing data of the Vitis vinifera V. orv. f.v. samples, we obtained sequences of the 16S rRNA gene from the leaves of six grape varieties (Vitis vinifera L., Vitis vinifera V., Vitis vinifera V., Vitis vinifera F., Vitis vinifera C., Vitis vinifera D, and Vitis vinifera F). We also extracted the sequences of the glycerol fermentation gene from the samples of the glycerol fermented in ethanol and methanol. The glycerol fermentation gene was named after the influence of the grape varieties, Vitis vinifera V. orv. f., Vitis vinifera V. orv. f.v., Vitis vinifera V. orv. v. v., Vitis vinifera V. orv. v. v., Vitis vinifera V. orv. v. v., and Vitis vinifera V. orv. v. v.). In brief, we have extracted the 16S rRNA gene of the five grape varieties, Vitis vinifera V. orv. f., Vitis vinifera V. orv. v. v., Vitis vinifera V. orv. v., and Vitis vinifera V. orv. v. v. v., respectively. The sequences of the glycerol fermentation gene from the samples of the six grape varieties were analyzed in order to obtain the phylogenetic trees, using the neighbor-joining method with MEGA version 5.0. In brief, we have extracted the sequence of the glycerol fermentation gene from the samples of the four grape varieties Vitis vinifera V. orv. f., Vitis vinifera V. orv. v. v., and Vitis vinifera V. orv. v. v., respectively.


## Results
gi-ps.org) to map the specific subtype-specific differences among grape varieties (Table 2 and Table S1).

As shown in Table 2, grapevine isolates were found to be closely related to grapevine varieties CBS27 and CBS50, and also with grapevine varieties CBS22, CBS23, CBS24, CBS25, and CBS26. The relative abundances of the three molecularly conserved subtypes in grapevine isolates varied greatly. On the other hand, the four grapevine isolates CBS23, CBS24, CBS25, and CBS26 were found to be related to the CBS11 isolate (Table S1).

The genes of the grapevine isolates were found to be differentially expressed (Table 2 and Table S1). However, the differential expression of the gene set was not related to grapevine variety, grapevine variety, or grapevine varieties. For instance, genes involved in the response to oxidative stress were found to be up-regulated in the CBS11 isolate (Table 2 and Table S1). Thus, the grapevine isolates CBS22, CBS24, CBS25, and CBS26 could potentially have different responses to environmental stresses. However, the significant up-regulation of genes involved in the response to osmotic stress (e.g., stress response, acyl-CoA dehydrogenase, and p-nitrophenylpyruvate production) and cell wall stress (e.g., glucan, chitin, and glucan-like structures) were not found to be significant among the grapevine isolates.

The DNA sequences of the clusters of clusters of genes differentially expressed in grapevine isolates were also found to be different (Table 2 and Table S1). The clusters were analyzed for a total of 841 clusters identified in all nine isolates and were then compared to the unsupervised multigene dataset of the PASHTES (http://parscript.sanger.ac.uk/PASHTES) using the cluster-based clustering algorithm (see Methods). The unsupervised clustering algorithm identified 644 clusters. Of these, 246 clusters were found to be unique to the CBS11 isolate (Table 2).


## Discussion
NMG-29 (Phy.org) was isolated from the soil of the same locality as the cultivated grape (Marin, 2008), which showed low activity against the same strain (WCC7069). The Mucurbitaria sp. was identified by the expression analysis in the metagenomics (Fig 4).

Mucurbitaria sp. was isolated in different parts of the world. In China, the strain was isolated from grape juice (Fig 4). In India, the strain was isolated from grape (Fig 5) and from grape juice (Fig 6). In Japan, the strain was isolated from grape juice (Fig 7). In Japan, the strain was isolated from grape juice (Fig 8). In Korea, the strain was isolated from grape (Fig 9). The same was also found in Japan (Fig 10).

Phylogenetic analyses of the DNA sequences of Mucurbitaria sp. have indicated that it belongs to a distinct group of species. Phylogenetically, the phylogenetic analysis revealed that Mucurbitaria sp. is the sister group to the Metschnikowia and Pneumocystis species. These two species are characterized by a high degree of sequence similarity (62.7% and 72.2% to Metschnikowia sp. and Pneumocystis sp. in comparison, 75% and 76.8% to Metschnikowia and Pneumocystis sp. and Pneumocystis sp. in comparison). The other species are present in high degree of sequence conservation (69.3% to Metschnikowia sp. and 86.4% to Metschnikowia sp. in comparison, 96.2% to Metschnikowia sp. and 94.2% to Metschnikowia sp. in comparison, 93.8% to Metschnikowia sp. and 93.6% to Metschnikowia sp. in comparison, 78.3% to Metschnikowia sp. and 78.1% to Metschnikowia sp. in comparison, 81.4% to Metschnikowia sp. and 82.7% to Metschnikowia sp. in comparison, 77.4% to Metschnikowia sp. and 80.4% to Metschnikowia sp. in comparison, 78.
