# Transcriptome Analysis of Sporothrix brasiliensis Mng1 Replication during Panored Oligodegradation of Polysaccharide from Wheat Bread Spice Cork
Quaglee Dragontacos


## Abstract
Fungal biofilms can play an important role in biofilm formation. The objective of this study was to elucidate the roles of the fungal polysaccharides in biofilm formation.

Fungal polysaccharides were extracted and identified in various biological matrices such as food, drinks, and medical implants. The respective compositions of fungal polysaccharides were analyzed by gas chromatography-mass spectrometry (GC-MS). Their chemical structures were elucidated by spectroscopic methods. The antifungal activity of different fungal polysaccharides was evaluated using the broth microdilution method. The antifungal activity of different fungal polysaccharides was further evaluated using the MTT assay.

Different biofilms were dominated by the glucosides glucoside A, gluconic acid, and lauric acid. In general, the glucosides glucoside A and gluconic acid dominated the fungal biofilms and were absent from all the matrices analyzed. The majority of fungal polysaccharides were preferentially hydroxycinnamic acid (7%), a-cyhalothiic acid (7%), ß-caryophyllene acid (7.5%), and <U+03B3>-caryophyllene (6.5%).


## Introduction
brasiliensis . gagei Dictyostelium [27].

The aim of this study was to evaluate the possibility of use of T. brasiliensis as a model in a comparative analysis of the application of T. brasiliensis as a tool for the functionalization of starch-like structures from a variety of food starch. We used the wheat starch as a model starch source to study protein secretion from T. brasiliensis using T. brasiliensis as a model. T. brasiliensis is a ubiquitous chitin-degrading fungus that can be isolated from soil, plant, and animal sources [28,29,30,31]. The starch content in the cell membrane of T. brasiliensis was found to be similar to that of other fungi, suggesting the presence of starch-like structures. Protein secretion in T. brasiliensis was reported to be robustly regulated during growth in maize and T. reesei. We also found that T. reesei is capable of degrading starch and glucose, which were found to be differentially secreted proteins. We hypothesize that T. brasiliensis will use T. reesei as a functional starch source and not be used as a functional starch source.

Results from the above studies indicated that T. reesei has a remarkable capacity to degrade starch and glucose. This enzyme is highly abundant in plants and is involved in the degradation of starch in many plant species including Arabidopsis thaliana, Drosophila melanogaster, wheat, Sorghum bicolor and S. cerevisiae [32,33]. We also observed that T. reesei produced starch-degrading enzymes, including T. reesei-DGGE and T. reesei-DGFR, that were able to degrade starch in comparison to T. reesei-DGGE. Protein degradation in T. reesei was similar to that in the other fungi, suggesting that T. reesei could degrade starch and glucose to proteins. Protein degradation of T. reesei-DGGE and T. reesei-DGFR was also similar. These findings indicate that T. reesei can be used as a functional starch source, and will be used as a functional starch source in future functionalization studies of T. reesei-DGFR.


## Methods
chrysosporium [32. The PCR products were purified from the matrix, cloned into the pGEM-T vector (Promega), and then transformed into Escherichia coli DH5a to obtain the pGEM-Mng1 construct. The resulting plasmid pGEM-Mng1-A was transformed into yeast (2 × 10^7) cells to generate pGEM-Mng1-B. The pGEM-Mng1-B pGEM-Mng1-B strain was used for RNA and DNA extraction. The pGEM-Mng1-B was transformed into pGEM-Mng1-A to generate pGEM-Mng1-A-B.

To construct a pGEM-Mng1-A-A, the plasmid pGEM-Mng1-A-A was amplified with PCR and homologous sequence sequencing of the genomic DNA of the FAS-B strain obtained from S. chrysosporium [32]. The PCR products were purified from the matrix, cloned into the pGEM-T vector (Promega), and then transformed into Escherichia coli DH5a to obtain the pGEM-Mng1-A-A construct. The pGEM-Mng1-A was transformed into Escherichia coli DH5a to obtain pGEM-Mng1-A-B.

To construct a pGEM-Mng1-B-A, the plasmid pGEM-Mng1-B-B was amplified with PCR and homologous sequence of the genomic DNA of the FAS-B strain obtained from S. chrysosporium [32]. The PCR products were purified from the matrix, cloned into the pGEM-T vector (Promega), and then transformed into Escherichia coli DH5a to obtain the pGEM-Mng1-B-B construct. The pGEM-Mng1-B-B was transformed into Escherichia coli DH5a to obtain pGEM-Mng1-B-B-B.


## Results
Our transcriptome analyses revealed that Mng1 was activated in wheat bread spice kernels by monocultured wheat germ agglutinin (MIG) (Figure 2A), which inhibits Mg^2+ levels in the wheat germ agglutinin-stimulated wheat kernels by up to 65% (Figure 2B. To further characterize the dynamics of Mng1 during the transition from wheat to oleaginous phase in wheat kernels, we performed transcriptomic analyses of the Mng1 transcript from O. sativa by quantitative real-time PCR (qRT-PCR) using an a-tubulin knockout strain (N50R) to determine the expression of Mng1 at a single time point during wheat oleaginous development. To this end, we performed quantitative real-time PCR of the O. sativa RNA samples for the upstream and downstream regions of Mng1. The expression of Mng1 was detected in the O. sativa RNA samples from wheat kernels at the time points studied, whereas expression of Mng1 was detected in the O. sativa RNA samples from the wheat kernels at the time points studied (Figure 2C). As expected, the transcript level of Mng1 was up-regulated at the time point studied in the wheat kernels. In contrast, Mng1 transcript levels were significantly decreased at the time points studied in the wheat kernels, especially at the time points studied in the wheat kernels compared to the O. sativa samples (Figure 2C). These data suggest that Mng1 is highly activated in the O. sativa transcriptome during wheat germogenesis and at the transition from wheat to oleaginous phase in the O. sativa RNA samples.

Transcriptome Analysis of Sporothrix brasiliensis Mng1 Expression in the Wheat Germ Extracellular Membrane
The mRNA expression level of Mng1 during wheat germogenesis was examined in the O. sativa RNA samples from the wheat kernels at the time points studied. The transcript level of Mng1 was significantly decreased at the time points studied in the wheat kernels compared to the O. sativa samples (Figure 2D). The transcript level of Mng1 in the O. sativa RNA samples from the wheat kernels was decreased at the time points studied in the wheat kernels compared to the O.


## Discussion
Expression of Mng2p is in the high range of transcript levels in the wheat phas. The Mng2p gene is only expressed during the growth phase, whereas Mng1p is not upregulated in response to growth. The transcriptome analysis shows that expression of Mng1p and Mng2p is similar to the expression of Mng1p in wheat.

The Mng2p gene expression is in the high range of transcript levels in the wheat phase. The expression of Mng2p is only upregulated in response to the wheat phase. The transcriptome analysis shows that expression of Mng2p is in the high range of transcript levels in the wheat phase.

To verify the existence of Mng1p and Mng2p transcripts in the wheat phase, we checked gene expression of the Mng1p gene by RT-PCR. The mRNA levels of the Mng1p gene are in the high range of the transcript levels in the wheat phase. The expression of the Mng1p gene is in the low range of the transcript levels in the wheat phase. Expression of Mng2p is only upregulated in the wheat phase. The transcriptome analysis shows that expression of the Mng2p gene is in the high range of transcript levels in the wheat phase.

The Mng1p gene expression is in the high range of transcript levels in the wheat phase. The transcriptome analysis shows that expression of the Mng1p gene is in the high range of transcript levels in the wheat phase.

The transcriptome analysis shows that the Mng2p gene is in the high range of transcript levels in the wheat phase. The transcriptome analysis shows that the mRNA level of the Mng2p gene is upregulated in the wheat phase. The transcriptome analysis shows that the Mng2p gene is upregulated in the wheat phase. The transcriptome analysis shows that the Mng2p gene is upregulated in the wheat phase.

The Mng2p gene is in the high range of transcript levels in the wheat phase. The transcriptome analysis shows that the mRNA level of the Mng2p gene is upregulated in the wheat phase.
