# Transcriptomic Analysis of the SREBP Gene and Its Impact on Host Defense against Cryptococcosis
Quaglee Dragontacos


## Abstract
Candida spp. are common colonizers of the oral cavity, and this paper describes one of the first clinical isolates of a Candida spp. associated to an urban center in China. This is the first report of a clinical isolate of a Candida spp. associated to a Chinese urban center.


## Introduction
The pathogenic yeast Cryptococcus neoformans is a major cause of morbidity and mortality in immunocompromised individuals. It is the most common cause of fungal meningoencephalitis in AIDS patients and accounts for up to 15% of all AIDS-related deaths (1-3). It is known to be ubiquitous, able to grow in various environmental conditions and to be able to survive in the serum and blood of immunosuppressed hosts. The global burden of cryptococcal disease is estimated to be over 1 million cases annually, with a mortality rate of 50% in sub-Saharan Africa and 40% in developed countries (4). The incidence of C. neoformans infection has been increasing over the past decades and its global burden is estimated to be in the millions of cases annually (5,6). C. neoformans infections are mainly acquired by inhalation of infectious propagules (spores) of the ascomycete fungus, which grow as yeast cells in the environment (7,8). The yeast cells enter the lungs as yeast-like form in the environment and subsequently disseminate to the central nervous system (CNS), causing meningoencephalitis and mortality. In the absence of an effective immune response, the yeast cells can cause meningoencephalitis and CNS involvement.

Candida albicans is a ubiquitous commensal of the human microbiome and the primary cause of infection of the central nervous system (CNS) (9). C. albicans is a harmless commensal organism and is known to be highly virulent (10-12). However, infections by C. albicans have emerged as an increasingly important cause of morbidity and mortality in immunocompromised patients and this has led to a shift in the focus toward antifungal prophylaxis and therapy (13-15).

In vitro studies have demonstrated that C. albicans cells exhibit distinct phenotypes compared to planktonic cells (16). C.


## Methods
Briefly, fungal cells were treated with either a 1% Triton X-100 solution (.01%) or 1.2% Triton X-100 for 30 min, followed by rinsing with distilled water and washing with sterile water. The DNA was lysed with a bead beater using the phenol method [40], and the number of nucleotides per million fragments (nt) was calculated by the Bradford method [41]. The PCR amplification of the SREBP gene was performed with the following primers: 5'-AACTCCTTCGAGGTACCTG-3' and 5'-TCCGTTGCTCGTGACGG-3', and the reaction was performed with an enzyme-to-plate-stream assay kit (Bioneer, Germany). The PCR products were checked on 1% agarose gel using an agarose gel electrophoresis system (SDS-PAGE) and were detected with a UV-visible spectrophotometer (SpectraMax M3R, Molecular devices, USA).

Quantitative Real-time PCR (qRT-PCR)
Total RNA was isolated from total mycelium of P. insidiosum wild-type, <U+0394>SREBP, and SREBP-DHA strain grown on YPD agar at 30°C. cDNA was synthesized using Reverse Transcription System (Promega, USA) with oligo dT primers (Table 2). qRT-PCR was performed in a 7500 Fast Real-Time PCR System (Bio-Rad, USA). Real-time PCR was performed in a Bio-Rad CFX96^TM Real-Time System (Bio-Rad, USA). The experiment was performed in triplicate.

Expression of SREBP in Cryptococcus neoformans
Cryptococcus neoformans strain CEA10 was used for the expression of SREBP in Cryptococcus neoformans. Cryptococcus cells were cultured on YPD agar at 30°C for 18 h and then cultured on SCM (1% yeast extract, 2% peptone, 2% glucose) at 37°C for 48 h.


## Results
We first used a BLASTP search against the SREBPs database to identify any orthologues with putative function and to validate our analysis (Figure 2A. However, no orthologues have been identified for SREBP gene in any other fungal genome. To validate our results, we first used the BLASTP analysis of the SREBPs database to search the SREBP gene and the genes that have not been shown to be in the BLASTP database. The BLASTP analysis revealed that SREBP genes from fungi could be identified (Figure 2B).


## Discussion
neoformans and other Cryptococcus species is the increased resistance of the two species to azole-resistant . neoformans (21). Azole-resistant C. neoformans has been associated with increased expression of SREBP-related genes (24, 26). A recent study, which compared the expression of SREBP-related genes in Cryptococcus neoformans and C. gattii (serotype A), revealed a significant increase in the expression of SREBP-related genes in serotype A (23). The increased susceptibility to fluconazole among serotype A cryptococcal isolates has also been shown to be linked to increased resistance to azole-resistant C. neoformans (25).

Azole resistance of C. neoformans has been associated with reduced expression of SREBP-related genes (22). Increased azole resistance in C. neoformans may reflect increased expression of azole-related genes (23). Consistent with this possibility, C. neoformans isolates from patients with AIDS and those with cryptococcal meningitis have been found to exhibit increased azole resistance (22, 25).

The increased azole resistance in C. neoformans may be due to the gene overexpression that encodes SREBP (24). In addition to the elevated azole resistance, we also found that the SREBP-related genes in C. neoformans may also be involved in the response to azole stress (26). This finding is in line with previous studies showing that SREBP-mediated gene expression can be induced by azole stress (25). The increased azole resistance in C. neoformans may be due to increased expression of SREBP-related genes (24).

This study also revealed that the azole resistance in C. neoformans is mainly driven by the overexpression of SREBP genes (23). The increased azole resistance in C. neoformans may be due to the gene overexpression that encodes SREBP (24).

In conclusion, C. neoformans strains that exhibit a high degree of azole resistance, but have less azole resistance than C. gattii, are more resistant to azole drugs. However, the mechanisms that contribute to this high azole resistance in C. neoformans are still not well understood.
