# Transcriptome analysis of tomato tissue during tomato blight disease
Quaglee Dragontacos


## Abstract
Background. Cryptococcal meningitis is the most common cause of adult meningitis in Africa. The immunocompromised host performs poorly in comparison with the immunocompetent one. Immune reconstitution inflammatory syndrome (IRIS) has been recently associated with the development of cryptococcal meningitis. We report a case of cryptococcal meningitis in a patient with Acquired Immunodeficiency Syndrome (AIDS) and the development of IRIS.

Methods. Five patients underwent computed tomography (CT) scans and two patients underwent immuno-restriction scans. Controls were analyzed by two-dimensional gel electrophoresis (2-DE) and two-dimensional histology (2-DE histogram).

Results. The patient was a 48-year-old Caucasian woman with a median age of 47 years with a history of living inwanetok (UIU) post-AIDS. The median CD4 was 8.1 in the baseline group (6.7, <U+03BA> group) and 4.


## Introduction
The transition from yeast (or filamentous fungi) to hyphae represents a major transition in plant and animal biology. The vast majority of plant pathogens colonize the host tissues by yeast-like budding, in which the infectious hyphae are formed to invade the hyphal cell and present a threat to the host [1], [2].


## Methods
.1). The BLAST program was used to search for homologs of known proteins in the genome of the selected P. expansum strain (a gift from A.N.C.) [64]. Genomic sequences of the selected P. expansum strains were aligned using the program CLUSTAL X version 1.8 [65]. The alignments were manually adjusted with the help of MUSCLE version 2.6 [66]. The alignments were visualized in Geneious 6.0.6.1 software.

Phylogenetic analysis
DNA sequences of the selected P. expansum strains were retrieved from NCBI (http://www.ncbi.nlm.nih.gov/) and aligned using the program Clustal X version 1.8. The DNA sequences of the selected P. expansum strains were retrieved from the online databases, including GeneBank and NCBI (http://www.ncbi.nlm.nih.gov/BLAST), and used for the phylogenetic analyses. The alignment of the P. expansum strains was performed using the program Clustal X version 1.8.

Morphological analysis
The leaves of P. expansum were stained with a mixture of ethidium bromide (0.1%), Congo Red (0.1%) and Melzerâ€™s reagent (0.1%). The leaves were examined under a light microscope with a charge-coupled device camera (Proxeon, Korea) and photographed. The photographs were taken in a ScanScope E600 (Nikon, Japan) using a Hamamatsu DMR-S camera (Hamamatsu Co., Ltd., Tokyo, Japan).

The molecular characterization of the P. expansum isolates was performed using MEGA 6.0 software (Tamura et al., 2005). The sequence of the P. expansum isolates was retrieved from GenBank and was compared with the taxonomic database of the National Center for Biotechnology Information (NCBI).

Quantitative real-time PCR
The 16S rRNA gene sequences of the selected P. expansum strains were retrieved from GenBank and used for qRT-PCR. The primers for the RT-PCR were designed based on the sequences of the P. expansum gene (Table 1).


## Results
2%), but also with a few transcripts associated with phosphatidylinositol (1.06%) and phosphatidylethanolamine (6.32%) in congenic isolates, and with phosphatidylinositol (8.04%) in non-congenic isolates (Table 3). The most abundant transcripts in the non-congenic and congenic isolates were involved in the degradation of phosphatidylinositol (PA), phosphatidylserine (PS), phosphatidylinositol (PI) and phosphatidylcholine (PC) (Table 3).

In terms of their co-occurrence in non-congenic and congenic isolates, the co-occurrence of the two transcripts for the most abundant transcripts in non-congenic and congenic isolates was significant for the phosphatidylinositol (PA) (Table 3). In contrast, the co-occurrence of the phosphatidylinositol (PA) and phosphatidylcholine (PC) transcripts did not group together, with the exception of the transcript for the phosphatidylcholine (PC) co-occurrence. Thus, co-occurrence of two transcripts from non-congenic and congenic isolates is not surprising.

Non-congenic and non-congenic isolates can be distinguished by their transcript abundance in non-congenic isolates
A total of 26 co-occurrences of the two transcripts (Table 3) for non-congenic and congenic isolates in non-congenic and congenic isolates were identified in both non-congenic and congenic isolates. The co-occurrence of the two transcripts for the most abundant transcripts in non-congenic and congenic isolates was significant for the phosphatidylinositol (PA) and phosphatidylcholine (PC) transcripts (Table 3). However, co-occurrence of the phosphatidylinositol (PA) and phosphatidylcholine (PC) transcripts did not group together, with the exception of the transcript for the phosphatidylcholine co-occurrence.


## Discussion
infected tomato tissue revealed a rapid increase in the transcript levels of genes related to secondary metabolism, including metalloproteinases, cellulases, hemicelluloses, and a-1, 3-beta-D-glucanases (Table 1 and Additional file 2. During the infection, Fusarium spp. first grew on its host and then to establish its infection, these two organisms interacted. Our RNA-seq analysis revealed that Fusarium spp. first appeared on tomato tomato, then on lettuce, and finally on tomato plants. Although we did not analyze the symbiotic relationship between Fusarium spp. and lettuce plants, the transcript levels of the secondary metabolites, including cellulases, hemicelluloses, and a-1, 3-beta-D-glucanases, were significantly higher during the early stages of infection. These results suggest that the Fusarium spp. are able to utilize a wide variety of plant-derived secondary metabolites to establish a successful infection in tomato plants.

During the late stages of infection, a significant number of genes involved in the secondary metabolism were upregulated in tomato plants, including a-1, 3-beta-D-glucanase, cellulases, hemicelluloses, and a-1, 3-beta-D-glucanases. These genes were found in all the Fusarium spp. infected plants. These genes were upregulated in the plant host. Our data suggest that the fungus is able to utilize a wide variety of plant-derived secondary metabolites to establish a successful infection in tomato plants.

Genes involved in secondary metabolism were also upregulated in tomato plants during the early stages of infection. These genes were found in all the Fusarium spp. infected plants. These genes were found in all the Fusarium spp. infected plants. However, the Fusarium spp. infection was a little more severe than the lettuce plant. The transcript levels of genes involved in secondary metabolism were significantly higher in the early stages of infection. The transcript levels of genes involved in secondary metabolism were significantly higher in the lettuce plant. These results suggest that the fungus is able to utilize a wide variety of plant-derived secondary metabolites to establish a successful infection in lettuce plants.

The transcript levels of genes involved in secondary metabolism were significantly higher in the tomato plant.
