# Urea Amidolyase (Dhy1) Contributes to Virulence and Plant Microbiome Properties
Quaglee Dragontacos


## Abstract
We previously found that a typical yeast-like fungus, Saccharomyces cerevisiae, can grow on ochracean substrates and to a lesser extent on oligopolymers. The aim of this study was to assess the contribution of the oligolytic substrate (ethyl-glucosamine) to yeast survival under different environmental conditions. One hundred twenty-four ochracean cultures were cultured under 30-60 mm salt, 30-60 mm exogenous salt, and 30-60 mm exogenous sugar conditions. The results showed that, during the initial phase of growth, a small fraction of the colonies were killed by the different ochracean classes. The proportion of these colonies that were killed by the ochracean combination was significantly higher during the initial phase of growth compared to the sugar-limited phase. The secreted aspartic proteinase (SSP) was also found to contribute to the yeast survival under salt stress and sugar stress conditions. The mean concentrations of sspp1, stp3, and ssp1 were significantly higher in the sugar-rich and ochracean-rich ochracean compared to the ochracean-poor ochracean. A correlation between ssp1 abundance and the fatty acids in the ochracean did not exist. These results suggest that the oligolytic substrate of S.


## Introduction
Invasive fungal infections (IFI) are a growing problem in nosocomial and community-acquired infections, where they affect both immunocompromised and immunocompetent individuals. The most common fungal pathogens are Candida albicans and Cryptococcus neoformans (1-3). Other Candida species include Debaryomyces hansenii, Pichia kudriavzevii, Kluyveromyces waltii, and Zygosaccharomyces rouxii. The most common pathogenic fungi in these species are Candida glabrata, Candida tropicalis, Candida krusei, and Candida parapsilosis. Other Candida species include Candida albicans, Candida tropicalis, Candida krusei, and Candida glabrata. The most common fungal pathogens in these species include Candida albicans, Candida tropicalis, Candida krusei, and Candida glabrata.

Candida glabrata, which is characterized by slow growth and multinucleated cells, is the most common yeast pathogen in candidiasis (4, 5). It has emerged as a major cause of septicemia, neonatal morbidity, and mortality in the United States (6-8). The incidence of Candida disease is increasing in non-candida species, especially C. parapsilosis, C. tropicalis, and C. krusei (6, 9). Although C. glabrata is generally harmless to humans, it can cause serious systemic infections in immunocompromised patients (6, 10). The most common agents used to treat Candida infections are azoles, polyenes, echinocandins, and pyrimidine analogues (11). Recently, the prevalence of C. glabrata infections has increased in the United States (12, 13).


## Methods
The disruption of genes with similar functions was achieved by using primers F-Z1 and F-Z2 [20. The primers used in this study are listed in Table 1.

Fungal strains were cultivated in potato dextrose broth (PDB) at 37°C. Fungal strains were grown on a shaken (200 rpm) slant for 24 h at 28°C. After centrifugation, the mycelium was collected and washed twice with phosphate-buffered saline (PBS), and the pellets were resuspended in 1 ml of PBS. The number of spores was counted on a haemocytometer, and the sample was diluted to 1 × 10^6 spores/ml. The diluted solution was inoculated on PDA plates (BD, Sparks, MD, USA) at 37°C for 24 h. After incubation for 48 h at 37°C, the mycelium was washed with PBS to remove unbound yeast and fixed with 3.7% formaldehyde at 4°C. After a 1-h incubation at 37°C, the plates were read on a LSM 510 fluorescence microscope (Nikon, Tokyo, Japan).

The method used in this study was similar to that previously described [21]. For genomic DNA extraction, 1.0 ml of a 0.1% w/v sterilized bovine serum albumin (BSA; Sigma-Aldrich, St. Louis, MO, USA) solution was added to 10 ml of liquid N_2, and the mixture was incubated at 35°C for 30 min, followed by centrifugation at 10,000 rpm for 15 min. The supernatant was removed, and the DNA pellet was washed with 70% ethanol. The pellet was then re-suspended in 1 ml of 5% v/v sodium dodecyl sulfate (SDS) (Sigma-Aldrich, St. Louis, MO, USA) and resuspended in 1 ml of TE buffer (10 mM Tris-HCl, pH 8.0, 1 mM EDTA, 1 mM PMSF, 100 mM sodium pyrophosphate, 0.


## Results
In this study, we found that the dhy1 gene was up-regulated in wild-type as well as dhy1<U+0394> as compared to dhy1<U+0394> mutant strains (Figure 3B,C. This suggests that the dhy1 gene was involved in regulating the physiological functions of the dhy1 mutant.

To further understand the effect of the dhy1 gene on the cellular functions of the fungus, we determined the protein expression levels of the genes encoding the cell surface and secreted proteins (susceptible effectors) (cna1, cna2, and cna3) and a set of proteins (cytokinesin, gliotoxin, and the cell wall-interacting proteins) (Figure 3D). The data suggested that the dhy1 gene was more effective at controlling the cellular functions of the fungus than at controlling other fungi.

The Dhy1-Derived Protein (Dhy1) Is Essential for Vegetative Growth and Its Expression Level Is Higher in dhy1<U+0394> and dhy1<U+0394>+Dhy1 Mutants
We investigated the function of the Dhy1-Derived Protein (Dhy1) in the virulence of B. dermatitidis. Dhy1 was expressed in D1/D2 region of the Aspergillus fumigatus genome and the P. jirovecii genome (Supplementary Table S4). As shown in Figure 4A, the Dhy1-Derived Protein showed lower expression level in the dhy1<U+0394> mutant compared to the dhy1<U+0394>+Dhy1 mutant strains. Dhy1 expression level was higher in dhy1<U+0394> than in the dhy1<U+0394>+Dhy1 strain (Figure 4A). Furthermore, the expression level of Dhy1 in the dhy1<U+0394> mutant was higher than that in the dhy1<U+0394> mutant (Figure 4B). These results suggested that the dhy1 gene was involved in controlling the virulence of the fungus.


## Discussion
The response of the fungi to salt was more pronounced in the deep than in the arid sites (Figure 7a,b), indicating that the high salinity plays a greater role in the fungal diversity in the deep than in the arid sites (Figure 7a. The fungal community composition and abundance of the arid sites were dominated by Ascomycota, which was also enriched at the deep end (Figure 7c). These data are consistent with previous observations on marine fungi that have shown that the fungal community in the deep environment was dominated by Ascomycota and Aspergillus species, and that the fungal community of the arid sites was dominated by Aspergillus species [42]. The fungal community composition of the deep sea was dominated by Aspergillus species (Figure 7c), and the fungal community of the arid sites was dominated by Aspergillus species (Figure 7d). These data suggest that the fungal community of the deep environment was not dominated by the fungi of the arid sites, but rather by the fungi of the deep and the arid sites. The fungal community composition of the arid sites was dominated by Aspergillus species (Figure 7e). In contrast, the fungal community of the deep site was dominated by Aspergillus species (Figure 7f). These data suggest that the fungal community of the deep environment was dominated by the fungi of the arid sites, but not by the fungi of the arid sites. This may be due to the fact that these fungal taxa are relatively more abundant in the arid sites than in the deep sites.

The response of the fungi to salt was less pronounced in the arid sites than in the arid sites (Figure 7d). These data indicate that the fungal community in the deep water is more dominated by the fungi of the arid sites, but not by the fungi of the arid sites. This may be due to the fact that the deep water is much smaller than the arid water, which could explain the difference in fungal diversity in the deep water and the fungal diversity in the arid sites.
