# Use of Liposomes as Biocontroovers: New Insights into the Mechanism of Intron Loss and Gain
Quaglee Dragontacos


## Abstract
The ability of microbes to evade their host immune surveillance is a prerequisite for pathogenic species invasion of vulnerable host tissues. Despite this, the host immune response to a common infectious disease has evolved in sophisticated and specific ways to evade or subvert host defense mechanisms. Here we review the host-pathogen relationship of fungal pathogens and the mechanisms of evasion and gain of immunity. We focus on how the host immune system uses and protects fungal pathogens and the mechanisms of this response. Host-pathogen interactions have evolved in ways that favor colonization and proliferation of the invading pathogen. While the host immune response can control many aspects of the fungal life-cycle, a limited degree of host-pathogen association is required to maintain normal pathogenesis. We highlight the host strategies that promote host protection of the infecting pathogen. We also review the current role of adaptive immunity and the mechanisms of host-pathogen interaction in fungal infections. This perspective highlights the importance of understanding the host-pathogen relationship in fungal infections.


## Introduction
The number of human fungal species has increased rapidly in recent years due to the widespread use of molecular techniques in fungal studies, the genomic tools, and the new “omics” techniques. According to the most recent World Database of Unidentified Symbols (WNV) database (http://www.wvn.org/), ~10,600 taxa are listed in the nomenclature (http://www.indexfungorum.org/Names/Names.asp) of the International Union for Conservation of Nature (COP). Of these, ~30% are recognized as belonging to Phylum Basidiomycota, including fungi such as fungi from the order Capnodiales, order of the Venturi, family and families of the Helotiales (Jain and Desai 2009).


## Methods
y [84], and the N-terminal gain and gain data were processed with N-terminal-stat.py. The B-terminal gain and the N-terminal gain data were processed with the program B-terminal-stats.py. The differences in the N-terminal gain and N-terminal gain data between the two types of growth were analyzed using the program P-N-G [85]. For the MHC class analysis, the normalized gain and N-terminal gain data were processed with the program P-MHC-stats.py. The differential abundance data from the two tests were analyzed using the program P-N-G [85].

MHC-based regression was performed using P-N-G [85] with default settings. A linear regression was performed using the program P-N-G [85].

Extracellular vesicle production
To examine extracellular vesicles, the N-terminal loss and the N-terminal gain data were processed with the program P-N-G. The E-test was performed to assess the influence of the chemical shift on vesicle production. The chemical shift was evaluated using a data-dependentlihood method, and the results were analyzed using the program P-N-G.

Liposome hydrophobicity
To examine the hydrophobicity of the liposomes, the N-terminal loss and the N-terminal gain data were processed with the program P-N-G. The E-test was performed to assess the influence of the chemical shift on lipid peroxidation. The hydrophobicity data were processed with the program P-N-G.

Heterologous protein identification
The protein sequences of the protein species in the N-terminal loss and the N-terminal gain data were used to identify the differentially expressed (D) proteins by the BLAST program. The putative orthologs of the proteins were analyzed by BLASTP analysis, using the program BLASTP. The assigned orthologs were clustered with S. cerevisiae as the outgroup.


## Results
.I. Videira & K.D. Hyde, Ph.D. nov. MycoBank MB 828752.

Etymology: Name derived from the host genus Fagus.

Conidiomata pycnidial, superficial, scattered, semi-immersed, oval, up to 400 µm diam, uniloculate, 120-250 × 100-220 µm; wall consisting of 3-6 layers of medium brown textura angularis. Conidiogenous cells phialidic, hyaline, smooth, subcylindrical, 5-7 × 5-7 µm. Conidia aseptate, hyaline, smooth, cylindrical to fusiform, 4-5 × 4-5 µm.

Type species: Fagus pseudocercospora (C.M. Shear) Videira & K.D. Hyde

Notes: Fagus pseudocercospora is a common pathogen causing leaf spots of Fagus magnifica (Morus cinerea) in South Africa. This fungus is morphologically similar to F. fagi (F. pseudocercospora C.M. Shear), but differs in having smaller conidia (4-5 × 4-5 µm) and the production of several small, globose, brown conidia with variable sizes. Fagus pseudocercospora is thus described as a pathogen for Fagus magnifica.

Fagus pseudocercospora (C.M. Shear) Videira & K.D. Hyde, comb. nov. MycoBank MB 828753. Facesoffungi number FoF03694. Fig 20.

Basionym: Fagus pseudocercospora C.M. Shear, Mycotaxon 29: 240. 1997.

Synonym: Fagus sclerotiorum C.M. Shear, Mycotaxon 29: 241. 1997.

Pseudocercospora pseudocercospora (C.M. Shear) Videira & K.D. Hyde, comb. nov. MycoBank MB 828754. Facesoffungi number FoF03694. Fig 21.

Basionym: Fagus pseudocercospora C.


## Discussion
.0 on the intron gain and loss of the introns of C. neoformans, C. deuterogattii, C. deneoformans, and C. albicans. We found that NcS. 3.0 could not induce the expression of the gene encoding the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase-ATPase (ATPase-ATPase-ATPase) (Fig 6). We further investigated the effects of NcS. 3.0 on the expression of the gene encoding the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase (ATPase-ATPase) (Fig 7). We found that NcS. 3.0 could not induce the expression of the gene encoding the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase (ATPase-ATPase) (Fig 7). It is likely that the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase (ATPase-ATPase) was not expressed in C. albicans. In addition, the expression of the intron-catalyzed gene (ATPase-ATPase) was not induced in C. deneoformans, C. deuterogattii, and C. deuterogattii, but it was induced in C. albicans (Fig 8).

NcS. 3.0 inhibited the expression of the gene encoding the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase (ATPase-ATPase) (Fig 8). In addition, NcS. 3.0 could not inhibit the expression of the gene encoding the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase (ATPase-ATPase) (Fig 8). We investigated the effects of NcS. 3.0 on the expression of the gene encoding the intron-catalyzed enzyme, CcS. cuniculi-ATPase-ATPase (ATPase-ATPase) (Fig 9).
