# Utility of a spiral arm method for the rapid detection of de novo genome assemblies in 10 super archaea from sugarcane thistle
Quaglee Dragontacos


## Abstract
We recently identified several fungal taxa that produce large-scale-size extracellular enzymes from specific non-redundant and drug-resistant mycelia grown in the absence of cell walls. This novel approach may be useful for understanding proteomic tinkering and drug resistance mechanisms in fungi. One such taxon is Mycosphaerellaceae, a widespread and ubiquitous plant pathogen that causes brown spot disease on a wide range of fungi, including many plant and human-related species. However, the taxonomic composition of this group has yet to be elucidated. In this study, we selected a subset of 18 taxa and compared them to the known fungal taxa and the known and known/known lack of enzymes from other fungi and bacteria that cause brown spots. We found that 10 and 6 fungi were resistant to the known and known/known pathogen, respectively, while 12 and 6 species were resistant to other fungi and bacteria. Functional annotation of the identified proteins revealed that the active site of secretory enzymes is located between the active site and the cell wall, and several pathogenic species have putatively secreted self-splicing elements that may function as pathogenicity factors. We also found that M. haustiae and M. phylum are good candidates for the M. oenophila brown spot model, where the pathogenicity of these species is equal parts fitness and virulence.


## Introduction
The detection of fungal pathogens in healthcare facilities is a critical issue in achieving effective patient management. In recent years, the number of reported infections by opportunistic pathogens has increased considerably. The main opportunistic pathogens associated with human disease include invasive fungal infections (IFIs), especially among immunocompromised patients, such as cryptococcus, candidiasis, histoplasmosis, and aspergillosis, as well as infections with aspergillosis and cytomegalovirus (C. neoformans). To date, only a few of these infections are diagnosed by conventional diagnostic methods, such as culture, microscopy, and polymerase chain reaction (PCR) (1-3). Among them, cytomegalovirus (CMV), an intracellular bacterium, is the most common, followed by cytomegalovirus type I (CTL) (4, 5). For these reasons, CMV infection can cause a wide spectrum of clinical manifestations. It is associated with chronic and acute symptoms, such as fever, anemia, diarrhea, and abdominal pain (6-9). In hospitalized patients, CMV is associated with high mortality rates (10).

A number of molecular diagnostic methods have been developed for the detection of CMV in the past decades, such as polymerase chain reaction (PCR), culture, and PCR. However, CMV detection in immunocompetent patients has been limited by the difficulties in obtaining samples for routine culture, and the lack of methods to obtain samples from immunocompromised patients. For the identification of fungal pathogens, it is important to understand the clinical spectrum of the disease. The Cryptococcus genus is known to be highly pathogenic to immunocompromised patients, including HIV-positive patients. In fact, it is responsible for approximately a million deaths each year (11). Moreover, C.


## Methods
 2016. Briefly, the sequenced raw reads from the 16S rRNA gene were generated using the Burrows-Wheeler Aligner (BWA) assembler (Bionumerics, Inc., Ashland, MA, USA) and the bamtools v0.8.9.0 assembler (Bionumerics, Inc., Ashland, MA, USA). The consensus sequences of the fungal and bacterial genomes were produced using the Blast2GO assembler (Bionumerics, Inc., Ashland, MA, USA). The sequencing reads were quality filtered using FastQC (v0.11.5) (http://www.bioinformatics.bbs.umich.edu/fastqc/). The resulting sequence data were analyzed using the Ribosomal Database Project (RDP) v2.0.1 (http://rdp.cme.msu.edu/web/rdp/) to identify transposable elements. The OTU table was organized using the RDP Classifier v2.1.1 (http://rdp.cme.msu.edu/classifier/) and the taxonomic units were assigned using the RDP Classifier. The OTU table was then imported into the RDP web tool v3.1.3 (http://rdp.cme.msu.edu/classifier/) for the classification of bacteria, fungi, and viruses. The variable modifications in the taxonomic units were classified using the RDP Classifier.

Phylogenetic analysis of the bacteria, fungi, and viruses from the rice, wheat, and maize seeds of N. benthamiana
All sequences and references used in this study were obtained from GenBank (Table 1). The sequences were aligned with MAFFT v7.864 (Katoh and Standley 2013, http://hmmer.org/index.html) and then the phylogenetic trees were constructed using the neighbor-joining method with bootstrap values equal to or greater than 100. The sequences were deposited in the NCBI's GenBank nucleotide database under the accession numbers KY636085 (C. cinerea), KY636090 (C. parapsilosis), KY636093 (C. krusei), and KY636094 (C. tropicalis).


## Results
All of the strains were cultured in YM liquid cultures for 2 day. The genome was sequenced using a forward/reverse mapping approach with a pair of forward/reverse BLAST-n primers. A total of 15.8 million reads were obtained and 16.1 million were mapped to the S. cerevisiae genome database. The reads were aligned using the BLAST tool with the appropriate open reading frame (ORF) and the corresponding data were used as queries in the NCBI non-redundant database (nr) at the following site: 10.2.1.1: GenBank accession number: XP_0039294610. The results of the NMR analyses are shown in Table 2. The nr and tsp are the standard diagnostic values for an NMR protein. A comprehensive analysis of the NMR data using the forward/reverse mapping approach revealed the presence of four peaks in the NMR spectrum of the sugarcane thistle genome: a single band at approximately 1348, an increase of ~40, and an increase of ~20 bp. These peaks were located at the nr position of the aromatic amino acid sequence of the yeast C. neoformans var. grubii (ACN) protein (NCBI accession number: XP_0039294610) and the tsp position of the aromatic amino acid sequence of the yeast C. neoformans var. grubii (ATL) protein (NCBI accession number: XP_0039294610). The nr peak at ~1348 corresponds to the oxygenation site of the UDP-N-acetylglucosamine-6-phosphate dehydrogenase (NAD) protein. The tsp peak at ~1423 corresponds to the oxygenation site of the UDP-N-acetylglucosamine-6-phosphate dehydrogenase (NAD) protein. The tsp peak at ~1423 corresponds to the oxygenation site of the UDP-N-acetylglucosamine-6-phosphate dehydrogenase (NAD) protein.


## Discussion
The number of fungal species isolated in this study is similar to that reported in previous studies [20], [21]. In contrast to previous studies, the frequency of fungal species in this study is significantly higher than those reported by others [18], [21], [23], [24].
